# Genetics

If you've benefitted from this document, [please consider tipping me](https://adaburrows.com/support-me/).

## Criticism

Genetic studies are an area fraught with criticism. Some of these criticisms are for good reason, some from fear ([and there can be good reason behind the fear, too](https://en.wikipedia.org/wiki/Human_Genome_Diversity_Project#Potential_problems)). A proper scientific understanding of genetics is necessary to draw conclusions about the following papers. Any understanding mired in the nonsense of racism is bound to produce further nonscientific nonsense. Any studies which do not respect that ownership and copyright are not concepts recognized by indigenous peoples sovereign nations. The earth and all of creation "owns" the genetics. No one can retain ownership or copyright over genetics. Those concepts applied to genes and genetics are rendered as nonsense in many indigenous worldviews, including my own.

Genetic studies provide an additional layer of information which might be missed if just looking at other measurements. For instance, there is the case of the Kennewick Man. The Kennewick Man is the skeleton of a male human from anywhere between approximately 7250 BCE &mdash; 6390 BCE in what is now known as SE Washington by the Columbia River. Initial studies measured various aspects of the man's bone structure. Based on that evidence and surrounding context, the study concluded that [NAGPRA](https://science.sciencemag.org/content/330/6001/166.1.full) did not apply and that the person was from the Aleutian Islands or even Asia. Based on a remark that measurements of the skull matched measurements of Ainu and Polynesian people, a Samoan living in California claimed that the "Ancient One" was one of the descendants of the first rulers of Polynesia. However, genetic testing confirmed that members of the Colvile Tribes where a close match to the "Ancient One". Their ancestor has since been repatriated.

* [The AP. (2001) KENNEWICK MAN: Descendant of Polynesian ruler claims rights to remains. Kitsap Sun. July 26.](https://web.archive.org/web/20200914065247/https://products.kitsapsun.com/archive/2001/07-26/0018_kennewick_man__descendant_of_poly.html)
* [NYT Staff. (2001) Northwest: Oregon: New Claim In Ancient-Bones Case. The New York Times. July 28.](https://web.archive.org/web/20110210102252/https://www.nytimes.com/2001/07/28/us/national-briefing-northwest-oregon-new-claim-in-ancient-bones-case.html)
* [Lawler, A. (2010). Grave disputes. Science. 330(6001): 166-170.](https://doi.org/10.1126/science.330.6001.166-a)
* [Achenbach, J. (2014) Scientists: Mysterious Kennewick Man looked Polynesian and came from far away.Washington Post. August 25.](https://web.archive.org/web/20140828073652/https://www.washingtonpost.com/national/health-science/scientists-mysterious-kennewick-man-looked-polynesian-and-came-from-far-away/2014/08/25/45411b2a-27b3-11e4-86ca-6f03cbd15c1a_story.html)
* [Rasmussen M, Sikora M, Albrechtsen A, Korneliussen TS, Moreno-Mayar JV, Poznik GD, et al. (2015) The ancestry and affiliations of Kennewick Man. Nature. 523: 455–458. 10.1038/nature14625](https://www.nature.com/articles/nature14625)
* [Raja, Tasneem. (2016) A Long, Complicated Battle Over 9,000-Year-Old Bones Is Finally Over. Code Switch. NPR. May 5.](https://web.archive.org/web/20160508050128/https://www.npr.org/sections/codeswitch/2016/05/05/476631934/a-long-complicated-battle-over-9-000-year-old-bones-is-finally-over)
* Great podcast: https://digpodcast.org/2019/02/03/skull-collectors-race-pseudoscience-native-american-bodies/

That being said, here is a the high-level overview of the very basics of human genetics:

* An average person has an estimated 37,200,000,000,000 cells containing some genetic material.
* Genetic material contains DNA and RNA.
* Only nucleated cells have DNA.
* Both RNA and DNA are passed on from parents to offspring, but the mechanism for some of the transfer is not fully understood.
* A complete DNA sequence has over 6,000,000,000 base pairs.
* Base pairs are two each of (A - adenine, C - cytosine, G - guanine, T - thymine). These have a particular way of matching up with each other so only certain combinations are possible.
* These base pairs form strands which are weakly bonded through the base pair bond. One stand is the sense strand, the other is the anti-sense strand.
* These two strands form a double helix.
* The DNA is stored in 23 pairs of chromosomes in the nucleus of a cell.
* Parts of chromosomes are broken down into genes, which code for specific molecules.
* The DNA is comprised of sequences which code for molecules (exons) and DNA which typically gets turned into RNA (introns).
* In exons, every three base pairs codes for a part of a protein. This unit is called a codon.
* Any gene coding for a protein can have mutations or several options (one might code for brown eyes, another for blue).
* The different possible genes (with mutations or changes between them) are called alleles.
* The differences between alleles are typically reduced to single nucleotide polymorphisms (SNPs). An SNP is the smallest change (mutation) one can make to a region that codes for molecules by swapping out one of the letters that code for a nucleotide. Many times, the alleles will have long sequences of SNPs.
* The exons (complete set of genetic material that codes for molecules) comprise about 1% of genetic material.
* Many genetic tests check a maximum of 10% of the exon SNPs.
* This means genetic tests only check a maximum of 0.1% of genetic material, unless they do full exon sequencing tests.
* The complete genetic sequence of any human being has not been sequenced. See the article: [Psst, the human genome was never completely sequenced. Some scientists say it should be](https://www.statnews.com/2017/06/20/human-genome-not-fully-sequenced/).
* Additionally, only populations which are sedentary and not traveling and continually mixing DNA develop distinct statistical differences. Those traveling together will likely receive similar mutations to SNPs through interacting with a similar environment.
* Additionally, there is heritable epigenetic coding. Most tests and research papers ignore this. Also, we don't fully understand how this happens since the epigenetic coding is wiped and reprogrammed through an as of yet undiscovered process.

You should take a look at [Whole genome sequencing](https://en.wikipedia.org/wiki/Whole_genome_sequencing) over at Wikipedia for more info.

## General Genetics

### Overview

The following Wikipedia articles contain the basics needed to progress to the next set of papers. You'll likely need to follow all of the links in the articles and fully explore the concepts.

* [Introduction to genetics](https://en.wikipedia.org/wiki/Introduction_to_genetics)
* [DNA](https://en.wikipedia.org/wiki/DNA)
* [RNA](https://en.wikipedia.org/wiki/RNA)
* [Genetic code](https://en.wikipedia.org/wiki/Genetic_code)
* [Genome](https://en.wikipedia.org/wiki/Genome)
* [Gene](https://en.wikipedia.org/wiki/Gene)
* [Gene expression](https://en.wikipedia.org/wiki/Gene_expression)
* [Gene–environment interaction](https://en.wikipedia.org/wiki/Gene%E2%80%93environment_interaction)
* [Epigenetics](https://en.wikipedia.org/wiki/Epigenetics)

### Not overview

The following journal papers contain most of the overview that one would need to understand genetics at a higher level necessary to understand the papers in the next section. In addition to these papers, a sound basis in math and statistics is needed to understand many of the methods used for creating the various graphs and relationship trees having the most parsimony.

* [Reich D (2016) The Genomic Ancient DNA Revolution. Edge. Feb.](https://www.edge.org/conversation/david_reich-the-genomic-ancient-dna-revolution)
* [Salem MSZ (2016) Formal genetic maps. Egyptian Journal of Medical Human Genetics. 16(2): 101-116.](https://doi.org/10.1016/j.ejmhg.2014.11.006)
* [Jo B, Chois SS (2015) Introns: The Functional Benefits of Introns in Genomes. Genomics & Informatics. 13(4): 112-118.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4742320/)

### Relatedness and Divergence over time.

* [Sethuraman A (2018) Estimating Genetic Relatedness in Admixed Populations. Genes Genomes Genetics. 8(10): 3203–3220.](https://dx.doi.org/10.1534%2Fg3.118.200485)
* [McIver LJ, McCormick JF, Martin A, Fondon III JW, Garner HR (2013) Population-scale analysis of human microsatellites reveals novel sources of exonic variation. Gene. 516(2)](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3815531/)
* [Kwong M, Pemberton TJ (2014) Sequence differences at orthologous microsatellites inflate estimates of human-chimpanzee differentiation. BMC Genomics. 15: 990.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4253012/)
* [Vaksman Z, Fonville NC, Tae H, Garner HR (2014) Exome-Wide Somatic Microsatellite Variation Is Altered in Cells with DNA Repair Deficiencies. PLoS ONE 9(11): e110263. https://doi.org/10.1371/journal.pone.0110263](https://doi.org/10.1371/journal.pone.0110263)
* [Pemberton TJ et al. (2013) Population Structure in a Comprehensive Genomic Data Set on Human Microsatellite Variation. Genes Genomes Genetics. 3(5): 891-907.](https://doi.org/10.1534/g3.113.005728)
* [López Herráez D, Bauchet M, Tang K, Theunert C, Pugach I, Li J, et al. (2009) Genetic Variation and Recent Positive Selection in Worldwide Human Populations: Evidence from Nearly 1 Million SNPs. PLoS ONE 4(11): e7888. https://doi.org/10.1371/journal.pone.0007888](https://doi.org/10.1371/journal.pone.0007888)
* [Rosenberg NA, Mahajan S, Ramachandran S, Zhao C, Pritchard JK, Feldman MW (2005) Clines, Clusters, and the Effect of Study Design on the Inference of Human Population Structure. PLoS Genet 1(6): e70. https://doi.org/10.1371/journal.pgen.0010070](https://doi.org/10.1371/journal.pgen.0010070)
* [Zhivotovsky LA, Rosenberg NA, Feldman MW (2003) Features of Evolution and Expansion of Modern Humans, Inferred from Genomewide Microsatellite Markers. American Journal of Human Genetics. 72(5): 1171–1186.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1180270/)
* [Driscoll CA et al. (2002) Genomic Microsatellites as Evolutionary Chronometers: A Test in Wild Cats. Genome Research. 12(3):414-23.](https://www.researchgate.net/publication/11487684_Genomic_Microsatellites_as_Evolutionary_Chronometers_A_Test_in_Wild_Cats)

### Accounting for Sequencing Errors

* [Ma X et al. (2019) Analysis of error profiles in deep next-generation sequencing data. Genome Biology. 20.](https://doi.org/10.1186/s13059-019-1659-6)
* [Bilton TP et al. (2018) Accounting for Errors in Low Coverage High-Throughput Sequencing Data When Constructing Genetic Maps Using Biparental Outcrossed Populations. 209(1): 65-76.](https://doi.org/10.1534/genetics.117.300627 )
* [Pfeiffer F et al. (2018) Systematic evaluation of error rates and causes in short samples in next-generation sequencing. Scientific Reports. 8(1): doi: 10.1038/s41598-018-29325-6](https://doi.org/10.1038/s41598-018-29325-6)
* [Wall JD et al. (2014) Estimating genotype error rates from high-coverage next-generation sequence data. Genome Research. 24(11): 1734-1739.](https://dx.doi.org/10.1101%2Fgr.168393.113)
* [Trimble WL et al. (2012) Short-read reading-frame predictors are not created equal: sequence error causes loss of signal. BMC Informatics. 13: 183. doi: 10.1186/1471-2105-13-183](https://doi.org/10.1186/1471-2105-13-183)
* [Wang XV et al. (2012) Estimation of sequencing error rates in short reads. BMC Bioinformatics. 13: 185. doi: 10.1186/1471-2105-13-185](https://dx.doi.org/10.1186%2F1471-2105-13-185)
* [Richterich T (1998) Estimation of Errors in "Raw" DNA Sequences: A Validation Study. Genome Research. 8: 251-259. doi: 10.1101/gr.8.3.251](https://genome.cshlp.org/content/8/3/251.full.html)

### The Myth of "Race"

* [Norton, H.L., Quillen, E.E., Bigham, A.W. et al. Human races are not like dog breeds: refuting a racist analogy. Evo Edu Outreach 12, 17 (2019). https://doi.org/10.1186/s12052-019-0109-y](https://doi.org/10.1186/s12052-019-0109-y)
* [Carja O et al. (2017) Worldwide patterns of human epigenetic variation. Nature Ecology & Evolution. 1: 1577–1583.](https://www.nature.com/articles/s41559-017-0299-z) &mdash; [bioRXiv](https://www.biorxiv.org/content/early/2015/07/05/021931)
* [Rosenberg NA (2011) A Population-Genetic Perspective on the Similarities and Differences among Worldwide Human Populations. Human Biology. 83(6): 659-684.](https://doi.org/10.3378/027.083.0601) &mdash; [Open Access](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3531797/)
* [Li JZ, Absher DD, et al. (2008) Worldwide Human Relationships Inferred from Genome-Wide Patterns of Variation. Science. 319(5866): 1100-1104.](https://science.sciencemag.org/content/319/5866/1100) &mdash; [PDF](https://pdfs.semanticscholar.org/b8de/1df47f30b54c57dce03fb14a4a94459b2e46.pdf) &mdash; Re-iterates the above finding.
  * `Together, these results reaffirm that within-population variation accounts for most of the genetic diversity in humans. However, the between-population variance is sufficient to reveal consistent population structure because subtle but nonrandom differences between populations accumulate over a large number of loci and yield principal components that can account for a major portion of the variation (21).`
* [Race, Ethnicity, and Genetics Working Group (2005) The Use of Racial, Ethnic, and Ancestral Categories in Human Genetics Research. American Journal of Human Genetics. 77(4): 519–532.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1275602/)
* [Excoffier, L., & Hamilton, G. (2003). Comment on genetic structure of human populations. Science, 300(5627), 1877. DOI: 10.1126/science.1083411](https://doi.or/10.1126/science.1083411)
* [Rosenberg NA et al. (2002) Genetic Structure of Human Populations. Science. 298(5602): 2381-2385.](https://science.sciencemag.org/content/298/5602/2381.full) &mdash; [PDF](https://www.researchgate.net/publication/10980474_Genetic_Structure_of_Human_Populations) &mdash; This is the original paper that started the meme that humans are roughly 99.9% the same. The biggest take away from the paper is that self reporting would likely be a reliable proxy for what genetic conditions they might get. The second take away was the intra-group genetic variation is greater than inter-group genetic variation.

## Australian, Melanesian, and Polynesian Connections

* [Castro, M. A., Ferraz, T., Bortolini, M. C., Comas, D., & Hünemeier, T. (2021). Deep genetic affinity between coastal Pacific and Amazonian natives evidenced by Australasian ancestry. Proceedings of the National Academy of Sciences, 118(14).](https://doi.org/10.1073/pnas.2025739118)
  * > This provides strong evidence that a significant variability of this signal exists not only at an interpopulation level but also between individuals from the same populations. These results suggest that the intrapopulation variability of this signal is not rare (Fig. 2) and is observed in several groups (Apalai, Guaraní Nãndeva, Karitiana, Munduruku, Parakanã, and Xavánte). Most significant tests detected this excess signal in Tupí-speaking individuals, but the signal was also detected in individuals from every major linguistic group (Fig. 2 and Dataset S4) and, at the same time, presented a widespread geographic distribution within South America (Fig. 1). Conversely, a considerable number of samples were inferred to have a deficit of allele sharing with Australasians (Fig. 2 and Dataset S4D). Strikingly, the individual PAR137 (Parakanã) presented an extremely high proportion of significant tests (31.64%), indicating a relative deficit. This individual is not an outlier neither in the principal component analysis of the Native American samples (Dataset S1 B and C), nor regarding its missingness rate (Dataset S1A), nor in a multidimensional scaling (MDS) of pairwise genetic distances between samples in the unrelated and unadmixed subset (Dataset S1D). Besides, the distribution of Y-population ancestry among present-day indigenous groups of South America showed no relationship with ethnolinguistic diversity or geographic location.
  * > As the Chotuna group in the Pacific coast also exhibited excess allele sharing (Fig. 1 and Dataset S3) with the Australasians as estimated by D statistic (Mbuti; Australasians: Y, Z), we created admixture graph models based on the scaffold of Skoglund et al. (2) (Fig. 3A) with the addition of the Pacific coastal groups Sechura, Chotuna, and Narihuala. The best-fitted model showed that the Pacific coast is a mixed group of South American ancestry and a small non-American contribution associated with a sister branch of Onge (Fig. 3C), as also observed for Karitiana and Suruí. When the Xavánte were included in the analysis, the best-fitted model showed a direct contribution of the Australasian component in the Pacific coast, followed by a strong drift of this signal, giving rise to Amazonian groups (Fig. 3D). Although Fig. 3D could indicate two independent events, the small genetic distances between the nodes in this model reinforced the single admixture event evidence. The Treemix (10) analysis also showed a pattern of diversification in which Pacific coastal and Andean groups diverged first (Dataset S6), followed by the eastern Andean slopes populations and then, finally, the Amazonians and other eastern South Americans. These findings suggest that the Y-population contribution was introduced before the formation of the Amazonian branch, likely in the ancestors of Pacific coastal/Amazonian populations.
  * > Different migration routes to the South American region have been previously proposed and evidenced. Archeological and genetic data demonstrated that both routes, Pacific coastal and inland, were likely used by the first migrants (11). Our models point to an ancient genetic affinity between the Pacific coast and Amazonian populations that could be explained by the presence of Y ancestry in both geographic regions. In addition, this shared ancestry seems to precede the separation of the Pacific and Amazon branches, showing an entry through the west coast, followed by successive events of genetic drift in the Brazilian populations. This genetic evidence for the presence of Y ancestry on the South American Pacific coast indicates that this ancestry likely reached this region through the Pacific coastal route, and therefore could explain absence of this genetic component in the populations of North and Central America studied so far.
  * [Price M. (2021) Earliest South American migrants had Indigenous Australian, Melanesian ancestry. Science News. March 29.](https://www.sciencemag.org/news/2021/03/earliest-south-american-migrants-had-australian-melanesian-ancestry)
    * > For the first time, scientists identified the Y signal in groups living outside the Amazon—in the Xavánte, who live on the Brazilian plateau in the country’s center, and in Peru’s Chotuna people, who descend from the Mochica civilization that occupied that country’s coast from about 100 C.E. to 800 C.E.
    * > Next, the researchers used software to test different scenarios that might have led to the current DNA dispersal. The best fit scenario involves some of the very earliest—possibly even the earliest—South American migrants carrying the Y signal with them, the researchers report today in the Proceedings of the National Academy of Sciences. Those migrants likely followed a coastal route, Hünemeier says, then split off into the central plateau and Amazon sometime between 15,000 and 8000 years ago. “[The data] match exactly what you’d predict if that were the case,” Raff agrees.
* [Ioannidis, A.G., Blanco-Portillo, J., Sandoval, K. et al. (2020) Native American gene flow into Polynesia predating Easter Island settlement. Nature.](https://doi.org/10.1038/s41586-020-2487-2)
  * > Here we analyse genome-wide variation in individuals from islands across Polynesia for signs of Native American admixture, analysing 807 individuals from 17 island populations and 15 Pacific coast Native American groups. We find conclusive evidence for prehistoric contact of Polynesian individuals with Native American individuals (around AD 1200) contemporaneous with the settlement of remote Oceania13,14,15. Our analyses suggest strongly that a single contact event occurred in eastern Polynesia, before the settlement of Rapa Nui, between Polynesian individuals and a Native American group most closely related to the indigenous inhabitants of present-day Colombia.
  * [Armitage, Hanae. Polynesians, Native Americans made contact before European arrival, genetic study finds. Stanford Medicine News Center. July 8. 2020.](https://med.stanford.edu/news/all-news/2020/07/polynesians-and-native-americans-made-early-contact.html)
    * > Before the study brought scientific evidence to the debate, the idea that Native Americans and Polynesians had crossed paths originated from a complex — both in its structure and origins — carbohydrate: the sweet potato. It turns out the sweet potato, which was originally domesticated in South and Central America, has also been known to grow in one other place prior to European contact. That place is known as Oceania, which consists of many islands, including the Polynesian islands.
    * > “The sweet potato is native to the Americas, yet it’s also found on islands thousands of miles away,” Ioannidis said. “On top of that, the word for sweet potato in Polynesian languages appears to be related to the word used in Indigenous American languages in the Andes.”
    * > The overlap in culture made some archaeologists and historians think it was not only feasible, but likely, that the potato’s arrival in Polynesia was the result of the two peoples mingling.
    * > The researchers believe that the Polynesians landed in what is now Colombia. It is also possible, though less likely due to their coastal travel norms, that one or two ships carrying Native Americans could have sailed off course and run into Polynesia, Ioannidis said.
    * > Without scientific evidence, the idea of overlap was just conjecture. Earlier, other groups of researchers turned to the genetics of the sweet potato, hoping to show that the domesticated potatoes from South America and Polynesia were genetically one and the same. But their efforts to trace the tubers have been inconclusive, as the sweet potato’s genetic origins were too complex to definitively point to human-mediated spread.
  * [Zimmer, Carl. Some Polynesians Carry DNA of Ancient Native Americans, New Study Finds. The New York Times. July 8. 2020.](https://www.nytimes.com/2020/07/08/science/polynesian-ancestry.html)
    * > But six people had no European ancestry at all. Their Native American ancestry had a different source: the Zenu population of Colombia. The scientists then found some of the same pieces of DNA in people on four other islands in eastern Polynesia.
    * > The researchers found that all of the Zenu-like stretches of DNA in the Polynesians were roughly the same size. They estimated that they came from Zenu relatives who lived about eight centuries ago.
* [Posth C et al. (2018) Reconstructing the Deep Population History of Central and South America. Cell. DOI:https://doi.org/10.1016/j.cell.2018.10.027](https://www.cell.com/cell/fulltext/S0092-8674(18)31380-1)
  * > Recent analyses have also shown that some groups in Brazil share more alleles with Australasians (indigenous New Guineans, Australians, and Andaman Islanders) (Raghavan et al., 2015, Skoglund et al., 2015) and an ∼40,000 BP individual from northern China (Yang et al., 2017) than do other Central and South Americans. Such patterns suggest that these groups do not entirely descend from a single homogeneous population and instead derive from a mixture of populations, one of which, Population Y, bore a distinctive affinity to Australasians. Notably, our study includes data from individuals such as those from the Lapa do Santo site who have a cranial morphology known as “Paleoamerican,” argued to indicate two distinct New-World-founding populations (von Cramon-Taubadel et al., 2017). Here, we test directly the hypothesis that a Paleoamerican cranial morphology was associated with a lineage distinct from the one that contributed to other Native Americans (whether the proposed Population Y or another).
* [Moreno-Mayar JV et al. (2018) Early human dispersals within the Americas. Science. 10.1126/science.aav2621](https://www.researchgate.net/publication/328815661_Early_human_dispersals_within_the_Americas) &mdash; This is a fantastic paper. I'd love to apply it's techniques to a broader sample of populations and early DNA sequences.
  * > To test for the Australasian genetic signal in NAs, we computed D statistics of the form D(NA, NA; Eurasian, Yoruba), where NA represents all newly sequenced and reference high-depth NA genomes (13). In agreement with previous results (6), we found that the Amazonian Suruí share a larger proportion of alleles with Australasian groups (represented by Papuans, Australians, and Andaman Islanders) than do the Mixe (13). Lagoa Santa yielded results similar to those obtained for the Suruí: The analyzed Lagoa Santa genome also shares a larger proportion of alleles with Australasian groups, but not with other Eurasians, than do Mesoamerican groups (the Mixe and Huichol) (Fig. 4) (13). However, the Australasian signal is not present in the Spirit Cave individual, and we include this distinction in the admixture graph modeling (Figs. 3A and 4A) (13). We inferred less than 3% European contamination in the Lagoa Santa genome (<3%) (13) and show that this finding is robust to potential European contamination by computing “contamination-corrected” f4 statistics (Fig. 4B) (13). The presence of the Australasian genomic signature in Brazil 10.4 ka ago and its absence in all genomes tested to date that are as old or older and located farther north present a challenge in accounting for its presence in Lagoa Santa.
* [Kanzawa-Kiriyama, H., Kryukov, K., Jinam, T. A., Hosomichi, K., Saso, A., Suwa, G., ... & Saitou, N. (2017). A partial nuclear genome of the Jomons who lived 3000 years ago in Fukushima, Japan. Journal of human genetics, 62(2), 213-221.](https://doi.org/10.1038/jhg.2016.110)
  * Suggests that genes flowed from Karitiana (Native American, Brazil) into Mal'ta (MA1), not the other way around like "Genetic evidence for two founding populations of the Americas".
* [Fehren-Schmitz, L., Jarman, C. L., Harkins, K. M., Kayser, M., Popp, B. N., & Skoglund, P. (2017). Genetic ancestry of Rapanui before and after European contact. Current Biology, 27(20), 3209-3215.](https://doi.org/10.1016/j.cub.2017.09.029)
* [Skoglund, P., Mallick, S., Bortolini, M. C., Chennagiri, N., Hünemeier, T., Petzl-Erler, M. L., ... & Reich, D. (2015). Genetic evidence for two founding populations of the Americas. Nature, 525(7567), 104-108.](https://www.nature.com/articles/nature14895) &mdash; [PDF](https://www.researchgate.net/publication/280236692_Genetic_evidence_for_two_founding_populations_of_the_Americas)
  * Shows that Pima and Mixe groups are likely descended from proto-MA1 and proto-Han. Population Y is admixture of proto-Onge and proto-Pima-Mixe, Xavante, Surui, Karitiana lineages derive from Mixe and Population Y.
  * > The geographic distribution of the shared genetic signal between South Americans and Australasians cannot be explained by post-Columbian African, European or Polynesian gene flow into Native American populations. If such gene flow produced signals strong enough to impact our statistics, our statistics would show their strongest deviations from zero for African, European or Polynesian populations, which is not observed. For example, a direct test is significant in showing that the Suruí-specific ancestry component is genetically closer to the Andamanese Onge than to Tongans from Polynesia (D = 0.0094, Z = 3.4).
  * > We are unable to fit Amazonians as forming a clade with the Mesoamericans, or as having a different proportion of ancestry related to Mal'ta or present-day East Asians. Thus, our signal cannot be explained by lineages that have previously been documented as having contributed to Native American populations. However, we do find that a model where Amazonians receive ancestry from the lineage leading to the Andamanese fits the data in the sense that its predicted f4-statistics are all within 2 standard errors of statistics computed on the empirical data (Extended Data Figure 6; Extended Data Figure 7; Extended Data Table 3). These results do not imply that an unmixed population related anciently to Australasians migrated to the Americas. While this is a formal possibility, an alternative model that we view as plausible is that the ‘Population Y’ (we use ‘Population Y’ after Ypykuéra, which means ‘ancestor’ in the Tupi language family spoken by the Suruí and Kartiana) that contributed Australian related ancestry to Amazonians was already mixed with a lineage related to First Americans at the time it reached Amazonia. When we model such a scenario, we obtain a fit for models that specify 2%-85% of the ancestry of the Suruí, Karitiana, and Xavante as coming from Population Y (Figure 2). These results show that quite a high fraction of Amazonian ancestry today plausibly comes from Population Y. At the same time, the results constrain the fraction of Amazonian ancestry that comes from an Australasian related population (via Population Y) to a much tighter range of 1%-2% (Figure 2).
  * > We have provided compelling evidence that a Population Y that has ancestry from a lineage more closely related to present-day Australasians than to present-day East Asians and Siberians, contributed a small fraction of the DNA of Native Americans from Amazonia and the Central Brazilian Plateau. This discovery is striking in light of interpretations of the morphology of some early Native American skeletons, which some authors have suggested have affinities to Australasian groups. The largest number of skeletons that have been described as having this craniofacial morphology and that date to younger than ten thousand years have been found in Brazil6, the home of the Suruí, Karitiana and Xavante who in genetic data show the strongest affinity to Australasians. However, in the absence of DNA directly extracted from a skeleton with this morphology, our results are not sufficient to conclude that the Population Y we have reconstructed from the genetic data had this morphology.
  * > An open question is when and how Population Y ancestry reached South America. There are several archaeological sites in the Americas that are substantially older than Clovis sites. Since one Clovis site is now known from ancient DNA analysis to have included an individual of entirely First American ancestry3, an interesting hypothesis is that Population Y ancestry may have been prevalent in the individuals of some of these earlier sites. Regardless, our results suggest that at least two different ancestry streams penetrated south of the Late Pleistocene ice sheets, perhaps taking different routes or arriving at different times from a structured Beringian or Northeast Asian source, or reflecting more longstanding gene flow. **The genetic data allow us to say with confidence that Population Y ancestry arrived south of the ice sheets anciently: the fact that the geographically diverse Andamanese, Australian and New Guinean populations are all similarly related to this source suggests that the population is no longer extant, and the absence of long-range admixture linkage disequilibrium suggests that the population mixture did not occur in the last few thousand years.** Further insight into the population movements responsible for these findings should be possible through genome-wide analysis of ancient remains from across the Americas.
* [Moreno-Mayar, J. V., Rasmussen, S., Seguin-Orlando, A., Rasmussen, M., Liang, M., Flåm, S. T., ... & Willerslev, E. (2014). Genome-wide ancestry patterns in Rapanui suggest pre-European admixture with Native Americans. Current Biology, 24(21), 2518-2525.](https://doi.org/10.1016/j.cub.2014.09.057)
  * > The possibility of trans-Pacific Polynesian-American contact has been proposed by archaeologists since the early 1900s ([28], p. 1). Although an early contact has been proposed by previous studies [15, 16, 17], no unequivocal genetic evidence has been reported to date. Our results suggest that Native American admixture occurred prior to European admixture. Specifically, we detect a lower variance in Native American ancestry components, and smaller ancestry tracts, as compared to Europeans. To characterize the timing of the admixture event in a statistical framework, we infer parameters from specific demographic scenarios, using a maximum-likelihood method that has been validated before in similar human data sets (e.g., [45]), as well as a new method based on the variance in admixture proportions. Using these approaches, we can statistically reject “European first” scenarios. It is noteworthy that models with several pulses are a generalization of single-pulse models and will therefore provide a better fit to the data. In our case, we find that (1) adding a second European pulse does not improve the fit significantly when assuming that the European gene flow postdated the first documented presence of Europeans in the Pacific in AD 1520 [46], and (2) the second pulse was found to be of such low magnitude that the associated admixture time spanned a whole range of times equally well (Figure S5). 
  * > Although our data suggest that modern-day Rapanui are the result of a pre-Columbian gene flow event between Native Americans and Polynesians, we cannot, with the data at hand, determine how this gene flow happened. Nevertheless, it is interesting to consider the evidence presented in other fields. There are two main scenarios compatible with the data: Native Americans sailing to Rapa Nui, or Polynesians sailing to the Americas and back. Although “boats on occasion can sail from almost anywhere to anywhere” [47], from a purely seafaring capability perspective, it is probably fair to say that the latter scenario is more likely. Let us consider the geographical situation of Rapa Nui and the feasibility of such a voyage, especially in the context of the Polynesian expansion.
  * > Jones et al. [28] reviewed in 2011 the archaeological, physical anthropological, linguistic, and navigational evidence for pre-Columbian contact. They identified three regions of potential contact, namely Chile, the Gulf of Guayaquil in Ecuador, and the Santa Barbara Channel in California, and constrained the contacts to a period between AD 700 and AD 1350. They argued that the most parsimonious explanation in all cases is a return voyage by Polynesians, but they could not exclude the scenario of a one-way raft trip, as suggested for example in the mid-20th century by Thor Heyerdahl [11]. One of their primary arguments is the lack of Native American influence in Polynesia, particularly in terms of Polynesian architecture ([28], pp. 263–265)—aside, perhaps, from one of the platform sites on Rapa Nui, Ahu Vinapu.

## Genetic Evidence for the Peopling of the Americas

* [Urban, M., & Barbieri, C. (2020). North and South in the ancient Central Andes: Contextualizing the archaeological record with evidence from linguistics and molecular anthropology. Journal of Anthropological Archaeology, 60, 101233.](https://doi.org/10.1016/j.jaa.2020.101233) &mdash; [PDF](https://pure.mpg.de/rest/items/item_3257439/component/file_3257446/content)
* [Nakatsuka, N., Lazaridis, I., Barbieri, C., Skoglund, P., Rohland, N., Mallick, S., ... & Michel, M. (2020). A paleogenomic reconstruction of the deep population history of the andes. Cell.](https://doi.org/10.1016/j.cell.2020.04.015) &mdash; [PDF](https://www.semanticscholar.org/paper/A-Paleogenomic-Reconstruction-of-the-Deep-History-Nakatsuka-Lazaridis/08d47345499ef6578de9aaa9ec98201549c6db41?p2df)
* [Blell M, Hunter MA (2019) Direct-to-Consumer Genetic Testing's Red Herring: “Genetic Ancestry” and Personalized Medicine. Frontiers in Medicine. 29 March.](https://doi.org/10.3389/fmed.2019.00048)
* [Walajahi H, Wilson DR, Hull SC (2019) Constructing Identities: The Implications of DTC Ancestry Testing for Tribal Communities. Genetic Medicine. 21(8): 1744-1750.](https://dx.doi.org/10.1038%2Fs41436-018-0429-2)
* [Pinotti T et al. (2019) Y Chromosome Sequences Reveal a Short Beringian Standstill, Rapid Expansion, and early Population structure of Native American Founders. Current Biology. 29(1): 149-157.](https://doi.org/10.1016/j.cub.2018.11.029)
* [Sikora, M., Pitulko, V. V., Sousa, V. C., Allentoft, M. E., Vinner, L., Rasmussen, S., ... & Yang, M. A. (2019). The population history of northeastern Siberia since the Pleistocene. Nature, 570(7760), 182-188.](https://doi.org/10.1038/s41586-019-1279-z)
* [Flegontov, P., Altınışık, N. E., Changmai, P., Rohland, N., Mallick, S., Adamski, N., ... & Flegontova, O. (2019). Palaeo-Eskimo genetic ancestry and the peopling of Chukotka and North America. Nature, 570(7760), 236-240.](https://doi.org/10.1038/s41586-019-1251-y)
* [Gnecchi-Ruscone GA et al. (2019) Dissecting the Pre-Columbian Genomic Ancestry of Native Americans along the Andes–Amazonia Divide, Molecular Biology and Evolution. 36(6): 1254–1269](https://doi.org/10.1093/molbev/msz066)
  * > Accordingly, recent evidence based on genomic data generated from ancient human remains retrieved in Central and South America confirmed the occurrence of multiple waves of diffusion into the south of the continent and suggested a complex scenario involving the spread of ancient populations that were already genetically structured (Moreno-Mayar, Vinner, et al. 2018; Posth et al. 2018). Among these samples, the oldest ones (dating to ∼11–10 ka), either from North America (Nevada) or from South America (West and East of the Andes), were those showing the highest genetic affinity with Anzick-1. This corroborates the hypothesis that the first diffusion from North into South America was extremely rapid (∼1–2 ka) and was not limited to the West coast since it is supported by samples from the entire continent. However, the genetic footprints of this first peopling event were found to be subtler in more recent samples, suggesting an extensive population replacement beginning from around 9 ka, by a different ancestral lineage with respect to that represented in the Clovis-associated Anzick-1 (Posth et al. 2018). Subsequent migrations after the initial diffusion were associated with expansion from Mesoamerica occurred sometime after ∼8.7 ka, which spread first southward (contributing to the ancestry of all present-day South Americans) and then northward, as suggested by ∼2 ka ancient samples from Nevada (Moreno-Mayar, Vinner, et al. 2018). Despite the common view that these processes contributed significantly to the formation of the modern South American genomic landscape, the two above-mentioned studies did not clarify in detail the relative proportions of these ancestries in the genomes of contemporary populations, being instead focused mostly on the relationship between ancient samples. Finally, minor contributions to the South American gene pool (i.e., <5% of ancestry) were ascribable to the affinity with an Austro Melanesian–related ancestry source already attested for some present-day Amazonian groups (Skoglund et al. 2015). This pattern was recognized also in a ∼10-ka sample from Brazil (Moreno-Mayar, Vinner, et al. 2018), coupled with a newly described connection between ancient samples from the California Channel Islands and the Late Central Andes since around 4.2 ka (Posth et al. 2018).
  * Goes on to show genetic admixture and possible gene flow amoung groups.
* [Barbieri, C., Barquera, R., Arias, L., Sandoval, J. R., Acosta, O., Zurita, C., ... & Mafessoni, F. (2019). The Current Genomic Landscape of Western South America: Andes, Amazonia, and Pacific Coast. Molecular biology and evolution, 36(12), 2698-2713.](https://doi.org/10.1093/molbev/msz174)
  * Examined gene flow using quite a few methods.
* [Grugni V et al. (2019) Analysis of the human Y-chromosome haplogroup Q characterizes ancient population movements in Eurasia and the Americas. BMC Biology. 17, 3.](https://bmcbiol.biomedcentral.com/articles/10.1186/s12915-018-0622-4)
* [Wright JL et al. (2018) Ancient nuclear genomes enable repatriation of Indigenous human remains. Science Advances. 4(12): eaau5064.](https://dx.doi.org/10.1126%2Fsciadv.aau5064)
* [Wade L (2018) Ancient DNA confirms Native Americans’ deep roots in North and South America. Science Magazine.](https://www.sciencemag.org/news/2018/11/ancient-dna-confirms-native-americans-deep-roots-north-and-south-america)
* [Barbieri C et al. (2018) The genomic landscape of western South America: Andes, Amazonia and Pacific Coast. bioRXiv preprint.](https://www.biorxiv.org/content/10.1101/505628v1.full)
* [Posth C et al. (2018) Reconstructing the Deep Population History of Central and South America. Cell. DOI:https://doi.org/10.1016/j.cell.2018.10.027](https://www.cell.com/cell/fulltext/S0092-8674(18)31380-1)
* > A second notable finding of this study is our evidence that the ancient individuals from the California Channel Islands have distinctive and significant allele sharing with groups that became widespread over the Central Andes after ∼4,200 BP. There is no archaeological evidence of large-scale cultural exchange between North and South America around this time, but it is important to recognize that ∼4,200 BP is a minimum date for the exchange between North and South American that drove this pattern; the gene flow itself could have occurred thousands of years before and the ancestry deriving from it could have persisted in a region of South America not yet sampled with ancient DNA. The evidence of an expansion of this ancestry type in the Central Andes by ∼4,200 BP is notable in light of the increasing density of sites in this region at approximately this time, a pattern that is consistent with a demographic expansion of a previously more restricted population (Goldberg et al., 2016).
* [Messina F, Di Corcia T, Ragazzo M, Sanchez Mellado C, Contini I, Malaspina P, et al. (2018) Signs of continental ancestry in urban populations of Peru through autosomal STR loci and mitochondrial DNA typing. PLoS ONE 13(7): e0200796.](https://doi.org/10.1371/journal.pone.0200796)
* [Moreno-Mayar JV et al. (2018) Early human dispersals within the Americas. Science. 10.1126/science.aav2621](https://www.researchgate.net/publication/328815661_Early_human_dispersals_within_the_Americas) &mdash; This is a fantastic paper. I'd love to apply it's techniques to a broader sample of populations and early DNA sequences.
  * > The ~5.1-ka-old Patagonian Ayayema genome is an exception; it forms a clade with the Lagoa Santa population. This suggests that the arrival of the Mesoamerican-related ancestry occurred post–5.1 ka ago and/or that it did not reach the remote region inhabited by the Ayayema individual’s ancestors (Fig. 3C) (13). This result is qualitatively mirrored by the 7.2-ka-old Punta Santa Ana individual (both cluster with present-day Patagonians and form a clade with Lagoa Santa). However, the low coverage of Punta Santa Ana may reduce our power to detect possible Mesoamerican-related admixture (Fig. 3C) (13). [Uses Mixe as a proxy for Mesoamerican allels]
  * > Mesoamerican-related expansion possibly had a bearing on a later, unresolved pattern seen in North America. In the western Great Basin of North America, paleoenvironmental evidence indicates decreased effective precipitation and increased aridity during the Middle Holocene, which led to a human population decline (45, 46). By ~5 ka ago, regional populations were rebounding, but whether these were descendants of the previous inhabitants is unknown. Unclear also is the relationship between those later Holocene groups and the people present in the region at the time of European contact and today. Linguistic evidence suggests that ancestors of Numic speakers presently inhabiting the region today arrived recently, perhaps ~1 ka ago. There is also archaeological evidence of changes in material culture around that time, though how those relate to the linguistic turnover is uncertain. Nor is it known whether these changes are related to population admixture or replacement. Patterns and changes in language, material culture, and genetics need not be congruent or causally linked (47). Thus, the so-called Numic Expansion hypothesis has been highly debated (46, 48); we address the population aspect by comparing genomes from Spirit Cave and Lovelock Cave (Fig. 1A) (13).
  * > MDS and ADMIXTURE analyses, as well as D statistics of the form D(SpiritCave, Lovelock2/3; NA, Yoruba), suggest that despite the ~9 ka separating the Spirit Cave and Lovelock individuals, they form a clade with respect to other NAs (Fig. 1, C to F) (13). We tested that topology through the same admixture graph search implemented for SNAs (13). We were not able to reject the model without Mesoamerican-related admixture for Lovelock 2 (~1.9 ka old). However, the ~0.7-ka-old Lovelock 3 individual received Mesoamerican admixture from a group that was likely not present in the region just ~1.2 ka earlier, at the time of Lovelock 2. Because we do not know the language(s) that may have been involved, we cannot securely attribute this admixture to arriving Numic speakers [the Mixe, whom we use as a proxy for Mesoamerican ancestry, fall in a separate language family from Numic (49)]. Notably, we also observe genetic continuity, suggesting that there was not a complete population replacement.
* [Scheib CL et al. (2018) Ancient human parallel lineages within North America contributed to a coastal expansion. Science. 360(6392): 1024-1027.](https://science.sciencemag.org/content/360/6392/1024) &mdash; [PDF](https://www.academia.edu/36866318/Ancient_human_parallel_lineages_within_North_America_contributed_to_a_coastal_expansion)
* [Gómez-Carballa A et al. (2018) The peopling of South America and the trans-Andean gene flow of the first settlers. Genome Research. 28: 767-779.](https://genome.cshlp.org/content/28/6/767.full)
* [Brandini S et al. (2018) The Paleo-Indian Entry into South America According to Mitogenomes. Molecular Biology and Evolution. 35(2): 299-311.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5850732/)
* [Hamilton-Brehm SD, Hristova LT, Edwards SR, Wedding JR, Snow M, Kruger BR, et al. (2018) Ancient human mitochondrial DNA and radiocarbon analysis of archived quids from the Mule Spring Rockshelter, Nevada, USA. PLoS ONE 13(3): e0194223. https://doi.org/10.1371/journal.pone.0194223](https://doi.org/10.1371/journal.pone.0194223)
* [KINY Radio (2018) DNA of Indigenous peoples explained. KINY Radio.](http://www.kinyradio.com/news/news-of-the-north/dna-of-indigenous-peoples-explained/?fbclid=IwAR0_VYafbGYaKQsvQaGldUY5lLVn3g9uhYzhy1tgWiN-xHupxOoG80Q4k1A)
* [Hohenstatt B (2018) DNA, oral tradition indicate 10,000 years of Native history in Southeast. Juneau Empire. November 13.](https://www.juneauempire.com/news/dna-oral-tradition-indicate-10000-years-of-native-history-in-southeast/)
* [de Saint Pierre M (2017) Antiquity of mtDNA lineage D1g from the southern cone of South America supports pre-Clovis migration. Quaternary International. 444 B: 19-25.](https://doi.org/10.1016/j.quaint.2017.05.054) &mdash; [PDF](https://www.semanticscholar.org/paper/Antiquity-of-mtDNA-lineage-D1g-from-the-southern-of-Pierre/66b3ebf226025bb8c54ab5ed13ff2bc3988cd18c)
* [Morrow JE (2017) After Anzick: Reconciling new genomic data and models with the archaeological evidence for peopling of the Americas. Quaternary International. Part B. 444, 1-3.](https://doi.org/10.1016/j.quaint.2017.07.002) &mdash; [PDF](https://www.researchgate.net/publication/318442263_After_Anzick_Reconciling_new_genomic_data_and_models_with_the_archaeological_evidence_for_peopling_of_the_Americas)
* [Lindo J et al. (2017) Ancient individuals from the North American Northwest Coast reveal 10,000 years of regional genetic continuity. PNAS. 114(16): 4093-4098.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5402414/)
* [González-Oliver A, Garfias-Morales E, Smith DG, Quinto-Sánchez M (2017) Mitochondrial DNA Analysis of Mazahua and Otomi Indigenous Populations from Estado de México Suggests a Distant Common Ancestry. Human Biology. 89(3): 3.](https://digitalcommons.wayne.edu/humbiol/vol89/iss3/3/)
* [Llamas B (2016) Ancient mitochondrial DNA provides high-resolution time scale of the peopling of the Americas. Science Advances. 2(4).](https://advances.sciencemag.org/content/2/4/e1501385)
* [Ochoa-Lugo MI, de Lourdez Muñoz M, et al. (2016) Genetic Affiliation of Pre-Hispanic and Contemporary Mayas through Maternal Linage. Human Biology. 88(2):136-167.](https://www.researchgate.net/publication/313483707_Genetic_Affiliation_of_Pre-Hispanic_and_Contemporary_Mayas_Through_Maternal_Linage)
* [López, John Emilio William (2016) Genetic Diversity in Urban and Rural Indigenous Mexico. Master's thesis, Harvard Extension School.](https://dash.harvard.edu/handle/1/33797263)
* [Rasmussen M, Sikora M, Albrechtsen A, Korneliussen TS, Moreno-Mayar JV, Poznik GD, et al. (2015) The ancestry and affiliations of Kennewick Man. Nature. 523: 455–458. 10.1038/nature14625](https://www.nature.com/articles/nature14625)
* [González-Martín A, Gorostiza A, Regalado-Liu L, Arroyo-Peña S, Tirado S, Nuño-Arana I, et al. (2015) Demographic History of Indigenous Populations in Mesoamerica Based on mtDNA Sequence Data. PLoS ONE 10(8): e0131791. https://doi.org/10.1371/journal.pone.0131791](https://doi.org/10.1371/journal.pone.0131791)
* [Vilar MG et al. (2014) Genetic diversity in Puerto Rico and its implications for the peopling of the Island and the West Indies. American Journal of Physical Anthropology. 155: 352-368.](https://www.academia.edu/20083458/Genetic_diversity_in_Puerto_Rico_and_its_implications_for_the_peopling_of_the_Island_and_the_West_Indies)
* [Brooks DR (2014) Genetic Diversity in Totonacan-Speaking Populations from Veracruz and Puebla States of Mexico. Proceedings of the National Conference of Undergraduate Research.](http://web.archive.org/web/20200218052502/http://www.ncurproceedings.org/ojs/index.php/NCUR2014/article/view/1087)
* [Raghavan M, Skoglund P, et al. (2014) Upper Palaeolithic Siberian genome reveals dual ancestry of Native Americans. Nature. 505(7481): 87-91.](https://dx.doi.org/10.1038%2Fnature12736) &mdash; [Open Access](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4105016/)
  * `The MA-1 mitochondrial genome belongs to haplogroup U, which has also been found at high frequency among Upper Palaeolithic and Mesolithic European hunter-gatherers10–12, and the Y chromosome of MA-1 is basal to modern-day western Eurasians and near the root of most Native American lineages5. Similarly, we find autosomal evidence that MA-1 is basal to modern-day western Eurasians and genetically closely related to modern-day Native Americans, with no close affinity to east Asians. This suggests that populations related to contemporary western Eurasians had a more north-easterly distribution 24,000 years ago than commonly thought. Furthermore, we estimate that 14 to 38% of Native American ancestry may originate through gene flow from this ancient population. This is likely to have occurred after the divergence of Native American ancestors from east Asian ancestors, but before the diversification of Native American populations in the New World.`
  * `The genetic affinity between Native Americans and MA-1 could be explained by gene flow after the split between east Asians and Native Americans, either from the MA-1 lineage into Native American ancestors or from Native American ancestors to the ancestors of MA-1. However, MA-1, at approximately 24,000 cal. bp, pre-dates time estimates of the Native American–east Asian population divergence event24,25. This presents little time for the formation of a diverged Native American gene pool that could have contributed ancestry to MA-1, suggesting gene flow from the MA-1 lineage into Native American ancestors. Such gene flow should also be detectable using modern-day western Eurasian populations in place of MA-1. Consistent with this, D-statistic tests estimated from outgroup-ascertained SNP data20 reveal significant evidence (Z == 3) for Middle Eastern, European, central Asian and south Asian populations being closer to Karitiana than to Han Chinese20 (Fig. 3b and Supplementary Information, section 14.5). Similar signals were also observed when we replaced modern-day Han Chinese with data from chromosome 21 from a 40,000-yearold east Asian individual (Tianyuan Cave, China), which has been found to be ancestral to modern-day Asians and Native Americans26 (Supplementary Information, section 14.5). Thus, if the gene flow direction was from Native Americans into western Eurasians it would have had to spread subsequently to European, Middle Eastern, south Asian and central Asian populations, including MA-1 before 24,000 years ago. Moreover, as Native Americans are closer to Han Chinese than to Papuans (Fig. 3c), Native American-related gene flow into the ancestors of MA-1is expected to result in MA-1 also being closer to Han Chinese than to Papuans. However, our results suggest that this is not the case (D (Papuan, Han; Sardinian, MA-1) = 20.002 ± 0.005 (Z = 20.36)), which is compatible with all or almost all of the gene flow being into Native Americans (Supplementary Information, section 14.6). Similar results are obtained when MA-1 is replaced with most modern-day western Eurasian populations, except populations with recent admixture from east Asia (Russian, Adygei and Burusho) and Africa (Middle Eastern populations) (Fig. 3c). The most parsimonious explanation for these results is that Native Americans have mixed origins, resulting from admixture between peoples related to modern-day east Asians and western Eurasians.`
* [Gravel S, Zakharia F, Moreno-Estrada A, Byrnes JK, Muzzio M, Rodriguez-Flores JL, et al. (2013) Reconstructing Native American Migrations from Whole-Genome and Whole-Exome Data. PLoS Genet 9(12): e1004023. https://doi.org/10.1371/journal.pgen.1004023](https://doi.org/10.1371/journal.pgen.1004023)
* [Amorim CEG, Bisso-Machado R, Ramallo V, Bortolini MC, Bonatto SL, Salzano FM, et al. (2013) A Bayesian Approach to Genome/Linguistic Relationships in Native South Americans. PLoS ONE 8(5): e64099. https://doi.org/10.1371/journal.pone.0064099](https://doi.org/10.1371/journal.pone.0064099)
* [Schurr TG et al. (2012) Clan, Language, and Migration History Has Shaped Genetic Diversity in Haida and Tlingit Populations From Southeast Alaska. American Journal of Physical Anthropology. 148(3): 422-435.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4335652/)
* [de Saint Pierre M, Bravi CM, Motti JMB, Fuku N, Tanaka M, Llop E, et al. (2012) An Alternative Model for the Early Peopling of Southern South America Revealed by Analyses of Three Mitochondrial DNA Haplogroups. PLoS ONE 7(9): e43486. https://doi.org/10.1371/journal.pone.0043486](https://doi.org/10.1371/journal.pone.0043486)
* [Gorostiza A, Acunha-Alonzo V, Regalado-Liu L, Tirado S, Granados J, Sámano D, et al. (2012) Reconstructing the History of Mesoamerican Populations through the Study of the Mitochondrial DNA Control Region. PLoS ONE 7(9): e44666. https://doi.org/10.1371/journal.pone.0044666](https://doi.org/10.1371/journal.pone.0044666)
* [Kemp BM, González-Oliver A, et al. (2010) Evaluating the Farming/Language Dispersal Hypothesis with genetic variation exhibited by populations in the Southwest and Mesoamerica. PNAS. 107(15): 6759-6764.](https://doi.org/10.1073/pnas.0905753107)
* [Rasmussen M, Li Y, et al. (2010) Ancient human genome sequence of an extinct Palaeo-Eskimo. Nature. 463: 757-762.](https://rdcu.be/bOjbe)
* [Achilli A, Perego UA, Bravi CM, Coble MD, Kong Q-P, Woodward SR, et al. (2008) The Phylogeny of the Four Pan-American MtDNA Haplogroups: Implications for Evolutionary and Disease Studies. PLoS ONE 3(3): e1764. https://doi.org/10.1371/journal.pone.0001764](https://doi.org/10.1371/journal.pone.0001764)
* [Melton, Philip Edward (2008) Genetic history and pre-Columbian Diaspora of Chibchan speaking populations: Molecular genetic evidence. Ph.D. thesis, University of Kansas.](https://kuscholarworks.ku.edu/handle/1808/4508?show=full)
* [Tamm E, Kivisild T, Reidla M, Metspalu M, Smith DG, Mulligan CJ, et al. (2007) Beringian Standstill and Spread of Native American Founders. PLoS ONE 2(9): e829. https://doi.org/10.1371/journal.pone.0000829](https://doi.org/10.1371/journal.pone.0000829)
* [Kemp BM et al. (2007), Genetic analysis of early holocene skeletal remains from Alaska and its implications for the settlement of the Americas. Am. J. Phys. Anthropol., 132: 605-621. doi:10.1002/ajpa.20543](https://doi.org/10.1002/ajpa.20543) &mdash; [PDF](https://www.igb.illinois.edu/sites/default/files/Kempetal2007.pdf)
* [Reséndez A, Kemp BM, et al. (2005) "An Analysis of Ancient Aztec mtDNA from Tlatelolco: Pre-Columbian Relations and the Spread of Uto-Aztecan." Biomolecular Archeology: Genetic Approaches to the Past. Center for Archeological Investigations, Ocassional Paper No. 32.](https://migrationcluster.ucdavis.edu/research/national-identity-shifting-border-texas-and-new-mexico-age-transition-1821-1848-0)
* [Malhi RS et al. (2003) Native American mtDNA prehistory in the American Southwest. American Journal of Physical Anthropology. 120(2): 108-124. <http://hdl.handle.net/2027.42/34277>](http://hdl.handle.net/2027.42/34277)
* [Hurles, M. E., Matisoo-Smith, E., Gray, R. D., & Penny, D. (2003). Untangling Oceanic settlement: the edge of the knowable. Trends in Ecology & Evolution, 18(10), 531-540.](https://doi.org/10.1016/S0169-5347(03)00245-3)
* [Torroni A, Schurr TG, et al. (1993) Asian affinities and continental radiation of the four founding Native American mtDNAs. American Journal of Human Genetics. American Journal of Human Genetics. 53(3): 563-590.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1682412/)
* [Torroni A, Sukernik RI, Schurr TG, et al. (1993). mtDNA variation of aboriginal Siberians reveals distinct genetic affinities with Native Americans. American journal of human genetics, 53(3), 591-608.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1682407/)
* [Shields GF, Schmiechen AM, Frazier BL, Redd A Voevoda MI, Reed JK, Ward RH (1993) mtDNA sequences suggest a recent evolutionary divergence for Beringian and northern North American populations. American Journal of Human Genetics. 53(3): 549-562.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1682422/)
* [Ward RH, Redd A, Valencia D, Pääbo S (1993) Genetic and linguistic differentiation in the Americas. PNAS. 90(22): 10663-10667.](https://dx.doi.org/10.1073%2Fpnas.90.22.10663)
* [Ward RH, Frazier BL, Dew-Jager K, Pääbo S (1991) Extensive mitochondrial diversity within a single Amerindian tribe. PNAS. 88(19): 8720-8724.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC52581/)
* [Schurr TG et al. (1990) Amerindian mitochondrial DNAs have rare Asian mutations at high frequencies, suggesting they derived from four primary maternal lineages. American Journal of Human Genetics. 46(3): 613-623.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1683611/)
* [Genetics and the History of Latin America](https://migrationcluster.ucdavis.edu/research/genetics-and-history-latin-america)
* [Genetics, Linguistics, and Prehistoric Migrations: An Analysis of California Indian Mitochondrial DNA Lineages](https://escholarship.org/uc/item/6g135010)
* [Genome-wide insights into the genetic history of human populations](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4381409/)
* [Genomic insights on the ethno-history of the Maya and the ‘Ladinos’ from Guatemala](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4422311/)
* [High-resolution mitochondrial DNA analysis sheds light on human diversity, cultural interactions, and population mobility in Northwestern Amazonia](https://www.researchgate.net/publication/320654352_High-resolution_mitochondrial_DNA_analysis_sheds_light_on_human_diversity_cultural_interactions_and_population_mobility_in_Northwestern_Amazonia)
* [HLA Class II Alleles in the Otomi Population of the Mezquital Valley: A Genetic Approach to the History of Interethnic Migrations in the Mexican Central Plateau](https://www.researchgate.net/publication/274396125_HLA_Class_II_Alleles_in_the_Otomi_Population_of_the_Mezquital_Valley_A_Genetic_Approach_to_the_History_of_Interethnic_Migrations_in_the_Mexican_Central_Plateau)
* [Large scale mitochondrial sequencing in Mexican Americans suggests a reappraisal of Native American origins](https://doi.org/10.1186/1471-2148-11-293)
* [Mitochondrial DNA and Prehistoric Settlements: Native Migrations on the Western Edge of North America](https://www.researchgate.net/publication/8485114_Mitochondrial_DNA_and_Prehistoric_Settlements_Native_Migrations_on_the_Western_Edge_of_North_America)
* [Mitochondrial DNA Studies of Native Americans: Conceptions and Misconceptions of the Population Prehistory of the Americas](https://pdfs.semanticscholar.org/a40f/fbea9601cea8b7c973a8cedbea297a6de20a.pdf)
* [Mitochondrial DNA Diversity in Indigenous Populations of the Southern Extent of Siberia, and the Origins of Native American Haplogroups](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3905771/)
* [Mitochondrial Population Genomics Supports a Single Pre-Clovis Origin with a Coastal Route for the Peopling of the Americas](https://doi.org/10.1016/j.ajhg.2007.11.013)
* [mtDNA haplogroup X: An ancient link between Europe/Western Asia and North America?](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1377656/)
* [mtDNA Variation among Greenland Eskimos: The Edge of the Beringian Expansion.](https://doi.org/10.1086/303038)
* [Origins and genetic legacies of the Caribbean Taino](https://doi.org/10.1073/pnas.1716839115)
* [Patterns of mtDNA Diversity in Northwestern North America](https://www.researchgate.net/publication/8485113_Patterns_of_mtDNA_Diversity_in_Northwestern_North_America)
* [The Genetic History Of The Otomi In The Central Mexican Valley](https://repository.upenn.edu/anthro_seniortheses/133/)
* [The Genetics of Mexico Recapitulates Native American Substructure and Affects Biomedical Traits](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4156478/)

## Early Hominid through Modern Human

This includes archeological and genetics papers to build a perspective on the diffusion of Hominids and modern humans.
* [Prüfer, K., Posth, C., Yu, H. et al. A genome sequence from a modern human skull over 45,000 years old from Zlatý kůň in Czechia. Nat Ecol Evol (2021). https://doi.org/10.1038/s41559-021-01443-x](https://doi.org/10.1038/s41559-021-01443-x)
  * [Gibbons, A. (2021) More than 45,000 years ago, modern humans ventured into Neanderthal territory. Here’s what happened next. Science News. April 7.](https://www.sciencemag.org/news/2021/04/more-45000-years-ago-modern-humans-ventured-neanderthal-territory-here-s-what-happened)
    * > Krause’s team also found that unlike the Bacho Kiro individuals, the Zlatý kůň skull was no more closely related to ancient Asians than to Europeans. This suggests she came from an ancient population that had not yet differentiated genetically into Asians and Europeans, Krause says.
    * > The new data show that all of the mod­ern human lineages in Europe vanished by the ad­vent of the last ice age, which reached its peak about 20,000 years ago. After the ice melted, other modern humans from Eur­asia repopulated the continent. “Multiple groups headed out [to Eurasia], but it seems like only a fraction of these early modern humans left progeny,” Novembre says.
* [Hajdinjak, M., Mafessoni, F., Skov, L. et al. Initial Upper Palaeolithic humans in Europe had recent Neanderthal ancestry. Nature 592, 253–257 (2021). https://doi.org/10.1038/s41586-021-03335-3](https://doi.org/10.1038/s41586-021-03335-3)
  * > When comparing the Bacho Kiro Cave individuals to present-day populations31, we found that the IUP individuals share more alleles (that is, more genetic variants) with present-day populations from East Asia, Central Asia and the Americas than with populations from western Eurasia (Fig. 2a, Supplementary Information 5), whereas the later BK1653 individual shares more alleles with present-day western Eurasian populations (Extended Data Figs. 3b, 4a).
  * > We found that the Ust’Ishim and Oase1 individuals showed no more affinity to western than to eastern Eurasian populations, suggesting that they did not contribute ancestry to later Eurasian populations, as previously shown7,8 (Supplementary Information 5, Extended Data Fig. 5). By contrast, the IUP Bacho Kiro Cave individuals shared more alleles with the roughly 40,000-year-old Tianyuan individual13 from China (Fig. 2b) and other ancient Siberians34,35 and Native Americans36,37,38,39 (Fig. 2c) than with the Kostenki14 individual (3.6 ≤ |Z| ≤ 5.3). Among other western Eurasian Upper Palaeolithic humans, the IUP Bacho Kiro Cave individuals shared more alleles with the Oase1 (3.6 ≤ |Z| ≤ 4.3) and roughly 35,000-year-old GoyetQ116-129 individuals than with the Kostenki14 individual (3.2 ≤ |Z| ≤ 4.3; Fig. 2c, Supplementary Information 5). Notably, the GoyetQ116-1 individual has previously been shown to share more alleles with early East Asians than other individuals of a similar age in Europe13.
  * > When we explored models of population history that are compatible with the observations above using admixture graphs28, we found that the IUP Bacho Kiro Cave individuals were related to populations that contributed ancestry to the Tianyuan individual in China as well as, to a lesser extent, to the GoyetQ116-1 and Ust’Ishim individuals (all |Z| < 3; Fig. 2d, Supplementary Information 6). This resolves the previously unclear relationship between the GoyetQ116-1 and Tianyuan individuals13 without the need for gene flow between these two geographically distant individuals. The models also suggest that the later BK1653 individual belonged to a population that was related, but not identical, to that of the GoyetQ116-1 individual (Fig. 2d, Extended Data Fig. 4, Supplementary Information 6) and that the Věstonice cluster, whose members were found in association with Gravettian assemblages29, derived part of their ancestry from such a population and the rest from populations related to the roughly 34,000-year-old ‘Sunghir’ individuals40 from Russia (Fig. 2d, Supplementary Information 6).
  * [Handwerk, B. (2021) Some of Europe’s Oldest-Known Modern Humans Are Distantly Related to Native Americans. Smithsonian Magazine. April 7.](https://www.smithsonianmag.com/science-nature/secrets-europes-oldest-known-modern-humans-revealed-genome-sequencing-180977437/)
* [Schierup, M. H. (2020). The last pieces of a puzzling early meeting. Science, 369(6511), 1565-1566](https://doi.org/10.1126/science.abe2766)
* [Petr, M., Hajdinjak, M., Fu, Q., Essel, E., Rougier, H., Crevecoeur, I., ... & de la Rasilla, M. (2020). The evolutionary history of Neandertal and Denisovan Y chromosomes. Science, 369(6511), 1653-1656](https://doi.org/10.1126/science.abb6460)
* [Haws, J. A., Benedetti, M. M., Talamo, S., Bicho, N., Cascalheira, J., Ellis, M. G., ... & Zinsious, B. K. (2020). The early Aurignacian dispersal of modern humans into westernmost Eurasia. Proceedings of the National Academy of Sciences, 117(41), 25414-25422.](https://doi.org/10.1073/pnas.2016062117)
* [Cox, T. J., Fazenda, B. M., & Greaney, S. E. (2020). Using scale modelling to assess the prehistoric acoustics of stonehenge. Journal of Archaeological Science, 122, 105218.](https://doi.org/10.1016/j.jas.2020.105218)
* [Groucutt, H. S., Breeze, P. S., Guagnin, M., Stewart, M., Drake, N., Shipton, C., ... & Petraglia, M. D. (2020). Monumental landscapes of the Holocene humid period in Northern Arabia: The mustatil phenomenon. The Holocene, 0959683620950449.](https://doi.org/10.1177/0959683620950449)
* [Hublin, J. J., Sirakov, N., Aldeias, V., Bailey, S., Bard, E., Delvigne, V., ... & Kromer, B. (2020). Initial Upper Palaeolithic Homo sapiens from Bacho Kiro Cave, Bulgaria. Nature, 1-4.](https://doi.org/10.1038/s41586-020-2259-z) &mdash; [PDF](https://www.researchgate.net/publication/341299437_Initial_Upper_Palaeolithic_Homo_sapiens_from_Bacho_Kiro_Cave_Bulgaria)
  * [David, A. (2020) A Tooth Changes Everything We Know About Humans Reaching Europe. Haaretz. 13 May.](https://web.archive.org/web/20200604133024if_/https://www.haaretz.com/science-and-health/.premium-a-tooth-changes-everything-we-know-about-humans-reaching-europe-1.8836555)
* [Fewlass, H., Talamo, S., Wacker, L., Kromer, B., Tuna, T., Fagault, Y., ... & Martisius, N. L. (2020). A 14 C chronology for the Middle to Upper Palaeolithic transition at Bacho Kiro Cave, Bulgaria. Nature Ecology & Evolution, 1-8.](https://doi.org/10.1038/s41559-020-1136-3)
* [Curry, A. (2020). Ancient DNA tracks Vikings across Europe. Science, 369(6510)](https://doi.org/10.1126/science.369.6510.1416)
* [Stewart, M., Clark-Wilson, R., Breeze, P. S., Janulis, K., Candy, I., Armitage, S. J., ... & Cuthbertson, P. (2020). Human footprints provide snapshot of last interglacial ecology in the Arabian interior. Science Advances, 6(38), eaba8940.](https://doi.org/10.1126/sciadv.aba8940) &mdash; Homo sapiens 120000 years ago in Levant and Arabia.
* [Wadley, L., Esteban, I., de la Peña, P., Wojcieszak, M., Stratford, D., Lennox, S., ... & Sievers, C. (2020). Fire and grass-bedding construction 200 thousand years ago at Border Cave, South Africa. Science, 369(6505), 863-866.](https://doi.org/10.1126/science.abc7239)
* [Burger, J., Link, V., Blöcher, J., Schulz, A., Sell, C., Pochon, Z., ... & Reyna-Blanco, C. S. (2020). Low Prevalence of Lactase Persistence in Bronze Age Europe Indicates Ongoing Strong Selection over the Last 3,000 Years. Current Biology.](https://doi.org/10.1016/j.cub.2020.08.033)
* [Albers, P. K., & McVean, G. (2020). Dating genomic variants and shared ancestry in population-scale sequencing data. PLoS biology, 18(1), e3000586.](https://dx.doi.org/10.1371%2Fjournal.pbio.3000586)
* [Duveau J, et al. (2019) The composition of a Neandertal social group revealed by the hominin footprints at Le Rozel (Normandy, France). PNAS. 116(39): 19409-19414.](https://doi.org/10.1073/pnas.1901789116)
* [Faith JT, Rowan J, Du A (2019) Early hominins evolved within non-analog ecosystems. PNAS. 116(43): 21478-21483.](https://doi.org/10.1073/pnas.1909284116)
* [Schuster R (2019) Prehistoric Superglue May Reveal Neanderthal Experiments 200,000 Years Ago. Haaretz. November 13.](https://www.haaretz.com/archaeology/.premium.MAGAZINE-prehistoric-superglue-may-reveal-neanderthal-experiments-200-000-years-ago-1.8068691?fbclid=IwAR3Fb2ksOsyf8ftioHvT4pubyfF-TSoMG6rIW8ZkfZdr4La03fzZe1TnglY)
* [Chan EKF, et al. (2019) Human origins in a southern African palaeo-wetland and first migrations. Nature. 575: 185-189.](https://www.nature.com/articles/s41586-019-1714-1) &mdash; [PDF](https://www.nature.com/articles/s41586-019-1714-1.epdf?referrer_access_token=iAfJZZ_lpZl3huBpr5zq1NRgN0jAjWel9jnR3ZoTv0Ow72lSjAIzdDOJcHUPoWIRGEOjMmthCa5BpP_dciaujFlTFsWPygPnm_eHl2AcrLFjeSO9PGAR2vyJhXma8rECK6S40cpYYTwflvHD4NiZ7HKsXBLLsVGkx8YcO8B0aOZu7MlerKh-hv7q75L8HMWR8nuZQfJdUEeusZGlVVZJe91EQF82zbGiMzbevXVwLkpyiENHRd_a17eb9Y1IDkDOLfVDnQfqUdMXlojsECkV6esUOl3P3GBBWSs3U57dPLc3Yz7vKowijwOo0uwLwoNK&tracking_referrer=www.theguardian.com)
* [Feldman, M., Master, D. M., Bianco, R. A., Burri, M., Stockhammer, P. W., Mittnik, A., ... & Krause, J. (2019). Ancient DNA sheds light on the genetic origins of early Iron Age Philistines. Science Advances, 5(7), eaax0061.](https://doi.org/10.1126/sciadv.aax0061)
* [li Z et al. (2019) Engraved bones from the archaic hominin site of Lingjing, Henan Province. Antiquity. 93(370): 886-900.](https://doi.org/10.15184/aqy.2019.81)
* [Price M (2019) Ancient DNA puts a face on the mysterious Denisovans, extinct cousins of Neanderthals. Science News. September 19.](https://www.sciencemag.org/news/2019/09/ancient-dna-puts-face-mysterious-denisovans-extinct-cousins-neanderthals)
* [Roberts P, Amano N (2019) Plastic pioneers: Hominin biogeography east of the Movius Line during the Pleistocene. Archaeological Research in Asia. 17: 181-192.](https://www.sciencedirect.com/science/article/pii/S2352226718300989?via%3Dihub)
* [Sahle Y, Brooks AS (2019) Assessment of complex projectiles in the early Late Pleistocene at Aduma, Ethiopia. PLoS ONE 14(5): e0216716.](https://doi.org/10.1371/journal.pone.0216716)
* [Braun DR et al. (2019) Earliest known Oldowan artifacts at >2.58 Ma from Ledi-Geraru, Ethiopia, highlight early technological diversity. PNAS. 116(24): 11712-11717.](https://doi.org/10.1073/pnas.1820177116)
* [Shinde V et al. (2019) An Ancient Harappan Genome Lacks Ancestry from Steppe Pastoralists or Iranian Farmers. Cell.](https://doi.org/10.1016/j.cell.2019.08.048)
* [Narasimhan VM et al. (2019) The formation of human populations in South and Central Asia. Science. 35(6457): eaat7487](https://science.sciencemag.org/content/365/6457/eaat7487.full)
* [Schaefer NK, Shapiro B (2019) New middle chapter in the story of human evolution. Science. 365(6457): 981-982.](https://science.sciencemag.org/content/365/6457/981) &mdash; Has great references for an updated and more complete phylogenetic tree.
* National Geographic has a nice looking, but crude, map which traces the general flow of modern genetics out of Africa and into the rest of the world: [Map of Human Migration](https://genographic.nationalgeographic.com/human-journey/)
* [Ragsdale AP, Gravel S (2019) Models of archaic admixture and recent history from two-locus statistics. PLoS Genet 15(6): e1008204.](https://doi.org/10.1371/journal.pgen.1008204)
* [Bokelmann L et al. (2019) A genetic analysis of the Gibralter Neanderthals. PNAS. 116(31): 15610-15615.](https://doi.org/10.1073/pnas.1903984116)
* [Harvati K et al. (2019) Apidima Cave fossila provide earliest evidence of Homo sapiens in Eurasia. Nature. 571: 500-504.](https://www.nature.com/articles/s41586-019-1376-z)
* [Delson E (2019) An early dispersal of modern humans from Africa to Greece. Nature News. July 10.](https://www.nature.com/articles/d41586-019-02075-9) &mdash; Has a great map of locations and times for various known dispersals.
* [Cepelewicz, J (2019) Fossil DNA Reveals New Twists in Modern Human Origins. Quanta Magazine. August 29. Web. Accessed September 7, 2019.](https://www.quantamagazine.org/fossil-dna-reveals-new-twists-in-modern-human-origins-20190829/)
* [Roger AR, Harris NS, Achenbach AA (2019) Neanderthal-Denisovan ancestors interbred with a distantly-related hominin. bioRxiv preprint. 657247.](https://doi.org/10.1101/657247)
* [Groucutt, H. S., Grün, R., Zalmout, I. A., Drake, N. A., Armitage, S. J., Candy, I., ... & Buck, L. T. (2018). Homo sapiens in Arabia by 85,000 years ago. Nature ecology & evolution, 2(5), 800-809.](https://doi.org/10.1038/s41559-018-0518-2) &mdash; [PDF](https://www.academia.edu/36508168/Homo_sapiens_in_Arabia_by_85_000_years_ago)
* [Potts R et al. (2018) Environmental dynamics during the onset of the Middle Stone Age in eastern Africa. Science. 360(6384): 86-90.](https://science.sciencemag.org/content/360/6384/86) &mdash; Obsidian obtained from far away created a technological change in eastern Africa around 320 ka ago.
* [Brooks AS et al. (2018) Long-distance stone transport and pigment use in the earliest Middle Stone Age. Science. 360(6384): 90-94.](https://science.sciencemag.org/content/360/6384/90)
  * `The Middle Stone Age (MSA) comprises a diverse group of African industries characterized by the absence or rarity of large cutting tools (LCTs, such as Acheulean handaxes and cleavers), the presence of prepared core (Levallois) technologies, and regional and/or temporal variation in weapon armatures (1–3). Later MSA industries contain varying frequencies of small retouched bladelets and geometric artifacts (4), small unifacially and bifacially worked points, and larger lanceolate and bone points (5, 6). African MSA sites are associated with the oldest known fossils attributed to Homo sapiens, which date between ~195 and ~350 thousand years (ka) ago (7–9). After 130 ka ago, late Pleistocene MSA sites, mostly from the north and south temperate zone extremities of the continent, document some of the oldest beads (10, 11), complex geometric designs (12, 13), lithic projectile armatures interpreted as arrow or dart tips (14, 15), paint production (16), heat treatment of lithics (17), and expanded resource use, including fishing and shellfish collecting (18, 19).`
  * `Although these late Pleistocene behaviors are thought to reflect advanced cognitive abilities for learning, imitation, working memory, planning, and recursive technological innovation, their middle Pleistocene or earlier antecedents have remained obscure. Evolutionary explanations for the middle Pleistocene enlargement of relative brain size, largely overlapping the modern H. sapiens range, include a shift to larger social groups or networks (20); greater foraging, technological, and social adaptability in the face of more rapid and unpredictable climate fluctuations (21); and the development of symbolic expression (22).`
* [Deino AL et al. (2018) Chronology of the Acheulean to Middle Stone Age transition in eastern Africa. Science. 360(6384): 95-98.](https://science.sciencemag.org/content/360/6384/95)
* [Zhu Z et al. (2018) Hominin occupation of the Chinese Loess Plateau since about 2.1 million years ago. Nature. 559: 608-612.](https://www.nature.com/articles/s41586-018-0299-4)
* [Karakostis FA et al. (2018) Evidence for precision grasping in Neandertal daily activities. Science Advances. 4(9): eaat2369](http://advances.sciencemag.org/content/4/9/eaat2369)
* [Hoffman DL et al. (2018) Symbolic use of marine shells and mineral pigments by Iberian Neandertals 115,000 years ago. Science Advances. 4(2): eaar5255](http://advances.sciencemag.org/content/4/2/eaar5255)
* [Browning SR et al. (2018) Analysis of Human Sequence Data Reveals Two Pulses of Archaic Denisovan Admixture. Cell. 173(1): 53-61.](https://doi.org/10.1016/j.cell.2018.02.031)
* [Wolf AB, Akey JM (2018) Outstanding questions in the study of archaic hominin admixture. PLoS Genet 14(5): e1007349.](https://doi.org/10.1371/journal.pgen.1007349)
* [Slon V et al. (2018) The genome of the offspring of a Neanderthal mother and a Denisovan father. Nature. 561: 113-116.](https://www.nature.com/articles/s41586-018-0455-x)
* [Groucutt HS, Grün R, Zalmout IAS, Drake NA, Armitage SJ, Candy I, et al. (2018). Homo sapiens in Arabia by 85,000 years ago. Nature Ecology & Evolution. doi:10.1038/s41559-018-0518-2](https://doi.org/10.1038/s41559-018-0518-2)
* [Scerri EML et al. (2018) Did Our Species Evolve in Subdivided Populations across Africa, and Why Does It Matter? Trends in Ecology & Evolution. 33(8): 582-594.](https://doi.org/10.1016/j.tree.2018.05.005)
* [Ohman J, Gomez-Olivencia A (2018) Neanderthals cared for each other and survived into old age – new research. HeritageDaily. April 11.](https://www.heritagedaily.com/2018/04/neanderthals-cared-for-each-other-and-survived-into-old-age-new-research/118812)
* [Gomez-Olivencia A et al. (2018) La Ferrassie 1: New perspectives on a “classic” Neandertal. Journal of Human Evolution. 117: 13-32.](https://doi.org/10.1016/j.jhevol.2017.12.004)
* [Daura J et al. (2017) New Middle Pleistocene hominin cranium from the Gruta da Aroeira (Portugal). PNAS. 114(13): 3397-3402.](https://doi.org/10.1073/pnas.1619040114)
* [Posth C et al. (2017) Deeply divergent archaic mitochondrial genome provides lower time boundary for African gene flow into Neanderthals. Nature Communications. 8: 16046.](https://www.nature.com/articles/ncomms16046)
* [Putt SS, Wijeakumar S, Franciscus RG, Spencer JP (2017) The functional brain networks that underlie Early Stone Age tool manufacture. Nature Human Behaviour. 1: 0102.](https://www.nature.com/articles/s41562-017-0102)
* [Scerri EML et al. (2017) Persistence of Middle Stone Age technology to the Pleistocene/Holocene transition supports a complex hominin evolutionary scenario in West Africa. Journal of Archeological Science: Reports. 11: 639-646.](https://doi.org/10.1016/j.jasrep.2017.01.003)
* [Schelbusch CM et al. (2017) Southern African ancient genomes estimate modern human divergence to 350,000 to 260,000 years ago. Science. 358(6363): 652-655.](https://science.sciencemag.org/content/358/6363/652)
* [Skoglund P et al. (2017) Reconstructing Prehistoric African Population Structure. Cell.1 71(1): 59-71.](https://doi.org/10.1016/j.cell.2017.08.049)
  * `Africa harbors more genetic diversity than any other part of the world (Cann et al., 1987, Tishkoff et al., 2009). This is reflected both in a higher average number of differences among sub-Saharan African genomes than among non-African genomes (Cann et al., 1987, Ramachandran et al., 2005) and in the fact that the ancestry found outside of Africa is largely a subset of that within it (Tishkoff et al., 2009).`
* [Hublin J-J et al. (2017) New fossils from Jebel Irhoud, Morocco and the pan-African origin of Homo sapiens. Nature. 546: 289–292.](https://www.nature.com/articles/nature22336)
* [Richter D (2017) The age of the hominin fossils from Jebel Irhoud, Morocco, and the origins of the Middle Stone Age. Nature. 546: 293-296.](https://doi.org/10.1038/nature22335) &mdash; Pushes back presence of modern humans to 286 ± 32 ka ago.
* [Platt DE, Haber M, Dagher-Kharrat MB, Douaihy B, Khazen G, Bonab MA, Salloum A, Mouzaya F, Luiselli D, Tyler-Smith C, Renfrew C, Matisoo-Smith E, Zalloua PA (2017) Mapping Post-Glacial expansions: The Peopling of Southwest Asia. Scientific Reports. 7.](https://www.nature.com/articles/srep40338)
* [Langley M (2017) World’s scientists turn to Asia and Australia to rewrite human history. The Conversation. 7 Dec. 2017.](http://theconversation.com/worlds-scientists-turn-to-asia-and-australia-to-rewrite-human-history-88697)
* [Gibbens S (2017) Humans Arrived in Australia 18,000 Years Earlier Than We Thought. National Geographic. 20 Jul.](https://news.nationalgeographic.com/2017/07/australia-aboriginal-early-human-evolution-spd/)
* [Gibbons A (2017) A find in Australia hints at very early human exit from Africa. 19 Jul.](http://www.sciencemag.org/news/2017/07/find-australia-hints-very-early-human-exit-africa)
* [Callaway E (2017) Oldest Homo sapiens fossil claim rewrites our species' history. Nature News. 07 Jun.](https://www.nature.com/news/oldest-homo-sapiens-fossil-claim-rewrites-our-species-history-1.22114)
* [Gibbons A (2017) World’s oldest Homo sapiens fossils found in Morocco. Science. 7 Jun.](http://www.sciencemag.org/news/2017/06/world-s-oldest-homo-sapiens-fossils-found-morocco)
* [Nielsen R et al. (2017) Tracing the peopling of the world through genomics. Nature. 541: 302-310.](https://www.nature.com/articles/nature21347) &mdash; Has a better map of possible migration routes which line up with indigenous histories. There's still some details missing, but it's the best 2017 had to offer.
* [Crevecoeur I et al. (2016) Late Stone Age human remains from Ishango (Democratic Republic of Congo): New insights on Late Pleistocene modern human diversity in Africa. Journal of Human Evolution. 96: 37-57.](https://doi.org/10.1016/j.jhevol.2016.04.003)
* [Stringer C (2016) The origin and evolution of Homo sapiens. Philosphical Transaction of Royal Society B. 371(1698)](https://doi.org/10.1098/rstb.2015.0237)
* [Mallick S et al. (2016) The Simons Genome Diversity Project: 300 genomes from 142 diverse populations. Nature. 538: 201-206.](https://www.nature.com/articles/nature18964) &mdash; This has an interesting neighbor joining tree based on pairwise divergence showing .
* [Malaspinas A-S et al. (2016) A genomic history of Aboriginal Australia. Nature. 538: 207–214.](https://www.nature.com/articles/nature18299)
* [Pagani L (2016) Genomic analyses inform on migration events during the peopling of Eurasia. Nature. 538: 238-242.](https://www.nature.com/articles/nature19792)
* [Qiu J (2016) How China is rewriting the book on human origins. Nature News. 12 Jul.](https://www.nature.com/news/how-china-is-rewriting-the-book-on-human-origins-1.20231)
* [(2016) Unprecedented study of Aboriginal Australians points to one shared Out of Africa migration for modern humans. 21 Sep.](https://www.cam.ac.uk/research/news/unprecedented-study-of-aboriginal-australians-points-to-one-shared-out-of-africa-migration-for)
* [Fu W et al. (2016) Robust Inference of Identity by Descent from Exome-Sequencing Data. American Journal of Human Gentics. 99(5): 1106-1116.](https://doi.org/10.1016/j.ajhg.2016.09.011)
* [Harris K, Nielson R (2016) The Genetic Cost of Neanderthal Introgression. Genetic. 203(2): 881-891.](https://doi.org/10.1534/genetics.116.186890)
* [Kuhlwilm M et al. (2016) Ancient gene flow from early modern humans into Eastern Neanderthals. Nature. 530: 429-433.](https://www.nature.com/articles/nature16544)
* [Lahr MM (2016) The shaping of human diversity: filters, boundaries and transitions. Philosphical Transaction of Royal Society B. 371(1698)](https://doi.org/10.1098/rstb.2015.0241)
* [Mathieson I et al. (2015) Eight thousand years of natural selection in Europe. bioRxiv preprint. 016477. doi: https://doi.org/10.1101/016477 Now published in Nature doi: 10.1038/nature16152](https://doi.org/10.1101/016477)
* [Roberts p et al. (2015) Direct evidence for human reliance on rainforest resources in late Pleistocene Sri Lanka. Science. 347(6227): 1246-1249.](https://science.sciencemag.org/content/347/6227/1246)
* [Haak W et al. (2015) Massive migration from the steppe was a source for Indo-European languages in Europe. Nature. 522(7555): 207-211](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5048219/)
* [Harmand S et al. (2015) 3.3-million-year-old stone tools from Lomekwi 3, West Turkana, Kenya. Nature. 521: 310-315.](https://www.nature.com/articles/nature14464)
* [Karmin M et al. (2015) A recent bottleneck of Y chromosome diversity coincides with a global change in culture. Genome Research. 25: 459-466.](https://genome.cshlp.org/content/25/4/459.full)
* [The 1000 Genomes Project Consortium (2015) A global reference for human genetic variation. Nature. 526: 68-74.](https://www.nature.com/articles/nature15393)
* [Miyagawa S et al. (2014) The integration hypothesis of human language evolution and the nature of contemporary languages. Frontiers in Psychology. 5.](https://doi.org/10.3389/fpsyg.2014.00564)
* [Fu Q et al. (2013) DNA analysis of an early modern human from Tianyuan Cave, China. PNAS. 110(6): 2223-2227.](https://doi.org/10.1073/pnas.1221359110)
* [Texier P-J et al. (2013) The context, form and significance of the MSA engraved ostrich eggshell collection from Diepkloof Rock Shelter, Western Cape, South Africa. Journal of Archeological Science. 40(9): 3412-3431.](https://doi.org/10.1016/j.jas.2013.02.021)
* [Blome MW et al. (2012) The environmental context for the origins of modern human diversity: A synthesis of regional variability in African climate 150,000–30,000 years ago. Journal of Human Evolution. 62(5): 563-592.](https://doi.org/10.1016/j.jhevol.2012.01.011)
* [Lachance J et al. (2012) Evolutionary History and Adaptation from High-Coverage Whole-Genome Sequences of Diverse African Hunter-Gatherers. Cell. 150(3): 457-469.](https://doi.org/10.1016/j.cell.2012.07.009)
* [Gronau I et al. (2011) Bayesian inference of ancient human demography from individual genome sequences. Nature Genetics. 43, 1031-1034.](https://www.nature.com/articles/ng.937)
* [McEvoy B et al. (2010) Whole-Genome Genetic Diversity in a Sample of Australians with Deep Aboriginal Ancestry. American Journal of Human Genetics. 13 Aug.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2917718/)
* [Texier P-J et al. (2010) A Howiesons Poort tradition of engraving ostrich eggshell containers dated to 60,000 years ago at Diepkloof Rock Shelter, South Africa. PNAS. 107(14): 6180-6185.](https://doi.org/10.1073/pnas.0913047107)
* [Bramanti B et al. (2009) Genetic Discontinuity Between Local Hunter-Gatherers and Central Europe’s First Farmers. Science. 326(5949): 137-140.](https://science.sciencemag.org/content/326/5949/137.abstract)
* [Hoffecker, J. F. (2009). The spread of modern humans in Europe. Proceedings of the National Academy of Sciences, 106(38), 16040-16045.](https://doi.org/10.1073/pnas.0903446106)
* [Lalueza-Fox C, Römpler H, et al. (2007) A Melanocortin 1 Receptor Allele Suggests Varying Pigmentation Among Neanderthals. Science. 318(5855): 1453-1455.](https://science.sciencemag.org/content/318/5855/1453)
* [Burger, J., Kirchner, M., Bramanti, B., Haak, W., & Thomas, M. G. (2007). Absence of the lactase-persistence-associated allele in early Neolithic Europeans. Proceedings of the National Academy of Sciences, 104(10), 3736-3741.](https://dx.doi.org/10.1073%2Fpnas.0607187104)
* [Plagnol V, Wall JD (2006) Possible Ancestral Structure in Human Populations. PLoS Genet 2(7): e105.](https://doi.org/10.1371/journal.pgen.0020105)

## In Case You Are Interested in Dog Genetics

* [Norton, H.L., Quillen, E.E., Bigham, A.W. et al. Human races are not like dog breeds: refuting a racist analogy. Evo Edu Outreach 12, 17 (2019). https://doi.org/10.1186/s12052-019-0109-y](https://doi.org/10.1186/s12052-019-0109-y)
* [Leathlobhair, M. N., Perri, A. R., Irving-Pease, E. K., Witt, K. E., Linderholm, A., Haile, J., ... & Brace, S. (2018). The evolutionary history of dogs in the Americas. Science, 361(6397), 81-85.](https://doi.org/10.1126/science.aao4776)
* [Hecht, J. (2018) What a Dog Geneticist Wants You to Know about Dog Genetics. Dog Spies Blog. Scientific American.](https://blogs.scientificamerican.com/dog-spies/what-a-dog-geneticist-wants-you-to-know-about-dog-genetics/)
* [Parker, H. G., Dreger, D. L., Rimbault, M., Davis, B. W., Mullen, A. B., Carpintero-Ramirez, G., & Ostrander, E. A. (2017). Genomic analyses reveal the influence of geographic origin, migration, and hybridization on modern dog breed development. Cell reports, 19(4), 697-708.](https://doi.org/10.1016/j.celrep.2017.03.079)
* [Larson, G., Karlsson, E. K., Perri, A., Webster, M. T., Ho, S. Y., Peters, J., ... & Comstock, K. E. (2012). Rethinking dog domestication by integrating genetics, archeology, and biogeography. Proceedings of the National Academy of Sciences, 109(23), 8878-8883.](https://doi.org/10.1073/pnas.1203005109)
* [Sutter, N. B., Bustamante, C. D., Chase, K., Gray, M. M., Zhao, K., Zhu, L., ... & Quignon, P. (2007). A single IGF1 allele is a major determinant of small size in dogs. Science, 316(5821), 112-115.](https://doi.org/10.1126/science.1137045)
* [Ostrander, E. A. (2007). Genetics and the shape of dogs: studying the new sequence of the canine genome shows how tiny genetic changes can create enormous variation within a single species. American Scientist, 95(5), 406-413.](https://www.americanscientist.org/article/genetics-and-the-shape-of-dogs)
* [Parker, H. G., & Ostrander, E. A. (2005). Canine genomics and genetics: running with the pack. PLoS Genet, 1(5), e58.](https://doi.org/10.1371/journal.pgen.0010058)

## Was farming a mistake? (Should this go somewhere else?)

Certain people have notice that in industrialized societies our jaw is getting smaller (and so are certain other features of our skulls). This is an extention of the process seen historically through evolution starting with our jaws becoming less pronounced and our skulls bigger. But since the adoption of farming it has accelerated. This is thought to be due to not using our jaw muscles and not having to work to eat processed food. However the mechanism, it poses existential risks for humanity.

* [Antrosio J (2011) Agriculture as ‘Worst Mistake in the History of the Human Race’? Living Anthropologically. June 1.](https://www.livinganthropologically.com/archaeology/agriculture-worst-mistake/)
* [Antrosio J (2011) Many Ways of Gathering and Hunting. Living Anthropologically. June 9.](https://www.livinganthropologically.com/archaeology/gathering-and-hunting/)
* [Diamond J (1987) The Worst Mistake in the History of the Human Race. Discover Magazine. May. Published online May 1999.](http://discovermagazine.com/1987/may/02-the-worst-mistake-in-the-history-of-the-human-race) &mdash; please read the last two articles first for proper context.
* [Pennisi E (2004) Weak Jaw, Big Brain. Sicence News. March 24.](https://www.sciencemag.org/news/2004/03/weak-jaw-big-brain)
* [Hopkin M (2004) Jaw-dropping theory of human evolution. Nature News. March 25.](https://www.nature.com/news/2004/040322/full/news040322-9.html)
* [Stedman H et al. (2004) Myosin gene mutation correlates with anatomical changes in the human lineage. Nature. 428: 415-418.](https://www.nature.com/articles/nature02358)
* [Ungar P (2017) It's not that your teeth are too big: your jaw is too small. Aeon. Marina Benjamin, editor. June 30.](https://aeon.co/ideas/its-not-that-your-teeth-are-too-big-your-jaw-is-too-small)
* [von Cramon-Taubadel N (2011) Global human mandibular variation reflects differences in agricultural and hunter-gatherer subsistence strategies. PNAS. 108(49) 19546-19551.](https://doi.org/10.1073/pnas.1113050108)
* [Crompton AW (1963) The Evolution of the Mammalian Jaw. Evolution. 17(4): 431-439.](http://www.jstor.org/stable/2407093) &mdash; [PDF](https://crompton.oeb.harvard.edu/files/crompton/files/crompt1963_evolmammjaw.pdf)
* [Alex B (2019) The Human Brain Has been Getting Smaller Since the Stone Age. The Crux Blog. Discover Magazine. April 8.](blogs.discovermagazine.com/crux/2019/04/08/human-brains-have-shrunk-since-the-stone-age/)
* [McAuliffe K (2010) If Modern Humans Are So Smart, Why Are Our Brains Shrinking? Discover Magazine. September.](http://discovermagazine.com/2010/sep/25-modern-humans-smart-why-brain-shrinking)
* [Henneberg M, Steyn M (1993) Trends in cranial capacity and cranial index in Subsaharan Africa during the Holocene. American Journal of Human Biology. 5(4): 473-479.](https://doi.org/10.1002/ajhb.1310050411)
* [Brown P, Maeda T (2004) Post-Pleistocene diachronic change in East Asian facial skeletons: the size, shape and volume of the orbits. Anthropological Science. 112(1): 29-40.](https://doi.org/10.1537/ase.00072)
* [Brown P (1987) Pleistocene Homogeneity and Holocene Size Reduction: The Australian Human Skeletal Evidence. Archeology in Oceania. 22(2): 41-67.](https://www.jstor.org/stable/40386774)
* [Stringer, C (2014) Why Have Our Brains Started to Shrink? Scientific American Mind. November 1.](https://www.scientificamerican.com/article/why-have-our-brains-started-to-shrink/)
* [Henneberg M (1988) Decrease of Human Skull Size in the Holocene. Human Biology. 60(3): 395-405.](https://www.jstor.org/stable/41464021)
* [Thompson H (2014) Switching to Farming Made Human Joint Bones Lighter. Smithsonian. December 22.](https://www.smithsonianmag.com/science-nature/switching-farming-made-human-joint-bones-lighter-180953711/#rzByrssq22fhuOva.99)
* [Pinhasi R, Eshed V, von Cramon-Taubadel N (2015) Incongruity between Affinity Patterns Based on Mandibular and Lower Dental Dimensions following the Transition to Agriculture in the Near East, Anatolia and Europe. PLoS ONE 10(2): e0117301.](https://doi.org/10.1371/journal.pone.0117301)
* [Carpenter J (2011) Jawbones are 'shaped by diet', a study finds. BBC News November 23.](https://www.bbc.com/news/science-environment-15823276)
* [University of Kent. (2011, November 23). Jaw size linked to diet: Could too soft a diet cause lower jaw to stay too short and cause orthodontic problems?. ScienceDaily. Retrieved October 2, 2019 from www.sciencedaily.com/releases/2011/11/111122112032.htm](www.sciencedaily.com/releases/2011/11/111122112032.htm)
* [Lewis KR (2019) Our Skulls Are Out-Evolving Us. OneZero. September 19.](https://onezero.medium.com/our-skulls-are-out-evolving-us-and-that-could-mean-a-public-health-crisis-f950faed696d) &mdash; This seems to conflate many issues. It is the articles that made me look up the rest of this.
* [Hidaka BH (2012) Depression as a disease of modernity: explanations for increasing prevalence. J Affect Disord. 140(3): 205-214.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3330161/)
* [Sadeh A, Gruber R, Raviv A (2003) The Effects of Sleep Restriction and Extension on School‐Age Children: What a Difference an Hour Makes. Child Development. 74(2): 444-455.](https://doi.org/10.1111/1467-8624.7402008)
* [Galland B et al. (2015) Sleep Disordered Breathing and Academic Performance: A Meta-analysis. Pediatrics. 136(4): e934-e946.](http://pediatrics.aappublications.org/cgi/pmidlookup?view=long&pmid=26347434)


Does this have anything to do with farming, or just general stress induced by modernity?
* [Berreby, D (2013) The obesity era. Aeon. June 19.](https://aeon.co/essays/blaming-individuals-for-obesity-may-be-altogether-wrong)
* [Shell ER (2019) A New Theory of Obesity. Scientific American. October.](https://www.scientificamerican.com/article/a-new-theory-of-obesity/)
