# Genetics

If you've benefitted from this document, [please consider tipping me](https://adaburrows.com/support-me/).

## Criticism

Genetic studies are an area fraught with criticism. Some of these criticisms are for good reason, some from fear ([and there can be good reason behind the fear, too](https://en.wikipedia.org/wiki/Human_Genome_Diversity_Project#Potential_problems)). A proper scientific understanding of genetics is necessary to draw conclusions about the following papers. Any understanding mired in the nonsense of racism is bound to produce further nonscientific nonsense. Any studies which do not respect that ownership and copyright are not concepts recognized by indigenous peoples sovereign nations. The earth and all of creation "owns" the genetics. No one can retain ownership or copyright over genetics. Those concepts applied to genes and genetics are rendered as nonsense in many indigenous worldviews, including my own.

Genetic studies provide an additional layer of information which might be missed if just looking at other measurements. For instance, there is the case of the Kennewick Man. The Kennewick Man is the skeleton of a male human from anywhere between approximately 7250 BCE &mdash; 6390 BCE in what is now known as SE Washington by the Columbia River. Initial studies measured various aspects of the man's bone structure. Based on that evidence and surrounding context, the study concluded that [NAGPRA](https://science.sciencemag.org/content/330/6001/166.1.full) did not apply and that the person was from the Aleutian Islands or even Asia. Based on a remark that measurements of the skull matched measurements of Ainu and Polynesian people, a Samoan living in California claimed that the "Ancient One" was one of the descendants of the first rulers of Polynesia. However, genetic testing confirmed that members of the Colvile Tribes where a close match to the "Ancient One". Their ancestor has since been repatriated.

An additional story of genetics helping out Indigenous peoples comes from the Uros who live on manufactured islands on Lake Titicaca. Much of the institutionalized knowledge of the Uros puts them in the same category as the Aymara and not a continuation of the ancient Uros culture. There may be several attributes of their culture that changed over time with interaction with the Aymara and Spanish, but genetic studies have shown they are still genetically distinct. The abstract of the most recent research paper states:

> Our results indicated that Uros populations stand out among the Altiplano populations, while appearing more closely related to the Aymara and Quechua from Lake Titicaca and surrounding regions than to the Amazon Arawaks. Moreover, the Uros populations from Peru and Bolivia are genetically differentiated from each other, indicating a high heterogeneity in this ethnic group. Finally, our results support the distinctive ancestry for the Uros populations of Peru and Bolivia, which are likely derived from ancient Andean lineages that were partially replaced during more recent farming expansion events and the establishment of complex civilizations in the Andes.

The paper itself goes on to say:

> The reduction of the Uros population has been suggested by linguists to have resulted from a gradual “transculturation” during pre-Inca, Inca and Spanish dominance, when many Uros were “puquinized”, “aymarized”, “quechuized”, and “castellanized” [6], [8], [9], [10]. Although the Uros’ original language (Uru or Uruquilla) was reportedly spoken in Puno Bay until 1929 [9], the current Peruvian population of approximately 219 families (∼2000 inhabitants) living in Lake Titicaca (Peruvian census at http://www.inei.gob.pe) speaks only Aymara and Spanish. Bolivian Uros (Uru-Chipaya, Uru-Poopo, and Uru-Irohito) are composed by about 2600 individuals (Bolivian census at http://www.ine.gob.bo), with the majority speaking only Aymara or Spanish [9]. However, about 1050 individuals from Santa Ana de Chipaya in Bolivia [12], [13] still speak their original language, Uru-Chipaya (likely derived from Uruquilla), although with a significant influence in the vocabulary from Aymara, Quechua, and more recently, Spanish language [8], [9].

And finally states:

> In conclusion, the genetic evidence obtained through analyses of autochthonous paternal and maternal lineages from the Andean Altiplano showed that Uros from Puno Bay in Peru and Bolivian Uros communities of Uru-Chipaya and Uru-Poopo are heterogeneous groups, bearing genetic lineages derived from divergent ancestors when compared to most of the current Andean populations. Our results also indicate more gene flow with neighboring Andeans (Aymara and Quechua) than with more distant Andean communities or Arawakan speaking populations. Moreover, the results suggest the Uros could be derived from ancestral Andean stocks that were intermingled and partially replaced by lineages arriving with populations expanding due to a more recent farming expansion and posterior establishment of complex civilizations on the Andes.

This shows a entwined genetic and cultural process that has played out many times where one genetic population gradually comes to replace an older genetic population, while parts of the old culture remain intact. Part of this played out with the Neanderthals in Eurasia: modern populations still have trace signals of Neandertal genetics, but appear to mostly related to their surrounding genetic groups.

* [The AP. (2001) KENNEWICK MAN: Descendant of Polynesian ruler claims rights to remains. Kitsap Sun. July 26.](https://web.archive.org/web/20200914065247/https://products.kitsapsun.com/archive/2001/07-26/0018_kennewick_man__descendant_of_poly.html)
* [NYT Staff. (2001) Northwest: Oregon: New Claim In Ancient-Bones Case. The New York Times. July 28.](https://web.archive.org/web/20110210102252/https://www.nytimes.com/2001/07/28/us/national-briefing-northwest-oregon-new-claim-in-ancient-bones-case.html)
* [Lawler, A. (2010). Grave disputes. Science. 330(6001): 166-170.](https://doi.org/10.1126/science.330.6001.166-a)
* [Achenbach, J. (2014) Scientists: Mysterious Kennewick Man looked Polynesian and came from far away.Washington Post. August 25.](https://web.archive.org/web/20140828073652/https://www.washingtonpost.com/national/health-science/scientists-mysterious-kennewick-man-looked-polynesian-and-came-from-far-away/2014/08/25/45411b2a-27b3-11e4-86ca-6f03cbd15c1a_story.html)
* [Rasmussen M, Sikora M, Albrechtsen A, Korneliussen TS, Moreno-Mayar JV, Poznik GD, et al. (2015) The ancestry and affiliations of Kennewick Man. Nature. 523: 455–458. 10.1038/nature14625](https://www.nature.com/articles/nature14625)
* [Raja, Tasneem. (2016) A Long, Complicated Battle Over 9,000-Year-Old Bones Is Finally Over. Code Switch. NPR. May 5.](https://web.archive.org/web/20160508050128/https://www.npr.org/sections/codeswitch/2016/05/05/476631934/a-long-complicated-battle-over-9-000-year-old-bones-is-finally-over)
* Great podcast: https://digpodcast.org/2019/02/03/skull-collectors-race-pseudoscience-native-american-bodies/
* [Kent, M. (2013). The importance of being Uros: Indigenous identity politics in the genomic age. Social Studies of Science, 43(4), 534-556.](https://www.jstor.org/stable/43284192)
* [Wikipedia contributors. (2020, December 11). Uru people. In Wikipedia, The Free Encyclopedia. Retrieved 06:22, May 13, 2021, from https://en.wikipedia.org/w/index.php?title=Uru_people&oldid=993517527](https://en.wikipedia.org/w/index.php?title=Uru_people&oldid=993517527)
* [Sandoval JR, Lacerda DR, Jota MSA, Salazar-Granara A, Vieira PPR, Acosta O, et al. (2013) The Genetic History of Indigenous Populations of the Peruvian and Bolivian Altiplano: The Legacy of the Uros. PLoS ONE 8(9): e73006. https://doi.org/10.1371/journal.pone.0073006](https://doi.org/10.1371/journal.pone.0073006)

That being said, here is a the high-level overview of the very basics of human genetics:

* An average person has an estimated 37,200,000,000,000 cells containing some genetic material.
* Genetic material contains DNA and RNA.
* Only nucleated cells have DNA.
* Both RNA and DNA are passed on from parents to offspring, but the mechanism for some of the transfer is not fully understood.
* A complete DNA sequence has over 6,000,000,000 base pairs.
* Base pairs are two each of (A - adenine, C - cytosine, G - guanine, T - thymine). These have a particular way of matching up with each other so only certain combinations are possible.
* These base pairs form strands which are weakly bonded through the base pair bond. One stand is the sense strand, the other is the anti-sense strand.
* These two strands form a double helix.
* The DNA is stored in 23 pairs of chromosomes in the nucleus of a cell.
* Parts of chromosomes are broken down into genes, which code for specific molecules.
* The DNA is comprised of sequences which code for molecules (exons) and DNA which typically gets turned into RNA (introns).
* In exons, every three base pairs codes for a part of a protein. This unit is called a codon.
* Any gene coding for a protein can have mutations or several options (one might code for brown eyes, another for blue).
* The different possible genes (with mutations or changes between them) are called alleles.
* The differences between alleles are typically reduced to single nucleotide polymorphisms (SNPs). An SNP is the smallest change (mutation) one can make to a region that codes for molecules by swapping out one of the letters that code for a nucleotide. Many times, the alleles will have long sequences of SNPs.
* The exons (complete set of genetic material that codes for molecules) comprise about 1% of genetic material.
* Many genetic tests check a maximum of 10% of the exon SNPs.
* This means genetic tests only check a maximum of 0.1% of genetic material, unless they do full exon sequencing tests.
* The complete genetic sequence of any human being has not been sequenced. See the article: [Psst, the human genome was never completely sequenced. Some scientists say it should be](https://www.statnews.com/2017/06/20/human-genome-not-fully-sequenced/).
* Additionally, only populations which are sedentary and not traveling and continually mixing DNA develop distinct statistical differences. Those traveling together will likely receive similar mutations to SNPs through interacting with a similar environment.
* Additionally, there is heritable epigenetic coding. Most tests and research papers ignore this. Also, we don't fully understand how this happens since the epigenetic coding is wiped and reprogrammed through an as of yet undiscovered process.

You should take a look at [Whole genome sequencing](https://en.wikipedia.org/wiki/Whole_genome_sequencing) over at Wikipedia for more info.

## General Genetics

### Overview

The following Wikipedia articles contain the basics needed to progress to the next set of papers. You'll likely need to follow all of the links in the articles and fully explore the concepts.

* [Introduction to genetics](https://en.wikipedia.org/wiki/Introduction_to_genetics)
* [DNA](https://en.wikipedia.org/wiki/DNA)
* [RNA](https://en.wikipedia.org/wiki/RNA)
* [Genetic code](https://en.wikipedia.org/wiki/Genetic_code)
* [Genome](https://en.wikipedia.org/wiki/Genome)
* [Gene](https://en.wikipedia.org/wiki/Gene)
* [Gene expression](https://en.wikipedia.org/wiki/Gene_expression)
* [Gene–environment interaction](https://en.wikipedia.org/wiki/Gene%E2%80%93environment_interaction)
* [Epigenetics](https://en.wikipedia.org/wiki/Epigenetics)

### Not overview

The following journal papers contain most of the overview that one would need to understand genetics at a higher level necessary to understand the papers in the next section. In addition to these papers, a sound basis in math and statistics is needed to understand many of the methods used for creating the various graphs and relationship trees having the most parsimony.

* [Reich D (2016) The Genomic Ancient DNA Revolution. Edge. Feb.](https://www.edge.org/conversation/david_reich-the-genomic-ancient-dna-revolution)
* [Salem MSZ (2016) Formal genetic maps. Egyptian Journal of Medical Human Genetics. 16(2): 101-116.](https://doi.org/10.1016/j.ejmhg.2014.11.006)
* [Jo B, Chois SS (2015) Introns: The Functional Benefits of Introns in Genomes. Genomics & Informatics. 13(4): 112-118.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4742320/)

### Relatedness and Divergence over time.

* [Kim, J., Edge, M. D., Goldberg, A., & Rosenberg, N. A. (2021). Skin deep: The decoupling of genetic admixture levels from phenotypes that differed between source populations. American journal of physical anthropology.](https://doi.org/10.1002/ajpa.24261)
* [Sethuraman A (2018) Estimating Genetic Relatedness in Admixed Populations. Genes Genomes Genetics. 8(10): 3203–3220.](https://dx.doi.org/10.1534%2Fg3.118.200485)
* [McIver LJ, McCormick JF, Martin A, Fondon III JW, Garner HR (2013) Population-scale analysis of human microsatellites reveals novel sources of exonic variation. Gene. 516(2)](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3815531/)
* [Kwong M, Pemberton TJ (2014) Sequence differences at orthologous microsatellites inflate estimates of human-chimpanzee differentiation. BMC Genomics. 15: 990.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4253012/)
* [Vaksman Z, Fonville NC, Tae H, Garner HR (2014) Exome-Wide Somatic Microsatellite Variation Is Altered in Cells with DNA Repair Deficiencies. PLoS ONE 9(11): e110263. https://doi.org/10.1371/journal.pone.0110263](https://doi.org/10.1371/journal.pone.0110263)
* [Pemberton TJ et al. (2013) Population Structure in a Comprehensive Genomic Data Set on Human Microsatellite Variation. Genes Genomes Genetics. 3(5): 891-907.](https://doi.org/10.1534/g3.113.005728)
* [López Herráez D, Bauchet M, Tang K, Theunert C, Pugach I, Li J, et al. (2009) Genetic Variation and Recent Positive Selection in Worldwide Human Populations: Evidence from Nearly 1 Million SNPs. PLoS ONE 4(11): e7888. https://doi.org/10.1371/journal.pone.0007888](https://doi.org/10.1371/journal.pone.0007888)
* [Rosenberg NA, Mahajan S, Ramachandran S, Zhao C, Pritchard JK, Feldman MW (2005) Clines, Clusters, and the Effect of Study Design on the Inference of Human Population Structure. PLoS Genet 1(6): e70. https://doi.org/10.1371/journal.pgen.0010070](https://doi.org/10.1371/journal.pgen.0010070)
* [Zhivotovsky LA, Rosenberg NA, Feldman MW (2003) Features of Evolution and Expansion of Modern Humans, Inferred from Genomewide Microsatellite Markers. American Journal of Human Genetics. 72(5): 1171–1186.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1180270/)
* [Driscoll CA et al. (2002) Genomic Microsatellites as Evolutionary Chronometers: A Test in Wild Cats. Genome Research. 12(3):414-23.](https://www.researchgate.net/publication/11487684_Genomic_Microsatellites_as_Evolutionary_Chronometers_A_Test_in_Wild_Cats)

### Accounting for Sequencing Errors

* [Ma X et al. (2019) Analysis of error profiles in deep next-generation sequencing data. Genome Biology. 20.](https://doi.org/10.1186/s13059-019-1659-6)
* [Bilton TP et al. (2018) Accounting for Errors in Low Coverage High-Throughput Sequencing Data When Constructing Genetic Maps Using Biparental Outcrossed Populations. 209(1): 65-76.](https://doi.org/10.1534/genetics.117.300627 )
* [Pfeiffer F et al. (2018) Systematic evaluation of error rates and causes in short samples in next-generation sequencing. Scientific Reports. 8(1): doi: 10.1038/s41598-018-29325-6](https://doi.org/10.1038/s41598-018-29325-6)
* [Wall JD et al. (2014) Estimating genotype error rates from high-coverage next-generation sequence data. Genome Research. 24(11): 1734-1739.](https://dx.doi.org/10.1101%2Fgr.168393.113)
* [Trimble WL et al. (2012) Short-read reading-frame predictors are not created equal: sequence error causes loss of signal. BMC Informatics. 13: 183. doi: 10.1186/1471-2105-13-183](https://doi.org/10.1186/1471-2105-13-183)
* [Wang XV et al. (2012) Estimation of sequencing error rates in short reads. BMC Bioinformatics. 13: 185. doi: 10.1186/1471-2105-13-185](https://dx.doi.org/10.1186%2F1471-2105-13-185)
* [Richterich T (1998) Estimation of Errors in "Raw" DNA Sequences: A Validation Study. Genome Research. 8: 251-259. doi: 10.1101/gr.8.3.251](https://genome.cshlp.org/content/8/3/251.full.html)

### The Myth of "Race"

* [Norton, H.L., Quillen, E.E., Bigham, A.W. et al. Human races are not like dog breeds: refuting a racist analogy. Evo Edu Outreach 12, 17 (2019). https://doi.org/10.1186/s12052-019-0109-y](https://doi.org/10.1186/s12052-019-0109-y)
* [Carja O et al. (2017) Worldwide patterns of human epigenetic variation. Nature Ecology & Evolution. 1: 1577–1583.](https://www.nature.com/articles/s41559-017-0299-z) &mdash; [bioRXiv](https://www.biorxiv.org/content/early/2015/07/05/021931)
* [Rosenberg NA (2011) A Population-Genetic Perspective on the Similarities and Differences among Worldwide Human Populations. Human Biology. 83(6): 659-684.](https://doi.org/10.3378/027.083.0601) &mdash; [Open Access](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3531797/)
* [Li JZ, Absher DD, et al. (2008) Worldwide Human Relationships Inferred from Genome-Wide Patterns of Variation. Science. 319(5866): 1100-1104.](https://science.sciencemag.org/content/319/5866/1100) &mdash; [PDF](https://pdfs.semanticscholar.org/b8de/1df47f30b54c57dce03fb14a4a94459b2e46.pdf) &mdash; Re-iterates the above finding.
  * `Together, these results reaffirm that within-population variation accounts for most of the genetic diversity in humans. However, the between-population variance is sufficient to reveal consistent population structure because subtle but nonrandom differences between populations accumulate over a large number of loci and yield principal components that can account for a major portion of the variation (21).`
* [Race, Ethnicity, and Genetics Working Group (2005) The Use of Racial, Ethnic, and Ancestral Categories in Human Genetics Research. American Journal of Human Genetics. 77(4): 519–532.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1275602/)
* [Excoffier, L., & Hamilton, G. (2003). Comment on genetic structure of human populations. Science, 300(5627), 1877. DOI: 10.1126/science.1083411](https://doi.or/10.1126/science.1083411)
* [Rosenberg NA et al. (2002) Genetic Structure of Human Populations. Science. 298(5602): 2381-2385.](https://science.sciencemag.org/content/298/5602/2381.full) &mdash; [PDF](https://www.researchgate.net/publication/10980474_Genetic_Structure_of_Human_Populations) &mdash; This is the original paper that started the meme that humans are roughly 99.9% the same. The biggest take away from the paper is that self reporting would likely be a reliable proxy for what genetic conditions they might get. The second take away was the intra-group genetic variation is greater than inter-group genetic variation.

## "Imagined Genetic Communities"

* [Kent, M., García-Deister, V., López-Beltrán, C., Santos, R. V., Schwartz-Marín, E., & Wade, P. (2015). Building the genomic nation:‘Homo Brasilis’ and the ‘Genoma Mexicano’in comparative cultural perspective. Social Studies of Science, 45(6), 839-861.](https://doi.org/10.1177/0306312715611262)
  * > This analysis addresses the complex ways in which genetic research becomes articulated with socially imagined identities and nations. Since the 1990s, the development of new molecular techniques has enabled direct biological connections between contemporary populations and (a proportion of) their ancestors, as well as the production of genetic profiles that indicate degrees of mixture and the relative contribution of ancestral populations (Amerindian, African and European in the case of Latin America). Both geneticists and non-scientists have at times used genetic knowledge as an element in their interpretations of social groups, their histories and their futures (e.g. Wailoo et al., 2012). In this article, we focus primarily on geneticists’ discourses about individual and collective identities, and the forms of ‘biosociality’ (Rabinow, 1996) and ‘imagined genetic communities’ (Simpson, 2000) that these discourses imply, in terms of possible collective identifications that are based on presumed genetic characteristics (Brodwin, 2002; Kent, 2013; Nelson, 2008; Pálsson, 2007). <br/><br/>Academic debates concerned with the relationship between genetics and identity – particularly those in the United States, but also those in Europe – have often focused on race and ethnicity. Some social scientists have emphasized the role of genetics in reproducing racial categories, albeit in altered form (Fujimura et al., 2008; Fullwiley, 2008; Koenig et al., 2008; Montoya, 2007; M’charek, 2013; Palmié, 2007). Others have raised concerns about the divide between notions of genetic ancestry, on the one hand, and indigenous conceptions of identity, origins and belonging, on the other (Reardon, 2005; TallBear, 2007). With some exceptions (M’charek et al., 2014b; Nash, 2013; Pálsson and Rabinow, 1999; Wailoo et al., 2012), the relationship between genetics and imagined national identities has received less attention, in spite of considerable research on genetic sovereignty, national databases and the nation as a collective of biological citizens with different levels of inclusion (Benjamin, 2009; Heath et al., 2004; Hinterberger, 2012a; Pálsson, 2007; Rabinow, 1999; Rose and Novas, 2005; Schwartz-Marín and Silva-Zolezzi, 2010). Yet in Latin America, the nation – as well as the mestizo that stands at the heart of national identity constructions – has figured as much or even more in human genetic research than has race or ethnicity (Hartigan, 2013; Santos and Maio, 2004). In Latin American countries, in particular in Brazil and Mexico, mixture is granted higher levels of interpretative priority in understanding the national population than racial categories, although the overtness of race varies widely within Latin America – for example, between Brazil and Mexico (Wade et al., 2014b). The figure of the mestizo allows for an ‘absent presence’ of race – in which race appears to be absent, but is present in the traces it has made (M’charek et al., 2014a) – insofar as there is a racialized structure embedded or hidden in the mestizo, but the nation occupies centre stage (Wade et al., 2014a). Brazil and Mexico offer a privileged context for exploring the relation between genetics, national identities and nation-building efforts, in part because of the powerful articulations between biological and social repertoires established by the mestizo as a long-standing scientific and social object.
  * > In Brazil and Mexico, there is a long tradition of thinking about the nation through its biological constitution (Stepan, 1991). In the late 19th and the early 20th centuries, physical anthropology played a central role in debates about national identity in both countries (García Murcia, 2013; Santos, 2012; Vergara Silva, 2013), analysing the history and constitution of national populations, and addressing the desirable future of the nation. In Brazil and Mexico, life scientists, social scientists and other intellectuals perceived their nations as racially heterogeneous and interpreted race-based differences as the natural bases of social hierarchies. Additionally, European and North American racialist theories influenced many (though not all) of these thinkers to interpret admixture as a form of degeneration, which damaged the fitness and productive potential of the nations’ populations. Close connections were established between the health of the nation’s social body and the health and vitality of citizens’ individual bodies (Appelbaum et al., 2003). In Brazil, ‘whitening’ was understood as a means of improving the nation, and state policies encouraged the immigration of approximately six million Europeans. Mexico had less immigration, but still ‘racial homogenization was an imperative on which everyone in nineteenth-century Mexico agreed’, even if ‘the type of citizen required by the new nation and the formula for creating such a citizen were a source of disagreement’ (López-Beltrán and García-Deister, 2013: 393).
  * > Mexico’s solution to racial heterogeneity involved protecting indigenous peoples and simultaneously incorporating them into the mestizo body of the nation. This assimilationist project, which was known as indigenismo, was facilitated by biological crossing and educational acculturation. Physical anthropology, which emerged as the key scientific discipline for forging national identity (García Murcia, 2013; Rutsch, 2001), was instrumental. The project was successful insofar as the brown mestizo – descendant of a mythified original encounter between Spanish male and female indígenas – came to be synonymous with the typical Mexican (Gómez Izquierdo and Sánchez Díaz, 2011). Influenced by the work of philosopher and politician José Vasconcelos (1997 [1925]) author of La raza cósmica, a group in the National Autonomous University of Mexico, under the name Hiperión, undertook the task of ‘making manifest the phenomena constituting what it meant to be Mexican, and ultimately, with prescribing a “mode of being” appropriate for the creation of an authentic and, consequently, a responsible homo mexicanus’ (Sanchez, 2008: 442).<br/><br/>Over a similar period, the idea of the raça brasileira was debated by Brazilian anthropologist Roquette-Pinto and sociologist Oliveira Vianna, while a Brazilian moreno (brown) ‘meta-race’ was discussed, somewhat later, by Gilberto Freyre (Hofbauer, 2006; Lima and Sá, 2008; Oliveira Vianna, 1923). In Brazil, Freyre’s work has been influential, with its argument that the intense and relatively consensual mixing of white Europeans, black slaves and Indians – mestiçagem – blurred differences so much that, nowadays, there are no clear-cut distinctions between them, only a racial continuum from the whitest to the blackest individual (Freyre, 1936). This point has often been pushed further to argue that Brazil has become a ‘racial democracy’, in which relations between people of different colours are relatively egalitarian and racism plays a minor role. This view became official ideology and was actively promoted as part of the nationalist policies of the Getúlio Vargas administration (1930–1945), and during the military dictatorship (1964–1985). In the 1950s, United Nations Educational, Scientific and Cultural Organization (UNESCO) sponsored studies of Brazil as a possible example of successful race relations, with mixed results (Maio, 2001; Wade, 2010: 52–59).
  * > These differences have led to distinct Brazilian and Mexican mobilizations of the genomic mestizo. In Mexico, the state, geneticists and the medical community agree that improvements to health policy are the main goal of genomic research, despite disagreements over strategies and the roles of INMEGEN. And while multiculturalism and indigenous rights are certainly at issue in Mexico – witness the Zapatista movement – race is not a major controversy in genomic research. INMEGEN has had little to say openly about multiculturalism or race. As we have illustrated, the GM project implicitly reproduced the old divide between indigenous and mestizo identities, repackaging it in genetic idiom. This divide continues to guide approaches to national identity in post-multicultural reform Mexico, framing the genetic mestizo as paradigmatic. The presence of race is still significant, but implicit.
* [Kent, M., Santos, R. V., & Wade, P. (2014). Negotiating imagined genetic communities: unity and diversity in Brazilian science and society. American Anthropologist, 116(4), 736-748.](https://anthrosource.onlinelibrary.wiley.com/doi/pdf/10.1111/aman.12142)
  * > The Brazilian material contributes various ideas to these debates. First, we argue that genomics creates a multiplicity of “imagined genetic communities” or nations, with race playing a varied role. Rather than generating a single mode of construing the nation and its racial aspects (or lack of them), geneticists and those who engage with their data produce several different genetic images, with varied political affordances. It is important to engage with the internal diversity of genomic ancestry research and avoid homogenizing its effects. The way genomics is mediated through lay people’s engagement with it creates some heterogeneity, but genomics itself also creates heterogeneous images of race and nation.<br/><br/>Second, the variety of genetic images of Brazil is distinctive in the way that images of homogenous mixture and heterogeneous mixture are generated. A good deal of the literature on genomics and racial identity highlights how genomics can lend a genetic meaning to specific racialized identities and connections—for example, in the U.S. case, people may search out African or Native American links (Hamilton 2012); the Lemba in Africa focused on Jewish genetic heritage (Abu El-Haj 2012). In addition, in the United States and United Kingdom, the politics of inclusion has institutionalized the recognition of ethnic and racial difference in medical research (Epstein 2007; Smart et al. 2008). In Brazil, we see a more varied picture, as genomic data underpin diverse representations of the nation, bringing race into and out of focus. Brazil is imagined as various genetic communities: as homogeneous and raceless; as a country differentiated by regions with very distinctive ancestral mixes; as rather European; and as connected to Africa. Genomics participates in long-standing debates about Brazil that have involved not only ideological and cultural narratives but also older modes of biological and physical anthropological investigation into diversity.
  * This paper makes some really fantastic points and deserves a full read.

## Genetics of Humans

* [Liu, Y., Mao, X., Krause, J., & Fu, Q. (2021). Insights into human history from the first decade of ancient human genomics. Science, 373(6562), 1479-1484.](https://doi.org/10.1126/science.abi8202)
  * > The genomes of Neanderthals and Denisovans have been reconstructed in 2010 (3, 6). Analyses of their autosomes indicate that they are more closely related to each other than to modern humans, their sister group (3). From the level of shared genetic variants, archaic humans are estimated to have separated from modern humans ~550 thousand years ago (ka), with Neanderthals and Denisovans splitting from each other ~400 ka (9–11) (Fig. 2). Genetic admixture with Neanderthals and Denisovans has been widely detected in present-day non-Africans, although admixture proportions and geographical distributions differ widely. Neanderthal-derived admixture has been detected in all present-day non-Africans, with East Asian genomes containing a higher amount of Neanderthal ancestries than that of western Eurasians (2.3 to 2.6% and 1.8 to 2.4%, respectively) (3, 6, 9), although a recent study suggests a smaller difference in levels of Neanderthal ancestry in non-African populations after correcting for back-migration into African populations (12). In addition, it has been hypothesized that some African populations carry ancestry from an even older archaic lineage that predates the divergence of Neanderthals and modern humans (13). Furthermore, ancient modern humans mated with extinct archaic lineages as multiple waves of introgression from Neanderthals and Denisovans to non-Africans have been suggested (Fig. 2) (14). This is supported by estimates of up to four admixture pulses from Denisovans and three pulses from Neanderthals to modern humans, accounting for the presence of archaic-derived DNA [including some adaptive introgressions, such as EPAS1 (endothelial Pas domain protein 1) gene (4)] in present-day populations (15–17).
  * > Admixture between modern humans and archaic lineages appears to have occurred in both directions. Studies suggest that both the mitochondrial DNA and Y chromosome of Neanderthals were replaced by that of early modern humans, which was likely a result of introgression from modern humans into Neanderthals between ~370 and 220 ka (Fig. 2) (18, 19). Neanderthal–modern human admixture occurring as recently as ~40 ka has been identified, although these lineages apparently did not survive to the present day (20, 21). Also, an ~50-kyr-old archaic individual (Denisova 11) had a Neanderthal mother and a Denisovan father (22). Considering the increasing number of ancient individuals identified with recent archaic ancestries, past hominins may have mixed frequently, opening the question of whether archaic and modern human should be regarded as distinct lineages or rather points taken from a continuous spectrum of genetic diversity that was genetically connected throughout the past ~500 kyr similar to that of present-day human populations (20, 22, 23).
* [Bergström, A., McCarthy, S. A., Hui, R., Almarri, M. A., Ayub, Q., Danecek, P., ... & Tyler-Smith, C. (2020). Insights into human genetic variation and population history from 929 diverse genomes. Science, 367(6484).](https://doi.org/10.1126/science.aay5012)
  * > Assuming a mutation rate of 1.25 × 10−8 per base pair per generation (30) and a generation time of 29 years (31), our midpoint estimates suggest (Fig. 5A) splits between the two central African rainforest hunter-gatherer groups, the Mbuti and the Biaka, at ~62 kya; between the Mbuti and the West African Yoruba at ~69 kya; between the Yoruba and the southern African San at ~126 kya; and between the San and both the Biaka and the Mbuti at ~110 kya. Non-Africans have separation midpoints from the Yoruba at ~76 kya, from the Biaka at ~96 kya, from the Mbuti at ~123 kya, and, representing the deepest split in the dataset, from the San at ~162 kya. However, all of these curves are clearly inconsistent with clean splits, suggesting a picture where genetic separations within Africa were gradual and shaped by ongoing gene flow over tens of thousands of years. For example, there is evidence of gene flow between the San and the Biaka until at least 50 kya, and between the Mbuti, the Biaka, and the Yoruba until the present day or as recently as the method can infer.
  * > [...] For the deepest splits, there is some evidence of genetic separation dating back to before 300 or even 500 kya, in the sense that even by that time, the rate of coalescence between populations still differs from that within populations. The implication of this is that there lived populations already at this time that contributed more to some present-day human ancestries than to others. We find that a small degree of such deep structure in MSMC2 curves might be spuriously caused by batch effects associated with sequencing and genotyping pairs of chromosomes from diploid human samples together, but that such effects are not large enough to fully explain the differences in coalescence rates at these time scales (fig. S7). However, even if this signal reflects actual ancient population structure, its magnitude is such that it would only apply to small fractions of present-day ancestries. An analogy to this is how Neanderthal and Denisovan admixture results in a few percent of non-African ancestries separating from some African ancestries approximately half a million years ago, whereas most of the ancestry was connected until much more recently. In light of such composite ancestries in present-day human populations and the clear deviation of our MSMC2 results from instant split behaviors, we argue that single point estimates are inadequate for describing the timing of early modern human population separations. A more meaningful summary of our results might be that the structure we observe among human populations today formed predominantly during the past 250,000 years, with continued genetic contact between all populations during much of this time, but also a small fraction of present-day ancestries retaining traces of structure that is older than this, potentially by hundreds of thousands of years.
  * > [...] The midpoint estimates suggest that modern and archaic populations separated 550 to 700 kya (Fig. 5A), which is consistent with, but potentially slightly earlier than, estimates obtained with other methods (16, 20). These results also provide relative constraints on the overall time depth of modern human structure that are independent of the mutation rate we use to scale the results, in the sense that the deepest modern human midpoints are less than one-third the age of the midpoints of the archaic curves. However, the deep tails of some modern human curves partly overlap a time period when genetic separation from the archaics might still not have been complete. The separation between archaic and modern humans appears more sudden than those between different modern human populations and only slightly less sudden than expected under an instant split scenario, suggesting a qualitatively different mode of separation between modern and archaic groups than between modern human groups within Africa. While the divergence time between modern human and Neanderthal mitochondrial genomes shows that there is at least some ancestry shared more recently than 500 kya (32), these MSMC2 results suggest that post-split gene flow to and from the archaic groups, likely geographically restricted to Eurasia, overall would have been limited.
  * > Outside of Africa, the time depths of population splits are consistent with previous estimates (3, 4, 22), with all populations sharing most of their ancestry within the past 70 kya (Fig. 5B). Our analyses of these physically phased genomes do not replicate a previously observed earlier divergence of West Africans from Oceanians than from Eurasians in MSMC analyses (4, 29), suggesting that those results were caused by some artifact of statistical phasing. Instead, all non-African populations display very similar histories of separation from African populations (fig. S6). Like those within Africa, many curves between non-African populations are more gradual than instant split simulations. However, some curves, including those between the Central American Pima and the South American Karitiana, between the Han Chinese and the Siberian Yakut, or between the European Sardinians and the Near Eastern Druze, do not deviate appreciably from those expected under instant splits. This suggests that once modern humans had expanded into the geographically diverse and fragmented continents outside of Africa, populations would sometimes separate suddenly and without much subsequent gene flow.
  * > We estimate an average of 2.4 and 2.1% Neanderthal ancestry in eastern non-Africans and western non-Africans, respectively. We estimate 2.8% (95% confidence interval: 2.1 to 3.6%) Denisovan ancestry in Papuan highlanders (15), substantially lower than the first estimate of 4 to 6% (35) based on less comprehensive modern and archaic data, but only slightly lower than more recent estimates (11, 36, 37). The proportion of ancestry that remains in present-day Oceanian populations after the Denisovan admixture is thus likely not much higher than the amount of Neanderthal ancestry that remains in non-Africans generally.
  * > We identified Neanderthal and Denisovan segments in non-African genomes using a hidden Markov model (15) and studied the diversity of these haplotypes to learn about the structure of these admixture events and whether they involved one or more source populations. [...] Coalescent simulations suggest that, genome-wide, as few as two to four founding haplotypes are sufficient to produce the observed distribution of haplotype network sizes.
  * > By contrast, Denisovan segments show evidence of a more complex admixture history. Segments in Oceania are distinct from those in East Asia, the Americas, and South Asia, as shown by their different distribution along the genome (fig. S18 and table S8), high DXY values (Fig. 6A), and a clear separation in most haplotype networks between these two geographical groups (fig. S24 and table S10), corresponding to a deep divergence between the Denisovan source populations. East Asian populations also harbor some Denisovan segments that are very similar to the Altai Denisovan genome, but which are absent from Oceania (fig. S19). This is consistent with the Denisovan ancestry in Oceania having originated from a separate gene flow event not experienced in other parts of the world (39). We do not, however, find clear evidence of more than one source in Oceanians (40). The more complicated structure of the Denisovan segments in East Asia (and likely also in the Americas and South Asia) is difficult to explain by one or even two admixture events and may possibly reflect encounters with multiple Denisovan populations by the ancestors of modern humans in Asia. Some Denisovan haplotypes found in Cambodians are somewhat distinct from those in the rest of East Asia with tentative connections to those in Oceania. Overall, these results paint a picture of an admixture history from Denisovan-related populations into modern humans that is substantially more complex than the history of admixture from Neanderthals.
  * > While there is an excess of haplotypes deriving from archaic admixture in non-Africans, many single variants present in archaic populations are also present in Africans due to their having segregated in the population ancestral to archaic and modern humans, and some of these variants were subsequently lost in non-Africans owing to increased genetic drift. Counting how many of the variants carried in heterozygote state in archaic individuals are segregating in balanced sets of African and non-African genomes, we find that more Vindija Neanderthal variants survive in non-Africans than in Africans (31.0 versus 26.4%). However, more Denisovan variants survive in Africans (18.9 versus 20.3%). These numbers might change if larger numbers of Oceanian populations were surveyed, but they highlight how the high levels of genetic diversity in African populations mean that, despite having received much less or no Neanderthal and Denisovan admixture, they still retain a substantial and only partly overlapping (Fig. 3E) subset of the variants that were segregating in late archaic populations.
* [Hellenthal, G., Busby, G. B., Band, G., Wilson, J. F., Capelli, C., Falush, D., & Myers, S. (2014). A genetic atlas of human admixture history. science, 343(6172), 747-751.](https://doi.org/10.1126/science.1243518)
  * > Diverse historical, archaeological, anthropological, and linguistic sources of information indicate that human populations have interacted throughout history, because of the rise and fall of empires, invasions, migrations, slavery, and trade. These interactions can result in sudden or gradual transfers of genetic material, creating admixed populations. However, the genetic legacy of these interactions remains unknown in most cases, and the historical record is incomplete. We have developed an approach that provides a detailed characterization of the mixture events in the ancestry of sampled populations on the basis of genetic data alone.
  * > Admixed populations should have segments of DNA from all contributing source groups (Fig. 1A), whose sizes decrease over successive generations because of recombination, and approaches have been developed to date admixture events by inferring the size of ancestry segments (1–5). Between-population frequency differences of individual alleles may provide information on ancestry sources (6, 7). On the basis of these principles, we developed an integrated approach by using genome-wide patterns of ancestry to infer jointly both fine-scale information about groups involved in admixture and its timing, allowing for the fact that migration and admixture events can occur at multiple times or involve numerous groups.
  * > Our results demonstrate that it is possible to elucidate the effect of ancient and modern migration events and to provide fine-scale details of the sources involved, the complexity of events, and the timing of mixing of groups by using genetic information alone. Where independent information exists from alternative historical or archaeological sources, our approach provides results consistent with known facts and determines the amount of genetic material exchanged. In other cases, novel mixture events we infer are plausible and often involve geographically nearby sources, supporting their validity. Admixture events within the past several thousand years affect most human populations, and this needs to be taken into account in inferences aiming to look at the more distant history of our species. Future improvements in whole-genome sequencing, greater sample sizes, and incorporation of ancient DNA, together with additional methodological extensions, are likely to allow better understanding of ancient events where little or no historical record exists, to identify many additional events, to infer sex biases, and to provide more precise event characterization than currently possible. We believe our approach will extend naturally to these settings, as well as to other species.
* [Rasmussen, M., Guo, X., Wang, Y., Lohmueller, K. E., Rasmussen, S., Albrechtsen, A., ... & Willerslev, E. (2011). An Aboriginal Australian genome reveals separate human dispersals into Asia. Science, 334(6052), 94-98.](https://doi.org/10.1126/science.1211177)
  * > We present an Aboriginal Australian genomic sequence obtained from a 100-year-old lock of hair donated by an Aboriginal man from southern Western Australia in the early 20th century. We detect no evidence of European admixture and estimate contamination levels to be below 0.5%. We show that Aboriginal Australians are descendants of an early human dispersal into eastern Asia, possibly 62,000 to 75,000 years ago. This dispersal is separate from the one that gave rise to modern Asians 25,000 to 38,000 years ago. We also find evidence of gene flow between populations of the two dispersal waves prior to the divergence of Native Americans from modern Asian ancestors. Our findings support the hypothesis that present-day Aboriginal Australians descend from the earliest humans to occupy Australia, likely representing one of the oldest continuous populations outside Africa.
  * > The genetic history of Aboriginal Australians is contentious but highly important for understanding the evolution of modern humans. All living non-African populations likely derived from a single dispersal of modern humans out of Africa, followed by subsequent serial founder effects (1, 2). Accordingly, eastern Asia is hypothesized to have been populated by a single early migration wave rather than multiple dispersals (3). In this “single-dispersal model,” Aboriginal Australians are predicted to have diversified from within the Asian cluster [for definitions of human populations and groups, see (4)] (Fig. 1A, top). Recent whole-genome studies reveal a split between Europeans and Asians dating to 17,000 to 43,000 years before the present (B.P.) (5, 6). Because greater Australia (Australia and Melanesia, including New Guinea) has some of the earliest archaeological evidence of anatomically modern humans outside Africa, dating back to ~50,000 years B.P. (7, 8), a divergence of aboriginal Australasians from within the Asian cluster is not compatible with population continuity in Australia. Alternatively, on the basis of archaeological and fossil evidence, it has been proposed that greater Australia was occupied by an early, possibly independent out-of-Africa dispersal, before the population expansion giving rise to the majority of present-day Eurasians (9, 10). According to this “multiple-dispersal model,” the descendants of the earlier migration became assimilated or replaced by the later-dispersing populations, with a few exceptions that include Aboriginal Australians (10, 11) (Fig. 1A, bottom).
  * > The degree to which a single individual is representative of the evolutionary history of Aboriginal Australians more generally is unclear. Nonetheless, we conclude that the ancestors of this Aboriginal Australian man—and possibly of all Aboriginal Australians—are as distant from Africans as are other Eurasians, and that the Aboriginal ancestors split 62,000 to 75,000 years B.P. from the gene pool that all contemporary non-African populations appear to descend from. Rather than supporting a single early human expansion into eastern Asia, our findings support the alternative model of Aboriginal Australians descending from an early Asian expansion wave some 62,000 to 75,000 years B.P. The data also fit this model’s prediction of substantial admixture and replacement of populations from the first wave by the second expansion wave, with a few populations such as Aboriginal Australians, and possibly PNG Highlands and Aeta, being remnants of the early dispersal (Fig. 2). This is compatible with mtDNA data showing that although all haplogroups observed in Australia are unique to this region, they derive from the same few founder haplogroups that are shared by all non-African populations (4). Finally, our data are in agreement with contemporary Aboriginal Australians being the direct descendants from the first humans to be found in Australia, dating to ~50,000 years B.P. (7, 8). This means that Aboriginal Australians likely have one of the oldest continuous population histories outside sub-Saharan Africa today.
  * Figure 1 is rather amazing and might be similar to other whole world genome projects, but it's smaller and shows different populations than the others I've seen.
* [Pemberton, T. J., Absher, D., Feldman, M. W., Myers, R. M., Rosenberg, N. A., & Li, J. Z. (2012). Genomic patterns of homozygosity in worldwide human populations. The American Journal of Human Genetics, 91(2), 275-292.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3415543/)
  * > **Abstract:** Genome-wide patterns of homozygosity runs and their variation across individuals provide a valuable and often untapped resource for studying human genetic diversity and evolutionary history. Using genotype data at 577,489 autosomal SNPs, we employed a likelihood-based approach to identify runs of homozygosity (ROH) in 1,839 individuals representing 64 worldwide populations, classifying them by length into three classes—short, intermediate, and long—with a model-based clustering algorithm. For each class, the number and total length of ROH per individual show considerable variation across individuals and populations. The total lengths of short and intermediate ROH per individual increase with the distance of a population from East Africa, in agreement with similar patterns previously observed for locus-wise homozygosity and linkage disequilibrium. By contrast, total lengths of long ROH show large interindividual variations that probably reflect recent inbreeding patterns, with higher values occurring more often in populations with known high frequencies of consanguineous unions. Across the genome, distributions of ROH are not uniform, and they have distinctive continental patterns. ROH frequencies across the genome are correlated with local genomic variables such as recombination rate, as well as with signals of recent positive selection. In addition, long ROH are more frequent in genomic regions harboring genes associated with autosomal-dominant diseases than in regions not implicated in Mendelian diseases. These results provide insight into the way in which homozygosity patterns are produced, and they generate baseline homozygosity patterns that can be used to aid homozygosity mapping of genes associated with recessive diseases.
  * > In each population, the size distribution of ROH appears to contain multiple components (Figure 2A). Using a three-component Gaussian mixture model, we classified ROH in each population into three size classes (Figure 2B): short (class A), intermediate (class B), and long (class C). Size boundaries between different classes vary across populations (Table S1); however, considering all populations, all A-B boundaries are strictly smaller than all B-C boundaries (Figure 2C). The mean sizes of class A and B ROH are similar among populations from the same geographic region (Figure S3), with the exception that Africa and East Asia have greater variability. The class C mean is generally largest in the Middle East, Central/South Asia, and the Americas and smallest in East Asia (Figure S3), with the exception that the Tujia population has the largest value. In the admixed Mexican population (MXL), mean ROH sizes are similar to those in European populations. In the admixed African American population (ASW), however, mean ROH sizes are among the smallest in our data set, notably smaller than in most Africans and Europeans.
  * > Several patterns emerge from a comparison of the per-individual total lengths of ROH across populations (Figure 3). First, the total lengths of class A (Figure 3A) and class B (Figure 3B) ROH generally increase with distance from Africa, rising in a stepwise fashion in successive continental groups. This trend is similar to the observed reduction in haplotype diversity with increasing distance from Africa.32,35,50,51 Second, total lengths of class C ROH (Figure 3C) do not show the stepwise increase. Instead, they are higher and more variable in most populations from the Middle East, Central/South Asia, Oceania, and the Americas than in most populations from Africa, Europe, and East Asia. This pattern suggests that a larger fraction of individuals from the Middle East, Central/South Asia, Oceania, and the Americas tend to have higher levels of parental relatedness, in accordance with demographic estimates of high levels of consanguineous marriage particularly in populations from the Middle East and Central/South Asia,52,53 and it is similar to that observed for inbreeding-coefficient54 and identity-by-descent55 estimates. Third, in the admixed ASW and MXL individuals, total lengths of ROH in each size class are similar to those observed in populations from Africa and Europe, respectively (Figure 3).
* [Kirin M, McQuillan R, Franklin CS, Campbell H, McKeigue PM, Wilson JF (2010) Genomic Runs of Homozygosity Record Population History and Consanguinity. PLoS ONE 5(11): e13996.](https://doi.org/10.1371/journal.pone.0013996)
  * **Note:** It's quite likely that these ROHs are more or signals about genetic bottlenecks.
  * > When an individual's parents share a relatively recent common ancestor they will share large parts of their genomes that are identical-by-descent. If both of the parents transmit the same segment to the child, the child will be homozygous for that segment, thus creating a run of homozygosity (ROH)[1]. However, parents can pass on identical chromosomal segments to a child even when the relationship between them is a very distant one. There is therefore a continuum of homozygous segment length, depending on the degree of shared parental ancestry and its age. ROH due to recent inbreeding will tend to be longer, because there has been little opportunity for recombination to break up the segments that are identical-by-descent. On the other hand, ROH of much older origin are generally much shorter because the chromosomal segments have been broken down by repeated meioses. Some longer segments may, however, persist due to locally low recombination rates in certain areas of the genome. Based on this observation it is of interest to compare the extent of homozygosity between populations with different degrees of isolation and consanguinity. The availability of genome scan technology, capable of genotyping individuals at hundreds of thousands of markers has made this observational approach possible.
  * > In common with recent studies of European-heritage populations [1], [4], [6], our analysis of this globally representative sample confirms that runs of homozygosity longer than 0.5 Mb are ubiquitous and frequent across all populations. Comparison of the number and sum length of ROH of various sizes to the mean inbreeding coefficient and effective population size indicate that ROH provide a valuable record of a population's demographic history. Moreover, ROH record the demographic history of the ancestors of the individual. In fact, in most populations individual ROH profiles cluster, particularly in terms of the number of ROH. Four classes of population can be distinguished on the basis of the length and number of ROH, which describe a demographic historical space. First the consanguineous populations from South and West Asia stand out in carrying significantly more very long ROH than the African and other Eurasian populations. Second the Oceanian populations are unique in having very large numbers of shorter ROH, but few long ROH, consistent with a reduced Ne in the past, but little inbreeding in recent times. Third the Native American populations have more short and long runs than any other population, indicating both ancient and recent parental relatedness. Fourth, the Europeans, East Asians and Africans show only the background level of shared ancestry relating to their continental Ne.
  * > The distribution of the shorter (0.5–2 Mb) ROH across the globe mirrors other aspects of human variation in reflecting our origin in sub-Saharan Africa and subsequent dispersals out of Africa, finally reaching Oceania and America. Analysis of ROH in three populations using 3.1 million markers revealed the same Africa, Europe, East Asia ranking [5]. The shortest category of ROH considered here (0.5–1 Mb) were inversely correlated with estimates of male Ne based on Y chromosome diversity. We note that the Y chromosome estimates of Ne are derived from independent data and thus avoid the circularity of comparing homozygosity with estimates of Ne derived from LD, which itself influences ROH. It must be remembered that the Y chromosome only records male history and furthermore represents only one realisation of the evolutionary process, so it will be interesting to compare to estimates of effective population size which also include mtDNA and/or whole genome variation. Longer ROH provide information on more recent ancestry, from population size and endogamy to recent inbreeding. However, unlike genetic diversity, linkage disequilibrium and other summary statistics of gene genealogies, ROH are an individual-level phenomenon and so provide a distinctive record of the demographic history of an individual's ancestors.
  * > The increased numbers and so total length of ROH from 0.5–2 Mb in certain populations is likely to be the result of common extended haplotypes reaching high frequencies in these small, isolated communities, such that they are frequently inherited from both parents. Such patterns of haplotype sharing – among individuals or among parents of individuals – can also be described as correlations among alleles at different loci or LD. Because we are interested in the potential expression of recessive effects, we include ROH regardless of local patterns of LD; the small sample sizes available also prevent unbiased estimates of LD. These shorter ROH do not arise from inbreeding in recent generations and are common in all of the populations represented in the HGDP. In all populations shorter ROH make up the bulk of the homozygosity present. Even in the most inbred populations, the total length of the genome in shorter ROH (0. 5–2 Mb) is more than that in longer ROH (over 2 Mb). This point can be illustrated more dramatically by using a panel of 3 million SNPs, which enables the reliable detection of ROH as short as 100 kb. Using this panel, the total ROH length among Han Chinese is ∼510 Mb [5], compared with ∼130 Mb using a panel of ∼400,000 SNPs). Given that shorter ROH account for more of the total homozygosity even in the most inbred people, any effect of ROH on disease risk could also be mediated by these shorter runs, and not only by the long ROH arising from recent parental relatedness.
  * > Individuals from particular populations stand somewhat apart from others of their continent due to a “right shift” (File S1), e.g. most of the very isolated Yakuts, and to a lesser degree some Sardinians, whilst the Maya have less homozygosity than the other Native Americans sampled, perhaps due to European admixture [9].
  * > ROH, like other aspects of our genetic variation including genetic diversity and linkage disequilibrium, demonstrate a strong correlation with distance from East Africa. This pattern was driven mainly by the shorter ROH, consistent with their origin long enough ago to be influenced by serial founder events as humans spread across the globe. Longer ROH were less correlated with walking distance from Africa. When the analyses were repeated without Native Americans, to exclude the possibility of an increase in their homozygosity levels for reasons other than their distance from Africa, the shorter ROH remained highly correlated, but there was no significant relationship between distance from Africa and sum of ROH over 16 Mb in length. The lack of relationship for longer ROH is not surprising, given their very recent origins. The correlation with Out-of-Africa distances for shorter ROH show that patterns of identity-by-descent are structured by our migrations from Africa, not just identity-by-state observed through single SNP analyses of diversity.
  * > Long ROH are a neglected feature of our genome, which we have shown here to be universally common in human populations and to correlate well with demographic history. ROH are, however, only partially predictable from an individual's background (due to the stochastic nature of inheritance). As well as conferring susceptibility to recessive Mendelian diseases, ROH are also potentially an underappreciated risk factor for common complex diseases, given the evidence for a recessive component in many complex disease traits [18]; they will allow quantification of the risk arising from recessive genetic variants in different populations.
* [Li JZ, Absher DD, et al. (2008) Worldwide Human Relationships Inferred from Genome-Wide Patterns of Variation. Science. 319(5866): 1100-1104.](https://science.sciencemag.org/content/319/5866/1100) &mdash; [PDF](https://pdfs.semanticscholar.org/b8de/1df47f30b54c57dce03fb14a4a94459b2e46.pdf)
  * > In the past 30 years, the ability to study DNA sequence variation has dramatically increased our knowledge of the relationships among and history of human populations. Analyses of mitochondrial, Y chromosomal, and autosomal markers have revealed geographical structuring of human populations at the continental level (1–3) and suggest that a small group of individuals migrated out of eastern Africa and their descendants subsequently expanded into most of today's populations (3–6). Despite this progress, these studies were limited to a small fraction of the genome, to limited populations, or both, and yield an incomplete picture of the relative importance of mutation, recombination, migration, demography, selection, and random drift (7–10). To substantially increase the genomic and population coverage of past studies (e.g., the HapMap Project), we have examined more than 650,000 single-nucleotide polymorphisms (SNPs) in samples from the Human Genome Diversity Panel (HGDP-CEPH), which represents 1064 fully consenting individuals from 51 populations from sub-Saharan Africa, North Africa, Europe, the Middle East, South/Central Asia, East Asia, Oceania, and the Americas (11). This data set is freely available (12) and allows a detailed characterization of worldwide genetic variation.
  * > In many populations, ancestry is derived predominantly from one of the inferred components, whereas in others, especially those in the Middle East and South/Central Asia, there are multiple sources of ancestry. For example, Palestinians, Druze, and Bedouins have contributions from the Middle East, Europe, and South/Central Asia. Burusho, Pathan, and Sindhi have an East Asian contribution. Hazara and Uygur share a similar profile of combined South/Central Asian, East Asian, and European ancestry. In East Asia, only the Yakuts share ancestry with both Europe and America, although these contributions are small. Although much of sub-Saharan Africa, Europe, and East Asia appears to be homogeneous in Fig. 1A, finer substructures can be detected when individual regions are analyzed separately. For example, we identified two components that separate the 16 East Asian populations and correspond to a north-south genetic gradient (fig. S2A). Han Chinese can be divided into a southern and a northern group. A similar analysis for South/Central Asia is shown in fig. S2B.

## Post Colonial Americas

* [Dai, C. L., Vazifeh, M. M., Yeang, C. H., Tachet, R., Wells, R. S., Vilar, M. G., ... & Martin, A. R. (2020). Population histories of the United States revealed through fine-scale migration and haplotype analysis. The American Journal of Human Genetics, 106(3), 371-388.](https://doi.org/10.1016/j.ajhg.2020.02.002)
  * > Previous genetic studies of the US population have sought to infer genetic ancestry and population history primarily in European Americans, African Americans, and Hispanics/Latinos.7, 8, 9,19,20 European American ancestry is characterized by substantial mixing between different ancestral European populations and, to a lesser extent, admixture with non-European populations.6,8,9 Isolation among certain European population, such as Ashkenazi Jewish, French Canadian, and Finnish populations, have also resulted in founder effects.21, 22, 23, 24 The mixing of European settlers with Native Americans has contributed to large variations in the admixture proportions of different Hispanic/Latino populations.1,4,9 Among Hispanics/Latinos, Mexicans and Central Americans have more Native American ancestry; Puerto Ricans and Dominicans have more African ancestry; and Cubans have more European ancestry.1,4 In African Americans, proportions of African, European, and Native American ancestry vary across the country and reflect migration routes, slavery, and patterns of segregation between states.2,3,7,9,25 Although much effort has been made to understand the genetic diversity in the US, fine-scale patterns of demography, migration, isolation, and founder effects are still being uncovered with the growing scale of genetic data, particularly for Latin American and African descendants with complex admixture histories.26,27 At the same time, there has been little research on the population structure of individuals with East Asian, South Asian, and Middle Eastern ancestry in the US.
  * > Here, we leverage these advantages over existing data to identify patterns of genetic ancestry by studying pairwise sharing among the project participants. We combine these comparative patterns with ancestral birth origin records and geographic information to uncover recent demographic and migration trends. By comprehensively analyzing these data to learn about recent migration events, we gain deeper insights into ancestral origins than in many existing studies, especially into Latin America. We also provide early insights into Asian Americans often ignored in genetic studies of the US, including South Asians, East Asians, and Middle Easterners. We also identify detailed patterns among European and African American populations, recapitulating some similar trends reported previously. Taken together, we use accessible individual-level genetic and birth record data to provide insights into the ancestral origins and complex population histories in the US.
  * > We assigned continental ancestry to each Genographic sample by using a random forest classifier. Using the PCs and known super population assignments (AFR, African; EUR, European; EAS, East Asian; AMR, admixed American; and SAS, South Asian) from the 1000 Genomes Project samples as training data, we applied the classifier to assign ancestry to each Genographic sample at 90% probability. We considered unassigned ancestries as “other” (OTH).
  * > We further examined the subcontinental structure of Genographic Project individuals who were classified as European ancestry individuals with data from the Population Reference Sample (POPRES).33 Similar to the analyses with the 1000 Genomes Project data, we performed dimensionality reduction with PCA and UMAP, keeping the same parameter values. Coloring of POPRES data was grouped by continental regions: Southeast Europeans = Croatia, Yugoslavia, Bosnia-Herzegovina, Serbia, Romania, Hungary, Albania, Macedonia; Central Europe = Switzerland, France, Germany, Germany, Swiss-Italian, Belgium, Swiss-French, Netherlands, Swiss-German; British Isle = Scotland, Ireland, United Kingdom; South Europe = Italy, Cyprus, Turkey, Greece; Iberian = Portugal, Spain; Eastern Europe = Austria, Czech Republic, Poland, Russia; Scandinavia = Sweden, Norway.
  * > We therefore sought to infer population structure across continuous space with the estimating effective migration surfaces (EEMS) method.41 EEMS statistically measures effective migration rates by overlaying a dense grid of evenly spaced demes and calculating genetic differentiation (i.e., resistance distance) between neighboring demes. Higher rates of migration are inferred in locations where genetic similarity is high (colored in blue in Figure 3) while lower rates of migration are inferred in locations where genetic similarity is low (colored in dark orange). Areas with low effective migration are also referred to in EEMS as “barriers,” which can be intuitively interpreted as regions in which neighboring populations are more genetically dissimilar than expected. In more homogeneous populations, these barriers tend to indicate isolation by distance, while in more heterogenous populations, they may reflect differences in population structure. We applied EEMS to genetically classified Europeans, African Americans, and Hispanic/Latinos across the contiguous 48 states. We excluded East Asians and South Asians due to low sample density.
  * > The patterns of genetic similarity for Europeans capture subcontinental structure. With the exception of the states in the Midwest and along the Atlantic coast, elevated levels of genetic similarity and relative migration rates are observed across most of the country. We find low effective migration rates surrounding Minnesota and Michigan, likely due to the genetic dissimilarity of Finnish and Scandinavian ancestry that is abundant in the region (Figures 3C, S8, and S9).8 We also find reduced migration rates across Ohio, West Virginia, and Virginia, suggesting the existence of genetic differentiation along the Appalachian Mountains. Many of the major cities, such as Washington, DC, Philadelphia, and Miami, also exhibit low genetic similarity, perhaps due to greater genetic diversity and admixture within cities.
  * > Genetic and geographic differences are greatest among Hispanic/Latino haplotype clusters. We identified a total of five Hispanic-related clusters (Figure 4). The largest of these cluster (n = 810; orange in Figure 4A) is strongly associated with south Florida (OR = 10.4; p = 2.5 × 10−25; Table S5) but is also found in California and Texas (OR ≥ 2; p < 0.05; Table S5). No single ancestral birthplace characterizes this cluster, as the US, Mexico, and Cuba each make up more than 10% of the birth origin labels (Figure 4B). Proportions of European ancestry tracts inferred with RFMix44 are higher in this cluster (mean = 72.7%, SD = 20.4%; Figure 4C) than in the other Hispanic/Latino clusters (mean = 48.0%–67.4%; Figure 4C). Puerto Ricans characterize a substantial proportion of another Hispanic/Latino cluster associated with Florida (OR > 4) as well as New York City (OR > 5). Unlike the other Hispanic clusters, the Puerto Rican cluster shares the same branch on the FST tree as the African American clusters (Figure S11), likely due to relatively high proportions of African ancestry (mean = 11.2%, SD = 9.0%) among Puerto Ricans. Median lengths of sROH and cumulative IBD in Puerto Ricans are also the highest among the Hispanic clusters (8.01 Mb and 26.23 cM, respectively; Table 1). Consistent with other studies,45,52 we found evidence of a strong bottleneck in Puerto Ricans approximately 9–14 generations ago (Figure S12), coinciding with the colonization of America and likely explaining the elevated levels of IBD and sROH.
    * Of course. People calssified into the Latino category only share language and possibly a portion of admixed European ancestry.
  * > Historical immigration of Europeans into the US occurred in successive waves, with northern and western Europeans making up one wave from the 1840s to 1880s and another wave comprised of southern and eastern Europeans occurring from the 1880s to 1910s.53 Consistent with this immigration pattern, haplotype clusters with ancestries from northwest and central Europe have higher proportions of US ancestral birth origins than haplotype clusters from southern and eastern Europe, suggesting earlier immigration (Figures 5A and 5B). The two clusters with the highest proportion (>75%) of US ancestral birth origin (“Northwest Europe 1” and “Northwest Europe 2”) have ∼4.5% of UK ancestral origins. The central European cluster and the Irish cluster both have 66.1% and 68.5% of US ancestral origins, respectively (Figure 5B). In contrast, the US makes up only 62.2% and 34.5% of ancestral birth origin for the clusters of southern Europeans and eastern Europeans, respectively.
  * > Unlike the larger European clusters, the smaller European clusters reflect the structure of recent immigrants and genetically isolated populations, recapitulating earlier findings.8 The geographic distributions of these subpopulations are more concentrated, and their ancestral birth origin proportions are overrepresented by specific countries and ethnicities (Figures 6A and 6B). Specifically, Finns and Scandinavians are abundant in the Upper Midwest and Washington; French Canadians are found in the Northeast; Acadians are present in the Northeast and Louisiana; and Italians, Greeks, and Jews are mostly located in the metropolitan area of New York City (Figure 6A).
  * > We inferred two haplotype clusters of African Americans separated along a north-south cline, recapitulating the EEMS migration barrier inference. One cluster is primarily distributed among the northern and western states (“African Americans North”), while the other is distributed among the states southeast of the Appalachian Mountains (“African Americans South”) (Figure S13). The proportion of US birth origin is higher in the northern cluster than the southern cluster, providing further evidence of isolation-by-distance among African Americans in the north.7 These two clusters share similar sROH lengths but differ in admixture proportions and median IBD sharing (Table 1), pointing to a cluster with consistent African American ancestors and a cluster with more admixed ancestors. Median cumulative IBD sharing is higher among African Americans in the south (median cumulative IBD = 19.6 cM, median sROH = 3.3 Mb) than in the north (median = 15.9 cM; Table 1), resulting in different patterns of effective population size over antecedent generations (Figure S12),45,46 while the average proportion of African ancestry is higher in the northern cluster than the southern cluster.
  * > Four of the clusters reflect recent immigrants from Asia (Figure S14), which grew rapidly in the mid-20th century after the elimination of national origin quotas.55 The recency of immigration among these clusters is supported by the observation that fewer than 30% of grandparents were born in the US. Geographically, individuals in these clusters primarily reside in major cities. East Asians predominantly inhabit the metropolitan areas of the west and northeast (OR > 2), Southeast Asians are enriched in the west (OR > 2.5), and South Asians are strongly associated with the northeast (OR > 2.5).
* [Han, E., Carbonetto, P., Curtis, R. E., Wang, Y., Granka, J. M., Byrnes, J., ... & Ball, C. A. (2017). Clustering of 770,000 genomes reveals post-colonial population structure of North America. Nature communications, 8(1), 1-12.](https://doi.org/10.1038/ncomms14238)
  * > Despite strides in characterizing human history from genetic polymorphism data, progress in identifying genetic signatures of recent demography has been limited. Here we identify very recent fine-scale population structure in North America from a network of over 500 million genetic (identity-by-descent, IBD) connections among 770,000 genotyped individuals of US origin. We detect densely connected clusters within the network and annotate these clusters using a database of over 20 million genealogical records. Recent population patterns captured by IBD clustering include immigrants such as Scandinavians and French Canadians; groups with continental admixture such as Puerto Ricans; settlers such as the Amish and Appalachians who experienced geographic or cultural isolation; and broad historical trends, including reduced north-south gene flow. Our results yield a detailed historical portrait of North America after European settlement and support substantial genetic heterogeneity in the United States beyond that uncovered by previous studies.
  * > In this study, we analyse genome-wide genotype data from over 777, 000 primarily US-born individuals. Among all pairs of individuals, we identify genetic connections defined by sharing a recent common ancestor; when these connections are aggregated into a network, our computational methods reveal densely connected clusters, in which the members of each cluster are subtly more related to each other. Using a unique collection of 20 million user-generated genealogical records, we annotate these densely connected clusters to identify the putative historical origins of such population substructure, and to infer temporal and geographic patterns of migration and settlement. With much greater granularity than previously possible, our analyses demonstrate the impact of subtle, complex demographic forces in shaping the patterns of genetic variation among contemporary North Americans.
  * > Our first indication that demography could be inferred from genomic sharing among present-day Americans was the relationship we observed between US geography and the projection of state-level IBD summary statistics onto their first two principal components (PCs); PC 1 is correlated with north-south geography, and PC 2 is correlated with east-west (Fig. 1; Supplementary Data 1). Following this initial observation, we turned to using IBD to discover previously unidentified population structure. Similar in some respects to Gusev et al.16, and based on principles developed in machine learning and statistical physics, we used a model-free approach to learn structure in a network22,23,24. We hypothesized that structural features of this network could be related to population demography16,25,26,27, analogous to the common use of PC analysis (PCA) to infer structure from genetic polymorphism data without specifying a demographic process11,14. To define the IBD network as a graph (Fig. 2a), each individual is represented as a vertex, and IBD between each pair of individuals is represented as an edge. We applied a weight function to each edge, setting the edge weight >0 only if the estimated total IBD was >12 cM (Supplementary Fig. 2). This choice allowed us to focus on IBD corresponding to more recent demography while reducing spuriously identified IBD28. On the basis of this choice, 769,444 (99.3%) of the vertices (individuals) formed a completely connected network; the remaining 0.7% of samples could correspond to populations poorly represented in our sample, and they were not included in our subsequent analyses.
  * > Given the IBD network clustering identified by hierarchical clustering and spectral analysis, we used a combination of admixture estimates and genealogical data to annotate these clusters and connect them to recent, fine-scale demography in North America (Fig. 2d,e). First, we estimated admixture proportions in 20 global populations from the genotypes using ADMIXTURE12 (Supplementary Figs 7–11; Supplementary Table 2). Characterization of the clusters with respect to this estimated worldwide population structure is summarized in Table 1. Second, user-generated pedigree data linked to a large proportion of individuals in the data set provided birth dates and locations the ancestors of cluster members (Supplementary Figs 12–17). Such data allowed us to understand the ancestral history of individuals assigned to identified clusters, in particular suggesting birth locations that were over-represented at particular time periods in the pedigrees of individuals assigned to each cluster (Fig. 3; Supplementary Figs 18–21). The cluster names in Table 1 and Fig. 3 were drawn from statistical summaries of these admixture and genealogical data.
  * > Taken together with the IBD network clustering results (Table 1), the visualizations of the genealogical data in North America (Fig. 3) highlight broad-scale demographic trends, as well as patterns specific to individual populations. While no two identified clusters have identical demographic histories, for convenience of discussion we divide the clusters into four broad categories, and present examples of each. The first grouping, which we label as intact immigrant clusters, are likely driven by population structure present before immigration that may have been maintained post immigration. These clusters often feature over-representation of a particular global admixture proportion, localized ancestral birth locations at both the source and destination of immigration, and genetic differentiation (FST) comparable to those of the source populations. We label the second grouping as continental admixed groups, the majority of which represent Hispanic/Latino populations. While these clusters all display a characteristic genetic mixture from two or more continents, admixture proportions alone do not distinguish them. Instead, we find that ancestral birth locations, primarily from outside current US boundaries, reveal the groups’ more recent origins. The vast majority of our samples are contained in the third set of clusters, which we label as assimilated immigrant groups. Although these clusters typically feature almost exclusively mixed European ancestry and very low genetic differentiation between one another, they can be distinguished primarily by geographic localization of their ancestral birth locations within the United States. Finally, the fourth set of clusters we label as post-migration isolated groups; these groups have historically resided in small or geographically isolated communities within the United States, and are distinguished by stable subsets within the IBD network, suggesting that reduced gene flow with neighbouring groups may have contributed to the identified structure.
* [Bryc, K., Durand, E. Y., Macpherson, J. M., Reich, D., & Mountain, J. L. (2015). The genetic ancestry of african americans, latinos, and european Americans across the United States. The American Journal of Human Genetics, 96(1), 37-53.](https://doi.org/10.1016/j.ajhg.2014.11.010)
  * > In the present work, we do not include individuals who self-report as having multiple identities, because this represents only a small fraction of individuals in our data set. Low rates of reporting as multiracial or multiethnic is in line with previous studies; an analysis of the 2000 US Census shows that 95 percent of blacks and 97 percent of whites acknowledge only a single identity.45 Future studies including multiracial individuals might further illuminate patterns of genetic ancestry and the complex relationship with self-identity.
  * > Self-reported survey data was used to generate cohorts of African Americans, Latinos, and European Americans. Out of 35,524 self-reported “European” individuals, 35,279 selected “white” on the ethnicity survey, yielding a per-survey error estimate of 0.2%. Out of 1,560 self-reported “Latino” individuals, 1,540 selected “Hispanic,” giving a per-survey error estimate of 0.7%. Lastly, out of 1,327 self-reported “African American” individuals, 1,287 selected “black,” resulting in a per-survey error rate estimate of 1.1%. For more details on our cross-survey validation, see Subjects and Methods.
  * > Genome-wide ancestry estimates of African Americans show average proportions of 73.2% African, 24.0% European, and 0.8% Native American ancestry (Table 1). [...] More than 5% of African Americans are estimated to carry at least 2% Native American ancestry genome-wide (Figures S1 and 1D). African Americans in the West and Southwest on average carry higher levels of Native American ancestry, a trend that is largely driven by individuals with less than 2% Native American ancestry (Figure 1B). With a lower threshold of 1% Native American ancestry, we estimate that about 22% of African Americans carry some Native American ancestry (Figure S2).
  * > We used the lengths of segments of European, African, and Native American ancestry to estimate a best-fit model of admixture history among these populations for African Americans (Figure S3). We estimate that initial admixture between Europeans and Native Americans occurred 12 generations ago, followed by subsequent African admixture 6 generations ago, consistent with other admixture inference methods dating African American admixture. A sex bias in African American ancestry, with greater male European and female African contributions, has been suggested through mtDNA, Y chromosome, and autosomal studies.6 On average, across African Americans, we estimate that the X chromosome has a 5% increase in African ancestry and 18% reduction in European ancestry relative to genome-wide estimates (see Table 1). Through comparison of estimates of X chromosome and genome-wide African and European ancestry proportions, we estimate that approximately 5% of ancestors of African Americans were European females and 19% were European males (Table S4).
  * > Latinos encompass nearly all possible combinations of African, Native American, and European ancestries, with the exception of individuals who have a mix of African and Native American ancestry without European ancestry (see Figures S4A and S1). On average, we estimate that Latinos in the US carry 18.0% Native American ancestry, 65.1% European ancestry, and 6.2% African ancestry. We find the highest levels of estimated Native American ancestry in self-reported Latinos from states in the Southwest, especially those bordering Mexico (Figure 2C). We find the highest mean levels of African ancestry in Latinos living in or born in states in the South, especially Louisiana, the Midwest, and Atlantic (Figure 2A). Further stratification of individuals by their self-reported population affiliation (e.g., “Mexican,” “Puerto Rican,” or “Dominican”) reveals a diversity in genetic ancestry, consistent with previous work studying these populations (see Figure S5 and Table S5).10,  20,  24,  25,  68,  69 We find that Latinos who, besides reporting as “Hispanic,” also self-report as Mexican or Central American, carry more Native American ancestry than Latinos overall; those also who self-report as black, Puerto Rican, or Dominican have higher levels of African ancestry; and those who additionally self-report as white, Cuban, or South American have on average higher levels of European ancestry.
  * > We find that many self-reported European Americans, predominantly those living west of the Mississippi River, carry Native American ancestry (Figure 3B). We estimate that European Americans who carry at least 2% Native American ancestry are found most frequently in Louisiana, North Dakota, and other states in the West. Using a less stringent threshold of 1%, our estimates suggest that as many as 8% of individuals from Louisiana and upward of 3% of individuals from some states in the West and Southwest carry Native American ancestry (Figure S7).<br/><br/>Admixture date estimates for Latino admixture suggest that Native American and European mixture occurred first, about 11 generations ago, followed by African admixture 7 generations ago. Consistent with previous studies that show a sex bias in admixture in Latino populations,12,  13,  14,  15,  16,  17,  18 we estimate 13% less European ancestry on the X chromosome than genome-wide (Table 1), showing proportionally greater European ancestry contributions from males. We inferred elevated African and Native American ancestry on the X chromosome, corresponding to higher female ancestry contributions from both Africans and Native Americans. Lastly, Latinos show higher proportions of inferred Iberian ancestry than both European Americans and African Americans (Figure S6).
  * > We find that many self-reported European Americans, predominantly those living west of the Mississippi River, carry Native American ancestry (Figure 3B). We estimate that European Americans who carry at least 2% Native American ancestry are found most frequently in Louisiana, North Dakota, and other states in the West. Using a less stringent threshold of 1%, our estimates suggest that as many as 8% of individuals from Louisiana and upward of 3% of individuals from some states in the West and Southwest carry Native American ancestry (Figure S7).
  * > [...] We estimate that a substantial fraction, at least 1.4%, of self-reported European Americans in the US carry at least 2% African ancestry. Using a less conservative threshold, approximately 3.5% of European Americans have 1% or more African ancestry (Figure S8). Individuals with African ancestry are found at much higher frequencies in states in the South than in other parts of the US: about 5% of self-reported European Americans living in South Carolina and Louisiana have at least 2% African ancestry. Lowering the threshold to at least 1% African ancestry (potentially arising from one African genealogical ancestor within the last 11 generations), European Americans with African ancestry comprise as much as 12% of European Americans from Louisiana and South Carolina and about 1 in 10 individuals in other parts of the South (Figure S8).
  * > Most individuals who have less than 28% African ancestry identify as European American, rather than as African American (Figures 4 and 5A ).
  * > Fitting a model of European and Native American admixture followed later by African admixture, we find the best fit with initial Native American and European admixture about 12 generations ago and subsequent African gene flow about 4 generations ago.
  * > We find that levels of Native American and African ancestry in 23andMe customers in each state are significantly correlated with the proportion of African Americans and Latinos in each state (Figures S11–S13). For example, levels of African ancestry in European Americans and Latinos in a state are highly correlated with proportion of African Americans in each state (both p values < 10−4). Levels of Native American ancestry in European Americans and Latinos in a state are highly correlated with proportion of Latinos in each state (p values < 10−6 and < 10−2, respectively).
  * > The frequency of Native American mtDNA haplogroups in European Americans and African Americans correlate with our estimates of genome-wide ancestry in European Americans and African Americans and are found in appreciable fractions of individuals who are estimated to carry Native American ancestry. The frequencies of haplogroups are shown in Table S8. These haplogroups are virtually absent in individuals with four grandparents from a European country (21 individuals out of 15,651). Furthermore, the majority of these Native American haplogroups in Europeans are found in individuals from Spain. Though it is possible these represent non-Native American haplogroups, prior literature and studies of genetic, archaeological, and paleontological evidence suggest that these haplogroups have Native American origins and is evidence of gene flow from the Americas to Spain. Excluding Spain, Native-American-specific haplogroups are detected in fewer than 0.05% of individuals with four grandparents from Europe and can be explained by survey errors in reporting all four grandparents’ birth places.
  * > Contrary to expectations under a social one-drop rule, or “Rule of Hypodescent,” which would mandate that individuals who knowingly carry African ancestry identify as African American, the probability of self-reporting as African American given a proportion of African ancestry follows a logistic probability curve (Figure S9A, Table S6), suggesting that individuals identify roughly with the majority of their genetic ancestry (Figures 4 and 5A). Individuals with more than 5% Native American ancestry are most likely to self-identify as Latino (Figures S9C and 5B), suggesting differences in sociological or historical factors associated with identifying with these groups. The transitions between Latino, African American, and European American self-reported identity by proportions of African and Native American ancestry illustrate both the complexity of how one self identifies as well as the overlapping ancestry profiles among groups (Figure 5B).

## Genetics of the Americas
* [A Genetic Chronicle of the First Peoples in the Americas](https://www.sapiens.org/archaeology/ancient-dna-native-americans/)
* [Y Not a Pacific Migration? Misunderstandings of Genetics in Service to Pseudoscience](http://digitaleditions.sheridan.com/publication/?m=16146&i=634462&view=articleBrowser&article_id=3531892&ver=html5)
  * This kind of research is also probably why this persists:
    * [Meggers, B. J. (1975). The transpacific origin of Mesoamerican civilization: A preliminary review of the evidence and its theoretical implications. American Anthropologist, 77(1), 1-27.](https://anthrosource.onlinelibrary.wiley.com/doi/pdf/10.1525/aa.1975.77.1.02a00020)
* [Lindo, J., De La Rosa, R., dos Santos, A. L. C., Sans, M., DeGiorgio, M., & Figueiro, G. (2021). The Genomic Prehistory of the Indigenous People of Uruguay. bioRxiv.](https://doi.org/10.1093/pnasnexus/pgac047)
* [Scott, G. R., O’Rourke, D. H., Raff, J. A., Tackney, J. C., Hlusko, L. J., Elias, S. A., ... & Hoffecker, J. F. (2021). Peopling the Americas: not “out of Japan”. PaleoAmerica, 7(4), 309-332.](https://doi.org/10.1080/20555563.2021.1940440)
  * This is an important point about population genetic drift and new genetics replacing older genetics:
    * > It should come as no surprise that ancient Jomon people exhibited genetic variation different than that seen in contemporary populations of the region. This has been observed not only throughout South and East Asia (Yang et al. 2020), but in South America (Llamas, Harkins, and Fehren-Schmitz 2017) and Europe (Fu et al. 2016). Contemporary, and even archaeologically recent, populations in any region may be poor indicators of patterns of genetic variation in past populations of the deeper past of the same regions.
  * Sumarizes new dental morphology based research that shows a relatively deep distinction between East-Asian (EA) groups dental morphologies and North American (NA) dental morphologies. Also summarizes current genetic research which shows that NA and EA groups do seem to have distant shared ancestry. All of the evidence stands against any hypothesis of Japanese populations being ancestral to NA.
    * > We have addressed the hypothesis that the First Peoples of the Western Hemisphere are derived from a population living in the Northeast Asian maritime region, including northern Japan, at the end of the LGM or about 16,000 cal yr BP. The hypothesis is based on striking similarities in a specific category of stone artifacts (stemmed points) made by (1) the people who occupied Japan and adjoining areas of mainland Asia 16,000–15,000 cal yr BP (known archaeologically as “Incipient Jomon”) and (2) early Native American people living in the western regions of mid-latitude North America (“Western Stemmed Point Tradition”).<br/><br/>We tested the hypothesis with the data and methods of biological anthropology, using a large body of data on teeth, which are proxies for neutral genomic data, and the growing body of pertinent genetic data itself (paleogenomics). We found that (1) quantitative biodistance measures using large samples of teeth and based on 21 crown and root traits show that the Jomon (and Ainu) cluster with Southeast Asian and Pacific groups; they are most divergent from Native Americans; (2) an analysis of individual ancestry through rASUDAS showed that roughly 40% of the Jomon sample were assigned to Austral-Melanesians and a significant percentage (24%) were assigned to American Arctic, but that only 7% were assigned to non-Arctic Native American (the Ainu sample exhibited a similar pattern); and (3) none of the genetic data (mtDNA, Y-DNA, or nuclear genome) suggests a direct relationship between the Jomon and Native Americans.<br/><br/>On the other hand, as described above, we found no evidence of links between the Holocene Jomon population and Native Americans in the dental and paleogenomic data (with the exception of the connection between the Ainu and Arctic Americans, discussed further below). If Native Americans were derived from the people who made the Incipient Jomon artifacts, only more-or-less complete replacement of the population of Japan between 16,000 and 9000 cal yr BP could account for this pattern. There is no indication of a wholesale replacement of the population of Japan during 16,000–9000 cal yr BP. Instead, multiple lines of evidence suggest both biological and cultural continuity in the Jomon population (e.g., Adachi, Shinoda, and Izuho 2015; Nakazawa 2017), which, as discussed earlier, appears to have diverged from its own Asian source population before the LGM as a lineage separate from that of the East Asian lineage of the Native American founder group (Figure 5; Gakuhari et al. 2020, figure 4; Ning et al. 2020; Yang et al. 2020, figure 2A).10<br/><br/>We conclude that the people who made stemmed points in the Northeast Asian maritime area during the millennia following the LGM are an unlikely source population for the peoples who spread through mid-latitude North America and South America after 16,000–15,000 cal yr BP. In our view, the Incipient Jomon population represents one of the least likely sources for Native American peoples of any of the non-African populations considered in our analyses.<br/><br/>The apparent link between the Jomon and at least some portion of the American Arctic population, warrants a comment. The pattern is apparent in the dental anthropological but equivocal in the human paleogenomics data and apparently indicates some gene flow between the two populations after the settlement of mid-latitude North America and South America had taken place (15,000–13,000 cal yr BP). We suggest that it probably represents one of several population movements/contacts between Beringia (or the Bering Strait region) and Northeast Asia that took place after the divergence of Native American Arctic and non-Arctic populations (e.g., Adachi et al. 2009; Sicoli and Holton 2014; Sikora et al. 2019; Tamm et al. 2007).
  * Intriguing point to add to microblade discussion:
    * > Finally, we note that there are archaeological remains in Hokkaido that may be related to the peopling of the Americas, but do not represent the people who made the artifacts assigned to Incipient Jomon. There are artifact assemblages in Hokkaido dating as early as ∼25,700–22,500 cal yr BP containing microblade cores (including Yubetsu type) that are similar to those found in the interior of Siberia and later in Beringia (Buvit et al. 2016; Gomez Coutouly 2018). Although the lineage associated with these assemblages and its relationship to Indigenous Americans is unknown, the presence of similar artifacts in eastern Beringia suggests a possible, if not likely, role in the origin of the latter (e.g., Graf and Buvit 2017; Potter et al. 2017). The early microblade assemblages in Hokkaido appear unrelated to Incipient Jomon, but rather reflect an LGM interior adaptation at a time when Hokkaido was connected to Sakhalin and the Asian mainland (but not Honshu) due to lower sea level (Buvit et al. 2016).
* [Popović, D., Molak, M., Ziołkowski, M., Vranich, A., Sobczyk, M., Vidaurre, D. U., ... & Baca, M. (2021). Ancient genomes reveal long range influence of the site and culture of Tiwanaku. bioRxiv.](https://doi.org/10.1101/2021.01.22.427554)
  * > Our results show that the population of the Titicaca basin was rather genetically homogeneous from at least 300 CE until the arrival of Europeans. This is well illustrated for the Lukurmata site, for which we have the widest time transect consisting of four individuals from between ca. 300 and 1500 CE. These individuals exhibit genetic similarity indicating that, over at least 12 centuries, no major genetic turnovers took place, despite the occurrence of substantial cultural and political changes. Earlier genetic studies of present-day South American populations, using either uniparental markers (28, 29) or genomic data (27), suggested that both precontact and present-day Central Andean populations were genetically homogeneous and characterized by high levels of gene flow between them. In contrast, we find little overlap between precontact individuals from central/southern Peru and those from the Lake Titicaca basin (Figs. 2 and 3). Our results corroborate the findings in (19) that also suggested homogeneity within distinct regions of the Central Andes during the past ca. 2000 years. However, contact between various parts of the Andes was not entirely absent, as shown by the individual from the Coropuna region who had affinity to northern Peru and an individual from the Totocachi site who showed genetic affinity to the populations from the south Peruvian Andes or northern Chile.
  * > Given the genetic homogeneity of the Bolivian Altiplano populations, the group of individuals from the Tiwanaku ritual core stands out because of its diversity. We found that several individuals show a close affinity to the Titicaca basin population, whereas others show distinct or admixed ancestry. At the peak of its influence, the Tiwanaku state reached outside the Titicaca basin to the fertile valleys of Moquegua (southern Peru), Cochabamba (Bolivia), and Azapa (northern Chile) (Fig. 1C) (3, 4). A pronounced presence of foreign ceramic clusters found at the Tiwanaku site supports a scenario in which ethnic groups settled in particular compounds around the complex (1, 4). Previous studies based on strontium isotopes (17) and cranial modifications (18) suggested the presence of individuals from neighboring areas, probably from Tiwanaku-affiliated sites at the peripheries of the polity. The evidence for contacts with populations from farther away, such as Amazonia, has been much more limited and based solely on isolated artifacts like a jaguar canine necklace (4), iconographic representations of tropical lowland fauna (30), tropical botanical remains (31), and the consumption of lowland hallucinogenic substances (32). The on-site presence of an individual with entirely Amazonian ancestry (TW056), although it is only one individual in the rather special circumstance of sacrificial offering, demonstrates that contact between the Tiwanaku site and Amazonia was not limited to the exchange of goods but included the physical movement of people. Contacts with remote regions were further confirmed by the finding of two individuals (TW059 and TW063) who showed a combination of Lake Titicaca basin and Amazonia and/or Gran Chaco ancestry. On the basis of the genomic data, we could not determine whether these individuals were already admixed incomers to Lake Titicaca basin or descendants of incomers and people with local ancestry. The latter scenario, in which the admixture between incomers and Titicaca individuals occurred locally, seems more probable, considering that no signs of foreign genetic ancestry have hitherto been found in Titicaca basin individuals outside the Tiwanaku ritual core.
* [Fernandes, D. M., Sirak, K. A., Ringbauer, H., Sedig, J., Rohland, N., Cheronet, O., ... & Reich, D. (2021). A genetic history of the pre-contact Caribbean. Nature, 590(7844), 103-110.](https://doi.org/10.1101/2020.06.01.126730)
  * [van Hoose, Natalie. (2020) Ancient DNA retells story of Caribbean’s first people. Florida Museum. December, 23.](https://www.floridamuseum.ufl.edu/science/ancient-dna-retells-story-of-caribbeans-first-people/)
* [Guevara EK, Palo JU, Översti S, King JL, Seidel M, Stoljarova M, et al. (2020) Genetic assessment reveals no population substructure and divergent regional and sex-specific histories in the Chachapoyas from northeast Peru. PLoS ONE 15(12): e0244497. https://doi.org/10.1371/journal.pone.0244497](https://doi.org/10.1371/journal.pone.0244497)
* [Urban, M., & Barbieri, C. (2020). North and South in the ancient Central Andes: Contextualizing the archaeological record with evidence from linguistics and molecular anthropology. Journal of Anthropological Archaeology, 60, 101233.](https://doi.org/10.1016/j.jaa.2020.101233) &mdash; [PDF](https://pure.mpg.de/rest/items/item_3257439/component/file_3257446/content)
* [Nakatsuka, N., Lazaridis, I., Barbieri, C., Skoglund, P., Rohland, N., Mallick, S., ... & Michel, M. (2020). A paleogenomic reconstruction of the deep population history of the andes. Cell.](https://doi.org/10.1016/j.cell.2020.04.015)
  * This has some interesting genetics research that, I think, shows how groups in central Mexico intermingled with their trade partners over the last 10Kya. It shows how the genetic groups which span Mexico into SW USA include genetics from Canada through Chile.
  * > We tested for gene flows between the Central Andes and other regions. We observe excess allele sharing between the Northwest Amazon and North Peru as shown by significantly more affinity of Indigenous peoples from the western Amazon (SanMartin, Ticuna, Wayku, and Surui) to NorthPeruCoast, NorthPeruHighlands, or CentralPeruCoast than to SouthPeruCoast or the earlier Lauricocha_5800BP individual (Table S7C). We used qpAdm to model Peru_Amazon_SanMartin_Modern as a mixture of 29% ancient North Highlands-related ancestry (related to Peru_NorthHighlands_LaGalgada_4100BP) and 71% Amazonian ancestry (related to Brazil_Amazon_Karitiana_Modern) (±11%, quoting one standard error), suggesting that at least some modern Northwest Amazonian groups harbor Andean-related ancestry. When we modeled Peruvian groups as a mixture of Peru_WestAmazon_SanMartin_Modern and Peru_NorthHighlands_LaGalgada_4100BP, we also identified coastal individuals with significantly non-zero Amazonian-related ancestry (e.g., 39% ± 14% in CentralPeruCoast) (Figure S6B; Table S7C), suggesting bi-directional gene flow with Amazonian-related ancestry affecting the Peruvian North and Central Coast more than the Highlands (Barbieri et al., 2014, Di Corcia et al., 2017, Gnecchi-Ruscone et al., 2019, Harris et al., 2018, Reich et al., 2012, Rothhammer et al., 2017). We detect no gene flow between the North Coast and Amazonian groups to the North, because we found no significant affinity of any modern Amazonian groups from Ecuador or Colombia to NorthPeruCoast relative to Lauricocha_5800BP.
  * > The stronger signal of Amazonian-related ancestry in the North and Central Coast relative to the North Highlands suggests that gene flow could have occurred over the low mountain ranges of North Peru (Huancabamba deflection), rather than across the high-altitude mountain ranges that dominate the Andes further south and potentially are a larger barrier to gene flow, or if Highlands groups maintained high social barriers to admixture from the Amazonian groups. We used the software DATES (Narasimhan et al., 2019), which models allele covariance over genetic distance to measure admixture dates, and found that the admixtures occurred ∼1,478 ± 252 years ago in NorthPeruCoast and ∼1,153 ± 90 years ago in CentralPeruCoast (Table S7C), consistent with the hypothesis of a southward migration pattern.
  * > We do not observe tropical lowlands-derived gene-flow into the Titicaca Basin or Northern Chile as reported in studies based on mitochondrial DNA (Rothhammer and Dillehay, 2009, Rothhammer et al., 2017). There is strong archaeological evidence for the exchange of food crops and other goods between the lowlands east of the Andes and the Chilean North coast (Rothhammer and Dillehay, 2009, Santoro, 1980), but it is possible this did not lead to gene flow detectable in the NorthChile individuals tested here, which post-date the postulated exchange by 2,000–3,000 years (Rothhammer and Dillehay, 2009). Because we do not have any DNA from ancient Amazonians, we cannot exclude gene flow from past groups carrying so-far undetected linages.
  * > We detect significantly more allele sharing of SouthPeruHighlands, SouthPeruCoast, CentralPeruCoast, and Titicaca Basin groups to Argentina_LagunaChica_1600BP relative to Argentina_LagunaChica_6800BP. This likely reflects gene flow between the Pampas and the Central Andes, consistent with previous claims (Gómez-Carballa et al., 2018, Muzzio et al., 2018). Using qpAdm, we fit Argentina_LagunaChica_1600BP as a mixture of 80% ± 12% ancestry related to Argentina_LagunaChica_6800BP and 20% ± 12% ancestry related to a representative Andes group giving the lowest standard error (CentralPeruCoast). We also fit CentralPeruCoast as 77% ± 17% related to Peru_Cuncaicha_4200BP and 23% ± 17% related to Argentina_LagunaChica_1600BP (Figure S6C; Table S7D). Pottery and metal objects of South Andean origin are found in the Araucania region in the western Pampas dating to at least ∼1,000 BP (Berón, 2007), and skeletons from Chenque 1 in the Pampas have been suggested to have South Andean isotopic signatures (Berón et al., 2013). Taken together, there is thus compelling evidence for human movements as well as cultural interactions between these regions at least ∼1,600 BP.
  * > We used a semi-automated procedure to build an admixture graph to model representative ancient Central and South-Central Andeans (Patterson et al., 2012) (Figures 4 and 5). Our best fit recapitulates key findings from this study. The earliest Peruvians do not share genetic drift with the later groups in our dataset, except for local continuity at the Lauricocha site. The differentiation between North and South Peru Highlands correlating to later structure is only evident by 5,800–4,100 BP. Post ∼2,000 BP South Peru Highlands individuals are modeled as a mixture of earlier South Highlands and North Highlands-related ancestry. Deep ancestry is inferred in Coast individuals, while North Chile individuals can only be fit with ancestry from a different basal lineage. Post ∼2,000 BP individuals from the socio-political center of Tiwanaku exhibit mixtures of ancestry related to contemporary people from the Central Peru Coast and South Peru Highlands. An important direction for future work is to obtain ancient DNA from the Coast prior to ∼1,600 BP, as well as equally rich ancient DNA data from regions to the north, west, and south of the Central Andes, which will provide further important insights.
* [Coppa A, Candilio F, Arganini C, de la Vega Machicao E, Moreno Terrazas EG, Lucci M, et al. (2020) Population dynamics in pre-Inca human groups from the Osmore Valley, the Azapa Valley and the coast of the South Central Andes. PLoS ONE 15(12): e0229370. https://doi.org/10.1371/journal.pone.0229370](https://doi.org/10.1371/journal.pone.0229370)
* [Hubbe M, Terrazas Mata A, Herrera B, Benavente Sanvicente ME, González González A, Rojas Sandoval C, et al. (2020) Morphological variation of the early human remains from Quintana Roo, Yucatán Peninsula, Mexico: Contributions to the discussions about the settlement of the Americas. PLoS ONE 15(1): e0227444. https://doi.org/10.1371/journal.pone.0227444](https://doi.org/10.1371/journal.pone.0227444)
  * > The high morphological diversity among the early Mexican material marks an interesting counterpoint to previous interpretations of early American diversity and as such has important implications for our understanding of the processes of early human movement across the continent. At the very least, it provokes researchers to reevaluate the validity of extrapolations made in the past. Since the beginning of the debate surrounding the settlement of the Americas, there has been a strong emphasis on grouping the human processes happening in the northern and southern continent as being similar. As such, most of the studies focusing on North America tend to assume that the occupation of South America can be extrapolated linearly from the northern continent (e.g., [17, 23, 25, 30]), while studies focusing on South America often assume that what is observed in that continent was also true for North America (e.g., [9, 15, 56, 63]), effectively ignoring the difference in archaeological evidence and eco-geographical realities between the two hemispherical regions for the sake of broad generalized models of human dispersion into the Americas.<br/><br/>The diversity seen in the Quintana Roo material suggests that, already at the Pleistocene/Holocene boundary, Native American individuals showed high morphological diversity, supporting studies of the few early specimens available from North America [41]. Moreover, this high diversity is not only restricted to the early populations, since recent groups in the Mexican region have also been described as presenting high morphological diversity [35, 41]. Together, these studies point to the Mexican territory being highly diverse across the entire time humans occupied it. In Baja California, the morphological affinities between the Pericues and Paleoamericans has been explained as the result of the former being isolated from other Native American populations during most of the Holocene [41]. However, in central Mexico, gene-flow barriers are not good explanations for the degree of morphological diversity reported by Herrera and colleagues [35], who suggest high diversity being present in the territory over longer periods of time.
  * > First, the high diversity in Quintana Roo, when compared to South American early remains, suggests that South America may have been occupied by groups carrying only a smaller portion of the total biological diversity in North America. This scenario is also supported by the study of early North Americans remains, which show different patterns of morphological affinities from what is observed in South America [41, 68]. This would explain the relative morphological homogeneity of South Paleoamericans, and also fits well with recent genetic data that shows Paleoamericans and other South American native groups share a common ancestor in North America [23]. Consequently, these results suggest that the abrupt change observed in the morphological pattern in South America does not need to be true in North America, and previous models of population replacement or multiple migration waves may only be applicable to South America.<br/><br/>Second, the Quintana Roo diversity supports previous studies that suggested similar levels of diversity in other parts of North America [41]. In other words, these individuals demonstrate that there is no reason to expect that all North American early individuals will share the Paleoamerican morphological pattern. In fact, this helps to contextualize some of the debated results found for some of the few early North American skulls, like Kennewick Man [65]. While this individual does not classify clearly with recent Native Americans, his morphology has been previously associated with Polynesian and European populations. It is important to clarify here that association with specific populations in the reference series does not imply a direct gene-flow or migration between them. In other words, strong morphological affinities between El Pit I (or other early North American specimens) with European populations does not imply that there was a migration from Europe to the Americas. It implies an unexpected level of morphological diversity, but it is not enough to establish ancestor-descendent relationships between reference series and the specimens analyzed.
* > [Ross, A. H., Keegan, W. F., Pateman, M. P., & Young, C. B. (2020). Faces divulge the origins of Caribbean prehistoric inhabitants. Scientific reports, 10(1), 1-9.](https://doi.org/10.1038/s41598-019-56929-3)
* [Waters, M. R. (2019). Late Pleistocene exploration and settlement of the Americas by modern humans. Science, 365(6449).](https://doi.org/10.1126/science.aat5447)
  * > Ancient genomes also reveal several poorly understood population connections. The Lagoa Santa individuals and some contemporary Amazonian tribes share a subtle genetic connection with Indigenous New Guineans, Australians, and Andaman Islanders (6, 8, 12, 20). This signal appears to be derived from an extinct ancient ancestor of both groups (Population Y), but does not represent a migration of a group of Australasian ancestors to the Americas (20). The contemporary Mixe population carries a distinctive genetic legacy from an outgroup called “Unsampled Population A,” which is neither AB, NNA, nor SNA and split from the ancestral Indigenous American population sometime between ∼30 and ~22 ka ago in Beringia and mixed with the Mixe population during the early Holocene (12). The ∼5600-year-old Big Bar remains from British Columbia represent a previously undetected outgroup that split from the ancestral Indigenous American population in Beringia after AB diverged, but before the NNA–SNA split (12). These findings show that the Late Pleistocene Beringian population was not homogeneous and even suggests genetic structure between groups in Beringia, perhaps because they were widely dispersed.
* [Blell M, Hunter MA (2019) Direct-to-Consumer Genetic Testing's Red Herring: “Genetic Ancestry” and Personalized Medicine. Frontiers in Medicine. 29 March.](https://doi.org/10.3389/fmed.2019.00048)
* [Walajahi H, Wilson DR, Hull SC (2019) Constructing Identities: The Implications of DTC Ancestry Testing for Tribal Communities. Genetic Medicine. 21(8): 1744-1750.](https://dx.doi.org/10.1038%2Fs41436-018-0429-2)
* [Pinotti T et al. (2019) Y Chromosome Sequences Reveal a Short Beringian Standstill, Rapid Expansion, and early Population structure of Native American Founders. Current Biology. 29(1): 149-157.](https://doi.org/10.1016/j.cub.2018.11.029)
* [Sikora, M., Pitulko, V. V., Sousa, V. C., Allentoft, M. E., Vinner, L., Rasmussen, S., ... & Yang, M. A. (2019). The population history of northeastern Siberia since the Pleistocene. Nature, 570(7760), 182-188.](https://doi.org/10.1038/s41586-019-1279-z)
  * This shows that a population descended from the Yana Siberian population admixed into USR1 Beringians and Kolyma1 Siberians anytime between 13.3 - 23.7 ka.
  * > Finally, we investigated the geographical extent of these processes of population flux across northern Eurasia. The notable spatial pattern of Ancestral Palaeo-Siberian and East Asian ancestry in present-day populations (Fig. 3) suggests that Ancient Palaeo-Siberian ancestry was once widespread, probably as far west as the Ural Mountains. At the western edge of northern Eurasia, genetic and strontium-isotope data from ancient individuals at the Levänluhta site (Supplementary Information 1) document the presence of Saami ancestry in southwestern Finland in the late Holocene, at about 1.5 ka. This ancestry component is currently limited to the northern fringes of the region, which mirrors the pattern that is observed for Ancient Palaeo-Siberian ancestry in northeastern Siberia. However, although the ancient Saami individuals contain ancestry from an eastern source, we find that this is modelled better by East Asians than by Ancient Palaeo-Siberians, which suggests that the influence of Ancient Palaeo-Siberians probably did not extend across the Ural Mountains into western Eurasia (Extended Data Fig. 7, Supplementary Table 5). East–west gene flow continued to shape the gene pool of the Finnish population into the very recent past. We observe West Eurasian admixture in present-day Saami; by contrast, present-day Finns have greater Siberian ancestry than the ancient Levänluhta individual (Extended Data Table 1), who may represent the Scandinavian component in the dual-origin (Uralic and Scandinavian) gene pool of Finns today.
  * > Our findings reveal that the population history of northeastern Siberia is far more complex than previously inferred from the contemporary genetic record. It involved, at a minimum, three major population expansions and subsequent large-scale replacements during the late Pleistocene and early Holocene, with smaller-scale population fluxes since then. These three major waves are also clearly documented in the archaeological record. The initial movement into the region represents a now-extinct ANS population diversifying at about 38 ka, soon after the basal split between West Eurasians and East Asians, which is represented by the archaeological culture found at Yana RHS4,29. This finding is consistent with other studies that have shown that this period was a time of rapid expansion of early modern humans across Eurasia13. The arrival of people who carried ancestry from East Asia and their admixture with descendants of the ANS lineage at about 20–18 ka led to the formation of the Ancient Palaeo-Siberian and Native American lineages. In the archaeological record, this is reflected by the spread of a microblade technology that accompanies the post-LGM contraction of the once-extensive mammoth steppe10. This group was, in turn, largely replaced by Neo-Siberians in the early to mid-Holocene. Our data suggest that the Neo-Siberians received ANS-related ancestry indirectly through admixture with Ancient Palaeo-Siberian groups at about 13 ka, and possibly later from Bronze Age groups from the central Asian steppe after around 5 ka. A signal of Australasian ancestry that has been observed at a very low frequency in some modern and ancient South American populations30–32 is not evident in any of the ancient Siberian or Beringian samples sequenced here, or in previous studies21.
    * This is a interesting point, as it seems to indicate any Australasian ancestry must have not travelled over land, but rather through an oceanic path.
  * > We find that—despite the complex pattern of population admixture throughout the past 40,000 years—the first inhabitants of northeastern Siberia (represented by the Yana RHS individuals) were not the direct ancestors of either Native Americans or present-day Siberians, although traces of their genetic legacy can be observed in ancient and modern genomes across America and northern Eurasia. These earliest ancient Siberians (the ANS) are known from a handful of other ancient genomes (those of the Mal’ta and Afontova Gora individuals); they are the descendants of one of the early modern human populations that diversified as Eurasia was first settled by our species, and are thus highly distinct. The ANS were later partially assimilated with a group with East Asian affinity who formed the Ancient Palaeo-Siberians (represented by Kolyma1); this group also probably once had a wide geographical distribution across northern Eurasia. The genetic legacy of Ancient Palaeo-Siberians among present-day Siberians is more limited, being restricted to groups in northeastern Siberia. Importantly, this legacy is also evident in the Americas, which implies that the majority of Native American genetic ancestry is likely to have originated in northeastern Siberia rather than south-central Siberia, as has been inferred from modern mitochondrial and Y chromosome DNA33. The Neo-Siberians, who occupy much of the range that was previously inhabited by ANSrelated and Ancient Palaeo-Siberian groups, represent a more recent arrival that originated further south. The replacement processes we have revealed for the northeastern portion of Siberia are mirrored in far-western Eurasia by the regional displacement and admixture of the Saami people during the late Holocene. This suggests that similar processes probably took place in many other parts of the northern hemisphere.
    * This likely explains some of the "European" signals found in Native American genetics.
* [Flegontov, P., Altınışık, N. E., Changmai, P., Rohland, N., Mallick, S., Adamski, N., ... & Flegontova, O. (2019). Palaeo-Eskimo genetic ancestry and the peopling of Chukotka and North America. Nature, 570(7760), 236-240.](https://doi.org/10.1038/s41586-019-1251-y)
  * > Present-day Native Americans descend from at least four distinct streams of ancient migration from Asia4,5,10–12. First, people related to present-day East Asians moved into North and South America by approximately 14,500 years ago5,13,14. Here we refer to these groups as ‘First Peoples’. Second, people with a higher degree of genetic relatedness to Australasians, termed ‘Population Y’, contributed distinct ancestry to Indigenous groups from Amazonia5,10–12. Third, a stream of ancestry that relates to Palaeo-Eskimos spread throughout the American Arctic after about 5,000 years ago1–3. Fourth, a lineage that we here call ‘Neo-Eskimo’ spread with the Thule and related archaeological cultures throughout the Arctic region around 800 years ago2,3 , and is today present in Yup’ik and Inuit groups. We use the terms Palaeo-Eskimo and Neo-Eskimo2,15 here, but recognize that this terminology is not universally accepted by all scholars and Indigenous groups in Canada and the USA16. For naming the Arctic metapopulations, we use names of recognized language families—Na-Dene, Eskimo–Aleut, and Chukotko-Kamchatkan. We chose these terms because genetic and linguistic relationship patterns are highly congruent in this region.
* [Gnecchi-Ruscone GA et al. (2019) Dissecting the Pre-Columbian Genomic Ancestry of Native Americans along the Andes–Amazonia Divide, Molecular Biology and Evolution. 36(6): 1254–1269](https://doi.org/10.1093/molbev/msz066)
  * > Accordingly, recent evidence based on genomic data generated from ancient human remains retrieved in Central and South America confirmed the occurrence of multiple waves of diffusion into the south of the continent and suggested a complex scenario involving the spread of ancient populations that were already genetically structured (Moreno-Mayar, Vinner, et al. 2018; Posth et al. 2018). Among these samples, the oldest ones (dating to ∼11–10 ka), either from North America (Nevada) or from South America (West and East of the Andes), were those showing the highest genetic affinity with Anzick-1. This corroborates the hypothesis that the first diffusion from North into South America was extremely rapid (∼1–2 ka) and was not limited to the West coast since it is supported by samples from the entire continent. However, the genetic footprints of this first peopling event were found to be subtler in more recent samples, suggesting an extensive population replacement beginning from around 9 ka, by a different ancestral lineage with respect to that represented in the Clovis-associated Anzick-1 (Posth et al. 2018). Subsequent migrations after the initial diffusion were associated with expansion from Mesoamerica occurred sometime after ∼8.7 ka, which spread first southward (contributing to the ancestry of all present-day South Americans) and then northward, as suggested by ∼2 ka ancient samples from Nevada (Moreno-Mayar, Vinner, et al. 2018). Despite the common view that these processes contributed significantly to the formation of the modern South American genomic landscape, the two above-mentioned studies did not clarify in detail the relative proportions of these ancestries in the genomes of contemporary populations, being instead focused mostly on the relationship between ancient samples. Finally, minor contributions to the South American gene pool (i.e., <5% of ancestry) were ascribable to the affinity with an Austro Melanesian–related ancestry source already attested for some present-day Amazonian groups (Skoglund et al. 2015). This pattern was recognized also in a ∼10-ka sample from Brazil (Moreno-Mayar, Vinner, et al. 2018), coupled with a newly described connection between ancient samples from the California Channel Islands and the Late Central Andes since around 4.2 ka (Posth et al. 2018).
  * Goes on to show genetic admixture and possible gene flow amoung groups.
* [Barbieri, C., Barquera, R., Arias, L., Sandoval, J. R., Acosta, O., Zurita, C., ... & Mafessoni, F. (2019). The Current Genomic Landscape of Western South America: Andes, Amazonia, and Pacific Coast. Molecular biology and evolution, 36(12), 2698-2713.](https://doi.org/10.1093/molbev/msz174)
  * Examined gene flow using quite a few methods.
* [Grugni V et al. (2019) Analysis of the human Y-chromosome haplogroup Q characterizes ancient population movements in Eurasia and the Americas. BMC Biology. 17, 3.](https://bmcbiol.biomedcentral.com/articles/10.1186/s12915-018-0622-4)
* [Wright JL et al. (2018) Ancient nuclear genomes enable repatriation of Indigenous human remains. Science Advances. 4(12): eaau5064.](https://dx.doi.org/10.1126%2Fsciadv.aau5064)
* [Wade L (2018) Ancient DNA confirms Native Americans’ deep roots in North and South America. Science Magazine.](https://www.sciencemag.org/news/2018/11/ancient-dna-confirms-native-americans-deep-roots-north-and-south-america)
* [Barbieri C et al. (2018) The genomic landscape of western South America: Andes, Amazonia and Pacific Coast. bioRXiv preprint.](https://www.biorxiv.org/content/10.1101/505628v1.full)
* [Posth C et al. (2018) Reconstructing the Deep Population History of Central and South America. Cell. DOI:https://doi.org/10.1016/j.cell.2018.10.027](https://www.cell.com/cell/fulltext/S0092-8674(18)31380-1)
  * > Genetic studies of present-day and ancient Native Americans have revealed that the great majority of ancestry in indigenous people in non-Arctic America derives from a homogeneous ancestral population. This population was inferred to have diversified 17,500–14,600 calendar years before present (BP) (Moreno-Mayar et al., 2018a) into two branches that have been called “Southern Native American” or “Ancestral A” (ANC-A) and “Northern Native American” or “Ancestral B” (ANC-B) (Moreno-Mayar et al., 2018a, Raghavan et al., 2015, Rasmussen et al., 2014, Reich et al., 2012, Scheib et al., 2018). An individual dating to ∼12,900–12,700 BP from the Anzick site in Montana and associated with the Clovis culture was on the ANC-A lineage, which is also heavily represented in present-day Central and South Americans and in ancient Californians. In contrast, ANC-B ancestry is heavily represented in eastern North Americans and in ancient people from southwest Ontario (Scheib et al., 2018). The original studies that documented these two deep lineages fit models in which Central and South Americans were of entirely ANC-A ancestry (Rasmussen et al., 2014, Reich et al., 2012). However, Scheib et al. (2018) suggested that all Central and South Americans harbor substantial proportions of both ancestries (at least ∼30% of each).
  * > Recent analyses have also shown that some groups in Brazil share more alleles with Australasians (indigenous New Guineans, Australians, and Andaman Islanders) (Raghavan et al., 2015, Skoglund et al., 2015) and an ∼40,000 BP individual from northern China (Yang et al., 2017) than do other Central and South Americans. Such patterns suggest that these groups do not entirely descend from a single homogeneous population and instead derive from a mixture of populations, one of which, Population Y, bore a distinctive affinity to Australasians. Notably, our study includes data from individuals such as those from the Lapa do Santo site who have a cranial morphology known as “Paleoamerican,” argued to indicate two distinct New-World-founding populations (von Cramon-Taubadel et al., 2017). Here, we test directly the hypothesis that a Paleoamerican cranial morphology was associated with a lineage distinct from the one that contributed to other Native Americans (whether the proposed Population Y or another).
  * > We detect long-standing continuity between ancient and present-day Native Americans in each of the regions of South America we analyzed beginning at least ∼5,800 BP, a pattern that is evident in heatmaps, neighbor-joining trees, and multi-dimensional scaling plots computed on outgroup-f3 statistics (Figures 2, S1, and S2; Table S1). In Peru, the most ancient individuals dating up to ∼9,000 BP from Cuncaicha and Lauricocha share alleles at the highest rate with present-day indigenous groups living in the Central Andes (Lindo et al., 2018, Llamas et al., 2016). Individuals dating up to ∼8,600 BP from Arroyo Seco 2 and Laguna Chica also show the strongest allele sharing with some present-day indigenous people in the Southern Cone. In Brazil, the evidence of continuity with present-day indigenous people begins with the Moraes individual at ∼5,800 BP. A striking pattern of continuity with present-day people is also observed in the ∼2,000 BP Jabuticabeira 2 individuals who were part of the Sambaqui shell-mound building tradition that was spread along the south Brazilian coast from around 8,000–1,000 BP. The Jabuticabeira 2 individuals share significantly more alleles with some Ge-speaking groups than they do with some Tupi-Guarani speaking groups who have been predominant on the coast during the post-Colonial period (Figure S3; Table S1). This supports the theory of shared ancestry between the makers of the Sambaqui culture and the speakers of proto-Ge who are hypothesized to have lived in the region ∼2,000 BP (Iriarte et al., 2017). These findings also support the theory of coastal replacement of Ge speakers by Tupi-Guarani speakers after ∼1,000 BP (Hubbe et al., 2009) (STAR Methods).
  * > Figure 1 plots the excess rate of allele sharing of ancient Central and South Americans with the ∼12,800 BP Anzick-1 individual from Montana compared to the ∼11,500 BP USR1 individual from Alaska, an Ancient Beringian who derives from a lineage that split from the one leading to all other known Native Americans before they separated from each other (Moreno-Mayar et al., 2018a) (Table S4). The distribution of this statistic f4(Mbuti, Test; USR1, Anzick-1) confirms previous findings that Anzick-1 relatedness is greatest in Central and South Americans and lowest in North American groups (Table S4) (Rasmussen et al., 2014), with the exception of the California Channel Islands, where the earliest individuals from San Nicolas Island around 4,900 BP show some of the highest Anzick-1 relatedness, consistent with an early spread of Anzick-1-related people to these islands followed by local isolation (Scheib et al., 2018) (Figure S2D).
  * > The fact that the three pairs each require two different sources of ancestry in order to produce a model fit could mean that they descend from a total of three (or more) distinct sources of ancestry differentially related to groups outside South America or alternatively that they are mixtures in different proportions of only two sources. To distinguish these possibilities, we used qpWave’s ability to test for consistency with the hypothesis that sets of three populations (Test1, Test2, Test3) derive from just two populations relative to the same set of outgroups. qpWave rejects the hypothesis of two sources (p = 0.0022), a result that is unlikely to be due to backflow from South America into Central America as the signal persists when we remove present-day Mexicans from the outgroup set (p = 0.001) (Table S5). Further evidence for the robustness of the finding of three source populations comes from the fact that the signal remains significant when we restrict to transversion polymorphisms that are not affected by cytosine-to-thymine errors (p = 0.01). We caution that we did not find significant signals of ancestry heterogeneity relative to North American outgroups when repeating the qpWave tests on pairs of present-day populations. We speculate that this may reflect more recent homogenization leading to variation in ancestry proportions too subtle for our methods to detect.
  * > When we add present-day Surui individuals into the analysis, there is evidence for a fourth source of ancestry (p = 0.03) (Table S5), likely reflecting the same signal that led to finding “Population Y” ancestry in this group (Raghavan et al., 2015, Skoglund et al., 2015).
  * > Our finding of no excess allele sharing with non-Native American populations in the ancient samples is also striking as many of these individuals—including those at Lapa do Santo—have a “Paleoamerican” cranial morphology that has been suggested to be evidence of the spread of a substructured population of at least two different Native American source populations from Asia to the Americas (von Cramon-Taubadel et al., 2017). Our finding that early Holocene individuals with such a morphology are consistent with deriving all their ancestry from the same homogeneous ancestral population as other Native Americans extends the finding of Raghavan et al. (2015) who came to a similar conclusion after analyzing Native Americans inferred to have Paleoamerican morphology who lived within the last millennium.
  * > Most important, our discovery that the Clovis-associated Anzick-1 genome at ∼12,800 BP shares distinctive ancestry with the oldest Chilean, Brazilian, and Belizean individuals supports the hypothesis that an expansion of people who spread the Clovis culture in North America also affected Central and South America, as expected if the spread of the Fishtail Complex in Central and South America and the Clovis Complex in North America were part of the same phenomenon (direct confirmation would require ancient DNA from a Fishtail-context) (Pearson, 2017). However, the fact that the great majority of ancestry of later South Americans lacks specific affinity to Anzick-1 rules out the hypothesis of a homogeneous founding population. Thus, if Clovis-related expansions were responsible for the peopling of South America, it must have been a complex scenario involving arrival in the Americas of sub-structured lineages with and without specific Anzick-1 affinity, with the one with Anzick-1 affinity making a minimal long-term contribution. While we cannot at present determine when the non-Anzick-1 associated lineages first arrived in South America, we can place an upper bound on the date of the spread to South America of all the lineages represented in our sampled ancient genomes as all are ANC-A and thus must have diversified after the ANC-A/ANC-B split estimated to have occurred ∼17,500–14,600 BP (Moreno-Mayar et al., 2018a).
  * > A second notable finding of this study is our evidence that the ancient individuals from the California Channel Islands have distinctive and significant allele sharing with groups that became widespread over the Central Andes after ∼4,200 BP. There is no archaeological evidence of large-scale cultural exchange between North and South America around this time, but it is important to recognize that ∼4,200 BP is a minimum date for the exchange between North and South American that drove this pattern; the gene flow itself could have occurred thousands of years before and the ancestry deriving from it could have persisted in a region of South America not yet sampled with ancient DNA. The evidence of an expansion of this ancestry type in the Central Andes by ∼4,200 BP is notable in light of the increasing density of sites in this region at approximately this time, a pattern that is consistent with a demographic expansion of a previously more restricted population (Goldberg et al., 2016).
* [Lindo, J., Haas, R., Hofman, C., Apata, M., Moraga, M., Verdugo, R. A., ... & Di Rienzo, A. (2018). The genetic prehistory of the Andean highlands 7000 years BP though European contact. Science advances, 4(11), eaau4921.](https://doi.org/10.1126/sciadv.aau4921)
  * > The Andean highlands of South America have long been considered a natural laboratory for the study of genetic adaptation of humans (1), yet the genetics of Andean highland populations remain poorly understood. People likely entered the highlands shortly after their arrival on the continent (2, 3), and while some have argued that humans lived permanently in the Central Andean highlands by 12,000 before the present (BP) (4), other research indicates that permanent occupation began between 9500 and 9000 BP (5, 6). Regardless of when the peopling of high-elevation environments began, selective pressure on the human genome was likely strong–not only because of challenging environmental factors, but also because social processes such as intensification of subsistence resources and residential sedentism (7) promoted the development of agricultural economies, social inequality, and relatively high population densities across much of the highlands. European contact initiated an array of economic, social, and pathogenic changes (8). Although it is known that the peoples of the Andean highlands experienced population contraction after contact (8), its extent is debated from both archeological and ethnohistorical perspectives, especially concerning the size of the indigenous populations at initial contact (9).
  * > Given that the above analyses provide an inference of genetic affinity in the Andes extending over roughly 4000 years BP, and perhaps up to 7000 years, we explored models relevant to the timing of the first permanent settlements in the Andes, as well as the extent of the population collapse occurring after European contact in the 1500s (Fig. 4). We used a composite likelihood method [Fastsimcoal2 (20)], which was informed by the site frequency spectra (SFS) of multiple populations, including the Rio Uncallane (n = 5; high altitude, pre-European contact), the Aymara (n = 25; high altitude, post-European contact), and the Huilliche-Pehuenche (n = 39; lowland population, post-European contact). We infer a split between low- and high-altitude populations in the Andes occurring at 8750 years [95% confidence interval (CI), 8200 to 9250]. We also infer the magnitude of the population collapse in the Andes after European contact to be a 27% reduction in effective population size and occurring 425 years ago (95% CI, 400 to 450). The model also inferred the collapse experienced by the Mixe (included in the model for the split between Northern and Southern Hemisphere populations) and the Huilliche-Pehuenche after European contact, which were much more severe. We infer a 94% (95% CI, 0.94 to 0.96) collapse for the Mixe and a comparable 96% (95% CI, 0.95 to 0.96) collapse for the Huilliche-Pehuenche. The collapse difference between high- and low-altitude populations from the Andes region is significant [Huilliche-Pehuenche versus Aymara, P < 0.0001 (95% CI, 0.66 to 0.71), χ2 test].
  * Shows interesting relationships in the ADMIXTURE clusturing between the Piapocco group and the Mixe, Ancient Andes (which branch out into Quechua and Aymara), Curui, Karitiana, and Huilliche-Pehuenche.
* [Messina F, Di Corcia T, Ragazzo M, Sanchez Mellado C, Contini I, Malaspina P, et al. (2018) Signs of continental ancestry in urban populations of Peru through autosomal STR loci and mitochondrial DNA typing. PLoS ONE 13(7): e0200796.](https://doi.org/10.1371/journal.pone.0200796)
* [Lindo, J., Rogers, M., Mallott, E. K., Petzelt, B., Mitchell, J., Archer, D., ... & DeGiorgio, M. (2018). Patterns of genetic coding variation in a Native American population before and after European contact. The American Journal of Human Genetics, 102(5), 806-815.](https://doi.org/10.1016/j.ajhg.2018.03.008)
* [Moreno-Mayar JV et al. (2018) Early human dispersals within the Americas. Science. 10.1126/science.aav2621](https://www.researchgate.net/publication/328815661_Early_human_dispersals_within_the_Americas) &mdash; This is a fantastic paper. I'd love to apply it's techniques to a broader sample of populations and early DNA sequences.
  * > The ~5.1-ka-old Patagonian Ayayema genome is an exception; it forms a clade with the Lagoa Santa population. This suggests that the arrival of the Mesoamerican-related ancestry occurred post–5.1 ka ago and/or that it did not reach the remote region inhabited by the Ayayema individual’s ancestors (Fig. 3C) (13). This result is qualitatively mirrored by the 7.2-ka-old Punta Santa Ana individual (both cluster with present-day Patagonians and form a clade with Lagoa Santa). However, the low coverage of Punta Santa Ana may reduce our power to detect possible Mesoamerican-related admixture (Fig. 3C) (13). [Uses Mixe as a proxy for Mesoamerican allels]
  * > Mesoamerican-related expansion possibly had a bearing on a later, unresolved pattern seen in North America. In the western Great Basin of North America, paleoenvironmental evidence indicates decreased effective precipitation and increased aridity during the Middle Holocene, which led to a human population decline (45, 46). By ~5 ka ago, regional populations were rebounding, but whether these were descendants of the previous inhabitants is unknown. Unclear also is the relationship between those later Holocene groups and the people present in the region at the time of European contact and today. Linguistic evidence suggests that ancestors of Numic speakers presently inhabiting the region today arrived recently, perhaps ~1 ka ago. There is also archaeological evidence of changes in material culture around that time, though how those relate to the linguistic turnover is uncertain. Nor is it known whether these changes are related to population admixture or replacement. Patterns and changes in language, material culture, and genetics need not be congruent or causally linked (47). Thus, the so-called Numic Expansion hypothesis has been highly debated (46, 48); we address the population aspect by comparing genomes from Spirit Cave and Lovelock Cave (Fig. 1A) (13).
  * > MDS and ADMIXTURE analyses, as well as D statistics of the form D(SpiritCave, Lovelock2/3; NA, Yoruba), suggest that despite the ~9 ka separating the Spirit Cave and Lovelock individuals, they form a clade with respect to other NAs (Fig. 1, C to F) (13). We tested that topology through the same admixture graph search implemented for SNAs (13). We were not able to reject the model without Mesoamerican-related admixture for Lovelock 2 (~1.9 ka old). However, the ~0.7-ka-old Lovelock 3 individual received Mesoamerican admixture from a group that was likely not present in the region just ~1.2 ka earlier, at the time of Lovelock 2. Because we do not know the language(s) that may have been involved, we cannot securely attribute this admixture to arriving Numic speakers [the Mixe, whom we use as a proxy for Mesoamerican ancestry, fall in a separate language family from Numic (49)]. Notably, we also observe genetic continuity, suggesting that there was not a complete population replacement.
* [Scheib CL et al. (2018) Ancient human parallel lineages within North America contributed to a coastal expansion. Science. 360(6392): 1024-1027.](https://science.sciencemag.org/content/360/6392/1024) &mdash; [PDF](https://www.academia.edu/36866318/Ancient_human_parallel_lineages_within_North_America_contributed_to_a_coastal_expansion)
* [Gómez-Carballa A et al. (2018) The peopling of South America and the trans-Andean gene flow of the first settlers. Genome Research. 28: 767-779.](https://genome.cshlp.org/content/28/6/767.full)
* [Brandini S et al. (2018) The Paleo-Indian Entry into South America According to Mitogenomes. Molecular Biology and Evolution. 35(2): 299-311.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5850732/)
* [Hamilton-Brehm SD, Hristova LT, Edwards SR, Wedding JR, Snow M, Kruger BR, et al. (2018) Ancient human mitochondrial DNA and radiocarbon analysis of archived quids from the Mule Spring Rockshelter, Nevada, USA. PLoS ONE 13(3): e0194223. https://doi.org/10.1371/journal.pone.0194223](https://doi.org/10.1371/journal.pone.0194223)
* [KINY Radio (2018) DNA of Indigenous peoples explained. KINY Radio.](http://www.kinyradio.com/news/news-of-the-north/dna-of-indigenous-peoples-explained/?fbclid=IwAR0_VYafbGYaKQsvQaGldUY5lLVn3g9uhYzhy1tgWiN-xHupxOoG80Q4k1A)
* [Hohenstatt B (2018) DNA, oral tradition indicate 10,000 years of Native history in Southeast. Juneau Empire. November 13.](https://www.juneauempire.com/news/dna-oral-tradition-indicate-10000-years-of-native-history-in-southeast/)
* [Achilli, A., Olivieri, A., Semino, O., & Torroni, A. (2018). Ancient human genomes—keys to understanding our past. Science, 360(6392), 964-965.](https://doi.org/10.1126/science.aat7257)
* [Moreno-Mayar, J. V., Potter, B. A., Vinner, L., Steinrücken, M., Rasmussen, S., Terhorst, J., ... & Willerslev, E. (2018). Terminal Pleistocene Alaskan genome reveals first founding population of Native Americans. Nature, 553(7687), 203-207.](https://doi.org/10.1038/nature25173)
* [Sutter, R. C. (2000). Prehistoric genetic and culture change: a bioarchaeological search for pre-Inka altiplano colonies in the coastal valleys of Moquegua, Peru, and Azapa, Chile. Latin American Antiquity, 11(1), 43-70.](https://doi.org/10.2307/1571670)
* [de Saint Pierre M (2017) Antiquity of mtDNA lineage D1g from the southern cone of South America supports pre-Clovis migration. Quaternary International. 444 B: 19-25.](https://doi.org/10.1016/j.quaint.2017.05.054) &mdash; [PDF](https://www.semanticscholar.org/paper/Antiquity-of-mtDNA-lineage-D1g-from-the-southern-of-Pierre/66b3ebf226025bb8c54ab5ed13ff2bc3988cd18c)
* [Morrow JE (2017) After Anzick: Reconciling new genomic data and models with the archaeological evidence for peopling of the Americas. Quaternary International. Part B. 444, 1-3.](https://doi.org/10.1016/j.quaint.2017.07.002) &mdash; [PDF](https://www.researchgate.net/publication/318442263_After_Anzick_Reconciling_new_genomic_data_and_models_with_the_archaeological_evidence_for_peopling_of_the_Americas)
* [Lindo J et al. (2017) Ancient individuals from the North American Northwest Coast reveal 10,000 years of regional genetic continuity. PNAS. 114(16): 4093-4098.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5402414/)
* [González-Oliver A, Garfias-Morales E, Smith DG, Quinto-Sánchez M (2017) Mitochondrial DNA Analysis of Mazahua and Otomi Indigenous Populations from Estado de México Suggests a Distant Common Ancestry. Human Biology. 89(3): 3.](https://digitalcommons.wayne.edu/humbiol/vol89/iss3/3/)
* [von Cramon-Taubadel, N., Strauss, A., & Hubbe, M. (2017). Evolutionary population history of early Paleoamerican cranial morphology. Science advances, 3(2), e1602289.](https://doi.org/10.1126/sciadv.1602289)
  * > Genomic data from early and contemporary Native Americans paint a complex picture of dispersal into and gene flow within the Americas (28). Many studies support a single migration into subarctic America with subsequent population divergence and gene flow (15, 29–31), whereas others suggest up to four separate waves of migration from northeast Asia (32–35). However, there is widespread agreement that at least one separate wave of dispersal from outside the Americas is required to explain the genetic variation of paleoarctic and modern arctic groups (34, 36–38), although the current consensus suggests that all indigenous South Americans descend from only one wave of dispersion (34, 39). Although (paleo)genomic data may provide important context for the debates surrounding the origins of cranial diversity, minimal direct genomic data are currently available for Paleoamerican specimens in South America [see the study of Fehren-Schmitz et al. (40)]. It should also be noted that, although all cranial populations have ancestors (that is, population history), not all fossil populations leave descendants, so Paleoamericans did not necessarily contribute to contemporary Native American genetic history. For these reasons, we take an explicit evolutionary population history approach based exclusively on the analysis of three-dimensional (3D) craniometric shape data, which have been repeatedly shown to provide an accurate proxy for neutral genetic expectation [see, for example, the study of von Cramon-Taubadel (41) for a recent review].
  * > The null hypothesis that Paleoamericans from Lagoa Santa share a MRCA with contemporary South Americans was rejected, irrespective of which cranial region was used. Moreover, alternative histories where Paleoamericans share a MRCA with subarctic Native Americans were also not statistically supported. In contrast, the best-fit history models suggest last common ancestry between Paleoamericans and other Native Americans outside the New World (Fig. 3). It is also worth noting that, although our results are consistent with the high levels of within-continent diversity noted previously, Lagoa Santa crania were not found to be outliers to contemporary modern human cranial variation. That is, their morphological variability falls within that observed among modern human populations, yet their overall morphology cannot be accounted for by a null hypothesis of shared common ancestry with all subarctic Native Americans.
  * > The best-fit model consistently suggested by the cranium, vault, and basicranium data sets is one where Lagoa Santa Paleoamericans and New World populations share a MRCA in northeast Asia with a population that later gave rise to the populations that colonized the American arctic and Greenland. Moreover, the face data set suggests a more ancient common ancestral link between Paleoamericans and the ancestors of all East Asians and New World populations. This is not inconsistent with the results from the vault and basicranium but does suggest that Paleoamerican facial shape is more generalized and plesiomorphic than vault or basicranial shape. This is also underlined by the fact that some significant alternative histories for the facial data set suggest common ancestry between Paleoamericans and Australasians, which would suggest the retention of a more plesiomorphic, generalized facial shape from the common ancestor of all Australasians and New World populations.
  * > Our results are also in accordance with the recent genomic link found between Amazonian populations and Australasians (35, 37). Skoglund et al. (35) suggest than an ancient Native American lineage, named “Population Y,” could have resulted from a highly substructured ancestral northeast Asian population that shared strong genetic affinities with the ancestors of modern Australasians. There is mounting genetic and morphological evidence for at least two major waves of dispersal into Asia from Africa, with Australomelanesians representing modern descendants of the earlier migration (52, 53). There is also genomic evidence that northeast Asia was continually occupied throughout the Last Glacial Maximum (~21 thousand years ago) (33). These spatiotemporal dynamics would have provided ample opportunity for population substructure to emerge within Siberia and Beringia and thus provide several distinct sources of Native American ancestry from the same geographic location through time. Earlier (Paleo)siberian populations would have shared greater genetic affinity with Australasians further south as an outcome of their shared out-of-Africa dispersal history. However, as time progressed, further dispersal from Africa along with differentiation and gene flow within Asia would have altered the genetic signature of the northeast Asian source populations that gave rise to later Paleoeskimo and (possibly) other Native American populations. If such a spatiotemporal scenario is borne out by future genomic studies of northeast Asians and Native Americans, then it would solve the apparent conundrum of how a single “source” population in Siberia/Beringia could have given rise to several distinct genetic and phenotypic lineages within the Americas over time.
* [Lindo, J., Huerta-Sánchez, E., Nakagome, S., Rasmussen, M., Petzelt, B., Mitchell, J., ... & Malhi, R. S. (2016). A time transect of exomes from a Native American population before and after European contact. Nature communications, 7(1), 1-11.](https://doi.org/10.1038/ncomms13175)
* [Llamas, B., Fehren-Schmitz, L., Valverde, G., Soubrier, J., Mallick, S., Rohland, N., ... & Haak, W. (2016). Ancient mitochondrial DNA provides high-resolution time scale of the peopling of the Americas. Science advances, 2(4), e1501385.](https://advances.sciencemag.org/content/2/4/e1501385)
  * Assumes that population expansion didn't occur until 16000 years ago, but since we've found footprints in the southwest dating to before then, we can safely say this assumption is faulty.
  * > The geographic isolation of the Americas delayed human settlement until the end of the Pleistocene [20 to 10 thousand years ago (ka)]; however, despite this relatively recent date, the specific time, place, and route of entry remain uncertain. It is likely that the first peoples moved from Asia across the Bering Land Bridge (1, 2), the landmass between Eurasia and America exposed by lowered sea levels during the Last Glacial Maximum (LGM). However, at this time, much of northern North America was covered by the Cordilleran and Laurentide ice sheets, which blocked access from eastern Beringia (Bering Land Bridge and Alaska/Yukon) southward to the rest of the Americas (Fig. 1A). Shortly after the Cordilleran ice sheet began to retreat ~17 ka (3), a potential Pacific coastal route became available ~15 ka (Fig. 1B) (3, 4), whereas an alternative route through an inland ice-free corridor along the eastern side of the Rocky Mountains opened around ~11.5 to 11 ka (4–6). The timing and route used in the migration event are important in understanding the size, number, and speed of the first migratory wave(s). Timing and route are also pivotal in resolving contentious issues such as the nature of peoples before Clovis—the first widespread archaeologically recognized culture in North America (13.2 to 12.8 ka) (1).
  * > The most recent genetic divergence observed between the ancestors of Siberians and Native Americans (24.9 ka; section S5 and Fig. 2B) is the last point at which we can detect apparent gene flow (that is, a shared lineage) between the Siberian population and the ancestral Native American population. We can assume that the real population divergence occurred after this point. In addition, if we accept that the estimated TMRCA of each of the five Native American haplogroups provides an independent estimate of the timing of the same small population’s isolation, we can use the 95% credible intervals to constrain the lower bound (section S5). The resulting estimate that the two populations became fully isolated between 24.9 and 18.4 ka is in accordance with calculations from modern complete genomes which indicate that Siberians and Native Americans split no later than ~23 ka (17). Gene flow to and from east Siberia certainly appears to have ceased by the height of the LGM (18.4 ka; Fig. 2B).
  * > Our data cannot determine whether the separation between Siberian and Native American ancestral populations occurred in Siberia or Beringia. However, the start of isolation (24.9 to 18.4 ka) closely coincides with the LGM. We hypothesize that cold arid conditions drove populations on the western (that is, Siberian) margins of the Bering Land Bridge to migrate to southern refugia (Fig. 1A), as suggested by the absence of megafauna kill sites younger than the far north Yana Rhinoceros Horn site 32 ka (1). In contrast, any populations east of the Kamchatka and Chukotka Peninsulas would not have been able to retreat farther south than the Aleutian ice belt and would thus remain isolated in eastern Beringia (Fig. 1A). We cannot accurately estimate the size of this founding population, but the effective female population that subsequently entered the Americas appears to be ~2000, which accords well with previous studies (9, 10, 25). Although this number cannot be directly translated into census population size, it suggests that the human population isolated in eastern Beringia was relatively small, probably not exceeding a few tens of thousands of people (section S6). The presence of large numbers of megafauna in eastern Beringia during the late Pleistocene, including the LGM, indicates an ice-free region dominated by shrub tundra (29), which would have been more than capable of sustaining such a population size (section S6). Thus, our observations are consistent with the idea that the founding Native American population used the exposed Bering Land Bridge and adjacent regions in Alaska/Yukon as a refugium during the height of the LGM, before climatic change and the retreat of the ice sheets allowed access to the remainder of the Americas. Unfortunately, the large temporal and geographic gaps in the archaeological record between the Yana Rhinoceros Horn site (~32 ka, western Beringia) and the Swan Point site (~14 ka, eastern Beringia) provide little additional information about this process (Fig. 1A) (1) or how the ancestral Native Americans were isolated from their Asian counterparts.
  * > The scenario of an Eastern Beringia refugium is consistent with the Beringian standstill hypothesis, which suggests that the ancestral Native Americans were isolated in the area for up to 15 thousand years (ky) (9, 10, 29). Our large data set of dated mitogenomes provides tight estimates for the duration of the standstill and the subsequent movement out of the area. The mitogenomic tree shows a sudden burst of lineage diversification starting ~16.0 to 13.0 ka (Fig. 3B). This is followed by a steep increase in the mean female effective population size (>10%) between adjacent time bins starting 16.0 ka (Fig. 3C). Overall, the population underwent a 60-fold increase between 16.0 and 13.0 ka, suggesting that 16.0 ka represents the initial entry into the Americas, where population size significantly increased in a more favorable environment. Considering the time between isolation (24.9 to 18.4 ka) and entry (16.0 ka), the improved temporal resolution provided by our data suggests that the Beringian Standstill could have been as short as ~2.4 ky, and no longer than ~9 ky, consistent with recent estimates based on autosomal data from complete modern-day genomes (17).
  * > The population burst at 16.0 ka is contemporaneous with the rapid retreat of coastal glaciers along the northwest Pacific coast associated with a phase of stepwise ocean warming (2° to 3°C) in the region (3). This date considerably predates the opening of the inland ice-free corridor ~11.5 to 11.0 ka (4–6) and indicates that the initial entry into the Americas took place via a southward expansion along recently emerged northwest Pacific coastal land (Fig. 1B) (3, 17, 28, 30, 31). Given the early archaeological sites in Monte Verde in southern Chile at 14.6 ka (32), the mitogenome data indicate that the transit of the full length of the Americas took around 1.4 ky.
  * > It has been suggested that European colonization resulted in a bottleneck in Native American genetic diversity (34, 35). In the present data set, it is notable that no ancient haplotype (Fig. 3B, red lineages) shared a common ancestor with a modern haplotype (Fig. 3B, blue lineages) more recently than ~9.0 ka (Fig. 3B, black triangle in haplogroup A2), despite the number of samples examined from both populations. Our ancient samples were principally derived from large population centers along the western coast of South America, which experienced high extinction rates following European colonization. Historic accounts have reported that the population decline was more rapid and intense in the Gulf of Mexico and the Pacific coast of Peru than in other areas such as the Mesoamerican plateau or the Andean highlands (7). The pronounced phylogeographic structure observed in Native American mitogenomes suggests that such demographic events could have removed major portions of genetic diversity in certain areas. As a result, it is possible that the lack of overlap between ancient and modern haplotypes is influenced by the undersampling of modern-day diversity in the source regions of the archaeological material.
  * > As a result, our ancient mitochondrial data suggest that European colonization was followed by local mass mortality and extinction of lineages associated with major population centers of the pre-Columbian past. Our results contrast with previous observations that Native American genetic diversity has been temporally and geographically stable for at least the past 2000 years (33). However, the apparent contradiction between our study and earlier work is likely attributable to a significant improvement of sequence resolution. We also caution that the demographic models tested in this study are likely too simple to encompass the particulars of the colonization of the Americas—despite exploring scenarios with early emergence of geographic structure, as well as ongoing haplotype loss resulting from either serial founder effects and drift, or population displacement and elimination during the expansion of major Central Andean imperial states. Finally, more complete mitochondrial sequence data from early and modern-day populations will be needed to further refine the demographic models. In particular, it will be critical to ensure a strong spatial overlap between archaeological sites and present-day population locations (for example, Peru).
* [Wade, Lizzie. Mummy genomes reveal just how catastrophic European contact was for New World. Science. Apr. 2016.](http://www.sciencemag.org/news/2016/04/mummy-genomes-reveal-just-how-catastrophic-european-contact-was-new-world)
* [Jota, M. S., Lacerda, D. R., Sandoval, J. R., Vieira, P. P. R., Ohasi, D., Santos-Júnior, J. E., ... & Santos, F. R. (2016). New native South American Y chromosome lineages. Journal of human genetics, 61(7), 593-603.](https://doi.org/10.1038/jhg.2016.26)
* [Valverde G, Barreto Romero MI, Flores Espinoza I, Cooper A, Fehren-Schmitz L, Llamas B, et al. (2016) Ancient DNA Analysis Suggests Negligible Impact of the Wari Empire Expansion in Peru’s Central Coast during the Middle Horizon. PLoS ONE 11(6): e0155508.](https://doi.org/10.1371/journal.pone.0155508)
* [Ochoa-Lugo MI, de Lourdez Muñoz M, et al. (2016) Genetic Affiliation of Pre-Hispanic and Contemporary Mayas through Maternal Linage. Human Biology. 88(2):136-167.](https://www.researchgate.net/publication/313483707_Genetic_Affiliation_of_Pre-Hispanic_and_Contemporary_Mayas_Through_Maternal_Linage)
* [López, John Emilio William (2016) Genetic Diversity in Urban and Rural Indigenous Mexico. Master's thesis, Harvard Extension School.](https://dash.harvard.edu/handle/1/33797263)
* [Raghavan, M., Steinrücken, M., Harris, K., Schiffels, S., Rasmussen, S., DeGiorgio, M., ... & Willerslev, E. (2015). Genomic evidence for the Pleistocene and recent population history of Native Americans. Science, 349(6250).](https://doi.org/10.1126/science.aab3884)
  * > We then applied the diCal2.0 model that allows for gene flow between populations after their split in order to estimate divergence times for Native Americans from more geographically and genetically distant East Asian groups, including the Siberian Nivkh and Han Chinese. As before, the divergence estimates for Amerindians and Athabascans were very similar to one another, ~23 ka (table S11B and figs. S18 and S21).
* [Gómez-Carballa, A., Catelli, L., Pardo-Seco, J., Martinón-Torres, F., Roewer, L., Vullo, C., & Salas, A. (2015). The complete mitogenome of a 500-year-old Inca child mummy. Scientific reports, 5(1), 1-11.](https://doi.org/10.1038/srep16462)
  * > Figure 4 shows that C1b mitogenomes were found around two main geographic locations, one in Mesoamerica and the other one around Peru and extending southwards from here along the Pacific coast. Figure 4 also shows a phylogenetic skeleton of the main C1b branches in the American chronology context. The observed pattern of geographic variation is compatible with the following broad migration scenario for C1b carriers: (i) early and rapid spread of C1b across the full American continent during the period of initial continental settlement; (ii) important population isolation of the Mesoamerican and the South American gene pools for long periods, as indicated by the presence of very old and very young clades exclusively found in each of these two sub-continental regions; and (iii) sporadic gene exchange between both sub-continental regions, as suggested by the existence of a few clades that are present today in Mesoamerica and South America; these clades have ages ranging from 15 kya (C1b3) to 2 kya (C1b2) ago. The main ancient American civilizations, such as the Maya and the Inca, could have contributed to the gene flow between the main continental regions.
  * > The haplotype of the Inca mummy belongs to a new clade that branches off from the root of haplogroup C1b, thus providing additional support for the authenticity of the haplotype (see M&M section). This new clade, C1bi (where ”i” stands for ‘Inca’), has 10 private mutations (Table 1). All private mutations were checked very carefully and replicated in confirmatory sequence analysis. Although there is no way to date C1bi using only one mitogenome, the amount of variation accumulated in the mummy’s haplotype is compatible with an old age, at least as old as other old branches within C1b that have accumulated a similar amount of variation. Figure S2 shows the number of mutations accumulated from the root of haplogroup C1b to all the tips of the phylogeny (Figure S1) and their relative frequency; the ten mutations observed in the mummy’s branch fall within expected values. The HVS-1 motif (Table 3) was used to search for members of C1bi in public haplotype databases (>170,000 partial mtDNA sequences). Only four samples belonging to C1b share the transition at position 16124 (Table 3; Fig. 2). A tentative dating can be carried out using these few control region haplotypes that fall within C1bi (Table 3). The TMRCA estimated from these haplotypes (based on ρ) is 14.3 (5.0-24.0) kya, which is consistent with the suggestion that C1bi constitutes an old clade. At the same time, the phylogeographic patterns of C1bi control region haplotypes point to a distribution of this lineage constrained to South America. Moreover, these patterns fit well with the maximum extension of the Inca Empire around 1525, when the mandate of Huayna Capac (eleventh “Sapa” of the Inca Empire) ended. Within C1b, there is a different sub-clade that shows very similar characteristics to C1bi, namely, C1b13. The TMRCA of this sub-clade is 11.8 (8.6–15.1) kya; it is virtually absent from North-Central America and its geographic location is mainly centered in Chile. C1b13 most likely arose in the Southern Cone region and differentiated locally soon after human arrival, during the tribalization and linguistic differentiation process25. The geographic distribution of C1b13 also fits well with the expansion of the Inca Civilization into the northern territories of Chile although its age is much older, thus suggesting that perhaps only some (still un-sampled) sub-lineages of C1bi might be related to the Inca’s timeline (as it occurs with other C1b sub-clades; Fig. 4).
  * [Wade, Lizzie. Inca child mummy reveals lost genetic history of South America. Science. Nov. 2015](http://www.sciencemag.org/news/2015/11/inca-child-mummy-reveals-lost-genetic-history-south-america)
* [Rasmussen M, Sikora M, Albrechtsen A, Korneliussen TS, Moreno-Mayar JV, Poznik GD, et al. (2015) The ancestry and affiliations of Kennewick Man. Nature. 523: 455–458. 10.1038/nature14625](https://www.nature.com/articles/nature14625)
* [González-Martín A, Gorostiza A, Regalado-Liu L, Arroyo-Peña S, Tirado S, Nuño-Arana I, et al. (2015) Demographic History of Indigenous Populations in Mesoamerica Based on mtDNA Sequence Data. PLoS ONE 10(8): e0131791. https://doi.org/10.1371/journal.pone.0131791](https://doi.org/10.1371/journal.pone.0131791)
* [Gómez-Carballa, A., Catelli, L., Pardo-Seco, J., Martinón-Torres, F., Roewer, L., Vullo, C., & Salas, A. (2015). The complete mitogenome of a 500-year-old Inca child mummy. Scientific reports, 5(1), 1-11.](https://doi.org/10.1038/srep16462)
* [Vilar MG et al. (2014) Genetic diversity in Puerto Rico and its implications for the peopling of the Island and the West Indies. American Journal of Physical Anthropology. 155: 352-368.](https://www.academia.edu/20083458/Genetic_diversity_in_Puerto_Rico_and_its_implications_for_the_peopling_of_the_Island_and_the_West_Indies)
* [Brooks DR (2014) Genetic Diversity in Totonacan-Speaking Populations from Veracruz and Puebla States of Mexico. Proceedings of the National Conference of Undergraduate Research.](http://web.archive.org/web/20200218052502/http://www.ncurproceedings.org/ojs/index.php/NCUR2014/article/view/1087)
* [Scliar, M. O., Gouveia, M. H., Benazzo, A., Ghirotto, S., Fagundes, N. J., Leal, T. P., ... & Tarazona-Santos, E. (2014). Bayesian inferences suggest that Amazon Yunga Natives diverged from Andeans less than 5000 ybp: implications for South American prehistory. BMC evolutionary biology, 14(1), 1-8.](https://doi.org/10.1186/s12862-014-0174-3)
  * > We found that Amazon Yunga Shimaas have a low genetic diversity that interestingly, is a subset of the higher diversity observed in the Andean Quechuas. In fact, more SNPs are found in the Quechuas than in the Shimaas and all the SNPs found in the Shimaa are shared with the Quechuas (Additional file 1: Table S3). The distribution of the mass probability of the divergence time posterior density (Figure 2 and Table 1) suggests that the Quechua and the Shimaa populations diverged recently: If the point estimate appears implausible, the upper limit of the 90% density interval excludes a divergence older than 5300 years ago. At the time of the split, individuals carrying s ≈ 96% (Figure 3, Table 1) of the effective population size of the ancestral population founded the Quechua population, while only a small fraction (s ≈ 4%) founded the Shimaa. This statement does not necessarily mean that the individuals that founded the Shimaa were around 1/25 of the ancestral population. Instead, in population genetics, the definition of effective population size implies that respect to the ancestral Quechua population, the ancestral Shimaa population behaved as an ideal Wright-Fisher model population that lost diversity due to the action of the genetic drift at a pace around twenty-five times faster. Therefore, the lower effective population of the Shimaa may have resulted from a combination of a certainly much smaller number of individuals together with other factors known to reduce the effective population size, such as a biased sex ratio or a high variance in the number of progeny [26].
  * > Our results indicate that the Shimaa diverged from the Quechua Andean population after the late Pleistocene peopling of South America, and very likely, less than 5300 years ago. The 5000–3000 years period BP was characterized by the development of complex societies in the Andean Region and the Pacific Coast, being a period of major cultural development, when large permanent communities settled, monumental architecture appeared, pottery came into use, and agriculture became the predominant source of food supply (see Figure 2 and its references in Additional file 2). Representative settlements of that time were Kotosh in the Amazon Yunga (Huanuco Region), La Galgada in the Central Andes, and Caral and El Paraiso in the Central Pacific Coast (Figure 2). The initial dispersal of the Arawak, which is the language currently spoken by the Shimaa, also seems to have occurred during this time [29], and some authors (see Figure 2), suggest this linguistic family has originated in the Peruvian Amazon Yunga [30]-[32]. The most plausible scenario compatible with our results is that a small subpopulation (the ancestors of the Shimaa) split from a larger Andean population, moved toward the Peruvian Amazon Yunga and then adapted to the different lifestyle of the Amazon Yunga, incorporating the culture of some of their neighbors and their language (Arawak), but not a substantial amount of their genes.
* [Raghavan M, Skoglund P, et al. (2014) Upper Palaeolithic Siberian genome reveals dual ancestry of Native Americans. Nature. 505(7481): 87-91.](https://dx.doi.org/10.1038%2Fnature12736) &mdash; [Open Access](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4105016/)
  * > The MA-1 mitochondrial genome belongs to haplogroup U, which has also been found at high frequency among Upper Palaeolithic and Mesolithic European hunter-gatherers10–12, and the Y chromosome of MA-1 is basal to modern-day western Eurasians and near the root of most Native American lineages5. Similarly, we find autosomal evidence that MA-1 is basal to modern-day western Eurasians and genetically closely related to modern-day Native Americans, with no close affinity to east Asians. This suggests that populations related to contemporary western Eurasians had a more north-easterly distribution 24,000 years ago than commonly thought. Furthermore, we estimate that 14 to 38% of Native American ancestry may originate through gene flow from this ancient population. This is likely to have occurred after the divergence of Native American ancestors from east Asian ancestors, but before the diversification of Native American populations in the New World.
  * > The genetic affinity between Native Americans and MA-1 could be explained by gene flow after the split between east Asians and Native Americans, either from the MA-1 lineage into Native American ancestors or from Native American ancestors to the ancestors of MA-1. However, MA-1, at approximately 24,000 cal. bp, pre-dates time estimates of the Native American–east Asian population divergence event24,25. This presents little time for the formation of a diverged Native American gene pool that could have contributed ancestry to MA-1, suggesting gene flow from the MA-1 lineage into Native American ancestors. Such gene flow should also be detectable using modern-day western Eurasian populations in place of MA-1. Consistent with this, D-statistic tests estimated from outgroup-ascertained SNP data20 reveal significant evidence (Z == 3) for Middle Eastern, European, central Asian and south Asian populations being closer to Karitiana than to Han Chinese20 (Fig. 3b and Supplementary Information, section 14.5). Similar signals were also observed when we replaced modern-day Han Chinese with data from chromosome 21 from a 40,000-yearold east Asian individual (Tianyuan Cave, China), which has been found to be ancestral to modern-day Asians and Native Americans26 (Supplementary Information, section 14.5). Thus, if the gene flow direction was from Native Americans into western Eurasians it would have had to spread subsequently to European, Middle Eastern, south Asian and central Asian populations, including MA-1 before 24,000 years ago. Moreover, as Native Americans are closer to Han Chinese than to Papuans (Fig. 3c), Native American-related gene flow into the ancestors of MA-1is expected to result in MA-1 also being closer to Han Chinese than to Papuans. However, our results suggest that this is not the case (D (Papuan, Han; Sardinian, MA-1) = 20.002 ± 0.005 (Z = 20.36)), which is compatible with all or almost all of the gene flow being into Native Americans (Supplementary Information, section 14.6). Similar results are obtained when MA-1 is replaced with most modern-day western Eurasian populations, except populations with recent admixture from east Asia (Russian, Adygei and Burusho) and Africa (Middle Eastern populations) (Fig. 3c). The most parsimonious explanation for these results is that Native Americans have mixed origins, resulting from admixture between peoples related to modern-day east Asians and western Eurasians.
* [Hoffecker, J. F., Elias, S. A., & O'Rourke, D. H. (2014). Out of Beringia?. Science, 343(6174), 979-980.](https://doi.org/10.1126/science.1250768)
  * > Based on the distribution of tundra plants around the Bering Strait region, Eric Hultén proposed in the 1930s that the now-submerged plain between Chukotka and Alaska—the Bering land bridge—became a refugium for shrub tundra vegetation during cold periods (1), which include the last glacial maximum (LGM) between ∼28,000 and 18,000 cal BP (calibrated radiocarbon years before the present). Adjoining areas to the west and east supported drier plant communities with a higher percentage of grasses during glacial periods. According to Hultén, when warmer and wetter conditions returned to these areas, the land bridge, which he named Beringia, became a center of dispersal for tundra plants. Now it appears that it also may have been a glacial refugium and postglacial center of dispersal for the people who first settled the Americas.
* [Gravel S, Zakharia F, Moreno-Estrada A, Byrnes JK, Muzzio M, Rodriguez-Flores JL, et al. (2013) Reconstructing Native American Migrations from Whole-Genome and Whole-Exome Data. PLoS Genet 9(12): e1004023. https://doi.org/10.1371/journal.pgen.1004023](https://doi.org/10.1371/journal.pgen.1004023)
* [Maschner, H., & Mason, O. K. (2013). The bow and arrow in northern North America. Evolutionary Anthropology: Issues, News, and Reviews, 22(3), 133-138.](https://doi.org/10.1002/evan.21357)
* [Amorim CEG, Bisso-Machado R, Ramallo V, Bortolini MC, Bonatto SL, Salzano FM, et al. (2013) A Bayesian Approach to Genome/Linguistic Relationships in Native South Americans. PLoS ONE 8(5): e64099. https://doi.org/10.1371/journal.pone.0064099](https://doi.org/10.1371/journal.pone.0064099)
* [Cui Y, Lindo J, Hughes CE, Johnson JW, Hernandez AG, Kemp BM, et al. (2013) Ancient DNA Analysis of Mid-Holocene Individuals from the Northwest Coast of North America Reveals Different Evolutionary Paths for Mitogenomes. PLoS ONE 8(7): e66948.](https://doi.org/10.1371/journal.pone.0066948)
* [Roewer L, Nothnagel M, Gusmão L, Gomes V, González M, Corach D, et al. (2013) Continent-Wide Decoupling of Y-Chromosomal Genetic Variation from Language and Geography in Native South Americans. PLoS Genet 9(4): e1003460.](https://doi.org/10.1371/journal.pgen.1003460)
  * > The way a certain habitat is first colonized by humans creates a primordial pattern of genetic variation that is subsequently attenuated by various demographic processes, including migration, population bottlenecks, fissions and fusions. A popular ramification of this paradigm is that most changes of the original genetic ‘make-up’ of a particular region follow trajectories established by geography and language [1] because, in addition to climatic conditions, the latter are the main conductors of gene flow. As a consequence, the type and degree of correlation observed between the genetic structure of an extant population on the one hand, and its linguistic and geographical structure on the other, should provide valuable information about the history of that population. Dissenting processes such as the adoption of a new language without substantial gene flow into the adopting population, for example, by ‘elite dominance’ are usually conceived as exceptions to the rule [2]. Following this line of arguments, any concordance between genetic, linguistic and geographic data should be indicative of steady settlement, isolation by distance and constant population growth whereas discordances suggest abrupt demographic changes such as major contractions or relocations [3].
    * There is an interesting assumption of either sedentary steady life or sudden abrupt migrations without a thought about the habbits of the House of the Jaguar and the trade networks which stem from that in the context of genetics. Just like how Jaguars form a more coherent genetic struture, South American Y-Chromosomes might form a similar structure.
  * > A plethora of culture anthropologic and population genetic studies have corroborated the above viewpoint for various geographical regions, with Europe providing a most illustrative example. Thus, genetic markers of different time depths, including rapidly mutating short tandem repeats (STRs) and slowly mutating single nucleotide polymorphisms (SNPs), have been employed to trace the settlement of Europe over the last 40,000 years, from the Middle Paleolithic through the Neolithic into historical times [4]–[8]. Studies of Y-chromosomal markers, in particular, revealed a substantial correlation between genetic and geographic as well as linguistic patterns for parts of Europe and Asia that did not however become similarly apparent with mitochondrial or autosomal markers [5], [7], [9]–[13]. Hence, an analysis of molecular variance (AMOVA) of Y-STRs identified clearly distinguishable sub-clusters of western and eastern European Y chromosomes that were largely congruent with the Slavic and Romance language domains [7], [9], [11], and Y-chromosomal genetic discontinuities throughout the continent were found to coincide with linguistic boundaries [7], [14]–[16]. Other in-depth studies in Asia [17]–[20], Africa [21], [22], Melanesia [23]–[27] and globally [28], [29] also suggested that paternal (i.e. Y-chromosomal) but not maternal (i.e. mtDNA) lineage formation was strongly related to language dispersal [30]. These findings notwithstanding, it must be noted that at least some instances of male-mediated gene flow over major linguistic barriers have been inferred as well, for example, in Iberia [31] and in the Balkans [32]. Therefore, the correlation between genetics, language and geography may vary, particularly across a highly differentiated region such as Europe, depending upon the local effective population size and the time-depth of the DNA markers used.
  * > Our study revealed that the extant Y-chromosomal genetic variation among native South America males is lacking any clear structure that could sensibly be related to their continent-wide geographic and linguistic relationships. Notable genetic similarity was only observed over a few hundred kilometers, and only (narrowly defined) language group, but not (broadly defined) language class, was found to explain some of the Y-chromosomal genetic variation observed. Such decoupling contrasts with the situation on other continents, particularly Europe and Asia, where many instances of a strong correlation between genetics on the one hand, and geography and spoken language on the other, have been reported. Random resampling from a pool of European Y chromosomes adopting the same number of sampling sites and the same distribution of the sampling size per site as in the present study yielded consistently higher levels of Y-STR genetic variation explicable by geographic structure. Therefore, the observed decoupling of Y-chromosomal genetic variation from geography in native South America males is unlikely to be due to the particular sampling scheme of our study. Despite a few regional examples of decoupling as, for example, in the Caucasus [84], “parallel linguistic and allele-frequency change” is generally considered to be a rule [85] to which pre-Columbian South America would thus be an exception at continental scale.
  * > The presence of Y-SNP haplogroup C-M217 (C3*) in the northwest of South America, and its concomitant absence from most of North and Central America, are intriguing in view of the high prevalence of this haplogroup in Central, East and Northeast Asia. Given the large population size of native North Americans, it appears unlikely that the early settlers of America carried C3* with them, and that the haplogroup got lost by genetic drift in the north, but not in the south. In fact, the locally confined occurrence of C3* in South America would require migration rates into the ancestral C3* and Q carrier populations that are so low (most likely only 2.5% out of the C3* carrier population) that they are hardly compatible with a long period of joint immigration from Asia. Instead, an independent introduction of C3* into South America appears plausible not the least because it would be consistent with the observed pattern of locally confined Y-STR autocorrelation as well. This view is further supported by the comparatively recent coalescence of the 14 C3* haplotypes from the present study, which appears to have occurred some 200 generations ago, corresponding to 6000 years. The above notwithstanding, inclusion of an isolated Tlingit C3* haplotype found in Alaska prolonged the coalescence time estimate by no more that approximately 40 generations which means that a North American origin of the Ecuadorian C3* haplotypes, albeit less likely prima facie, cannot be ruled out.
  * > In view of the above, two scenarios for the introduction of C3* into Ecuador seem credible: (i) one or more late migratory waves that quickly passed North and Central America without leaving a trace of C3*, and (ii) long-distance contact with East Asia. As regards the second scenario, there appears to be at least some archaeological evidence for a pre-Columbian contact between East Asia and South America [43]. In particular, the similarity of ceramic artifacts found in both regions led to the hypothesis of a trans-Pacific connection between the middle Jōmon culture of Kyushu (Japan) and the littoral Valdivia culture in Ecuador at 4400–3300 BC. In view of the close proximity of the spotty C3* cluster to the Valdivia site, which was considered at the time to represent the earliest pottery in the New World [40], it may well be that C3* was introduced into the northwest of South America from East Asia by sea, either along the American west coast or across the Pacific (with some help by major currents). The considerable differences between the extant Y-STR haplotypes of Ecuadorian and Asian C3* carriers would clearly be explicable in terms of their long divergence time. The differences between C3* chromosomes carried by different ethnic groups in Ecuador, on the other hand, highlight that population splits followed by limited gene flow are characteristic of the genetic structure of South American natives [88].
  * > In summary, our study revealed that the Y-chromosomal genetic variation of South American natives lacks a clear structure that could be related to the continent-wide geographic and linguistic relationship, suggesting a history of rapid peopling and subsequent evolution in small groups. Moreover, it appears unlikely that the South American natives are descendants of a single terrestrial wave of migration. Instead of being confined to a major founding lineage of the Q branch of the human phylogeny, as has been widely held to be the case in the past [60], the continent hosts other Asian haplogroups as well (e.g. those belonging to the basal C3* clade). Further characterization of their distribution is likely to provide new insights into the demographic history of South America.
* [Sandoval JR, Lacerda DR, Jota MSA, Salazar-Granara A, Vieira PPR, Acosta O, et al. (2013) The Genetic History of Indigenous Populations of the Peruvian and Bolivian Altiplano: The Legacy of the Uros. PLoS ONE 8(9): e73006. https://doi.org/10.1371/journal.pone.0073006](https://doi.org/10.1371/journal.pone.0073006)
  * > In conclusion, the genetic evidence obtained through analyses of autochthonous paternal and maternal lineages from the Andean Altiplano showed that Uros from Puno Bay in Peru and Bolivian Uros communities of Uru-Chipaya and Uru-Poopo are heterogeneous groups, bearing genetic lineages derived from divergent ancestors when compared to most of the current Andean populations. Our results also indicate more gene flow with neighboring Andeans (Aymara and Quechua) than with more distant Andean communities or Arawakan speaking populations. Moreover, the results suggest the Uros could be derived from ancestral Andean stocks that were intermingled and partially replaced by lineages arriving with populations expanding due to a more recent farming expansion and posterior establishment of complex civilizations on the Andes.
* [Schurr TG et al. (2012) Clan, Language, and Migration History Has Shaped Genetic Diversity in Haida and Tlingit Populations From Southeast Alaska. American Journal of Physical Anthropology. 148(3): 422-435.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4335652/)
* [de Saint Pierre M, Bravi CM, Motti JMB, Fuku N, Tanaka M, Llop E, et al. (2012) An Alternative Model for the Early Peopling of Southern South America Revealed by Analyses of Three Mitochondrial DNA Haplogroups. PLoS ONE 7(9): e43486. https://doi.org/10.1371/journal.pone.0043486](https://doi.org/10.1371/journal.pone.0043486)
* [Gorostiza A, Acunha-Alonzo V, Regalado-Liu L, Tirado S, Granados J, Sámano D, et al. (2012) Reconstructing the History of Mesoamerican Populations through the Study of the Mitochondrial DNA Control Region. PLoS ONE 7(9): e44666. https://doi.org/10.1371/journal.pone.0044666](https://doi.org/10.1371/journal.pone.0044666)
* [Casas‐Vargas, A., Gómez, A., Briceño, I., Díaz‐Matallana, M., Bernal, J. E., & Rodríguez, J. V. (2011). High genetic diversity on a sample of pre‐Columbian bone remains from Guane territories in northwestern Colombia. American journal of physical anthropology, 146(4), 637-649.](https://www.researchgate.net/publication/51709646_High_genetic_diversity_on_a_sample_of_pre-Columbian_bone_remains_from_Guane_territories_in_northwestern_Colombia)
* [Kemp BM, González-Oliver A, et al. (2010) Evaluating the Farming/Language Dispersal Hypothesis with genetic variation exhibited by populations in the Southwest and Mesoamerica. PNAS. 107(15): 6759-6764.](https://doi.org/10.1073/pnas.0905753107)
* [Rasmussen M, Li Y, et al. (2010) Ancient human genome sequence of an extinct Palaeo-Eskimo. Nature. 463: 757-762.](https://rdcu.be/bOjbe)
* [Achilli A, Perego UA, Bravi CM, Coble MD, Kong Q-P, Woodward SR, et al. (2008) The Phylogeny of the Four Pan-American MtDNA Haplogroups: Implications for Evolutionary and Disease Studies. PLoS ONE 3(3): e1764. https://doi.org/10.1371/journal.pone.0001764](https://doi.org/10.1371/journal.pone.0001764)
* [Melton, Philip Edward (2008) Genetic history and pre-Columbian Diaspora of Chibchan speaking populations: Molecular genetic evidence. Ph.D. thesis, University of Kansas.](https://kuscholarworks.ku.edu/handle/1808/4508?show=full)
* [Goebel T, Waters MR, O'Rourke DH (2008) The Late Pleistocene Dispersal of Modern Humans in the Americas. Science. 319(5869): 1497-1502.](https://science.sciencemag.org/content/319/5869/1497)
* [Tamm E, Kivisild T, Reidla M, Metspalu M, Smith DG, Mulligan CJ, et al. (2007) Beringian Standstill and Spread of Native American Founders. PLoS ONE 2(9): e829. https://doi.org/10.1371/journal.pone.0000829](https://doi.org/10.1371/journal.pone.0000829)
* [Kemp BM et al. (2007), Genetic analysis of early holocene skeletal remains from Alaska and its implications for the settlement of the Americas. Am. J. Phys. Anthropol., 132: 605-621. doi:10.1002/ajpa.20543](https://doi.org/10.1002/ajpa.20543) &mdash; [PDF](https://www.igb.illinois.edu/sites/default/files/Kempetal2007.pdf)
* [Reséndez A, Kemp BM, et al. (2005) "An Analysis of Ancient Aztec mtDNA from Tlatelolco: Pre-Columbian Relations and the Spread of Uto-Aztecan." Biomolecular Archeology: Genetic Approaches to the Past. Center for Archeological Investigations, Ocassional Paper No. 32.](https://migrationcluster.ucdavis.edu/research/national-identity-shifting-border-texas-and-new-mexico-age-transition-1821-1848-0)
* [Malhi RS et al. (2003) Native American mtDNA prehistory in the American Southwest. American Journal of Physical Anthropology. 120(2): 108-124. <http://hdl.handle.net/2027.42/34277>](http://hdl.handle.net/2027.42/34277)
* [Hurles, M. E., Matisoo-Smith, E., Gray, R. D., & Penny, D. (2003). Untangling Oceanic settlement: the edge of the knowable. Trends in Ecology & Evolution, 18(10), 531-540.](https://doi.org/10.1016/S0169-5347(03)00245-3)
* [Torroni A, Schurr TG, et al. (1993) Asian affinities and continental radiation of the four founding Native American mtDNAs. American Journal of Human Genetics. American Journal of Human Genetics. 53(3): 563-590.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1682412/)
* [Torroni A, Sukernik RI, Schurr TG, et al. (1993). mtDNA variation of aboriginal Siberians reveals distinct genetic affinities with Native Americans. American journal of human genetics, 53(3), 591-608.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1682407/)
* [Shields GF, Schmiechen AM, Frazier BL, Redd A Voevoda MI, Reed JK, Ward RH (1993) mtDNA sequences suggest a recent evolutionary divergence for Beringian and northern North American populations. American Journal of Human Genetics. 53(3): 549-562.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1682422/)
* [Ward RH, Redd A, Valencia D, Pääbo S (1993) Genetic and linguistic differentiation in the Americas. PNAS. 90(22): 10663-10667.](https://dx.doi.org/10.1073%2Fpnas.90.22.10663)
* [Ward RH, Frazier BL, Dew-Jager K, Pääbo S (1991) Extensive mitochondrial diversity within a single Amerindian tribe. PNAS. 88(19): 8720-8724.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC52581/)
* [Schurr TG et al. (1990) Amerindian mitochondrial DNAs have rare Asian mutations at high frequencies, suggesting they derived from four primary maternal lineages. American Journal of Human Genetics. 46(3): 613-623.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1683611/)
* [Genetics and the History of Latin America](https://migrationcluster.ucdavis.edu/research/genetics-and-history-latin-america)
* [Genetics, Linguistics, and Prehistoric Migrations: An Analysis of California Indian Mitochondrial DNA Lineages](https://escholarship.org/uc/item/6g135010)
* [Genome-wide insights into the genetic history of human populations](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4381409/)
* [Genomic insights on the ethno-history of the Maya and the ‘Ladinos’ from Guatemala](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4422311/)
* [High-resolution mitochondrial DNA analysis sheds light on human diversity, cultural interactions, and population mobility in Northwestern Amazonia](https://www.researchgate.net/publication/320654352_High-resolution_mitochondrial_DNA_analysis_sheds_light_on_human_diversity_cultural_interactions_and_population_mobility_in_Northwestern_Amazonia)
* [HLA Class II Alleles in the Otomi Population of the Mezquital Valley: A Genetic Approach to the History of Interethnic Migrations in the Mexican Central Plateau](https://www.researchgate.net/publication/274396125_HLA_Class_II_Alleles_in_the_Otomi_Population_of_the_Mezquital_Valley_A_Genetic_Approach_to_the_History_of_Interethnic_Migrations_in_the_Mexican_Central_Plateau)
* [Large scale mitochondrial sequencing in Mexican Americans suggests a reappraisal of Native American origins](https://doi.org/10.1186/1471-2148-11-293)
* [Mitochondrial DNA and Prehistoric Settlements: Native Migrations on the Western Edge of North America](https://www.researchgate.net/publication/8485114_Mitochondrial_DNA_and_Prehistoric_Settlements_Native_Migrations_on_the_Western_Edge_of_North_America)
* [Mitochondrial DNA Studies of Native Americans: Conceptions and Misconceptions of the Population Prehistory of the Americas](https://pdfs.semanticscholar.org/a40f/fbea9601cea8b7c973a8cedbea297a6de20a.pdf)
* [Mitochondrial DNA Diversity in Indigenous Populations of the Southern Extent of Siberia, and the Origins of Native American Haplogroups](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3905771/)
* [Mitochondrial Population Genomics Supports a Single Pre-Clovis Origin with a Coastal Route for the Peopling of the Americas](https://doi.org/10.1016/j.ajhg.2007.11.013)
* [mtDNA haplogroup X: An ancient link between Europe/Western Asia and North America?](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1377656/)
* [mtDNA Variation among Greenland Eskimos: The Edge of the Beringian Expansion.](https://doi.org/10.1086/303038)
* [Origins and genetic legacies of the Caribbean Taino](https://doi.org/10.1073/pnas.1716839115)
* [Patterns of mtDNA Diversity in Northwestern North America](https://www.researchgate.net/publication/8485113_Patterns_of_mtDNA_Diversity_in_Northwestern_North_America)
* [The Genetic History Of The Otomi In The Central Mexican Valley](https://repository.upenn.edu/anthro_seniortheses/133/)
* [The Genetics of Mexico Recapitulates Native American Substructure and Affects Biomedical Traits](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4156478/)

## Europe

* [Aragón, E., Montero-Ruiz, I., Polzer, M. E., & van Duivenvoorde, W. (2022). Shipping metal: Characterisation and provenance study of the copper ingots from the Rochelongue underwater site (Seventh–Sixth century BC), West Languedoc, France. Journal of Archaeological Science: Reports, 41, 103286.](https://doi.org/10.1016/j.jasrep.2021.103286)
* [Flynn, P. (2021) Oldest ink pen in Ireland is discovered by archaeologist during excavation in Burren. Independent House. 9 December.](https://www.independent.ie/irish-news/news/oldest-ink-pen-in-ireland-is-discovered-by-archaeologist-during-excavation-in-burren-41132749.html)
* [Toolis, R. (2021). Shifting perspectives on 1st-millennia Scotland. Proceedings of the Society of Antiquaries of Scotland, 150, 247-278.](https://doi.org/10.9750/PSAS.150.1316)
* [Durangit - The Prehistoric Heart of Galloway](https://www.guard-archaeology.co.uk/DunragitBlog/Dunragit_monograph.pdf)
* [Patterson, N., Isakov, M., Booth, T., Büster, L., Fischer, C. E., Olalde, I., ... & Mihovilić, K. (2021). Large-scale migration into Britain during the Middle to Late Bronze Age. Nature, 1-14.](https://doi.org/10.1038/s41586-021-04287-4)
  * > Present-day people from England and Wales have more ancestry derived from early European farmers (EEF) than did people of the Early Bronze Age1. To understand this, here we generated genome-wide data from 793 individuals, increasing data from the Middle to the Late Bronze Age and Iron Age in Britain by 12-fold, and western and central Europe by 3.5-fold. Between 1000 and 875 BC, EEF ancestry increased in southern Britain (England and Wales) but not northern Britain (Scotland) due to incorporation of migrants who arrived at this time and over previous centuries, and who were genetically most similar to ancient individuals from France. These migrants contributed about half the ancestry of people of England and Wales from the Iron Age, thereby creating a plausible vector for the spread of early Celtic languages into Britain. These patterns are part of a broader trend of EEF ancestry becoming more similar across central and western Europe in the Middle to the Late Bronze Age, coincident with archaeological evidence of intensified cultural exchange2,3,4,5,6. There was comparatively less gene flow from continental Europe during the Iron Age, and the independent genetic trajectory in Britain is also reflected in the rise of the allele conferring lactase persistence to approximately 50% by this time compared to approximately 7% in central Europe where it rose rapidly in frequency only a millennium later. This suggests that dairy products were used in qualitatively different ways in Britain and in central Europe over this period.
* [Villalba-Mouco, V., Oliart, C., Rihuete-Herrada, C., Childebayeva, A., Rohrlach, A. B., Fregeiro, M. I., ... & Haak, W. (2021). Genomic transformation and social organization during the Copper Age–Bronze Age transition in southern Iberia. Science advances, 7(47), eabi7038.](https://doi.org/10.1126/sciadv.abi7038)
* [Lull, V., Rihuete-Herrada, C., Risch, R., Bonora, B., Celdrán-Beltrán, E., Fregeiro, M. I., ... & Micó, R. (2021). Emblems and spaces of power during the Argaric Bronze Age at La Almoloya, Murcia. Antiquity, 95(380), 329-348.](https://doi.org/10.15184/aqy.2021.8)
* [Talamo, S., Urbanowski, M., Picin, A., Nowaczewska, W., Vazzana, A., Binkowski, M., ... & Hublin, J. J. (2021). A 41,500 year-old decorated ivory pendant from Stajnia Cave (Poland). Scientific reports, 11(1), 1-11.](https://doi.org/10.1038/s41598-021-01221-6)
* [Rivollat, M., Jeong, C., Schiffels, S., Küçükkalıpçı, I., Pemonge, M. H., Rohrlach, A. B., ... & Haak, W. (2020). Ancient genome-wide DNA from France highlights the complexity of interactions between Mesolithic hunter-gatherers and Neolithic farmers. Science Advances, 6(22), eaaz5344.](https://doi.org/10.1126/sciadv.aaz5344)
* [Sano, K., Arrighi, S., Stani, C., Aureli, D., Boschin, F., Fiore, I., ... & Benazzi, S. (2019). The earliest evidence for mechanically delivered projectile weapons in Europe. Nature ecology & evolution, 3(10), 1409-1414.](https://doi.org/10.1038/s41559-019-0990-3)
  * > Microscopic analysis of backed lithic pieces from the Uluzzian technocomplex (45–40 thousand yr ago) at Grotta del Cavallo (southern Italy) reveals their use as mechanically delivered projectile weapons, attributed to anatomically modern humans. Use-wear and residue analyses indicate that the lithics were hunting armatures hafted with complex adhesives, while experimental and ethnographic comparisons support their use as projectiles. The use of projectiles conferred a hunting strategy with a higher impact energy and a potential subsistence advantage over other populations and species.
* [Blakemore, E. (2019) Artificial islands older than Stonehenge stump scientists. National Geographic. 12 June.](https://www.nationalgeographic.co.uk/culture-history/2019/06/artificial-islands-older-stonehenge-stump-scientists?utm_source=Facebook&utm_medium=Social&utm_campaign=JARVIS&fbclid=IwAR0L-pT2MB_byXT04YwHQ2cxZJB5N6fjLWiJA83Np4T1Ims--fOaf-eaGSE)
* [O’Sullivan, N., Posth, C., Coia, V., Schuenemann, V. J., Price, T. D., Wahl, J., ... & Maixner, F. (2018). Ancient genome-wide analyses infer kinship structure in an Early Medieval Alemannic graveyard. Science advances, 4(9), eaao1262.](https://doi.org/10.1126/sciadv.aao1262)
* [Ebenesersdóttir, S. S., Sandoval-Velasco, M., Gunnarsdóttir, E. D., Jagadeesan, A., Guðmundsdóttir, V. B., Thordardóttir, E. L., ... & Helgason, A. (2018). Ancient genomes from Iceland reveal the making of a human population. Science, 360(6392), 1028-1032.](https://doi.org/10.1126/science.aar2625)
* [Gibbons, A. (2017). Thousands of horsemen may have swept into Bronze Age Europe, transforming the local population. Science. 21 February.](https://www.sciencemag.org/news/2017/02/thousands-horsemen-may-have-swept-bronze-age-europe-transforming-local-population)
* [Little, A., van Gijn, A., Collins, T., Cooney, G., Elliott, B., Gilhooly, B., ... & Warren, G. (2017). Stone dead: uncovering early Mesolithic mortuary rites, Hermitage, Ireland. Cambridge Archaeological Journal, 27(2), 223-243.](https://doi.org/10.1017/S0959774316000536)
* [Fu, Q., Posth, C., Hajdinjak, M., Petr, M., Mallick, S., Fernandes, D., ... & Reich, D. (2016). The genetic history of ice age Europe. Nature, 534(7606), 200-205.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4943878/)
* [Balter, M., & Gibbons, A. (2015). Indo-European languages tied to herders. Science, 347(6224), 814-815.](https://doi.org/10.1126/science.347.6224.814)
* [Seguin-Orlando, A., Korneliussen, T. S., Sikora, M., Malaspinas, A. S., Manica, A., Moltke, I., ... & Willerslev, E. (2014). Genomic structure in Europeans dating back at least 36,200 years. Science, 346(6213), 1113-1118.](https://doi.org/10.1126/science.aaa0114)
* [Gibbons, A. (2014). Three-part ancestry for Europeans. Science, 345(6201), 1106-1107.](https://doi.org/10.1126/science.345.6201.1106)
* [Brandt, G., Haak, W., Adler, C. J., Roth, C., Szécsényi-Nagy, A., Karimnia, S., ... & Genographic Consortium. (2013). Ancient DNA reveals key stages in the formation of central European mitochondrial genetic diversity. Science, 342(6155), 257-261.](https://doi.org/10.1126/science.1241844)
  * > The processes that shaped modern European mitochondrial DNA (mtDNA) variation remain unclear. The initial peopling by Palaeolithic hunter-gatherers ~42,000 years ago and the immigration of Neolithic farmers into Europe ~8000 years ago appear to have played important roles but do not explain present-day mtDNA diversity. We generated mtDNA profiles of 364 individuals from prehistoric cultures in Central Europe to perform a chronological study, spanning the Early Neolithic to the Early Bronze Age (5500 to 1550 calibrated years before the common era). We used this transect through time to identify four marked shifts in genetic composition during the Neolithic period, revealing a key role for Late Neolithic cultures in shaping modern Central European genetic diversity.
* [Ralph P, Coop G (2013) The Geography of Recent Genetic Ancestry across Europe. PLoS Biol 11(5): e1001555.](https://doi.org/10.1371/journal.pbio.1001555)
* [Skoglund, P., Malmström, H., Raghavan, M., Storå, J., Hall, P., Willerslev, E., ... & Jakobsson, M. (2012). Origins and genetic legacy of Neolithic farmers and hunter-gatherers in Europe. Science, 336(6080), 466-469.](https://doi.org/10.1126/science.1216304)
* [Bramanti, B., Thomas, M. G., Haak, W., Unterländer, M., Jores, P., Tambets, K., ... & Burger, J. (2009). Genetic discontinuity between local hunter-gatherers and central Europe’s first farmers. science, 326(5949), 137-140.](https://doi.org/10.1126/science.1176869)
* [Ammerman, A. J., Pinhasi, R., & Bánffy, E. (2006). Comment on" Ancient DNA from the first European farmers in 7500-year-old Neolithic sites". Science, 312(5782), 1875-1875.](https://doi.org/10.1126/science.1123936)
* [Haak, W., Forster, P., Bramanti, B., Matsumura, S., Brandt, G., Tanzer, M., ... & Burger, J. (2005). Ancient DNA from the first European farmers in 7500-year-old Neolithic sites. Science, 310(5750), 1016-1018.](https://doi.org/10.1126/science.1118725)
* [Gibbons, A. (2000). Europeans trace ancestry to Paleolithic people. Science, 290(5494), 1080-1081.](https://doi.org/10.1126/science.290.5494.1080)

## Doggerland

* [Curry, A. (2020). Europe's lost frontier. Science. 30 January.](https://www.science.org/content/article/relics-washed-beaches-reveal-lost-world-beneath-north-sea)

## Eurasia

* [Massilani, D., Skov, L., Hajdinjak, M., Gunchinsuren, B., Tseveendorj, D., Yi, S., ... & Pääbo, S. (2020). Denisovan ancestry and population history of early East Asians. Science, 370(6516), 579-583.](https://doi.org/10.1126/science.abc1166)
  * > We present analyses of the genome of a ~34,000-year-old hominin skull cap discovered in the Salkhit Valley in northeastern Mongolia. We show that this individual was a female member of a modern human population that, following the split between East and West Eurasians, experienced substantial gene flow from West Eurasians. Both she and a 40,000-year-old individual from Tianyuan outside Beijing carried genomic segments of Denisovan ancestry. These segments derive from the same Denisovan admixture event(s) that contributed to present-day mainland Asians but are distinct from the Denisovan DNA segments in present-day Papuans and Aboriginal Australians.
  * > Modern humans may have been present in East Asia as early as 80,000 years before the present (BP) (1, 2), but how they eventually settled in the region remains largely unknown (3–5). To date, genomic data from the first half of the Upper Paleolithic in East Asia exist only for a single human, a ~40,000-year-old individual from Tianyuan Cave in the Beijing area in China (6). This individual was more closely related to present-day East Asians than to ancient Europeans but, surprisingly, shared more alleles with a ~35,000-year-old individual from Belgium (Goyet Q116-1) than with other ancient Europeans (7). In Siberia, which neighbors East Asia to the north, four modern human individuals older than 20,000 years BP have been studied: a ~45,000-year-old individual from Ust’-Ishim in West Siberia who did not contribute ancestry to present-day populations (8); a ~24,000-year-old individual from Mal’ta 1 in South Central Siberia who was more related to Western Europeans than to East Asians and was part of a population that contributed approximatively one-third of the ancestry of present-day Native Americans (9); and two ~31,000-year-old individuals from the Yana Rhinoceros Horn Site in northeastern Siberia who show affinities to early modern humans in both West and East Eurasia (10).
  * > We inferred the extent of genetic similarity [using “outgroup” f3 statistics and D statistics (25)] between the Salkhit individual, modern human individuals older than 20,000 years (table S1), and 131 present-day populations (19). The Salkhit individual, similar to the ~40,000-year-old Tianyuan individual from China, is more related to present-day East Eurasians and Native Americans than to West Eurasians (Fig. 1B, table S10, and fig. S10). Both early East Asians are equally related to most present-day East Eurasians and Native Americans (Fig. 1C and table S12) but differ in their affinity to West Eurasians; present-day West Eurasians share more alleles with the Salkhit individual than with the Tianyuan individual (Fig. 1C). Additionally, the Salkhit individual shares as many alleles with the Tianyuan individual as with the ~31,000-year-old Yana individuals from northeastern Siberia (tables S12 and S14), yet the Tianyuan and Yana individuals share fewer alleles with each other than with the Salkhit individual (Fig. 1C and tables S12 to S14). Those observations suggest that gene flow occurred between populations ancestral to the Salkhit individual and the Yana individuals before ~34,000 years BP—that is, between early populations in East Asia and in Siberia following the divergence of East and West Eurasians. The ~35,000-year-old Goyet Q116-1 individual from Belgium shares more alleles with the Salkhit and Tianyuan individuals (7) than do other Europeans analyzed to date (tables S15 and S16). The fact that the Salkhit individual shares even more alleles with the Goyet Q116-1 individual than does the Tianyuan individual (Fig. 1C) is probably due to gene flow bringing West Eurasian ancestry into the ancestors of the Salkhit individual.
  * > Population admixture models that are compatible with genomic data from modern human individuals older than 20,000 years were evaluated using qpGraph (25) (Fig. 2 and fig. S14). Our models suggest that the Tianyuan individual and the ~37,000-year-old Kostenki14 individual from Russia are unadmixed representatives of early East and West Eurasian populations, respectively. The Salkhit individual, who lived in Mongolia about 6000 years after the Tianyuan individual, carries ~75% of its ancestry from a Tianyuan-related East Eurasian population and the remaining ~25% from a population related to the Siberian Yana individuals, who lived some 3000 years later than the Salkhit individual. In agreement with previous results (10), the Yana individuals are estimated to have about one-third of their ancestry from early East Eurasians and the remaining two-thirds from the early West Eurasians. Their relationship to the Salkhit individual is complex: Models without bidirectional gene flow between an East Asian population ancestral to the Salkhit and a population related to the Yana individuals do not fit the data (fig. S14). Thus, some time before 34,000 years ago, gene flow from West to East Eurasia occurred, probably mediated by ancestors of the early colonizers of Siberia as represented by the Yana individuals.
  * > In addition to the Neanderthal ancestry, present-day individuals in East Asia carry ancestry from Denisovans, although in mainland Asia the amount of Denisovan ancestry in present-day populations is less than the amount of Neanderthal ancestry by a factor of 10 (28–30). This has hitherto made it impossible to determine whether ancient genomes from Asia, which are of lower quality than present-day genomes, carry Denisovan ancestry. We applied a novel hidden Markov approach (31) that is able to identify introgressed Neanderthal and Denisovan genomic segments in low-coverage ancient genomes; this method uses a genotype likelihood approach that incorporates contamination, so that all fragments can be used for this analysis. Using data from ~1.7 million SNPs where Neanderthal and/or Denisovan genomes differ from present-day African genomes, we detected 18 segments of Denisovan ancestry longer than 0.2 cM in the Salkhit genome (Fig. 3, table S18, and figs. S17 and S28) and 20 such segments in the Tianyuan genome (table S18 and figs. S19 and S28). We detected about one-third as many segments of Denisovan DNA in the genomes of the ancient Siberians Yana 1, Yana 2, and Mal’ta 1 (table S18, figs. S19 to S21, and fig. S28), consistent with the proportion of their East Asian ancestry. In contrast, no Denisovan ancestry was detected in the genome of the ~45,000-year-old Siberian individual from Ust’Ishim in West Siberia, nor in any European individual older than 20,000 years (table S18, figs. S22 to S25, and fig. S28). Thus, the Salkhit and Tianyuan genomes provide direct evidence that ancestors of modern humans who lived in East Asia 40,000 years ago had met and mixed with Denisovans. The small number of these segments does not provide enough power to date the introgression event. However, given their relatively short length (≤1.3 cM), the Denisovan introgression is likely to have happened at least 10,000 years before these individuals lived.
  * > We compared the Denisovan segments in the Salkhit and Tianyuan genomes to those in present-day people to estimate whether introgressed segments between genomes overlap more often than expected by chance. Significance was assessed using 500 bootstrap reshuffles, where segments are randomly relocated across the analyzed genomes (supplementary text). The Denisovan DNA segments in the ancient East Asian genomes overlap more than expected with Denisovan segments detected in the genomes of several present-day populations in Asia and in populations with some Asian ancestry, such as Hawaiians (Fig. 4, table S19, and figs. S29 to S31). In contrast, we find no significant overlap with Denisovan segments detected in Papuans or Aboriginal Australians, although these groups carry on the order of 20 times as much Denisovan DNA as mainland Asians.
* [Narasimhan, V. M., Patterson, N., Moorjani, P., Rohland, N., Bernardos, R., Mallick, S., ... & Reich, D. (2019). The formation of human populations in South and Central Asia. Science, 365(6457).](https://doi.org/10.1126/science.aat7487)
  * > By sequencing 523 ancient humans, we show that the primary source of ancestry in modern South Asians is a prehistoric genetic gradient between people related to early hunter-gatherers of Iran and Southeast Asia. After the Indus Valley Civilization’s decline, its people mixed with individuals in the southeast to form one of the two main ancestral populations of South Asia, whose direct descendants live in southern India. Simultaneously, they mixed with descendants of Steppe pastoralists who, starting around 4000 years ago, spread via Central Asia to form the other main ancestral population. The Steppe ancestry in South Asia has the same profile as that in Bronze Age Eastern Europe, tracking a movement of people that affected both regions and that likely spread the distinctive features shared between Indo-Iranian and Balto-Slavic languages.
  * > In the Late Bronze Age and Iron Age individuals of the Swat Valley, we detect a significantly lower proportion of Steppe admixture on the Y chromosome (only 5% of the 44 Y chromosomes of the R1a-Z93 subtype that occurs at 100% frequency in the Central_Steppe_MLBA males) compared with ~20% on the autosomes (Z = −3.9 for a deficiency from males under the simplifying assumption that all the Y chromosomes are unrelated to each other since admixture and thus are statistically independent), documenting how Steppe ancestry was incorporated into these groups largely through females (Fig. 4). However, sex bias varied in different parts of South Asia, as in present-day South Asians we observe a reverse pattern of excess Central_Steppe_MLBA–related ancestry on the Y chromosome compared with the autosomes (Z = 2.7 for an excess from males) (13, 56) (Fig. 4). Thus, the introduction of lineages from Steppe pastoralists into the ancestors of present-day South Asians was mediated mostly by males. This bias is similar in direction to what has been documented for the introduction of Steppe ancestry into Iberia in far western Europe, although it is less extreme than the bias reported in that case (57).
  * Has an interesting admixture tree.
* [de Barros Damgaard, P., Martiniano, R., Kamm, J., Moreno-Mayar, J. V., Kroonen, G., Peyrot, M., ... & Willerslev, E. (2018). The first horse herders and the impact of early Bronze Age steppe expansions into Asia. Science, 360(6396).](https://doi.org/10.1126/science.aar7711)
  * > We analyzed whole-genome sequence data of 74 ancient humans (14, 15) (tables S1 to S3) ranging from the Mesolithic (~9000 BCE) to Medieval times, spanning ~5000 km across Eastern Europe, Central Asia, and Western Asia (Anatolia) (Fig. 1). Our genome data includes 3 Copper Age individuals (~3500 to 3300 BCE) from Botai in northern Kazakhstan (Botai_CA; 13.6X, 3.7X, and 3X coverage, respectively); 1 Early Bronze Age (~2900 BCE) Yamnaya sample from Karagash, Kazakhstan (16) (YamnayaKaragash_EBA; 25.2X); 1 Mesolithic (~9000 BCE) EHG from Sidelkino, Russia (SidelkinoEHG_ML; 2.9X); 2 Early/Middle Bronze Age (~2200 BCE) central steppe individuals (~4200 BP) (CentralSteppe_EMBA; 4.5X and 9.1X average coverage, respectively) from burials at Sholpan and Gregorievka that display cultural similarities to Yamnaya and Afanasievo (12); 19 individuals of the Bronze Age (~2500 to 2000 BCE) Okunevo culture of the Minusinsk Basin in the Altai region (Okunevo_EMBA; ~1X average coverage; 0.1 to 4.6X); 31 Baikal hunter-gatherer genomes (~1X average coverage; 0.2 to 4.5X) from the cis-Baikal region bordering on Mongolia and ranging in time from the Early Neolithic (~5200 to 4200 BCE; Baikal_EN) to the Early Bronze Age (~2200 to 1800 BCE; Baikal_EBA); 4 Copper Age individuals (~3300 to 3200 BCE; Namazga_CA; ~1X average coverage; 0.1 to 2.2X) from Kara-Depe and Geoksur in the Kopet Dag piedmont strip of Turkmenistan, affiliated with the period III cultural layers at Namazga-Depe (fig. S1), plus 1 Iron Age individual (Turkmenistan_IA; 2.5X) from Takhirbai in the same area dated to ~800 BCE; and 12 individuals from Central Turkey (figs. S2 to S4), spanning from the Early Bronze Age (~2200 BCE; Anatolia_EBA) to the Iron Age (~600 BCE; Anatolia_IA), and including 5 individuals from presumed Hittite-speaking settlements (~1600 BCE; Anatolia_MLBA), and 2 individuals dated to the Ottoman Empire (1500 CE; Anatolia_Ottoman; 0.3 to 0.9X). All the population labels including those referring to previously published ancient samples are listed in table S4 for contextualization. Additionally, we sequenced 41 high-coverage (30X) present-day Central Asian genomes, representing 17 self-declared ethnicities (fig. S5), and collected and genotyped 140 individuals from five IE-speaking populations in northern Pakistan.
  * > To investigate ancestral relationships between these populations, we used coalescent modeling with the momi (Moran Models for Inference) program (28) (Fig. 4, figs. S16 to S22, and tables S6 to S11). This exploits the full joint-site frequency spectrum and can separate genetic drift into divergence-time and population-size components, in comparison to PCA, admixture, and qpAdm approaches, which are based on pairwise covariances. We find that Botai_CA, CentralSteppe_EMBA, Okunevo_EMBA, and Baikal populations are deeply separated from other ancient and present-day populations and are best modeled as mixtures in different proportions of ANE ancestry and an Ancient East Asian (AEA) ancestry component represented by Baikal_EN, with mixing times dated to ~5000 BCE. Although some modern Siberian samples lie under the Baikal samples in Fig. 2A, these are separated out in a more limited PCA, involving just those populations and the ancient samples (fig. S23). Our momi model infers that the ANE lineage separated ~15,000 years ago in the Upper Paleolithic from the EHG lineage to the west, with no independent drift assigned to MA1. This suggests that MA1 may represent their common ancestor. Similarly, the AEA lineage to the east also separated ~15,000 years ago, with the component that leads to Baikal_EN and the AEA component of the steppe separating from the lineage leading to present-day East Asian populations represented by Han Chinese (figs. S19 to S21). The ANE and AEA lineages themselves are estimated as having separated approximately 40,000 years ago, relatively soon after the peopling of Eurasia by modern humans.
  * > The deep population structure among the local populations in Inner Asia around the Copper Age/Bronze Age transition is in line with distinct origins of central steppe hunter-herders related to Botai of the central steppe and those related to Altaian hunter-gatherers of the eastern steppe (30). Furthermore, this population structure, which is best described as part of the ANE metapopulation, persisted within Inner Asia from the Upper Paleolithic to the end of the Early Bronze Age. In the Baikal region, the results show that at least two genetic shifts occurred: first, a complete population replacement of the Upper Paleolithic hunter-gatherers belonging to the ANE by Early Neolithic communities of Ancient East Asian ancestry, and second, an admixture event between the latter and additional members of the ANE clade, occurring during the 1500-year period that separates the Neolithic from the Early Bronze Age. These genetic shifts complement previously observed severe cultural changes in the Baikal region (18–22).
  * > Our results indicate that the early spread of IE languages into Anatolia was not associated with any large-scale steppe-related migration, as previously suggested (62). Additionally, and in agreement with the later historical record of the region (63), we find no correlation between genetic ancestry and exclusive ethnic or political identities among the populations of Bronze Age Central Anatolia, as has previously been hypothesized (64).
  * > Thus, while the steppe hypothesis, in the light of ancient genomics, has so far successfully explained the origin and dispersal of IE languages and culture in Europe, we find that several elements must be reinterpreted to account for Asia. First, we show that the earliest unambiguous example of horse herding emerged among hunter-gatherers, who had no substantial genetic interaction with western steppe herders. Second, we demonstrate that the Anatolian IE language branch, including Hittite, did not derive from a substantial steppe migration into Anatolia. And third, we conclude that Early Bronze Age steppe pastoralists did not migrate into South Asia but that genetic evidence fits better with the Indo-Iranian IE languages being brought to the region by descendants of Late Bronze Age steppe pastoralists.
* [Krzewińska, M., Kılınç, G. M., Juras, A., Koptekin, D., Chyleński, M., Nikitin, A. G., ... & Götherström, A. (2018). Ancient genomes suggest the eastern Pontic-Caspian steppe as the source of western Iron Age nomads. Science advances, 4(10), eaat4457.](https://doi.org/10.1126/sciadv.aat4457)
  * > The genomic structure of the Bronze and Iron Age (1800 BCE–400 CE) populations in the Pontic-Caspian steppe has not been fully resolved. While earlier genomic studies have suggested close links between the Srubnaya and the central European Late Neolithic and Bronze Age populations (9), our knowledge of the genetic origins of the Cimmerians is limited. Genetic analyses of maternal lineages of Scythians suggest a mixed origin and an east-west admixture gradient across the Eurasian steppe (10–12). The genomics of two early Scythian Aldy-Bel individuals (13) showed genetic affinities to eastern populations of Central Asia (12). However, population interactions and the origin of Scythians of the Pontic-Caspian steppe remain poorly understood. Similarly, little is known about the origins and genetic affinities of the Sarmatians. Genomic studies suggest that the latter group may have been genetically similar to the eastern Yamnaya and Poltavka Bronze Age groups (12). To investigate the demographic dynamics in the Pontic-Caspian steppe, we generated and analyzed genomes of the Late Bronze and Iron Age individuals from the region (Fig. 1, A and B).
  * > The Bronze Age Srubnaya-Alakulskaya individuals from Kazburun 1/Muradym 8 presented genetic similarities to the previously published Srubnaya individuals. However, in f4 statistics, they shared more drift with representatives of the Andronovo and Afanasievo populations compared to the published Srubnaya individuals. Those apparently West Eurasian people lacked significant Siberian components (NEA and SEA) in ADMIXTURE analyses but carried traces of the SA component that could represent an earlier connection to ancient Bactria. The presence of an SA component (as well as finding of metals imported from Tien Shan Mountains in Muradym 8) could therefore reflect a connection to the complex networks of the nomadic transmigration patterns characteristic of seasonal steppe population movements [see (2), figure 6.1, p. 205]. These movements, although dictated by the needs of the nomads and their animals, shaped the economic and social networks linking the outskirts of the steppe and facilitated the flow of goods between settled, semi-nomadic, and nomadic peoples. In contrast, all Cimmerians carried the Siberian genetic component. Both the PCA and f4 statistics supported their closer affinities to the Bronze Age western Siberian populations (including Karasuk) than to Srubnaya. It is noteworthy that the oldest of the Cimmerians studied here (cim357) carried almost equal proportions of Asian and West Eurasian components, resembling the Pazyryks, Aldy-Bel, and Iron Age individuals from Russia and Kazakhstan (12). The second oldest Cimmerian (cim358) was also the only one with both uniparental markers pointing toward East Asia. The Q1* Y chromosome sublineage of Q-M242 is widespread among Asians and Native Americans and is thought to have originated in the Altai Mountains (24). It has previously been identified in numerous ancient samples from Siberia, the Americas, and in representatives of the Siberian Bronze Age and nomadic populations (4, 24). This is the first indication that Cimmerians did not originate in the PCS region but were nomads tracing their origin to the Far East.
  * > In conclusion, our genomic analyses revealed that both the Bronze Age and Iron Age were highly dynamic periods in the Pontic-Caspian steppe. The time span between 1800 BCE and 400 CE was characterized by mobility, population movements, and replacements, which shaped the complex demography of the region through time. Our results showed that the Western Eurasian steppe nomads were not direct descendants of the Bronze Age Srubnaya-Alakulskaya individuals but shared elements of common ancestry with contribution from different peoples. The early nomads could thus be referred to as a “cultural and chronological horizon” represented by various cultures of the Scythian-Siberian world that was not composed of a genetically homogeneous and/or isolated group. Quite the contrary is observed. We observe little evidence of mobility from the Far East, suggesting that the main source of most Western nomads is likely found in eastern Pontic-Caspian steppe and southern Urals. Thus, we propose that the region, similar to the so-called Mongolian steppe generator of peoples during the Middle Ages, served as the generator of the west nomadic peoples that sustained the western nomadic horizon in the Iron Age.
* [Siska, V., Jones, E. R., Jeon, S., Bhak, Y., Kim, H. M., Cho, Y. S., ... & Manica, A. (2017). Genome-wide data from two early Neolithic East Asian individuals dating to 7700 years ago. Science advances, 3(2), e1601877.](https://doi.org/10.1126/sciadv.1601877)
  * > These results suggest a relatively high degree of continuity in this region; the Ulchi are likely descendants of Devil’s Gate (or a population genetically very close to it), but the geographic and genetic connectivity among populations in the region means that this modern population also shows increased association with related modern populations. Compared to Europe, these results suggest a higher level of genetic continuity in northern East Asia over the last ~7.7 thousand years (ky), without any major population turnover since the early Neolithic.
  * > By analyzing genome-wide data from two early Neolithic East Asians from Devil’s Gate, in the Russian Far East, we could demonstrate a high level of genetic continuity in the region over at least the last 7700 years. The cold climatic conditions in this area, where modern populations still rely on a number of hunter-gatherer-fisher practices, likely provide an explanation for the apparent continuity and lack of major genetic turnover by exogenous farming populations, as has been documented in the case of southeast and central Europe. Thus, it seems plausible that the local hunter-gatherers progressively added food-producing practices to their original lifestyle. However, it is interesting to note that in Europe, even at very high latitudes, where similar subsistence practices were still important until very recent times, the Neolithic expansion left a significant genetic signature, albeit attenuated in modern populations, compared to the southern part of the continent. Our ancient genomes thus provide evidence for a qualitatively different population history during the Neolithic transition in East Asia compared to western Eurasia, suggesting stronger genetic continuity in the former region. These results encourage further study of the East Asian Neolithic, which would greatly benefit from genetic data from early agriculturalists (ideally, from areas near the origin of wet rice cultivation in southern East Asia), as well as higher-coverage hunter-gatherer samples from different regions to quantify population structure before intensive agriculture.

## Asia
* [Zhang, H., Cheng, H., Sinha, A., Spötl, C., Cai, Y., Liu, B., ... & Edwards, R. L. (2021). Collapse of the Liangzhu and other Neolithic cultures in the lower Yangtze region in response to climate change. Science advances, 7(48), eabi9275.](https://doi.org/10.1126/sciadv.abi9275)
* [Zhang, F., Ning, C., Scott, A., Fu, Q., Bjørn, R., Li, W., ... & Cui, Y. (2021). The genomic origins of the Bronze Age Tarim Basin mummies. Nature, 1-6.](https://doi.org/10.1038/s41586-021-04052-7)
  * > Although human activities in Xinjiang can be traced back to around 40,000 years ago24,39, the earliest evidence for sustained human habitation in the Tarim Basin dates only to the late third to early second millennium BC﻿. There, at the sites of Xiaohe, Gumugou and Beifang, well-preserved mummified human remains buried within wooden coffins and associated with rich organic grave good assemblages represent the earliest known archaeological cultures of the region. Since their initial discovery in the early twentieth century and subsequent large-scale excavations beginning in the 1990s (ref. 16), the Tarim mummies have been at the centre of debates with regard to their origins, their relationship to other Bronze Age steppe (Afanasievo), oasis (BMAC) and mountain (IAMC and Chemurchek) groups, and their potential connection to the spread of Indo-European languages into this region3,4,40.
  * > The palaeogenomic and proteomic data we present here suggest a very different and more complex population history than previously proposed. Although the IAMC may have been a vector for transmitting cultural and economic factors into the Tarim Basin, the known sites from the IAMC do not provide a direct source of ancestry for the Xiaohe populations. Instead, the Tarim mummies belong to an isolated gene pool whose Asian origins can be traced to the early Holocene epoch. This gene pool is likely to have once had a much wider geographic distribution, and it left a substantial genetic footprint in the EMBA populations of the Dzungarian Basin, IAMC and southern Siberia. The Tarim mummies’ so-called Western physical features are probably due to their connection to the Pleistocene ANE gene pool, and their extreme genetic isolation differs from the EBA Dzungarian, IAMC and Chemurchek populations, who experienced substantial genetic interactions with the nearby populations mirroring their cultural links, pointing towards a role of extreme environments as a barrier to human migration.
  * > In contrast to their marked genetic isolation, however, the populations of the Xiaohe horizon were culturally cosmopolitan, incorporating diverse economic elements and technologies with far-flung origins. They made cheese from ruminant milk using a kefir-like fermentation37, perhaps learned from descendants of the Afanasievo, and they cultivated wheat, barley and millet37,41, crops that were originally domesticated in the Near East and northern China and which were introduced into Xinjiang no earlier than 3500 BC﻿ (refs. 8,42), probably via their IAMC neighbours24. They buried their dead with Ephedra twigs in a style reminiscent of the BMAC oasis cultures of Central Asia, and they also developed distinctive cultural elements not found among other cultures in Xinjiang or elsewhere, such as boat-shaped wooden coffins covered with cattle hides and marked by timber poles or oars, as well as an apparent preference for woven baskets over pottery43,44. Considering these findings together, it appears that the tightknit population that founded the Xiaohe horizon were well aware of different technologies and cultures outside the Tarim Basin and that they developed their unique culture in response to the extreme challenges of the Taklamakan Desert and its lush and fertile riverine oases4.
  * > This study illuminates in detail the origins of the Bronze Age human populations in the Dzungarian and Tarim basins of Xinjiang. Notably, our results support no hypothesis involving substantial human migration from steppe or mountain agropastoralists for the origin of the Bronze Age Tarim mummies, but rather we find that the Tarim mummies represent a culturally cosmopolitan but genetically isolated autochthonous population. This finding is consistent with earlier arguments that the IAMC served as a geographic corridor and vector for regional cultural interaction that connected disparate populations from the fourth to the second millennium BC﻿ (refs. 24,25). While the arrival and admixture of Afanasievo populations in the Dzungarian Basin of northern Xinjiang around 3000 BC﻿ may have plausibly introduced Indo-European languages to the region, the material culture and genetic profile of the Tarim mummies from around 2100 BC﻿ onwards call into question simplistic assumptions about the link between genetics, culture and language and leave unanswered the question of whether the Bronze Age Tarim populations spoke a form of proto-Tocharian. Future archaeological and palaeogenomic research on subsequent Tarim Basin populations—and most importantly, studies of the sites and periods where first millennium AD Tocharian texts have been recovered—are necessary to understand the later population history of the Tarim Basin. Finally, the palaeogenomic characterization of the Tarim mummies has unexpectedly revealed one of the few known Holocene-era genetic descendant populations of the once widespread Pleistocene ANE ancestry profile. The Tarim mummy genomes thus provide a critical reference point for genetically modelling Holocene-era populations and reconstructing the population history of Asia.
  * [Gibbons, A. (2021) Western China’s mysterious mummies were local descendants of ice age ancestors. Science. 27 October.](https://www.science.org/content/article/western-china-s-mysterious-mummies-were-local-descendants-ice-age-ancestors)
* [Wang, W., Ding, M., Gardner, J. D., Wang, Y., Miao, B., Guo, W., ... & Fu, Q. (2021). Ancient Xinjiang mitogenomes reveal intense admixture with high genetic diversity. Science Advances, 7(14), eabd6690.](https://doi.org/10.1126/sciadv.abd6690)
* [Bourgon, N., Jaouen, K., Bacon, A. M., Dufour, E., McCormack, J., Tran, N. H., ... & Tütken, T. (2021). Trophic ecology of a Late Pleistocene early modern human from tropical Southeast Asia inferred from zinc isotopes. Journal of Human Evolution, 161, 103075.](https://doi.org/10.1016/j.jhevol.2021.103075)
  * > Tam Pà Ling, a cave site in northeastern Laos, has yielded the earliest skeletal evidence of Homo sapiens in mainland Southeast Asia. The reliance of Pleistocene humans in rainforest settings on plant or animal resources is still largely unstudied, mainly due to poor collagen preservation in fossils from tropical environments precluding stable nitrogen isotope analysis, the classical trophic level proxy. However, isotopic ratios of zinc (Zn) in bioapatite constitute a promising proxy to infer trophic and dietary information from fossil vertebrates, even under adverse tropical taphonomic conditions. Here, we analyzed the zinc isotope composition (66Zn/64Zn expressed as δ66Zn value) in the enamel of two teeth of the Late Pleistocene (63–46 ka) H. sapiens individual (TPL1) from Tam Pà Ling, as well as 76 mammal teeth from the same site and the nearby Nam Lot cave. The human individual exhibits relatively low enamel δ66Zn values (+0.24‰) consistent with an omnivorous diet, suggesting a dietary reliance on both plant and animal matter. These findings offer direct evidence of the broad utilization of resources from tropical rainforests by one of the earliest known anatomically modern humans in Southeast Asia.
* [Cooke, N. P., Mattiangeli, V., Cassidy, L. M., Okazaki, K., Stokes, C. A., Onbe, S., ... & Nakagome, S. (2021). Ancient genomics reveals tripartite origins of Japanese populations. Science Advances, 7(38), eabh2419.](https://doi.org/10.1126/sciadv.abh2419)
* [Teixeira, J. C., Jacobs, G. S., Stringer, C., Tuke, J., Hudjashov, G., Purnomo, G. A., ... & Helgen, K. M. (2021). Widespread Denisovan ancestry in Island Southeast Asia but no evidence of substantial super-archaic hominin admixture. Nature Ecology & Evolution, 5(5), 616-624.](https://doi.org/10.1038/s41559-021-01408-0)
* [Yang, M. A., Fan, X., Sun, B., Chen, C., Lang, J., Ko, Y. C., ... & Fu, Q. (2020). Ancient DNA indicates human population shifts and admixture in northern and southern China. Science, 369(6501), 282-288.](https://doi.org/10.1126/science.aba0909)
* [Narasimhan VM et al. (2019) The formation of human populations in South and Central Asia. Science. 35(6457): eaat7487](https://science.sciencemag.org/content/365/6457/eaat7487.full)
* [McColl, H., Racimo, F., Vinner, L., Demeter, F., Gakuhari, T., Moreno-Mayar, J. V., ... & Willerslev, E. (2018). The prehistoric peopling of Southeast Asia. Science, 361(6397), 88-92.](https://doi.org/10.1126/science.aat3628)
  * > We performed a principal component analysis (PCA) of worldwide present-day populations (12, 13) to find the strongest axes of genetic variation in our data and projected the ancient individuals onto the first two principal components. The two oldest samples—Hòabìnhians from Pha Faen, Laos [La368; 7950 with 7795 calendar years before the present (cal B.P.)] and Gua Cha, Malaysia (Ma911; 4415 to 4160 cal B.P.)—henceforth labeled “group 1,” cluster most closely with present-day Önge from the Andaman Islands and away from other East Asian and Southeast Asian populations (Fig. 2), a pattern that differentiates them from all other ancient samples. We used ADMIXTURE (14) and fastNGSadmix (15) to model ancient genomes as mixtures of latent ancestry components (11). Group 1 individuals differ from the other Southeast Asian ancient samples in containing components shared with the supposed descendants of the Hòabìnhians: the Önge and the Jehai (Peninsular Malaysia), along with groups from India and Papua New Guinea.
  * > We also find a distinctive relationship between the group 1 samples and the Ikawazu Jōmon of Japan (IK002). Outgroup f3 statistics (11, 16) show that group 1 shares the most genetic drift with all ancient mainland samples and Jōmon (fig. S12 and table S4). All other ancient genomes share more drift with present-day East Asian and Southeast Asian populations than with Jōmon (figs. S13 to S19 and tables S4 to S11). This is apparent in the fastNGSadmix analysis when assuming six ancestral components (K = 6) (fig. S11), where the Jōmon sample contains East Asian components and components found in group 1. To detect populations with genetic affinities to Jōmon, relative to present-day Japanese, we computed D statistics of the form D(Japanese, Jōmon; X, Mbuti), setting X to be different present-day and ancient Southeast Asian individuals (table S22). The strongest signal is seen when X = Ma911 and La368 (group 1 individuals), showing a marginally nonsignificant affinity to Jōmon (11). This signal is not observed with X = Papuans or Önge, suggesting that the Jōmon and Hòabìnhians may share group 1 ancestry (11).
* [Lipson, M., Cheronet, O., Mallick, S., Rohland, N., Oxenham, M., Pietrusewsky, M., ... & Reich, D. (2018). Ancient genomes document multiple waves of migration in Southeast Asian prehistory. Science, 361(6397), 92-95.](https://doi.org/10.1126/science.aat3188)
  * > We initially analyzed the data by performing principal component analysis (PCA) using two different sets of present-day populations (19). First, compared to a set of diverse non-Africans (East and Southeast Asians, Australasians, Central Americans, and Europeans), the ancient individuals fall close to present-day Chinese and Vietnamese when projected onto the first two axes, with Man Bac, Ban Chiang, and Vat Komnou shifted slightly in the direction of Onge (Andaman Islanders) and Papuan (fig. S1). To focus on East and Southeast Asian diversity, we then used a panel of 16 present-day populations from the region, with three primary directions in the first two dimensions represented by Han Chinese, Austroasiatic-speaking groups (Mlabri and Htin from Thailand, Nicobarese, and Cambodian, but not Kinh), and aboriginal (Austronesian-speaking) Taiwanese [right, left, and top, respectively; Fig. 1B; compare (20)]. Man Bac, Ban Chiang (all periods), and Vat Komnou cluster with Austroasiatic speakers, whereas Nui Nap projects close to present-day Vietnamese and Dai near the center, and Oakaie projects close to present-day Myanmar and other Sino-Tibetan speakers. Present-day Lao are intermediate between Austroasiatic speakers and Dai, and western Indonesians (Semende from southern Sumatra and Barito from southeastern Borneo) fall intermediate between Austroasiatic speakers and aboriginal Taiwanese.
  * > Next, we built admixture graph models to test the relationships between the Vietnam Neolithic samples and present-day Southeast Asians in a phylogenetic framework. We began with a scaffold model containing the Upper Paleolithic Siberian Ust’-Ishim individual (21) as an outgroup and present-day Mixe, Onge, and Atayal, to which we added Man Bac, Nicobarese, and Mlabri. The latter three were inferred to have ancestry from a Southeast Asian farmer–related source (∼70%, forming a clade with Atayal) and a deeply diverging eastern Eurasian source [∼30%, sharing a small amount of drift with Onge; f-statistics indicate that this source is also not closely related to Papuans, South Asians, or the 40,000 yr B.P. Tianyuan individual (22); table S3]. The allele sharing demonstrated by outgroup f3-statistics can be accommodated along the farmer lineage, the deeply splitting lineage, or a combination of the two, but given the closeness of the mixture proportions among the three groups, we found that the most parsimonious model (Fig. 3 and fig. S2) involved a shared ancestral admixture event (29% deep ancestry; 28% omitting VN29), followed by divergence of Man Bac from the present-day Austroasiatic speakers, and lastly, a second pulse of deep ancestry (5%) into Nicobarese (19).
  * > Finally, to assess the relationships among additional present-day populations, we fit two extended admixture graphs (figs. S3 and S4), with the first including Dai, Semende, Barito, Lebbo (from east-central Borneo), and Juang (an Austroasiatic-speaking group from India), and the second including Dai, Semende, Barito, and Lao. The western Indonesians could be fit well with three (but not two) sources of ancestry: Austronesian-related, Austroasiatic-related, and Papuan-related (table S3), in respective proportions of ∼67, 29, and 4% (Semende); ∼37, 60, and 2% (Barito); and ∼55, 23, and 22% (Lebbo) (19). The Austroasiatic-associated component was inferred to be closer to Nicobarese than to Mlabri or Man Bac, forming a “southern” Austroasiatic subclade (Fig. 3B). For Juang, we also obtained a good fit with three ancestry components: one western Eurasian, one deep eastern Eurasian (interpreted as an indigenous South Asian lineage), and one from the Austroasiatic clade (fig. S3). The Austroasiatic source for Juang (proportion 35%) was inferred to be closest to Mlabri, as supported by statistics f4(Juang, Palliyar; Mlabri, X) > 0 for X = Atayal, Man Bac, or Nicobarese (Z = 5.1, 2.8, 2.3), creating a “northern” Austroasiatic subclade. Separately, we found that Lao also possesses ancestry from the Austroasiatic clade (47%; fig. S4) but cannot be fit as a simple mixture of the same two components found in Nicobarese and Mlabri (residual statistic Z = 3.4 without a source to explain **distantly shared ancestry between Lao and Mixe**) (19).
  * > Our results provide genetic support for the hypothesis that agriculture was first practiced in mainland Southeast Asia by (proto-) Austroasiatic-speaking migrants from southern China (4–6, 11–13). We find that all eight of our sampled individuals from Man Bac (as well as individuals from Ban Chiang and Vat Komnou) are closely related to present-day Austroasiatic speakers, including a shared pattern of admixture, with one, VN29, exhibiting significantly elevated indigenous ancestry. By comparison, studies of cranial and dental morphology have placed Man Bac either close to present-day East and Southeast Asians (“Neolithic”), intermediate between East Asians and a cluster containing more ancient hunter-gatherers from the region plus present-day Onge and Papuan (“indigenous”), or split between the two clusters (7, 8, 23). The simplest explanation for our results is that the majority of our Man Bac samples represent a homogeneous Neolithic cluster, with recent local contact between farmers and hunter-gatherers leading to additional hunter-gatherer ancestry in VN29 and perhaps VN40 (7, 8). This model would imply that the incoming farmers had already acquired 25 to 30% hunter-gatherer ancestry, either in China or Southeast Asia, establishing the characteristic Austroasiatic-affiliated genetic profile seen in multiple populations today. The wide distribution of this profile across Southeast Asia (in some cases in admixed form) also supports a coherent migration with early shared admixture. The symmetric position of aboriginal Taiwanese and the majority East Asian ancestral lineage in Man Bac (and Austroasiatic speakers) with respect to Native Americans points to an origin for the farming migration specifically in southern China [contrasting with f4(X, Atayal; Mixe, Dinka) > 0 for northern East Asians; X = Han, Japanese, or Korean, Z > 4.5). Conversely, the signal of allele sharing between Lao and Native Americans points to admixture in Lao from a population affected by Han Chinese migrations, with a plausible explanation for our results being a mixture between resident Austroasiatic speakers and incoming Tai speakers within historical times (5).
  * > Our findings also have implications for genetic transformations linked to later cultural and linguistic shifts in Southeast Asia and beyond. We observe substantial genetic turnover between the Neolithic period and Bronze Age in Vietnam, likely reflecting a new influx of migrants from China (24). Late Neolithic to Bronze Age Myanmar individuals from Oakaie also do not possess an Austroasiatic genetic signature, in their case being closer to populations speaking Sino-Tibetan languages (including present-day Myanmar), pointing to an independent East Asian origin. Outside of mainland Southeast Asia, we document admixture events involving Austroasiatic-related lineages in India (where Austroasiatic languages continue to be spoken) and in Borneo and Sumatra (where all languages today are Austronesian). In the latter case, the shared ancestry with Nicobarese (in addition to separate Papuan-related and Austronesian-associated components) supports previous genetic results and archaeological hints of an early Austroasiatic-associated Neolithic expansion to western Indonesia (25, 26). Overall, Southeast Asia shares common themes with Europe, Oceania, and sub-Saharan Africa, where ancient DNA studies of farming expansions and language shifts have revealed similar instances of genetic turnover associated with archaeologically attested transitions in culture.

## Africa

* [Scerri, E. M., Niang, K., Candy, I., Blinkhorn, J., Mills, W., Cerasoni, J. N., ... & Groucutt, H. S. (2021). Continuity of the Middle Stone Age into the Holocene. Scientific reports, 11(1), 1-11.](https://doi.org/10.1038/s41598-020-79418-4)
* [Sirak, K. A., Fernandes, D. M., Lipson, M., Mallick, S., Mah, M., Olalde, I., ... & Reich, D. (2021). Social stratification without genetic differentiation at the site of Kulubnarti in Christian Period Nubia. Nature Communnications, 12(7283).](https://doi.org/10.1038/s41467-021-27356-8)
* [Silva, M., Oteo-García, G., Martiniano, R., Guimarães, J., von Tersch, M., Madour, A., ... & Richards, M. B. (2021). Biomolecular insights into North African-related ancestry, mobility and diet in eleventh-century Al-Andalus. Scientific reports, 11(1), 1-13.](https://doi.org/10.1038/s41598-021-95996-3)
* [Esoh, K. K., Apinjoh, T. O., Nyanjom, S. G., Wonkam, A., Chimusa, E. R., Amenga-Etego, L., ... & Achidi, E. A. (2021). Fine scale human genetic structure in three regions of Cameroon reveals episodic diversifying selection. Scientific reports, 11(1), 1-13.](https://doi.org/10.1038/s41598-020-79124-1)
* [Lipson, M., Ribot, I., Mallick, S., Rohland, N., Olalde, I., Adamski, N., ... & Reich, D. (2020). Ancient West African foragers in the context of African population history. Nature, 577(7792), 665-670.](https://doi.org/10.1038/s41586-020-1929-1)
  * > Our knowledge of ancient human population structure in sub-Saharan Africa, particularly prior to the advent of food production, remains limited. Here we report genome-wide DNA data from four children—two of whom were buried approximately 8,000 years ago and two 3,000 years ago—from Shum Laka (Cameroon), one of the earliest known archaeological sites within the probable homeland of the Bantu language group1,2,3,4,5,6,7,8,9,10,11. One individual carried the deeply divergent Y chromosome haplogroup A00, which today is found almost exclusively in the same region12,13. However, the genome-wide ancestry profiles of all four individuals are most similar to those of present-day hunter-gatherers from western Central Africa, which implies that populations in western Cameroon today—as well as speakers of Bantu languages from across the continent—are not descended substantially from the population represented by these four people. We infer an Africa-wide phylogeny that features widespread admixture and three prominent radiations, including one that gave rise to at least four major lineages deep in the history of modern humans.
  * > The oldest human-occupied layers at the site date to about 30,000 calendar years before present (bp; taken as ad 1950 in accordance with radiocarbon calibration convention), but of special interest are artefacts and skeletons dating to between the end of the Later Stone Age (about 8,000 bp) and the beginning of the Iron Age (about 2,500 bp) (Extended Data Fig. 1, Supplementary Information section 1).
  * > A total of 18 human skeletons have been discovered at Shum Laka, comprising 2 distinct burial phases1–3 (Supplementary Information section 1). We attempted to retrieve DNA from six petrous-bone samples and obtained working data from two individuals of the early Stone to Metal Age and two of the late Stone to Metal Age (about 8,000 and 3,000 bp, respectively) (Table 1, Supplementary Table 2). The two earlier individuals—a boy of 4 ± 1 years old at time of death (given the identifying code 2/SE I) lying on top of the lower limbs of an adolescent male of 15 ± 3 years old (denoted 2/SE II)2—were recovered from a primary double burial, and the two later individuals—a boy of 8 ± 2 years (denoted 4/A) and a girl of 4 ± 1 years (denoted 5/B)2—were in adjacent primary single burials.
  * > All of the mtDNA and Y chromosome haplogroups we observe at Shum Laka are associated today with sub-Saharan Africans. The two earlier individuals carry mtDNA haplogroup L0a (specifically L0a2a1), which is widespread in Africa, and the two later individuals carry L1c (specifically L1c2a1b), which is found among both farmers and hunter-gatherers in Central and West Africa15,16. Individuals 2/SE I and 4/A have Y chromosomes from macrohaplogroup B (often found today in hunter-gatherers from Central Africa17), and 2/SE II has the rare Y chromosome haplogroup A00, which was discovered in 2013 and is present at appreciable frequencies only in Cameroon—in particular, among the Mbo and Bangwa in the western part of the country12,13. A00 is the oldest known branch of the modern human Y chromosome tree, with a split time of about 300,000–200,000 bp from all other known lineages12,18,19. At 1,666 positions (from whole-genome sequence data; Supplementary Table 4) that differ between present-day A0018 and all other Y chromosomes, the sequence of the Shum Laka individual carries the nonreference allele at a total of 1,521, translating to a within-A00 split at about 37,000–25,000 bp (95% confidence interval) (Fig. 1, Methods).
  * > Finally, we built an admixture graph (Fig. 4a, Extended Data Fig. 4, Methods) comodelling the Shum Laka, Mota and ancient South African hunter-gatherer individuals; present-day Mbuti, Aka, Agaw (speakers of an Afroasiatic language, from Ethiopia20), Yoruba, Mende and Lemande; non-Africans (French); and two outgroups (Neanderthal and chimpanzees). We also fit versions of the model using alternative SNP ascertainments and additional populations (Hadza, Mbo, Herero, Chewa, Mursi, Baka, Bakola, Bedzan, Mada, Fulani and ancient individuals from Taforalt in Morocco26) and obtained similar results (Extended Data Table 3, Supplementary Information section 3).
  * > The geography of the populations involved suggests the centre of this radiation was plausibly in East Africa (Fig. 4b), and estimated divergences of African and non-African populations place its date at about 80,000–60,000 bp24,35. Such an expansion is also consistent with mtDNA phylogeography—specifically the diversification of haplogroup L3, which probably originated in East Africa about 70,000 bp36,37—and potentially with the origins of clade CT in the Y chromosome tree at a similar time depth18,38.
  * > Second, we infer a phase of divergences that involved at least four lineages early in the history of modern humans (point (1) in Fig. 4a). Recent consensus has been that southern African hunter-gatherers, who split from other populations about 250,000–200,000 bp, represent the deepest sampled branch of modern human variation21,24,25. Our results suggest that Central African hunter-gatherers split at close to the same time (or perhaps slightly earlier), and thus that both clades—as well as the lineage that would later diversify at point (2) (Fig. 4a)—originated as part of a large-scale radiation.
  * > In addition to the well-characterized deep lineages, we also detect at least one deep ghost source that contributed to West Africans and East African hunter-gatherers. This signal corroborates previous evidence for the Hadza and Sandawe39 and for West Africans22, although we find that the best fit is a source that splits near the same point as southern and Central African hunter-gatherers. Our results are also consistent with previous reports of archaic ancestry in African populations27–31, specifically in West Africans. The presence of deep ancestry in the West African clade is notable in light of the Pleistocene archaeological record5,40, which is limited but includes Homo sapiens fossils dated to about 300,000 bp in northwestern Africa41, as well as an individual with archaic features buried about 12,000 bp in southwestern Nigeria (the oldest known human fossil from West Africa proper)42. Middle Stone Age artefacts have also been found in parts of West Africa into the terminal Pleistocene43, despite the development of Later Stone Age technologies elsewhere (as, for example, at Shum Laka). Thus, the available material and fossil evidence is concordant with our genetic results in indicating long-term African population structure and admixture44,45.
* [Wang, K., Goldstein, S., Bleasdale, M., Clist, B., Bostoen, K., Bakwa-Lufu, P., ... & Schiffels, S. (2020). Ancient genomes reveal complex patterns of population movement, interaction, and replacement in sub-Saharan Africa. Science Advances, 6(24), eaaz0183.](https://doi.org/10.1126/sciadv.aaz0183)
* [Prendergast, M. E., Lipson, M., Sawchuk, E. A., Olalde, I., Ogola, C. A., Rohland, N., ... & Reich, D. (2019). Ancient DNA reveals a multistep spread of the first herders into sub-Saharan Africa. Science, 365(6448), eaaw6275.](https://doi.org/10.1126/science.aaw6275)
  * > Archaeological and now genetic evidence suggest complex spreads of herding and farming in eastern Africa involving multiple movements of ancestrally distinct peoples as well as gene flow among these groups. Models formulated on the basis of ancient DNA are a starting point for further exploration through additional archaeological, linguistic, and genetic research.
  * > To address these debates, we generated genome-wide ancient DNA data from individuals buried at sites associated with LSA (n = 3), early pastoral and PN (n = 31), IA (n = 1), and PIA (n = 6) archaeological traditions in what are now Kenya and Tanzania (Fig. 1, Table 1, and table S1). We extracted DNA from a combination of tooth and bone samples and enriched for a targeted set of ~1.2 million single nucleotide polymorphisms (SNPs) (21). Surprisingly, given the tropical climate and variable curatorial conditions, we obtained excellent data quality, with a median of approximately 0.51× coverage, or 440,000 SNPs covered by at least one sequence, for the 41 newly reported individuals (from a total of 67 sequencing libraries; table S2). The data scored well in standard ancient DNA authenticity metrics for all but two individuals [I12391 and I13970, whom we excluded from genome-wide analyses but for whom we obtained Y chromosome and mitochondrial DNA (mtDNA) haplogroups (21)]. We also generated direct radiocarbon dates for 35 individuals (tables S3 and S4 and fig. S1). We analyzed these data jointly with sequences from published ancient African individuals (22–27) as well as from people living in eastern Africa today (28–31).
  * > From these results, we formulated a four-part hypothesis to explain the origins of the ancestries in the sampled eastern African groups. First, admixture in northeastern Africa, likely associated with the spread of pastoralism, created groups (as yet unsampled with ancient DNA) with approximately equal proportions of ancestry related to (i) present-day Nilotic speakers such as Dinka and Nuer and (ii) sampled ancient and present-day groups from northern Africa and the Levant. We refer to this combination as early northeastern African pastoralist-associated ancestry (henceforth “early northeastern pastoralist,” or ENP) and the two subcomponents as EN1 and EN2. Second, descendants of these groups mixed with local foragers in eastern Africa, leading to the ~20% Mota-related ancestry in the PN individuals. Third, an additional period of Sudan-related gene flow occurred before the Iron Age and contributed to PIA groups. Fourth, close to the same time, western African–related ancestry related to present-day Bantu speakers [also seen in an individual buried on Pemba Island ~600 B.P. (22)] appeared in the Rift Valley (notably at Deloraine Farm), in association with the spread of farming.
* [Van de Loosdrecht, M., Bouzouggar, A., Humphrey, L., Posth, C., Barton, N., Aximu-Petri, A., ... & Krause, J. (2018). Pleistocene North African genomes link near Eastern and sub-Saharan African human populations. Science, 360(6388), 548-552.](https://doi.org/10.1126/science.aar8380)
  * > We investigated whether two first-hand proxies, Natufians and West Africans, are sufficient to explain the Taforalt gene pool or whether a more complex admixture model is required. We thus tested whether Natufians could be a sufficient proxy for the Eurasian ancestry in Taforalt without explicit modeling of its African ancestry (fig. S18). This line of investigation was inspired by proposed archaeological connections between the Iberomaurusian and Upper Paleolithic cultures in Southern Europe, either via the Strait of Gibraltar (19) or Sicily (20). If this connection is true, both the Upper Paleolithic European and Natufian ancestries will be required to explain the Taforalt gene pool. For our admixture modeling with the program qpAdm (16), we chose outgroups that can distinguish sub-Saharan African, Natufian, and Paleolithic European ancestries but are blind to differences between sub-Saharan African lineages (11). A two-way admixture model, comprising Natufian and sub-Saharan African populations, does not significantly deviate from our data (χ2 P ≥ 0.128), with 63.5% Natufian and 36.5% sub-Saharan African ancestry, on average (table S8). Adding Paleolithic European lineages as a third source only marginally increased the model fit (χ2 P = 0.019 to 0.128; table S9). Consistently, by using the qpGraph package (21), we find that a mixture of Natufian and Yoruba reasonably fits the Taforalt gene pool (|z| ≤ 3.7; fig. S19 and table S10). Adding gene flow from Paleolithic Europeans does not improve the model fit and provides an ancestry contribution estimate of 0% (fig. S19). We thus find no evidence of gene flow from Paleolithic Europeans into Taforalt within the resolution of our data.
  * > We further characterized the sub-Saharan African–related ancestry in the Taforalt individuals by using f4 statistics in the form f4(Chimpanzee, African; Yoruba/Mende, Natufian). We find that Yoruba or Mende and Natufians are symmetrically related to two deeply divergent outgroups, an ancient South African group from 2000 yr B.P. (aSouthAfrica) and Mbuti Pygmy, respectively (|z| ≤ 1.564 SE; table S11). Because f4 statistics are linear under admixture, we expect the Taforalt population not to be any closer to these outgroups than Yoruba or Natufians if the two-way admixture model is correct. However, we find instead that the Taforalt group is significantly closer to both outgroups (aSouthAfrica and Mbuti) than any combination of Yoruba and Natufians (z ≥ 2.728 SE; Fig. 4). A similar pattern is observed for the East African outgroups Dinka, Mota, and Hadza (table S11 and fig. S20). These results can only be explained by Taforalt harboring an ancestry that contains additional affinity with South, East, and Central African outgroups. None of the present-day or ancient Holocene African groups serve as a good proxy for this unknown ancestry, because adding them as the third source is still insufficient to match the model to the Taforalt gene pool (table S12 and fig. S21). However, we can exclude any branch in human genetic diversity more basal than the deepest known one represented by aSouthAfrica (4) as the source of this signal: it would result in a negative affinity to aSouthAfrica, not a positive one as we find (Fig. 4). Both an unknown archaic hominin and the recently proposed deep West African lineage (4) belong to this category and therefore cannot explain the Taforalt gene pool.
* [Llorente, M. G., Jones, E. R., Eriksson, A., Siska, V., Arthur, K. W., Arthur, J. W., ... & Manica, A. (2015). Ancient Ethiopian genome reveals extensive Eurasian admixture in Eastern Africa. Science, 350(6262), 820-822.](https://doi.org/10.1126/science.aad2879)
  * > The ability to sequence ancient genomes has revolutionized our understanding of human evolution. However, genetic analyses of ancient material have focused on individuals from temperate and Arctic regions, where ancient DNA is preserved over longer time frames (1). Africa has so far failed to yield skeletal remains with much ancient DNA, with the exception of a few poorly preserved specimens from which only mitochondrial DNA could be extracted (2). This is particularly unfortunate, as African genetic diversity is crucial to most analyses reconstructing the evolutionary history of anatomically modern humans, by providing the baseline against which other events are defined. In the absence of ancient DNA, geneticists rely on contemporary African populations, but a number of historic events, in particular a genetic backflow from West Eurasia into Eastern Africa (3, 4), act as confounding factors.
  * > The age of Mota means that he should predate the West Eurasian backflow, which has been dated to ~3000 years ago (3, 4). We formally tested this proposition by using an f4 ratio estimating the West Eurasian component (6), following the approach adopted by Pickrell et al. (3). As expected, we failed to find any West Eurasian component in Mota (table S5), thus providing support for previous dating of that event (3, 4).
  * > Given that Mota predates the backflow, we searched for its most likely source by modeling the Ari, the contemporary population closest to our ancient genome, as a mixture of Mota and another West Eurasian population (6). We investigated both contemporary sources (3) and other Eurasian ancient genomes (5, 9). In this analysis, contemporary Sardinians and the early Neolithic LBK (Stuttgart) genome stand out (Fig. 2A). Previous analyses have shown Sardinians to be the closest modern representatives of early Neolithic farmers (10, 11), implying that the backflow came from the same genetic source that fueled the Neolithic expansion into Europe from the Near East/Anatolia, before recent historic events changed the genetic makeup of populations living in that region. An analysis with haplotype sharing also identified a connection between contemporary Ethiopians and Anatolia (4, 12). Interestingly, archaeological evidence dates the arrival of Near Eastern domesticates (such as wheat, barley, and lentils) to the same time period (~3000 years ago) (13, 14), suggesting that the direct descendants of the farmers that earlier brought agriculture into Europe may have also played a role in the development of new forms of food production in the Horn of Africa.
  * > Using Mota as an unadmixed African reference and the early farmer LBK as the source of the West Eurasian component, it is possible to reassess the magnitude and geographic extent of historical migrations, avoiding the complications of using admixed contemporary populations (6). We estimated a substantially higher Eurasian backflow admixture than previously detected (3), with an additional 4 to 7% of the genome of most African populations tracing back to a Eurasian source. Moreover, we detected a much broader geographical impact of the backflow, going all the way to West and Southern Africa (Fig. 2B). Even though the West Eurasian component in these regions is smaller than in Eastern Africa, it is still sizable, with Yoruba and Mbuti, who are often used as African reference populations (15, 16), showing 7% and 6%, respectively, of their genomes to be of Eurasian origin (table S5).

## Austronesia: Australia, Melanesia, and Polynesia

* [Larena, M., McKenna, J., Sanchez-Quinto, F., Bernhardsson, C., Ebeo, C., Reyes, R., ... & Jakobsson, M. (2021). Philippine Ayta possess the highest level of Denisovan ancestry in the world. Current Biology, 31(19), 4219-4230.](https://doi.org/10.1016/j.cub.2021.07.022)
* [Teixeira, J. C., Jacobs, G. S., Stringer, C., Tuke, J., Hudjashov, G., Purnomo, G. A., ... & Helgen, K. M. (2021). Widespread Denisovan ancestry in Island Southeast Asia but no evidence of substantial super-archaic hominin admixture. Nature Ecology & Evolution, 5(5), 616-624.](https://doi.org/10.1038/s41559-021-01408-0) &mdash; [View](https://www.nature.com/articles/s41559-021-01408-0.epdf?sharing_token=zjMY6v0U0osKunNt1nMYcdRgN0jAjWel9jnR3ZoTv0MMfjkv7RDXtRSpDa6GHVksk43DIqUAK_idH314cHs2-_z7jY7UCeqEbtZHd-3h5QwLwo17n4vHFnlqlGYVcx7qgy0K-Z9QflLKZA3Y01HUfFFnxqSIWbgEuhWSv63tIbdnDrW_XB8D_A9gErWVqHavfnmaVltEfDrVD1g5CP4eZFhctYzVUGYNiQ7HPpbsXE8%3D&tracking_referrer=www.sciencenews.org)
* [Larena, M., Sanchez-Quinto, F., Sjödin, P., McKenna, J., Ebeo, C., Reyes, R., ... & Jakobsson, M. (2021). Multiple migrations to the Philippines during the last 50,000 years. Proceedings of the National Academy of Sciences, 118(13).](https://doi.org/10.1073/pnas.2026132118)
* [Ioannidis, A. G., Blanco-Portillo, J., Sandoval, K., Hagelberg, E., Barberena-Jonas, C., Hill, A. V., ... & Moreno-Estrada, A. (2021). Paths and timings of the peopling of Polynesia inferred from genomic networks. Nature, 597(7877), 522-526.](https://doi.org/10.1038/s41586-021-03902-8) &mdash; [PDF](https://www.nature.com/articles/s41586-021-03902-8.epdf?sharing_token=FAP3TN9wUjOT_P1NGpqHvNRgN0jAjWel9jnR3ZoTv0Nz5yPyOTL29YLfDPttn0US2JmNKXl70BZQCQuwhlNd_aAdfnCdl0sJ0RBGl5QogLfCXl7TU1QV-sKaR1PtSqY73x7DukYVi5LbjvNXQuMQw05uqGKxO5ibbL-2Be1cEl-RT1PNUfsyBR_t6RqnCT74jfMSzwaGha0CSlLL1ngn90S-ZmQCxOYmAf3EWAMC31o%3D&tracking_referrer=www.sciencenews.org)
* [Choin, J., Mendoza-Revilla, J., Arauna, L. R., Cuadros-Espinoza, S., Cassar, O., Larena, M., ... & Quintana-Murci, L. (2021). Genomic insights into population history and biological adaptation in Oceania. Nature, 592(7855), 583-589.](https://doi.org/10.1038/s41586-021-03236-5) &mdash; [PDF](https://hal-pasteur.archives-ouvertes.fr/pasteur-03205291/document)
* [Carlhoff, S., Duli, A., Nägele, K., Nur, M., Skov, L., Sumantri, I., ... & Brumm, A. (2021). Genome of a middle Holocene hunter-gatherer from Wallacea. Nature, 596(7873), 543-547.](https://doi.org/10.1038/s41586-021-03823-6)
  * > Modern humans crossed through Wallacea (Fig. 1a) to Sahul5,6,7,8 at least 50 thousand years ago (kya)9, and possibly by up to 65 kya10. Presently, however, the earliest archaeological evidence for our species in Wallacea dates to at least 45.5 kya for figurative art in Sulawesi11, and 47–43 kyr cal BP for a behavioural shift at Liang Bua (Flores, Indonesia)12. The oldest Homo sapiens skeletal remains date to 13 kya13. The route modern humans used to enter Sahul is not known14. Demographic models infer a population split between the ancestors of Oceanian and Eurasian groups approximately 58 kya, whereas Papuan and Aboriginal Australian groups separated around 37 kya5. Within this time interval, modern humans admixed multiple times with groups related to Denisovans15,16,17,18,19,20,21,22,23, and potentially other unknown hominins24,25. The genetic ancestry of the two Hòabìnhian-associated foragers from Pha Faen and Gua Cha1 shows the highest similarity to modern Andamanese peoples. These ancient and present-day peoples lack substantial amounts of Denisovan-related ancestry, suggesting that the Hòabìnhian-associated and Onge-related lineage diverged before the main archaic human introgression events1. Current Wallacean individuals carry larger proportions of Denisovan-related ancestry, but at substantially lower frequencies than is the case in Papuan and Indigenous Australian individuals20. This is probably due to admixture with the East Asian Neolithic farmers (‘Austronesian peoples’) who arrived in Wallacea around 4 kya20,26.
  * > We initiated our genomic investigation by principal component analyses (PCAs), comparing the Leang Panninge genome with present-day individuals from East Asia, southeast Asia and Near Oceania (comprising Indigenous Australia, Papua New Guinea and Bougainville) genotyped on the Human Origins SNP panel18,32,33,34. The newly generated genome and relevant published genomes from ancient individuals from eastern Eurasia were then projected on the PCA1,34,35,36,37,38. Leang Panninge falls into PCA space not occupied by any present-day or ancient individuals, but is broadly located between Indigenous Australian peoples and the Onge (Fig. 2a, Extended Data Fig. 6). F3-statistics33 of the form f3 (Mbuti; Leang Panninge, X), where X is replaced with present-day Asian-Pacific groups, indicated that the new genome shares most genetic drift with Near Oceanian individuals (Fig. 2b). We confirmed these results with f4-statistics33, suggesting similar affinity of Leang Panninge and Papuan individuals to present-day Asian individuals, despite Near Oceanian groups forming a clade to the exclusion of Leang Panninge (Extended Data Fig. 7a, b). All present-day groups from the region, with the exception of the Mamanwa and the Lebbo26, carry only a minor contribution of Papuan-related ancestry (Supplementary Fig. 4).
  * > Genome-wide analyses of the Leang Panninge individual show that most genetic drift is shared with present-day groups from New Guinea and Aboriginal Australia (Fig. 2b, Extended Data Fig. 7a). However, this Toalean-associated genome represents a previously undescribed ancestry profile, one that branched off after Onge-related and Hòabìnhian-related lineages but around the time that Papuan and Indigenous Australian groups split (Fig. 3b, c, Extended Data Fig. 8, Supplementary Figs. 6–9). It is possible that this Toalean individual carries a local ancestry that was present in Sulawesi before the initial peopling of Sahul at least 50 kya9, although whether this population produced the Late Pleistocene rock art in the south of the island11,44,45 is unknown.
  * > The lower amount of Denisovan ancestry in the Leang Panninge individual than in Papuan and Indigenous Australian individuals could result from: (1) an additional admixture with Denisovan ancestry into the common ancestors of Near Oceanian groups, or (2) a dilution of the Denisovan-related ancestry in the Leang Panninge genome through admixture with lineages carrying less or no such ancestry. Our allele frequency-based analyses do not support the first scenario (Supplementary Fig. 11), but they do favour the latter. The scarcity of pre-Neolithic genomes from across Asia prevents us from defining the exact source and admixture proportions of this gene flow event. It is noteworthy, however, that despite the reconstructed population trees (TreeMix and qpGraph) suggesting a genetic influence on middle Holocene Sulawesi from mainland East Asia, our qpAdm modelling cannot rule out a southeast Asian contribution from a group related to present-day Andamanese peoples (Fig. 3, Supplementary Figs. 6–11, Supplementary Table 26). This is consistent with a recent study that describes widespread admixtures across Asia between Onge-related and Tianyuan-related ancient populations47. However, the presence of this type of ancestry in a middle Holocene forager from Wallacea suggests that the Asian-related admixture could have taken place long before the expansion of Austronesian societies into the region.
* [Brumm, A., Oktaviana, A. A., Burhan, B., Hakim, B., Lebe, R., Zhao, J. X., ... & Aubert, M. (2021). Oldest cave art found in Sulawesi. Science Advances, 7(3), eabd4648.](https://doi.org/10.1126/sciadv.abd4648)
  * > Indonesia harbors some of the oldest known surviving cave art. Previously, the earliest dated rock art from this region was a figurative painting of a Sulawesi warty pig (Sus celebensis). This image from Leang Bulu’ Sipong 4 in the limestone karsts of Maros-Pangkep, South Sulawesi, was created at least 43,900 years ago (43.9 ka) based on Uranium-series dating. Here, we report the Uranium-series dating of two figurative cave paintings of Sulawesi warty pigs recently discovered in the same karst area. The oldest, with a minimum age of 45.5 ka, is from Leang Tedongnge. The second image, from Leang Balangajia 1, dates to at least 32 ka. To our knowledge, the animal painting from Leang Tedongnge is the earliest known representational work of art in the world. There is no reason to suppose, however, that this early rock art is a unique example in Island Southeast Asia or the wider region.
* [Langley, M. C., Amano, N., Wedage, O., Deraniyagala, S., Pathmalal, M. M., Perera, N., ... & Roberts, P. (2020). Bows and arrows and complex symbolic displays 48,000 years ago in the South Asian tropics. Science Advances, 6(24), eaba3831.](https://doi.org/10.1126/sciadv.aba3831)
  * > Archaeologists contend that it was our aptitude for symbolic, technological, and social behaviors that was central to Homo sapiens rapidly expanding across the majority of Earth’s continents during the Late Pleistocene. This expansion included movement into extreme environments and appears to have resulted in the displacement of numerous archaic human populations across the Old World. Tropical rainforests are thought to have been particularly challenging and, until recently, impenetrable by early H. sapiens. Here, we describe evidence for bow-and-arrow hunting toolkits alongside a complex symbolic repertoire from 48,000 years before present at the Sri Lankan site of Fa-Hien Lena—the earliest bow-and-arrow technology outside of Africa. As one of the oldest H. sapiens rainforest sites outside of Africa, this exceptional assemblage provides the first detailed insights into how our species met the extreme adaptive challenges that were encountered in Asia during global expansion.
* [Roberts, P., Louys, J., Zech, J., Shipton, C., Kealy, S., Carro, S. S., ... & OʼConnor, S. (2020). Isotopic evidence for initial coastal colonization and subsequent diversification in the human occupation of Wallacea. Nature communications, 11(1), 1-11.](https://doi.org/10.1038/s41467-020-15969-4)
  * > Our extensive faunal baseline demonstrates that terrestrial and marine environments can be clearly distinguished isotopically in Wallacea on the basis of stable isotope analysis of fossil tooth enamel. We show that a tooth of the earliest preserved H. sapiens fossil found from the region c. 42–39,000 years ago shows that this individual made significant use of coastal resources. From 20,000 years ago, human populations show an increasing reliance on interior, terrestrial environments on the islands of both Timor and Alor at a time of increasing forest expansion in Island Southeast Asia more generally, though some individuals continue to intensively use marine resources. We argue that our data further demonstrates the huge adaptive flexibility of our species, acutely visible as it rapidly and persistently colonized Wallacean environments. Its ability to specialize in the use of more extreme environments seems to stand in contrast to other hominin species known from Island Southeast Asia based on current evidence.
  * > Our large, robust faunal baseline enables the long-term ecological niches of Late Pleistocene/Holocene human foragers to be directly determined for Asitau Kuru, Matja Kuru 2, Makpan, and Tron Bon Lei, as well as the additional sites of Lene Hara and Matja Kuru 1 where only human samples were available (Figs. 2–6; n = 26). Sampled human δ13C and δ18O ranges between −14.1 and −5.6‰ and −6.2 to −3.2‰, respectively (Fig. 2, Supplementary Data 2). The earliest human sample in the study, and the earliest recovered from Wallacea, from context B63 at Asitau Kuru, dated to c. 42,440–38,853 cal. years BP, has a δ13C value of −5.6‰ (Fig. 3). This is indicative of a high reliance on marine resources, given the lack of any evidence for C4 resources in this part of Timor at this time and zooarchaeological evidence for abundant marine resources (Fig. 2). The majority of the remaining humans sampled from the Terminal Pleistocene and Holocene contexts of Asitau Kuru (Fig. 3), Lene Hara (Fig. 3), Matja Kuru 1 and 2 (Fig. 4), and the site of Makpan (Fig. 5) on Alor, have δ13C values between −14.1‰ and −9.6‰, indicating human reliance on a mixture of terrestrial tropical forest resources and more open C3 environments (Fig. 2).
  * > Nevertheless, it is clear that two Terminal Pleistocene/Holocene individuals at Makpan, Alor (−8.1‰; 15,000−11,000 cal. years BP; Fig. 5), and Matja Kuru 2, Timor (−5.6‰; Fig. 4; 11,000–4000 cal. years BP) incorporated a significant proportion of C4/marine and marine resources into their diets, respectively, based on associated faunal data (Fig. 2), suggesting that diets were diverse between individuals and societies during the Terminal Pleistocene and Holocene.
  * > Our stable isotope data allow us to directly assess the ecological reliance on different categories of resources that accompanied our species’ arrival and subsequent settling of Wallacea. The earliest humans to arrive in this part of the world seemingly specialized in the use of coastal resources. While we cannot currently distinguish between pelagic and other forms of offshore resource use13, and our sample size from this early period is limited, we can be confident that the δ13C value for this individual indicates a reliance on marine resources. Following 20,000 cal. years BP, a clear diversification in human resource use across Wallacea emerges. While some coastal reliance is indicated by one individual at Matja Kuru 2, and perhaps also Makpan, most individuals demonstrate broader use of interior environments, including closed tropical forest habitats. This may be considered surprising, particularly given the ongoing presence of fish and shellfish9, the symbolic burial of an individual at Tron Bon Lei with fishhooks (Supplementary Note 1), as well as archaeological evidence for increased transfer of material culture between islands from this time46,47. However, tooth enamel δ13C reflects the whole diet of an individual, and our data highlights the necessity of paying more attention to the contribution of plant (and terrestrial animal) resources to human diets on tropical islands, particularly as those populations become more established—something also recently urged even for the study of the later Lapita expansion in the Pacific48,49.
* [Roberts P, Amano N (2019) Plastic pioneers: Hominin biogeography east of the Movius Line during the Pleistocene. Archaeological Research in Asia. 17: 181-192.](https://www.sciencedirect.com/science/article/pii/S2352226718300989?via%3Dihub)
  * > Irrespective of technological remnants it is clear that Early/Middle Pleistocene Homo, as well as our own species in the Late Pleistocene, was able to get beyond the Movius Line and even beyond the formal, bioegographical Wallace Line. This is potentially adaptively significant because modern biogeographic divisions (Holt et al., 2012) map onto the regional division brought into focus by the Movius Line. In particular, Southeast Asia represents the largest tropical rainforest block to be encountered by hominins following their expansion beyond Africa (Whitmore, 1998; Ghazoul and Sheil, 2010), while Island Southeast Asia and Oceania also include insular settings that necessitate ocean crossings and adaptations to depauperate fauna (Samper Carro et al., 2016). Given theoretical debate regarding the attractiveness of these habitats to hominin occupation (Bailey et al., 1989; Gamble 1993; O'Connor et al., 2011; Roberts and Petraglia, 2015; Samper Carro et al., 2016), as well as potential fluctuations in the extent of these environments in the past, the Movius Line may represent a useful regional device for comparing the adaptive capacities of Early/Middle Pleistocene Homo taxa (namely H. erectus and H. floresiensis) to those of our own species, Homo sapiens. Moving beyond a technological, cultural, or cognitive definition, we seek to evaluate whether the Movius Line represents a useful theoretical focus when reviewing current climatic, environmental, and ecological data associated with Pleistocene hominins in Southeast Asia and Melanesia.
  * > It is perhaps telling in this regard that H. erectus and H. floresiensis apparently become extinct in Southeast Asia alongside a variety of grassland-woodland fauna c. 125 ka with the appearance and expansion of modern rainforest fauna (Westaway et al., 2007; Marwick, 2009) (Fig. 2). A premolar possibly dating to earlier than c. 100 ka on the island of Java, associated with Punung tropical rainforest fauna, has been argued to represent early human rainforest presence in Indonesia (Storm et al., 2005; Storm and de Vos, 2006; Westaway et al., 2007). Indeed, if the proposed date of c. 115–125 ka is correct, this would represent the earliest fossil of our species beyond Africa. However, while the tooth may indeed belong to H. sapiens, there remains intense debate relating to the species identification and the date should be treated with skepticism until further work and additional definitive sites of the same age are discovered (Storm et al., 2005; Barker et al., 2007; Bacon et al., 2008; Polanski et al., 2016). Yet, what is clear, regardless of the position taken on the Punung tooth, is that H. sapiens is the only hominin that is found in association with any of this Late Pleistocene fauna in other contexts. This is suggestive of a novel adaptation when compared to the palaeoecological context earlier members of the genus Homo in the region reviewed above (Fig. 2). Indeed, Southeast Asia is potentially home to some of the earliest fossils of our species in tropical rainforest contexts globally (Sémah and Sémah, 2015).
  * > Four teeth at Panxian Dadong in South China, dated to 300–130 ka, based on biostratigraphy, U-series, and Electron Spin Resonance dating (Shen et al., 1997; Rink et al., 2003; Schepatz and Miller-Antonio, 2004), have been identified as H. sapiens (Schepartz et al., 2000), and associated with fauna indicative of mixed woodland, bamboo forest, and grassland (Schepartz et al. 2000) (Fig. 3). However, the teeth show a clear mix of archaic and derived features, while their chronological association also remains debatable (Liu et al., 2013). Confirmed H. sapiens has been found at Fuyan Cave in the same region in association with mixed tropical rainforest, woodland, and grassland fauna (Liu et al., 2015) though the date remains disputed (Michel et al., 2016). Finds of H. sapiens associated with rainforest fauna at Lida Ajer in Sumatra c. 73–63 ka (Westaway et al., 2017), at Tam Pa Ling, Laos c. 60–46 ka associated with some elements of tropical forest microfauna (Demeter et al., 2012, 2015), and a human metatarsal found within a rainforest river valley environment at Callao Cave in the Philippines (Mijares et al., 2010), potentially also represent early human presence in the tropical forests of Southeast Asia (Fig. 2). Famously, the Niah Caves in Sarawak, Borneo have also yielded one of the earliest human fossils, the ‘Deep Cave Skull’ in an archaeological context in this region (c. 44,000–40,000 cal. years BP) (Barker et al., 2007; Higham et al., 2009; Reynolds and Barker, 2015) (Fig. 3).
  * > Beyond the Wallace Line, particularly the islands of Timor and Alor that were never connected to Sahul (the continent connecting Melanesia and Australia during glacial declines in sea-level) during the Late Pleistocene, tropical forest use has been considered an unviable source of subsistence for the earliest colonists given depauperate Wallacean insular terrestrial forest environments (O'Connor et al., 2011; Samper Carro et al., 2016). Here, faunal remains, and cultural evidence for pelagic fishing behaviours, from the coastal sites of Jerimalai (45,000–6000 years ago) and Matja Kuru 2 (36,000–10,000 years ago) imply a reliance on marine shellfish, fish, and large marine mammals and turtles (O'Connor et al., 2011; O'Connor, 2015; Samper Carro et al., 2016) (Fig. 3). Shell artefacts, fishhooks, and faunal distributions in the southern Japanese Archipelago dated to c. 35,000–13,000 years ago (Fujita et al., 2016), and similar finds in the Philippines (Pawlik, 2015), further highlight the specialized nature of our species' interaction with marine environments in the Pacific region during this time. Nevertheless, on Timor and Alor, giant rat taxa, with postulated preferences for forest environments, are also present at a number of Late Pleistocene sites (Aplin and Helgen, 2010), while other sites, such as Makpan on Alor, are located inland from the coast (J. Louys pers. comm.), suggesting that our species may also have developed specialized terrestrial adaptations to these island contexts (Fig. 3).
  * > Prey mortality profiles have been used to suggest that Early Pleistocene Homo was hunting prime adult bovids at the FLK site of Zinj, Tanzania (Bunn and Gurtov, 2014), perhaps in groups (Bennett et al., 2009), rather than scavenging. The expansion of H. erectus beyond Africa and into Eurasia has also been associated with the extinction (perhaps through competition) of large carnivores (Turner, 1992; Turner and O'Regan, 2007). This adaptive model would appear to favour large or middle-sized mammalian game living in open and woodland habitats. H. floresiensis also appears to have prioritized the hunting or scavenging of stegodon communities, in competition with other large scavengers, that appear to have favoured similar habitats (Westaway et al., 2009; Meijer and Awe Due, 2010). The general focus of Early to Middle Pleistocene hominins in regions with reliable access to water, in the form of lakes and rivers, would have provided access to large game populations. Moreover, lakes and rivers would have presented key migration routes for large to medium sized mammals through the tropical region of Southeast Asia (Roberts et al., 2016). Although H. erectus was clearly able to make sea crossings into Flores during the Early Pleistocene, this was not accompanied by maritime adaptations and appears to be part of a broader faunal spread and perhaps accidental voyaging (Smith, 2001; Dennell et al., 2014; Van den Bergh et al., 2016b). It therefore seems that for Early-Middle Pleistocene Homo, the following of complex grassland-woodland mosaics represented a useful strategy for the crossing of the Movius and Wallace Lines.
  * > By contrast, from at least 45,000 years ago, and perhaps 74,000 years ago or even earlier, our own species crossed the Movius Line using a variety of different environmental adaptations (Kaifu et al., 2015). While in many cases Late Pleistocene sites are associated with complex grassland and forest mosaics, there is also clear evidence for the dedicated use of rainforest plants and animals (Barker et al., 2007; Barker and Farr, 2016) (Fig. 2) and specialized maritime adaptation (O'Connor et al., 2011). Furthermore, our species developed regionspecific behaviours in a huge variety of ‘tropical rainforest’ formations, including mobile lifestyles focused on the use of forest trees and marsupial game in montane New Guinea, the hunting of giant rats on the isolated islands of Timor and Alor, the hunting of arboreal primates and wild boar on Borneo, and pelagic fishing documented in Timor and Alor. In some cases there is also clear evidence for human ‘niche’ construction and the manipulation of environments, including forest burning (Hunt et al., 2012) and faunal and plant translocation (Gosden and Robertson, 1991; Paz, 2005; Boivin et al., 2016). This engagement with a huge variety of different environments, often with relatively ‘simple’ core and flake technology (though see Ford, 2017), appears to set apart our species beyond the Movius Line, allowing it to deal with a variety of ecological challenges as the previous hominin inhabitants of Southeast Asia disappeared from the face of the Earth.
  * > It is clear that the Movius Line is no longer relevant as a definitive technological or cognitive boundary for Pleistocene hominins (Lycett and Bae, 2010; Dennell, 2014, 2016). Its geographical focus has also been, in part, dictated by a lack of internationally-available EarlyMiddle Pleistocene research in eastern and southeastern Asia until recently, and Acheulean and core and flake technologies should not be seen as definitive markers of different cognitive abilities. Instead, technological assemblages should be seen as part of cultural and ecological adaptive strategies to different local contexts and environments. Indeed, the longevity of the Movius Line as a theoretical construct is, in part, a result of palaeoanthropolgocial assumptions of the adaptive challenges involved in crossing this line. To its west, Saharo-Arabian and Afro-tropical biogeographic zones may have represented, at different points in the past, corridors of environmental continuity across much of Africa, Arabia, and western Eurasia in the form of grasslandwoodland mosaics (Dennell and Roebroeks, 2005). By contrast, to the east, tropical rainforest and isolated insular settings represent a very different prospect: one that has often been assumed to be an obstacle to hominin occupation and subsistence (Gamble, 1993; Bird et al., 2005; Boivin et al., 2013). Although we do not wish to argue that the Movius Line represents an absolute environmental and adaptive separation, it is potentially useful in considering the varied colonization strategies adopted by Pleistocene hominin species crossing it.
  * > H. erectus, and its diminutive descendant H. floresiensis, was able to cross the Movius Line and also the biogeographic division of the Wallace Line. Yet, these species appear to have done so alongside a variety of woodland- and grassland-adapted fauna. While there is tentative evidence for the use of lacustrine and riverine tropical forest settings by these hominins, definitive evidence for adaptations to dense tropical rainforests, or maritime adaptations, during the Early-Middle Pleistocene of Southeast Asia and Melanesia are currently lacking. This suggests that they maintained a relatively homogeneous and specialized adaptation during the colonization of these new regions. By contrast, our species, H. sapiens, appeared in Southeast Asia during the expansion of evergreen rainforest fauna that characterizes the humid forests of the region today. Not only that but our species specialized in the colonization of, and adaptation to, a diversity of tropical rainforest settings including high-altitude montane rainforest, depauperate insular forests, peat-swamp forests, as well as wet-dry tropical rainforest mosaics across Southeast Asia and Melanesia. Furthermore, it developed specialized maritime subsistence strategies that aided its expansion beyond the Wallace Line and Sahul. This environmental and adaptive plasticity appears to be unique to our species and may be the reason behind its status as the last hominin on Earth (Roberts and Stewart, 2018).
* [Mondal, M., Bertranpetit, J., & Lao, O. (2019). Approximate Bayesian computation with deep learning supports a third archaic introgression in Asia and Oceania. Nature communications, 10(1), 1-9.](https://doi.org/10.1038/s41467-018-08089-7)
* [Aubert, M., Lebe, R., Oktaviana, A. A., Tang, M., Burhan, B., Jusdi, A., ... & Brumm, A. (2019). Earliest hunting scene in prehistoric art. Nature, 576(7787), 442-445.](https://doi.org/10.1038/s41586-019-1806-y) &mdash; [PDF](https://www.nature.com/articles/s41586-019-1806-y.epdf?sharing_token=4Uk7y5fpMkRickcxo9lcKdRgN0jAjWel9jnR3ZoTv0OrHLJXX3frqDE-UdL7M7YyKWotJktQ9H-bxocXSBjzKV95hrVg7jBcpyDJegxKmqeN_tmlyDf4CB6aBKnbr_trRo2lQpeFgCTh6dkMCIfXquHZ69meu7dclYf-WuVPjgFjikX9jx9D0g-2PbM1Hq5qjy_PFKxs5bFv_OdTh1Ev2Obf-Vk2Uy6t8NUlSYI0aKQ%3D&tracking_referrer=www.sciencenews.org)
* Putting this here for now: [Swift, J. A., Roberts, P., Boivin, N., & Kirch, P. V. (2018). Restructuring of nutrient flows in island ecosystems following human colonization evidenced by isotopic analysis of commensal rats. Proceedings of the National Academy of Sciences, 115(25), 6392-6397.](https://doi.org/10.1073/pnas.1805787115)
* [Langley M (2017) World’s scientists turn to Asia and Australia to rewrite human history. The Conversation. 7 Dec. 2017.](http://theconversation.com/worlds-scientists-turn-to-asia-and-australia-to-rewrite-human-history-88697)
* [Gibbens S (2017) Humans Arrived in Australia 18,000 Years Earlier Than We Thought. National Geographic. 20 Jul.](https://news.nationalgeographic.com/2017/07/australia-aboriginal-early-human-evolution-spd/)
* [Gibbons A (2017) A find in Australia hints at very early human exit from Africa. 19 Jul.](http://www.sciencemag.org/news/2017/07/find-australia-hints-very-early-human-exit-africa)
* [Malaspinas A-S et al. (2016) A genomic history of Aboriginal Australia. Nature. 538: 207–214.](https://www.nature.com/articles/nature18299)
* [(2016) Unprecedented study of Aboriginal Australians points to one shared Out of Africa migration for modern humans. 21 Sep.](https://www.cam.ac.uk/research/news/unprecedented-study-of-aboriginal-australians-points-to-one-shared-out-of-africa-migration-for)
* [McEvoy B et al. (2010) Whole-Genome Genetic Diversity in a Sample of Australians with Deep Aboriginal Ancestry. American Journal of Human Genetics. 13 Aug.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2917718/)

### Australian, Melanesian, and Polynesian Connections to the Americas

* [Castro, M. A., Ferraz, T., Bortolini, M. C., Comas, D., & Hünemeier, T. (2021). Deep genetic affinity between coastal Pacific and Amazonian natives evidenced by Australasian ancestry. Proceedings of the National Academy of Sciences, 118(14).](https://doi.org/10.1073/pnas.2025739118)
  * > This provides strong evidence that a significant variability of this signal exists not only at an interpopulation level but also between individuals from the same populations. These results suggest that the intrapopulation variability of this signal is not rare (Fig. 2) and is observed in several groups (Apalai, Guaraní Nãndeva, Karitiana, Munduruku, Parakanã, and Xavánte). Most significant tests detected this excess signal in Tupí-speaking individuals, but the signal was also detected in individuals from every major linguistic group (Fig. 2 and Dataset S4) and, at the same time, presented a widespread geographic distribution within South America (Fig. 1). Conversely, a considerable number of samples were inferred to have a deficit of allele sharing with Australasians (Fig. 2 and Dataset S4D). Strikingly, the individual PAR137 (Parakanã) presented an extremely high proportion of significant tests (31.64%), indicating a relative deficit. This individual is not an outlier neither in the principal component analysis of the Native American samples (Dataset S1 B and C), nor regarding its missingness rate (Dataset S1A), nor in a multidimensional scaling (MDS) of pairwise genetic distances between samples in the unrelated and unadmixed subset (Dataset S1D). Besides, the distribution of Y-population ancestry among present-day indigenous groups of South America showed no relationship with ethnolinguistic diversity or geographic location.
  * > As the Chotuna group in the Pacific coast also exhibited excess allele sharing (Fig. 1 and Dataset S3) with the Australasians as estimated by D statistic (Mbuti; Australasians: Y, Z), we created admixture graph models based on the scaffold of Skoglund et al. (2) (Fig. 3A) with the addition of the Pacific coastal groups Sechura, Chotuna, and Narihuala. The best-fitted model showed that the Pacific coast is a mixed group of South American ancestry and a small non-American contribution associated with a sister branch of Onge (Fig. 3C), as also observed for Karitiana and Suruí. When the Xavánte were included in the analysis, the best-fitted model showed a direct contribution of the Australasian component in the Pacific coast, followed by a strong drift of this signal, giving rise to Amazonian groups (Fig. 3D). Although Fig. 3D could indicate two independent events, the small genetic distances between the nodes in this model reinforced the single admixture event evidence. The Treemix (10) analysis also showed a pattern of diversification in which Pacific coastal and Andean groups diverged first (Dataset S6), followed by the eastern Andean slopes populations and then, finally, the Amazonians and other eastern South Americans. These findings suggest that the Y-population contribution was introduced before the formation of the Amazonian branch, likely in the ancestors of Pacific coastal/Amazonian populations.
  * > Different migration routes to the South American region have been previously proposed and evidenced. Archeological and genetic data demonstrated that both routes, Pacific coastal and inland, were likely used by the first migrants (11). Our models point to an ancient genetic affinity between the Pacific coast and Amazonian populations that could be explained by the presence of Y ancestry in both geographic regions. In addition, this shared ancestry seems to precede the separation of the Pacific and Amazon branches, showing an entry through the west coast, followed by successive events of genetic drift in the Brazilian populations. This genetic evidence for the presence of Y ancestry on the South American Pacific coast indicates that this ancestry likely reached this region through the Pacific coastal route, and therefore could explain absence of this genetic component in the populations of North and Central America studied so far.
  * [Price M. (2021) Earliest South American migrants had Indigenous Australian, Melanesian ancestry. Science News. March 29.](https://www.sciencemag.org/news/2021/03/earliest-south-american-migrants-had-australian-melanesian-ancestry)
    * > For the first time, scientists identified the Y signal in groups living outside the Amazon—in the Xavánte, who live on the Brazilian plateau in the country’s center, and in Peru’s Chotuna people, who descend from the Mochica civilization that occupied that country’s coast from about 100 C.E. to 800 C.E.
    * > Next, the researchers used software to test different scenarios that might have led to the current DNA dispersal. The best fit scenario involves some of the very earliest—possibly even the earliest—South American migrants carrying the Y signal with them, the researchers report today in the Proceedings of the National Academy of Sciences. Those migrants likely followed a coastal route, Hünemeier says, then split off into the central plateau and Amazon sometime between 15,000 and 8000 years ago. “[The data] match exactly what you’d predict if that were the case,” Raff agrees.
* [Ioannidis, A.G., Blanco-Portillo, J., Sandoval, K. et al. (2020) Native American gene flow into Polynesia predating Easter Island settlement. Nature.](https://doi.org/10.1038/s41586-020-2487-2) &mdash; [PDF](https://www.nature.com/articles/s41586-020-2487-2.epdf?sharing_token=WVZkoDgz4sHTVRni8v2oDNRgN0jAjWel9jnR3ZoTv0NQ-GCGXs3b23XyInVhVtT1AdcNQHZodtkKNjeo_SR6zo1Y_cdhNwxAhOvOCbNzWOLXBkCusSoFcGBXnVspTQdLKo5xpm13eg2ly61A5zDZJFQYqLrgXNIeb5N3hkEGhwEi4vFnWpN4633lNvJKJJd3Me9cNPvG3dcjoW4JFV1YmO6MqiQ5PHfrrfTUdSYW9rA%3D&tracking_referrer=www.sciencenews.org)
  * > Here we analyse genome-wide variation in individuals from islands across Polynesia for signs of Native American admixture, analysing 807 individuals from 17 island populations and 15 Pacific coast Native American groups. We find conclusive evidence for prehistoric contact of Polynesian individuals with Native American individuals (around AD 1200) contemporaneous with the settlement of remote Oceania13,14,15. Our analyses suggest strongly that a single contact event occurred in eastern Polynesia, before the settlement of Rapa Nui, between Polynesian individuals and a Native American group most closely related to the indigenous inhabitants of present-day Colombia.
  * > Our earliest estimated date of contact is AD 1150 for Fatu Hiva, South Marquesas. This is close to the date estimated by radiocarbon dating for settlement of that island group13, raising the intriguing pos-sibility that, upon their arrival, Polynesian settlers encountered a small, already established, Native American population. It was on the island of Fatu Hiva—the easternmost island in equatorial Polynesia—that Thor Heyerdahl hypothesized that Native American and Polynesian individuals might have contacted one another, based on islanders’ legends stating that their forefathers had come from the east39. The Marquesas lie at the latitude of Ecuador, and wind- and current-based simulations indicate that they are the islands most likely to be reached from South America via the strong east-to-west currents and winds at these equatorial latitudes4,40,41.
  * > We cannot discount an alternative explanation: a group of Polyne-sian people voyaged to northern South America and returned42 together with some Native American individuals, or with Native American admix-ture, as speculated in ref. 10. We have dated the contact event to the time when Polynesian explorers were, according to some studies, making their longest-range voyages (the century surrounding AD 1200)—a time when these studies suggest that the Polynesian settlers discovered all remaining island groups in the Pacific, from Hawaii to New Zealand to Rapa Nui13,38,42. The Tuamotu Archipelago, which lies at the centre of the Polynesian islands in which we found a Native American component, is known to have been a Polynesian voyaging hub, and according to simulations it is the second most likely location to be reached when voyaging from South America4. Further population genetics collabo-rations with these genetically understudied island populations are needed to resolve these alternative hypotheses.
  * > In conclusion, we find strong genetic evidence for pre-Columbian human trans-Pacific voyaging contact (at the turn of the twelfth cen-tury), contemporaneous with the Polynesian voyages of discovery in the remote eastern Pacific13,14. Previous studies of putative Polynesian–Native American contact have focused on Rapa Nui, whose modern genetic history has been influenced by a recent Chilean admixture event, and have missed the possibility, which we show to be more likely, that prehistoric contact occurred before the settlement of Rapa Nui. We show that evidence for early Native American contact is found on widely separated islands across easternmost Polynesia, including islands not influenced by more recent Native American contact events. Our results show the usefulness of genetic studies of modern populations, which allow for large sample sizes to unravel complex prehistoric questions, and demonstrate the importance of combining anthropological, mathematical and biological approaches to answer these questions.
  * [Armitage, Hanae. Polynesians, Native Americans made contact before European arrival, genetic study finds. Stanford Medicine News Center. July 8. 2020.](https://med.stanford.edu/news/all-news/2020/07/polynesians-and-native-americans-made-early-contact.html)
    * > Before the study brought scientific evidence to the debate, the idea that Native Americans and Polynesians had crossed paths originated from a complex — both in its structure and origins — carbohydrate: the sweet potato. It turns out the sweet potato, which was originally domesticated in South and Central America, has also been known to grow in one other place prior to European contact. That place is known as Oceania, which consists of many islands, including the Polynesian islands.
    * > “The sweet potato is native to the Americas, yet it’s also found on islands thousands of miles away,” Ioannidis said. “On top of that, the word for sweet potato in Polynesian languages appears to be related to the word used in Indigenous American languages in the Andes.”
    * > The overlap in culture made some archaeologists and historians think it was not only feasible, but likely, that the potato’s arrival in Polynesia was the result of the two peoples mingling.
    * > The researchers believe that the Polynesians landed in what is now Colombia. It is also possible, though less likely due to their coastal travel norms, that one or two ships carrying Native Americans could have sailed off course and run into Polynesia, Ioannidis said.
    * > Without scientific evidence, the idea of overlap was just conjecture. Earlier, other groups of researchers turned to the genetics of the sweet potato, hoping to show that the domesticated potatoes from South America and Polynesia were genetically one and the same. But their efforts to trace the tubers have been inconclusive, as the sweet potato’s genetic origins were too complex to definitively point to human-mediated spread.
  * [Zimmer, Carl. Some Polynesians Carry DNA of Ancient Native Americans, New Study Finds. The New York Times. July 8. 2020.](https://www.nytimes.com/2020/07/08/science/polynesian-ancestry.html)
    * > But six people had no European ancestry at all. Their Native American ancestry had a different source: the Zenu population of Colombia. The scientists then found some of the same pieces of DNA in people on four other islands in eastern Polynesia.
    * > The researchers found that all of the Zenu-like stretches of DNA in the Polynesians were roughly the same size. They estimated that they came from Zenu relatives who lived about eight centuries ago.
* [Posth C et al. (2018) Reconstructing the Deep Population History of Central and South America. Cell. DOI:https://doi.org/10.1016/j.cell.2018.10.027](https://www.cell.com/cell/fulltext/S0092-8674(18)31380-1)
  * > Recent analyses have also shown that some groups in Brazil share more alleles with Australasians (indigenous New Guineans, Australians, and Andaman Islanders) (Raghavan et al., 2015, Skoglund et al., 2015) and an ∼40,000 BP individual from northern China (Yang et al., 2017) than do other Central and South Americans. Such patterns suggest that these groups do not entirely descend from a single homogeneous population and instead derive from a mixture of populations, one of which, Population Y, bore a distinctive affinity to Australasians. Notably, our study includes data from individuals such as those from the Lapa do Santo site who have a cranial morphology known as “Paleoamerican,” argued to indicate two distinct New-World-founding populations (von Cramon-Taubadel et al., 2017). Here, we test directly the hypothesis that a Paleoamerican cranial morphology was associated with a lineage distinct from the one that contributed to other Native Americans (whether the proposed Population Y or another).
* [Moreno-Mayar JV et al. (2018) Early human dispersals within the Americas. Science. 10.1126/science.aav2621](https://www.researchgate.net/publication/328815661_Early_human_dispersals_within_the_Americas) &mdash; This is a fantastic paper. I'd love to apply it's techniques to a broader sample of populations and early DNA sequences.
  * > To test for the Australasian genetic signal in NAs, we computed D statistics of the form D(NA, NA; Eurasian, Yoruba), where NA represents all newly sequenced and reference high-depth NA genomes (13). In agreement with previous results (6), we found that the Amazonian Suruí share a larger proportion of alleles with Australasian groups (represented by Papuans, Australians, and Andaman Islanders) than do the Mixe (13). Lagoa Santa yielded results similar to those obtained for the Suruí: The analyzed Lagoa Santa genome also shares a larger proportion of alleles with Australasian groups, but not with other Eurasians, than do Mesoamerican groups (the Mixe and Huichol) (Fig. 4) (13). However, the Australasian signal is not present in the Spirit Cave individual, and we include this distinction in the admixture graph modeling (Figs. 3A and 4A) (13). We inferred less than 3% European contamination in the Lagoa Santa genome (<3%) (13) and show that this finding is robust to potential European contamination by computing “contamination-corrected” f4 statistics (Fig. 4B) (13). The presence of the Australasian genomic signature in Brazil 10.4 ka ago and its absence in all genomes tested to date that are as old or older and located farther north present a challenge in accounting for its presence in Lagoa Santa.
* [Yang, M. A., Gao, X., Theunert, C., Tong, H., Aximu-Petri, A., Nickel, B., ... & Fu, Q. (2017). 40,000-year-old individual from Asia provides insight into early population structure in Eurasia. Current Biology, 27(20), 3202-3208.](https://doi.org/10.1016/j.cub.2017.09.030)
  * This paper shows there is a deep genetic connection between an ancestral population of the Papuan, Tianyuan, [Surui](https://www.wikiwand.com/en/Suru%C3%AD), [Karitiana](https://www.wikiwand.com/en/Karitiana), and [Chané](https://www.wikiwand.com/es/Chan%C3%A9s).
  * Figure 3 shows some interesting likely relationships between ancestors of the [Amis](https://www.wikiwand.com/en/Amis_people) ([mas información en Espanol](https://www.wikiwand.com/es/Ami_(pueblo))), ancestors of the [Papuan Peoples](https://www.wikiwand.com/en/Indigenous_people_of_New_Guinea) and ancestors of the Mixe (which likely form a proxy for much of Native American genetics).
    * Amis:
      * [Amis Music | International Conference on Formosan Indigenous Peoples | Academia Sinica | Taiwan](https://www.youtube.com/watch?v=qdJBb5466fY)
      * [Ami People - Elder's Drinking Song](https://youtu.be/8OkihQez4Dw?t=59) &mdash; Also see [A real “Return to Innocence” - Amis Elder's Drinking Song by Difang Duana (Story behind Enigma song)](https://www.youtube.com/watch?v=1g-Q6GPsMII)
      * [＜Taiwan＞Amis Traditional Dance Festpac in Guam](https://www.youtube.com/watch?v=K__n2hlzVsg)
      * [Amis tribe of Taiwan singing a greetingg song "milalaliu gami"](https://www.youtube.com/watch?v=YLzP4TILqNQ)
      * [Amis Aboriginal music, Taiwan (3)](https://www.youtube.com/watch?v=espIlnVQZm0)
      * [Austronesian Language Introduction - Amis Tribe - Taiwan](https://www.youtube.com/watch?v=Mkf5Sa7X8Kg)
    * [Crossing the Wallace Line - languages, genes and a forgotten history](https://www.youtube.com/watch?v=cvWDO4QSMuQ)
    * [Indigenous Music of Taiwan: Angelic Voices of Motherland](https://beehy.pe/indigenous-music-taiwan-angelic-voices-motherland/)
    * Intriguingly, [on this papuan regalia](https://en.wikipedia.org/wiki/File:Mall_culture_jakarta102.jpg), there is the xicalcoliuhqui (twisted gourd) motif. Perhaps this is how Lagenaria siceraria came to be in the Americas so long ago?
      * [Erickson, D. L., Smith, B. D., Clarke, A. C., Sandweiss, D. H., & Tuross, N. (2005). An Asian origin for a 10,000-year-old domesticated plant in the Americas. Proceedings of the National Academy of Sciences, 102(51), 18315-18320.](https://doi.org/10.1073/pnas.0509279102)
      * [Lepš, J. (2015) ETHNOBOTANY OF PENIS SHEATHES (PHALLOCRYPTS) IN PAPUA NEW GUINEA. Botanical Electronic News, 490.](http://web.archive.org/web/20180821181414/https://www.ou.edu/cas/botany-micro/ben/ben490.html)
      * [Heiser Jr, C. B. (1973). The penis gourd of New Guinea. Annals of the Association of American Geographers, 63(3), 312-318.](https://www.jstor.org/stable/2561996)
  * > By at least 45,000 years before present, anatomically modern humans had spread across Eurasia [1, 2, 3], but it is not well known how diverse these early populations were and whether they contributed substantially to later people or represent early modern human expansions into Eurasia that left no surviving descendants today. Analyses of genome-wide data from several ancient individuals from Western Eurasia and Siberia have shown that some of these individuals have relationships to present-day Europeans [4, 5] while others did not contribute to present-day Eurasian populations [3, 6]. As contributions from Upper Paleolithic populations in Eastern Eurasia to present-day humans and their relationship to other early Eurasians is not clear, we generated genome-wide data from a 40,000-year-old individual from Tianyuan Cave, China, [1, 7] to study his relationship to ancient and present-day humans. We find that he is more related to present-day and ancient Asians than he is to Europeans, but he shares more alleles with a 35,000-year-old European individual than he shares with other ancient Europeans, indicating that the separation between early Europeans and early Asians was not a single population split. We also find that the Tianyuan individual shares more alleles with some Native American groups in South America than with Native Americans elsewhere, providing further support for population substructure in Asia [8] and suggesting that this persisted from 40,000 years ago until the colonization of the Americas. Our study of the Tianyuan individual highlights the complex migration and subdivision of early human populations in Eurasia.
  * > Most Asian and Native American populations share similar numbers of alleles with the Tianyuan individual (Tables S2Bv and S2Dviii). **However, three South American populations—the Surui and Karitiana in Brazil (“Amazonians”) and the Chane in northern Argentina and southern Bolivia—share more alleles with the Tianyuan individual than other Native American populations do (Figure 3A; Tables S2E, S2H, and S2J). The two Amazonian populations were recently shown to share more alleles with the present-day Papuan and Andamanese Onge than with other Native Americans [8, 30, 31] (Figures 3B and 3C), suggesting that at least two populations contributed ancestry to Native Americans in Central and South America.** A 12,000-year-old individual from North America (Anzick-1 [32]) does not share more alleles with the Tianyuan individual (or with Oceanians or the Andamanese [8]) than with other Native Americans (Figures 3A–3C; Tables S2Dviii and S2J). The Surui and Chane show the highest levels of allele sharing with the Tianyuan individual (D(Surui/Chane, Mixe, Tianyuan, Mbuti) = 0.02, Z > 3; Table S2J), which is higher than, or similar to, levels of allele sharing with the Papuan or Onge (Table S2J). Using an analysis robust to uncertainty of the exact population history [9], we find that the Amazonians can be described as a mixture of other Native American populations and 9%–15% of an ancestral population related to the Tianyuan individual, the Papuan, or the Onge (SE 4%–10%; Table S2G). **Although the SE is high, we note that the Amazonians are consistently modeled as a mixture of other Native Americans and the Tianyuan individual, the Papuan, or the Onge. The mixture proportion estimates are also similar across all analyses, indicating that the relationship between the Tianyuan individual and the Amazonians is similar to that reported between the Papuan and the Amazonians and Onge and the Amazonians.**
  * > We also studied a model relating the Tianyuan individual to other ancient individuals and present-day populations using a base model including the Altai Neanderthal, Denisovan, Ust’-Ishim, and Kostenki14 from Mallick et al. [33]. We added the Tianyuan individual, Mal’ta1, and the present-day Ami, Mixe, Surui, and Papuan. Because present-day European populations have recent ancestry from an unknown Basal Eurasian population, we use Kostenki14, which has recently been shown to have no Basal Eurasian ancestry [4] to represent Europeans. We caution that our model (Figure 3D) is unlikely to reflect the true population relationships, as we cannot model many demographic features, such as population structure or continuous migration [11]. Intriguingly, however, within this simple model, we find that all three South American populations can be modeled as sharing ancestry not only with other Native Americans, but also with populations related to the Tianyuan individual and the Papuan (Figure 3D; Table S2I). The fact that the Tianyuan individual, who lived in mainland Asia about 40,000 years ago, has affinities to some South American populations that is as strong as or stronger than that observed for the Papuan and Onge suggests that a population related to the Tianyuan individual, as well as to the present-day Papuan and Onge, was once widespread in eastern Asia. This group or another Asian population related to this group persisted at least until the colonization of the Americas and contributed to the genomes of some Native American populations.
* [Kanzawa-Kiriyama, H., Kryukov, K., Jinam, T. A., Hosomichi, K., Saso, A., Suwa, G., ... & Saitou, N. (2017). A partial nuclear genome of the Jomons who lived 3000 years ago in Fukushima, Japan. Journal of human genetics, 62(2), 213-221.](https://doi.org/10.1038/jhg.2016.110)
  * Suggests that genes flowed from Karitiana (Native American, Brazil) into Mal'ta (MA1), not the other way around like "Genetic evidence for two founding populations of the Americas".
* [Fehren-Schmitz, L., Jarman, C. L., Harkins, K. M., Kayser, M., Popp, B. N., & Skoglund, P. (2017). Genetic ancestry of Rapanui before and after European contact. Current Biology, 27(20), 3209-3215.](https://doi.org/10.1016/j.cub.2017.09.029)
  * > Here we present, to our knowledge, the first genome-wide data from ancient Rapanui that allow direct testing of the timing of Native American admixture observed in Rapanui today. While some previous genetic studies of contemporary indigenous inhabitants of Rapa Nui concluded that Native American admixture occurred prior to European contact [5, 6, 8], none of our autosomal DNA (aDNA) analyses of pre- or post-European-contact Rapanui individuals detected Native American ancestry. Although the genomic data we obtained are limited due to unfavorable DNA preservation, the data are statistically well-powered to confirm that all ancient Rapanui samples analyzed here fall into the genomic variation found within ancient and modern Polynesians [2, 15]. Our direct aDNA evidence from three pre-European-contact and two post-European-contact Rapanui suggests that any European-mediated genetic contact between Native Americans and Easter Islanders had not spread to the entire population earlier than the 19th century. However, by use of contemporary Rapanui samples, Moreno-Mayar et al. [6] estimated the Native American admixture event at ∼22 generations ago, which translates to 1310–1420 CE if assuming a generation time of 25–30 years [32]. Although it is theoretically possible that pre-European-contact admixture with Native Americans occurred only within some spatiopolitical groups on Rapa Nui [3], to which the five individuals investigated here did not belong, we do not regard this as plausible. Rapa Nui is a very small island, and Anakena is one of the few beaches that are suitable for landing boats [3], meaning that cross-island interactions would be likely at this site. Furthermore, given the wide time span represented by our samples as well as the intrusive burial of the post-European-contact individuals, it is unlikely that all of our individuals represent the same group.
* [Skoglund, P., Mallick, S., Bortolini, M. C., Chennagiri, N., Hünemeier, T., Petzl-Erler, M. L., ... & Reich, D. (2015). Genetic evidence for two founding populations of the Americas. Nature, 525(7567), 104-108.](https://www.nature.com/articles/nature14895) &mdash; [PDF](https://www.researchgate.net/publication/280236692_Genetic_evidence_for_two_founding_populations_of_the_Americas)
  * Shows that Pima and Mixe groups are likely descended from proto-MA1 and proto-Han. Population Y is admixture of proto-Onge and proto-Pima-Mixe, Xavante, Surui, Karitiana lineages derive from Mixe and Population Y.
  * > The geographic distribution of the shared genetic signal between South Americans and Australasians cannot be explained by post-Columbian African, European or Polynesian gene flow into Native American populations. If such gene flow produced signals strong enough to impact our statistics, our statistics would show their strongest deviations from zero for African, European or Polynesian populations, which is not observed. For example, a direct test is significant in showing that the Suruí-specific ancestry component is genetically closer to the Andamanese Onge than to Tongans from Polynesia (D = 0.0094, Z = 3.4).
  * > We are unable to fit Amazonians as forming a clade with the Mesoamericans, or as having a different proportion of ancestry related to Mal'ta or present-day East Asians. Thus, our signal cannot be explained by lineages that have previously been documented as having contributed to Native American populations. However, we do find that a model where Amazonians receive ancestry from the lineage leading to the Andamanese fits the data in the sense that its predicted f4-statistics are all within 2 standard errors of statistics computed on the empirical data (Extended Data Figure 6; Extended Data Figure 7; Extended Data Table 3). These results do not imply that an unmixed population related anciently to Australasians migrated to the Americas. While this is a formal possibility, an alternative model that we view as plausible is that the ‘Population Y’ (we use ‘Population Y’ after Ypykuéra, which means ‘ancestor’ in the Tupi language family spoken by the Suruí and Kartiana) that contributed Australian related ancestry to Amazonians was already mixed with a lineage related to First Americans at the time it reached Amazonia. When we model such a scenario, we obtain a fit for models that specify 2%-85% of the ancestry of the Suruí, Karitiana, and Xavante as coming from Population Y (Figure 2). These results show that quite a high fraction of Amazonian ancestry today plausibly comes from Population Y. At the same time, the results constrain the fraction of Amazonian ancestry that comes from an Australasian related population (via Population Y) to a much tighter range of 1%-2% (Figure 2).
  * > We have provided compelling evidence that a Population Y that has ancestry from a lineage more closely related to present-day Australasians than to present-day East Asians and Siberians, contributed a small fraction of the DNA of Native Americans from Amazonia and the Central Brazilian Plateau. This discovery is striking in light of interpretations of the morphology of some early Native American skeletons, which some authors have suggested have affinities to Australasian groups. The largest number of skeletons that have been described as having this craniofacial morphology and that date to younger than ten thousand years have been found in Brazil6, the home of the Suruí, Karitiana and Xavante who in genetic data show the strongest affinity to Australasians. However, in the absence of DNA directly extracted from a skeleton with this morphology, our results are not sufficient to conclude that the Population Y we have reconstructed from the genetic data had this morphology.
  * > An open question is when and how Population Y ancestry reached South America. There are several archaeological sites in the Americas that are substantially older than Clovis sites. Since one Clovis site is now known from ancient DNA analysis to have included an individual of entirely First American ancestry3, an interesting hypothesis is that Population Y ancestry may have been prevalent in the individuals of some of these earlier sites. Regardless, our results suggest that at least two different ancestry streams penetrated south of the Late Pleistocene ice sheets, perhaps taking different routes or arriving at different times from a structured Beringian or Northeast Asian source, or reflecting more longstanding gene flow. **The genetic data allow us to say with confidence that Population Y ancestry arrived south of the ice sheets anciently: the fact that the geographically diverse Andamanese, Australian and New Guinean populations are all similarly related to this source suggests that the population is no longer extant, and the absence of long-range admixture linkage disequilibrium suggests that the population mixture did not occur in the last few thousand years.** Further insight into the population movements responsible for these findings should be possible through genome-wide analysis of ancient remains from across the Americas.
* [Raghavan, M., Steinrücken, M., Harris, K., Schiffels, S., Rasmussen, S., DeGiorgio, M., ... & Willerslev, E. (2015). Genomic evidence for the Pleistocene and recent population history of Native Americans. Science, 349(6250), aab3884.](https://doi.org/10.1126/science.aab3884)
  * > To ascertain the population history of present-day Native American populations in relation to worldwide populations, we generated admixture graphs with TreeMix (28, 37). All of the modern Siberian and Native American genomes sequenced in this study, except for the North American Tsimshian genome that showed evidence of recent western Eurasian admixture (28), were used for this analysis, together with previously published genomes from Africa (Yoruba) (38), Europe (Sardinian and French) (38), East Asia (Dai and Han) (38), Siberia (Nivkh) (39) and the Americas (Karitiana, Athabascan, and Greenlandic Inuit) (table S1) (5, 38, 39). The ancient individuals included in the analysis were Saqqaq, Anzick-1, and the 24,000-year-old Mal’ta child from south-central Siberia (4). By use of TreeMix, we affirmed that all Native Americans form a monophyletic group across all 10 migration parameter values, with further diversification into two branches, one representing Amerindians (represented in this analysis by Amerindians from southern North America and Central and South America) and the other representing Athabascans (Fig. 1B and fig. S5). Paleo-Eskimos and Inuit were supported as a separate clade relative to the Native Americans, as reported previously (Fig. 1B and fig. S5) (29, 39). Our results show that the Siberian Yupik and Koryak are the closest Eurasian populations to the Americas, with the Yupik likely representing back-migration of the Inuit into Siberia (Fig. 1B and fig. S5).
  * > Overall, our results support a common Siberian origin for all Native Americans, contradicting claims for an early migration to the Americas from Europe (49), with their initial isolation and entrance into the Americas occurring no earlier than 23 ka, but with subsequent admixture with East Asian populations. This additionally suggests that the Mal’ta-related admixture into the early Americans (4), representing ancestors of both Amerindians and Athabascans (Fig. 1 and fig. S5), occurred sometime after 23 ka, after the Native American split from East Asians.
  * > When testing for gene flow between Athabascans and Inuit with masked SNP chip genotype data–based D-statistics (47) (fig. S8), we observed a weak tendency for the Inuit to be much closer to the Athabascans than to certain Amerindians such as the North American Algonquin and Cree, and the Yaqui and Arhuaco of Central and South America (respectively), as compared with other Amerindians such as the Palikur and Surui of Brazil (fig. S8).
  * > To further investigate this trend, we tested for additional gene flow from Eurasian populations into the Americas with D-statistics using the masked SNP chip genotype data set (47). We found that some American populations—including the Aleutian Islanders, Surui, and Athabascans—are closer to Australo-Melanesians as compared with other Native Americans, such as North American Ojibwa, Cree, and Algonquin and the South American Purepecha, Arhuaco, and Wayuu (fig. S10). The Surui are, in fact, one of closest Native American populations to East Asians and Australo-Melanesians, the latter including Papuans, non-Papuan Melanesians, Solomon Islanders, and South East Asian hunter-gatherers such as Aeta (fig. S10). We acknowledge that this observation is based on the analysis of a small fraction of the whole-genome and SNP chip genotype data sets—especially for the Aleutian Islander data, which is heavily masked owing to recent admixture with Europeans (28)—and that the trends in the data are weak.
  * > Nonetheless, if it proves correct, **these results suggest that there may be a distant Old World signal related to Australo-Melanesians and East Asians in some Native Americans. The widely scattered and differential affinity of Native Americans to the Australo-Melanesians, ranging from a strong signal in the Surui to a much weaker signal in northern Amerindians such as Ojibwa, points to this gene flow occurring after the initial peopling by Native American ancestors.**
  * > However, how this signal may have ultimately reached South America remains unclear. One possible means is along a northern route via the Aleutian Islanders, previously found to be closely related to the Inuit (39), who have a relatively greater affinity to East Asians, Oceanians, and Denisovan than Native Americans in both whole-genome and SNP chip genotype data–based D tests (table S10 and figs. S10 and S11). On the basis of archaeological evidence and mtDNA data from ancient and modern samples, the Aleutian Islands are hypothesized to have been peopled as early as ~9 ka by “Paleo-Aleuts” who were succeeded by the “Neo-Aleuts,” with present-day Aleutian Islanders potentially resulting from admixture between these two populations (52, 53). Perhaps their complex genetic history included input from a population related to Australo-Melanesians through an East Asian continental route, and this genomic signal might have been subsequently transferred to parts of the Americas, including South America, through past gene flow events (Fig. 1). **Evidence for this gene flow is supported with diCal2.0 and MSMC analyses showing a weak but recent gene flow into South Americans from populations related to present-day Northeast Asians (Koryak) (Fig. 2C and table S11C), who might be considered a proxy for the related Aleutian Islanders.**
  * > That Native Americans diverged from their East Asian ancestors during the LGM and no earlier than 23 ka provides an upper bound, and perhaps the climatic and environmental context, for the initial isolation of their ancestral population and a maximum estimate for the entrance and subsequent spread into the Americas. This result is consistent with the model that people entered the Americas before the development of the Clovis complex and had reached as far as southern South America by 14.6 ka. Because archaeological evidence provides only a minimum age for human presence in the Americas, we can anticipate the possible discovery of sites that approach the time of the divergence of East Asians and Native Americans. However, our estimate for the initial divergence and entry of Native American ancestors does not support archaeological claims for an initial peopling substantially earlier than the LGM (8–10).
  * > Although our data cannot provide the precise geographical context for the initial peopling process, it has allowed us to more accurately estimate its temporal dynamics. This, in turn, has enabled us to reassess the Beringian Incubation Model, which, based on mtDNA data and the timing and geographical distribution of archaeological sites, hypothesized a ~15,000-year-long period of isolation of ancestral Native Americans in Beringia during the LGM (19–21). Our results, along with recent findings of mtDNA haplogroup C1 in Iceland and ancient northwest Russia (60), do not fit with the proposed 15,000-year span of the Beringian Incubation Model (19–21). It is possible that a shorter period of isolation occurred (~8000 years), but whether it occurred in Siberia or Beringia will have to be determined from future ancient DNA and archaeological findings. **Given the genetic continuity between Native Americans and some East Asian populations (figs. S4 and S5), other demographic factors, such as surfing during population expansions into unoccupied regions (61), may ultimately need to be taken into account to better understand the presence of a large number of high-frequency private variants in the indigenous populations of the Americas.**
  * > The data presented here are consistent with a single initial migration of all Native Americans and with later gene flow from sources related to East Asians and, more distantly, Australo-Melanesians. From that single migration, there was a diversification of ancestral Native Americans leading to the formation of northern and southern branches, which appears to have taken place ~13 ka within the Americas. This split is consistent with the patterns of uniparental genomic regions of mtDNA haplogroup X and some Y chromosome C haplotypes being present in northern, but not southern, populations in the Americas (18, 62). This diversification event coincides roughly with the opening of habitable routes along the coastal and the interior corridors into unglaciated North America some 16 and 14 ka, respectively (63, 64), suggesting a possible role of one or both of these routes in the isolation and subsequent dispersal of Native Americans across the continent.
* [Moreno-Mayar, J. V., Rasmussen, S., Seguin-Orlando, A., Rasmussen, M., Liang, M., Flåm, S. T., ... & Willerslev, E. (2014). Genome-wide ancestry patterns in Rapanui suggest pre-European admixture with Native Americans. Current Biology, 24(21), 2518-2525.](https://doi.org/10.1016/j.cub.2014.09.057)
  * > The possibility of trans-Pacific Polynesian-American contact has been proposed by archaeologists since the early 1900s ([28], p. 1). Although an early contact has been proposed by previous studies [15, 16, 17], no unequivocal genetic evidence has been reported to date. Our results suggest that Native American admixture occurred prior to European admixture. Specifically, we detect a lower variance in Native American ancestry components, and smaller ancestry tracts, as compared to Europeans. To characterize the timing of the admixture event in a statistical framework, we infer parameters from specific demographic scenarios, using a maximum-likelihood method that has been validated before in similar human data sets (e.g., [45]), as well as a new method based on the variance in admixture proportions. Using these approaches, we can statistically reject “European first” scenarios. It is noteworthy that models with several pulses are a generalization of single-pulse models and will therefore provide a better fit to the data. In our case, we find that (1) adding a second European pulse does not improve the fit significantly when assuming that the European gene flow postdated the first documented presence of Europeans in the Pacific in AD 1520 [46], and (2) the second pulse was found to be of such low magnitude that the associated admixture time spanned a whole range of times equally well (Figure S5). 
  * > Although our data suggest that modern-day Rapanui are the result of a pre-Columbian gene flow event between Native Americans and Polynesians, we cannot, with the data at hand, determine how this gene flow happened. Nevertheless, it is interesting to consider the evidence presented in other fields. There are two main scenarios compatible with the data: Native Americans sailing to Rapa Nui, or Polynesians sailing to the Americas and back. Although “boats on occasion can sail from almost anywhere to anywhere” [47], from a purely seafaring capability perspective, it is probably fair to say that the latter scenario is more likely. Let us consider the geographical situation of Rapa Nui and the feasibility of such a voyage, especially in the context of the Polynesian expansion.
  * > Jones et al. [28] reviewed in 2011 the archaeological, physical anthropological, linguistic, and navigational evidence for pre-Columbian contact. They identified three regions of potential contact, namely Chile, the Gulf of Guayaquil in Ecuador, and the Santa Barbara Channel in California, and constrained the contacts to a period between AD 700 and AD 1350. They argued that the most parsimonious explanation in all cases is a return voyage by Polynesians, but they could not exclude the scenario of a one-way raft trip, as suggested for example in the mid-20th century by Thor Heyerdahl [11]. One of their primary arguments is the lack of Native American influence in Polynesia, particularly in terms of Polynesian architecture ([28], pp. 263–265)—aside, perhaps, from one of the platform sites on Rapa Nui, Ahu Vinapu.
* [Gonçalves, V. F., Stenderup, J., Rodrigues-Carvalho, C., Silva, H. P., Gonçalves-Dornelas, H., Líryo, A., ... & Pena, S. D. J. (2013). Identification of Polynesian mtDNA haplogroups in remains of Botocudo Amerindians from Brazil. Proceedings of the National Academy of Sciences, 110(16), 6465-6469.](https://dx.doi.org/10.1073%2Fpnas.1217905110)
  * > There is a consensus that modern humans arrived in the Americas 15,000–20,000 y ago during the Late Pleistocene, most probably from northeast Asia through Beringia. However, there is still debate about the time of entry and number of migratory waves, including apparent inconsistencies between genetic and morphological data on Paleoamericans. Here we report the identification of mitochondrial sequences belonging to haplogroups characteristic of Polynesians in DNA extracted from ancient skulls of the now extinct Botocudo Indians from Brazil. The identification of these two Polynesian haplogroups was confirmed in independent replications in Brazil and Denmark, ensuring reliability of the data. Parallel analysis of 12 other Botocudo individuals yielded only the well-known Amerindian mtDNA haplogroup C1. Potential scenarios to try to help understand these results are presented and discussed. The findings of this study may be relevant for the understanding of the pre-Columbian and/or post-Columbian peopling of the Americas.
  * **NOTE**: The Botocudo are not fully extinct. As of [2010, 350 Krenak (their descendants) were alive in Minas Gerais](https://en.wikipedia.org/wiki/Aimor%C3%A9).

## Genetic Clues from Lice

* [Pedersen, M. W., Antunes, C., De Cahsan, B., Moreno-Mayar, J. V., Sikora, M., Vinner, L., ... & Perotti, M. A. (2021). Ancient human genomes and environmental DNA from the cement attaching 2,000 year-old head lice nits. Molecular biology and evolution.](https://doi.org/10.1093/molbev/msab351)
  * [Handwerk, B. (2021) DNA Preserved in Lice Glue Reveals South American Mummies’ Secrets. Smithsonian Magazine. 28 December.](https://www.smithsonianmag.com/science-nature/mummies-secrets-revealed-by-ancient-dna-preserved-in-lice-cement-180979300/)
* [Arriaza, B., Standen, V. G., Heukelbach, J., Cassman, V., & Olivares, F. (2014). Head combs for delousing in ancient arican populations: scratching for the evidence. Chungara, Revista de Antropología Chilena, 46(4), 693-706.](https://www.redalyc.org/articulo.oa?id=32633279013)
  * [Loewy, M. (2015) Oldest Lice Combs in the Americas Discovered. Scientific American. 20 August.](https://www.scientificamerican.com/article/oldest-lice-combs-in-the-americas-discovered/)
* [Ascunce MS, Toups MA, Kassu G, Fane J, Scholl K, Reed DL (2013) Nuclear Genetic Diversity in Human Lice (Pediculus humanus) Reveals Continental Differences and High Inbreeding among Worldwide Populations. PLoS ONE 8(2): e57619.](https://doi.org/10.1371/journal.pone.0057619)
  * > Understanding the processes that shape the genetic structure of human parasite populations is important to both basic and applied evolutionary biology. In human lice, knowing the processes and mechanisms that maintain and generate human louse genetic diversity within and among host populations is critical for our ability to design effective control methods and to predict how louse-borne diseases can spread through human populations. In this study, we showed that human louse populations are genetically structured based on geography (Figure 6). The high degree of genetic structure may lead to the evolution of different resistance alleles among different populations, thus suggesting the need of regional epidemiological studies to control lice. Furthermore, this work has shown that the study of the genetic diversity of human lice could help us better understand patterns of human migrations worldwide at different temporal scales, and could also be used to test hypotheses about human evolution such as ecological interactions between modern and archaic hominins.
  * [Nuwer, R. (2013) Nitpicking the Lice Genome to Track Humanity’s Past Footsteps. Smithsonian Magazine. 27 February.](https://www.smithsonianmag.com/science-nature/nitpicking-the-lice-genome-to-track-humanitys-past-footsteps-29121653/)
* [Arriaza, B., Standen, V., Reinhard, K., Araújo, A., Heukelbach, J., & Dittmar, K. (2013). On head lice and social interaction in archaic Andean coastal populations. International Journal of Paleopathology, 3(4), 257-268.](https://digitalcommons.unl.edu/cgi/viewcontent.cgi?article=1000&context=natresreinhard)
* [Arriaza, B., Orellana, N. C., Barbosa, H. S., Menna-Barreto, R. F., Araújo, A., & Standen, V. (2012). Severe head lice infestation in an Andean mummy of Arica, Chile. Journal of Parasitology, 98(2), 433-436.](https://www.researchgate.net/profile/Helene-Barbosa/publication/51729750_Severe_Head_Lice_Infestation_in_an_Andean_Mummy_of_Arica_Chile/links/55a7cbbf08ae1dca686f9269/Severe-Head-Lice-Infestation-in-an-Andean-Mummy-of-Arica-Chile.pdf)

## Genetic Clues through H. Pylori

* [Moodley, Y., Brunelli, A., Ghirotto, S., Klyubin, A., Maady, A. S., Tyne, W., ... & Achtman, M. (2021). Helicobacter pylori’s historical journey through Siberia and the Americas. Proceedings of the National Academy of Sciences, 118(25).](https://doi.org/10.1073/pnas.2015523118)
  * > hspIndigenousAmericas suffered a very severe demographic bottleneck during the initial stage of its colonization of North America, comprising no more than 10 to 854 effective bacteria. However, our models cannot distinguish whether this bottleneck occurred during the migration from Siberia into Beringia and/or from isolation and genetic drift after crossing the Bering Strait. Our analyses estimate an increase in population size about 7 kya (95% HPD 680 to 16,849 y), but we consider the upper limit to be 12,500 y, when the recession of Cordilleran (west) and Laurentide (east) ice sheets created a corridor for human migration (58, 59). Alternatively, the earliest North American settlers would have to have migrated south along a coastal route (49) or by boats (60).


## Early Homid through Modern Human Genetics
* [Yang, M. A., & Fu, Q. (2018). Insights into modern human prehistory using ancient genomes. Trends in Genetics, 34(3), 184-196.](https://doi.org/10.1016/j.tig.2017.11.008)
* [Massilani, D., Morley, M. W., Mentzer, S. M., Aldeias, V., Vernot, B., Miller, C., ... & Meyer, M. (2022). Microstratigraphic preservation of ancient faunal and hominin DNA in Pleistocene cave sediments. Proceedings of the National Academy of Sciences, 119(1).](https://doi.org/10.1073/pnas.2113666118)
* [Zhang, P., Zhang, X., Zhang, X., Gao, X., Huerta-Sanchez, E., & Zwyns, N. (2021). Denisovans and Homo sapiens on the Tibetan Plateau: dispersals and adaptations. Trends in ecology & evolution.](https://doi.org/10.1016/j.tree.2021.11.004)
  * > Recent archaeological discoveries suggest that both archaic Denisovans and Homo sapiens occupied the Tibetan Plateau earlier than expected. Genetic studies show that a pulse of Denisovan introgression was involved in the adaptation of Tibetan populations to high-altitude hypoxia. These findings challenge the traditional view that the plateau was one of the last places on earth colonized by H. sapiens and warrant a reappraisal of the population history of this highland. Here, we integrate archaeological and genomic evidence relevant to human dispersal, settlement, and adaptation in the region. We propose two testable models to address the peopling of the plateau in the broader context of H. sapiens dispersal and their encounters with Denisovans in Asia.
* [Brown, S., Massilani, D., Kozlikin, M. B., Shunkov, M. V., Derevianko, A. P., Stoessel, A., ... & Douka, K. (2021). The earliest Denisovans and their cultural adaptation. Nature ecology & evolution, 1-8.](https://doi.org/10.1038/s41559-021-01581-2)
  * > We identified five new hominin bones, four of which contained sufficient DNA for mitochondrial analysis. Three carry mitochondrial DNA of the Denisovan type and one was found to carry mtDNA of the Neanderthal type. The former come from the same archaeological layer near the base of the cave’s sequence and are the oldest securely dated evidence of Denisovans at 200 ka (thousand years ago) (205–192 ka at 68.2% or 217–187 ka at 95% probability).
* [Schaefer, N. K., Shapiro, B., & Green, R. E. (2021). An ancestral recombination graph of human, Neanderthal, and Denisovan genomes. Science advances, 7(29), eabc0776.](https://doi.org/10.1126/sciadv.abc0776)
* [Vernot, B., Zavala, E. I., Gómez-Olivencia, A., Jacobs, Z., Slon, V., Mafessoni, F., ... & Meyer, M. (2021). Unearthing Neanderthal population history using nuclear and mitochondrial DNA from cave sediments. Science, 372(6542).](https://doi.org/10.1126/science.abf1667)
* [Yuan, K., Ni, X., Liu, C., Pan, Y., Deng, L., Zhang, R., ... & Xu, S. (2021). Refining models of archaic admixture in Eurasia with ArchaicSeeker 2.0. Nature communications, 12(1), 1-15.](https://doi.org/10.1038/s41467-021-26503-5)
* [Svensson, E., Günther, T., Hoischen, A., Hervella, M., Munters, A. R., Ioana, M., ... & Jakobsson, M. (2021). Genome of Peştera Muierii skull shows high diversity and low mutational load in pre-glacial Europe. Current Biology.](https://doi.org/10.1016/j.cub.2021.04.045)
* [Prüfer, K., Posth, C., Yu, H. et al. A genome sequence from a modern human skull over 45,000 years old from Zlatý kůň in Czechia. Nat Ecol Evol (2021). https://doi.org/10.1038/s41559-021-01443-x](https://doi.org/10.1038/s41559-021-01443-x)
  * Young dates are likely from contamination. Reasons are laid out in the paper.
  * Has an interesting maximum-parsimony phylogenetic tree and admixture graph.
  * > We extracted DNA from ~15 mg bone powder from the Zlatý kůň petrous portion of the temporal bone and first enriched for and sequenced the mitochondrial genome (mtDNA) to ~150-fold coverage (Methods). Around 4% of the mtDNA sequences were estimated to stem from human contamination (Supplementary Section 3). The reconstructed mtDNA belongs to haplogroup N and its branch length, measured as the number of accumulated substitutions, is similar to those of the currently oldest sequenced modern human mtDNA genomes (Fig. 2a and Extended Data Fig. 3), including the recently published mtDNAs from Bacho Kiro, a cave in Bulgaria with remains dating to 43–47 ka1. Bayesian tip dating suggests that Zlatý kůň lived ~43 ka (95% highest posterior density = 31.5–52.6 ka).
  * [Gibbons, A. (2021) More than 45,000 years ago, modern humans ventured into Neanderthal territory. Here’s what happened next. Science News. April 7.](https://www.sciencemag.org/news/2021/04/more-45000-years-ago-modern-humans-ventured-neanderthal-territory-here-s-what-happened)
    * > Krause’s team also found that unlike the Bacho Kiro individuals, the Zlatý kůň skull was no more closely related to ancient Asians than to Europeans. This suggests she came from an ancient population that had not yet differentiated genetically into Asians and Europeans, Krause says.
    * > The new data show that all of the mod­ern human lineages in Europe vanished by the ad­vent of the last ice age, which reached its peak about 20,000 years ago. After the ice melted, other modern humans from Eur­asia repopulated the continent. “Multiple groups headed out [to Eurasia], but it seems like only a fraction of these early modern humans left progeny,” Novembre says.
* [Hajdinjak, M., Mafessoni, F., Skov, L. et al. Initial Upper Palaeolithic humans in Europe had recent Neanderthal ancestry. Nature 592, 253–257 (2021). https://doi.org/10.1038/s41586-021-03335-3](https://doi.org/10.1038/s41586-021-03335-3)
  * > When comparing the Bacho Kiro Cave individuals to present-day populations, we found that the IUP individuals share more alleles (that is, more genetic variants) with present-day populations from East Asia, Central Asia and the Americas than with populations from western Eurasia (Fig. 2a, Supplementary Information 5), whereas the later BK1653 individual shares more alleles with present-day western Eurasian populations (Extended Data Figs. 3b, 4a).
  * > We found that the Ust’Ishim and Oase1 individuals showed no more affinity to western than to eastern Eurasian populations, suggesting that they did not contribute ancestry to later Eurasian populations, as previously shown (Supplementary Information 5, Extended Data Fig. 5). By contrast, the IUP Bacho Kiro Cave individuals shared more alleles with the roughly 40,000-year-old Tianyuan individual13 from China (Fig. 2b) and other ancient Siberians and Native Americans (Fig. 2c) than with the Kostenki individual (3.6 ≤ |Z| ≤ 5.3). Among other western Eurasian Upper Palaeolithic humans, the IUP Bacho Kiro Cave individuals shared more alleles with the Oase1 (3.6 ≤ |Z| ≤ 4.3) and roughly 35,000-year-old GoyetQ116-129 individuals than with the Kostenki14 individual (3.2 ≤ |Z| ≤ 4.3; Fig. 2c, Supplementary Information 5). Notably, the GoyetQ116-1 individual has previously been shown to share more alleles with early East Asians than other individuals of a similar age in Europe.
  * > When we explored models of population history that are compatible with the observations above using admixture graphs, we found that the IUP Bacho Kiro Cave individuals were related to populations that contributed ancestry to the Tianyuan individual in China as well as, to a lesser extent, to the GoyetQ116-1 and Ust’Ishim individuals (all |Z| < 3; Fig. 2d, Supplementary Information 6). This resolves the previously unclear relationship between the GoyetQ116-1 and Tianyuan individuals13 without the need for gene flow between these two geographically distant individuals. The models also suggest that the later BK1653 individual belonged to a population that was related, but not identical, to that of the GoyetQ116-1 individual (Fig. 2d, Extended Data Fig. 4, Supplementary Information 6) and that the Věstonice cluster, whose members were found in association with Gravettian assemblages, derived part of their ancestry from such a population and the rest from populations related to the roughly 34,000-year-old ‘Sunghir’ individuals from Russia (Fig. 2d, Supplementary Information 6).
  * [Handwerk, B. (2021) Some of Europe’s Oldest-Known Modern Humans Are Distantly Related to Native Americans. Smithsonian Magazine. April 7.](https://www.smithsonianmag.com/science-nature/secrets-europes-oldest-known-modern-humans-revealed-genome-sequencing-180977437/)
* [Weiss, C. V., Harshman, L., Inoue, F., Fraser, H. B., Petrov, D. A., Ahituv, N., & Gokhman, D. (2021). The cis-regulatory effects of modern human-specific variants. Elife, 10, e63713.](https://doi.org/10.7554/eLife.63713)
* [Stepanova, V., Moczulska, K. E., Vacano, G. N., Kurochkin, I., Ju, X., Riesenberg, S., ... & Pääbo, S. (2021). Reduced purine biosynthesis in humans after their divergence from Neandertals. Elife, 10, e58741.](https://doi.org/10.7554/eLife.58741)
* [Hubisz MJ, Williams AL, Siepel A (2020) Mapping gene flow between ancient hominins through demography-aware inference of the ancestral recombination graph. PLoS Genet 16(8): e1008895. https://doi.org/10.1371/journal.pgen.1008895](https://doi.org/10.1371/journal.pgen.1008895)
* [Schierup, M. H. (2020). The last pieces of a puzzling early meeting. Science, 369(6511), 1565-1566](https://doi.org/10.1126/science.abe2766)
* [Petr, M., Hajdinjak, M., Fu, Q., Essel, E., Rougier, H., Crevecoeur, I., ... & de la Rasilla, M. (2020). The evolutionary history of Neandertal and Denisovan Y chromosomes. Science, 369(6511), 1653-1656](https://doi.org/10.1126/science.abb6460)
* [Hublin, J. J., Sirakov, N., Aldeias, V., Bailey, S., Bard, E., Delvigne, V., ... & Kromer, B. (2020). Initial Upper Palaeolithic Homo sapiens from Bacho Kiro Cave, Bulgaria. Nature, 1-4.](https://doi.org/10.1038/s41586-020-2259-z) &mdash; [PDF](https://www.researchgate.net/publication/341299437_Initial_Upper_Palaeolithic_Homo_sapiens_from_Bacho_Kiro_Cave_Bulgaria)
  * Has an interesting maximum parsimony tree.
  * > The Bacho Kiro Cave site clearly demonstrates that the IUP in this region was made by H.sapiens, and is consistent with models that attrib-ute the spread of the IUP to the dispersal of our species throughout large parts of Eurasia. The presence of IUP assemblages documents a wave of peopling that precedes the spread of the first Upper Palaeolithic bladelet techno-complexes—such as the Early Ahmarian industry in the Levant, the Early Kozarnikan industry in the eastern Balkans and the Pro-toaurignacian industry in western and central Europe—by several mil-lennia1,34. At Bacho Kiro Cave, the IUP starts before 45,000 cal. and, as the assemblage of the upper part of layer J is identical to that from layer I, it may begin as early as 47,000 cal. 3. We now have evidence for H.sapiens in Eurasia spanning from Ust'-Ishim28 in western Siberia to Bacho Kiro Cave in eastern Europe, directly dated to approximately 45,000 cal. Together, the behavioural and biological evidence strongly suggest a relatively rapid dispersal of IUP assemblages from southwest Asia35 into mid-latitude Eurasia by groups that—contrary to Aurignacian populations—seem unrelated to present-day European populations28. Direct contact with Neanderthals must have occurred much earlier in eastern Europe than in western Europe, where the latest Neanderthals and their associated assemblages persisted until at least about 40,000 cal. In Romania, the Pestera cu Oase H. sapiens individual had a Neanderthal ancestor as recently as four-to-six genera-tions back in his family tree. In light of the Bacho Kiro Cave results, the 42,000–37,000 cal. yr radiocarbon age of the Pestera cu Oase fossil implies an extended period of contact between Neanderthals and H.sapiens in eastern Europe. Alternatively, it may be that the direct date of Pestera cu Oase—which was obtained before recent improvements in pretreatment techniques—is an underestimate, and that local coexist-ence was more ancient and ephemeral. The IUP pendants of Bacho Kiro Cave (Fig.3) are notably similar to artefacts produced by late Nean-derthals of the Châtelperronian layers at Grotte du Renne (France)4. Whatever the cognitive complexity of the last Neanderthals might have been, the earlier age of the Bacho Kiro Cave material supports the notion that these specific behavioural novelties seen in declining Nean-derthal populations resulted from contacts with migrant H.sapiens.
  * [David, A. (2020) A Tooth Changes Everything We Know About Humans Reaching Europe. Haaretz. 13 May.](https://web.archive.org/web/20200604133024if_/https://www.haaretz.com/science-and-health/.premium-a-tooth-changes-everything-we-know-about-humans-reaching-europe-1.8836555)
* [Albers, P. K., & McVean, G. (2020). Dating genomic variants and shared ancestry in population-scale sequencing data. PLoS biology, 18(1), e3000586.](https://dx.doi.org/10.1371%2Fjournal.pbio.3000586)
* [Burger, J., Link, V., Blöcher, J., Schulz, A., Sell, C., Pochon, Z., ... & Reyna-Blanco, C. S. (2020). Low Prevalence of Lactase Persistence in Bronze Age Europe Indicates Ongoing Strong Selection over the Last 3,000 Years. Current Biology.](https://doi.org/10.1016/j.cub.2020.08.033)
* [Chan EKF, et al. (2019) Human origins in a southern African palaeo-wetland and first migrations. Nature. 575: 185-189.](https://www.nature.com/articles/s41586-019-1714-1) &mdash; [PDF](https://www.nature.com/articles/s41586-019-1714-1.epdf?referrer_access_token=iAfJZZ_lpZl3huBpr5zq1NRgN0jAjWel9jnR3ZoTv0Ow72lSjAIzdDOJcHUPoWIRGEOjMmthCa5BpP_dciaujFlTFsWPygPnm_eHl2AcrLFjeSO9PGAR2vyJhXma8rECK6S40cpYYTwflvHD4NiZ7HKsXBLLsVGkx8YcO8B0aOZu7MlerKh-hv7q75L8HMWR8nuZQfJdUEeusZGlVVZJe91EQF82zbGiMzbevXVwLkpyiENHRd_a17eb9Y1IDkDOLfVDnQfqUdMXlojsECkV6esUOl3P3GBBWSs3U57dPLc3Yz7vKowijwOo0uwLwoNK&tracking_referrer=www.theguardian.com)
* [Price M (2019) Ancient DNA puts a face on the mysterious Denisovans, extinct cousins of Neanderthals. Science News. September 19.](https://www.sciencemag.org/news/2019/09/ancient-dna-puts-face-mysterious-denisovans-extinct-cousins-neanderthals)
* [Feldman, M., Master, D. M., Bianco, R. A., Burri, M., Stockhammer, P. W., Mittnik, A., ... & Krause, J. (2019). Ancient DNA sheds light on the genetic origins of early Iron Age Philistines. Science Advances, 5(7), eaax0061.](https://doi.org/10.1126/sciadv.aax0061)
* [Roger AR, Harris NS, Achenbach AA (2019) Neanderthal-Denisovan ancestors interbred with a distantly-related hominin. bioRxiv preprint. 657247.](https://doi.org/10.1101/657247)
* [Shinde V et al. (2019) An Ancient Harappan Genome Lacks Ancestry from Steppe Pastoralists or Iranian Farmers. Cell.](https://doi.org/10.1016/j.cell.2019.08.048)
* [Schaefer NK, Shapiro B (2019) New middle chapter in the story of human evolution. Science. 365(6457): 981-982.](https://science.sciencemag.org/content/365/6457/981) &mdash; Has great references for an updated and more complete phylogenetic tree.
* National Geographic has a nice looking, but crude, map which traces the general flow of modern genetics out of Africa and into the rest of the world: [Map of Human Migration](https://genographic.nationalgeographic.com/human-journey/)
* [Ragsdale AP, Gravel S (2019) Models of archaic admixture and recent history from two-locus statistics. PLoS Genet 15(6): e1008204.](https://doi.org/10.1371/journal.pgen.1008204)
* [Bokelmann L et al. (2019) A genetic analysis of the Gibralter Neanderthals. PNAS. 116(31): 15610-15615.](https://doi.org/10.1073/pnas.1903984116)
* [Cepelewicz, J (2019) Fossil DNA Reveals New Twists in Modern Human Origins. Quanta Magazine. August 29. Web. Accessed September 7, 2019.](https://www.quantamagazine.org/fossil-dna-reveals-new-twists-in-modern-human-origins-20190829/)
* [Callaway, E. (2018) Divided by DNA: The uneasy relationship between archaeology and ancient genomics. Nature News. 28 March.](https://www.nature.com/articles/d41586-018-03773-6)
* [Browning SR et al. (2018) Analysis of Human Sequence Data Reveals Two Pulses of Archaic Denisovan Admixture. Cell. 173(1): 53-61.](https://doi.org/10.1016/j.cell.2018.02.031)
* [Wolf AB, Akey JM (2018) Outstanding questions in the study of archaic hominin admixture. PLoS Genet 14(5): e1007349.](https://doi.org/10.1371/journal.pgen.1007349)
* [Slon V et al. (2018) The genome of the offspring of a Neanderthal mother and a Denisovan father. Nature. 561: 113-116.](https://www.nature.com/articles/s41586-018-0455-x)
* [Brewer, B. (2021) 2021 research reinforced that mating across groups drove human evolution. 13 December.](https://www.sciencenews.org/article/human-evolution-mating-2021-research)
* [Scerri EML et al. (2018) Did Our Species Evolve in Subdivided Populations across Africa, and Why Does It Matter? Trends in Ecology & Evolution. 33(8): 582-594.](https://doi.org/10.1016/j.tree.2018.05.005)
* [Slon, V., Hopfe, C., Weiß, C. L., Mafessoni, F., De La Rasilla, M., Lalueza-Fox, C., ... & Meyer, M. (2017). Neandertal and Denisovan DNA from Pleistocene sediments. Science, 356(6338), 605-608.](https://doi.org/10.1126/science.aam9695)
* [Posth C et al. (2017) Deeply divergent archaic mitochondrial genome provides lower time boundary for African gene flow into Neanderthals. Nature Communications. 8: 16046.](https://www.nature.com/articles/ncomms16046)
  * > The presence of modern human admixture into archaic humans has already been detected in the high coverage Neanderthal genome from the Altai region but not in sequences of chromosome 21 of two Neanderthals from Spain and Croatia14. The authors therefore suggested that a genomic contribution estimated between 0.1 and 2.1% occurred after the divergence of Altai from other late Neanderthals. However, there is a high level of uncertainty around the time of the inferred gene flow event since only one high coverage Neanderthal nuclear genome has been analysed so far. Moreover, the divergence time of the introgressing African population was estimated to date before or right after the TMRCA of modern-day humans (∼200 ka)14, while the mtDNA coalescence time between Neanderthals and modern humans is calculated at least twice as old (∼400 ka). The evolutionary scenario responsible for providing the mtDNA to the Late Pleistocene Neanderthals might have been an even earlier Middle Pleistocene gene flow from Africa, occurring in a time interval that we date between 413 and 268 ka (460–219 ka including upper and lower 95% HPD). It should be highlighted that this additional genomic contribution might have already been accounted for in ref. 14, which effectively measures the total amount of African introgression into Neanderthals after their split from Denisovans (473–381 ka; ref. 5).
  * > In conclusion, the HST mtDNA provided insights into the mtDNA diversity of Neanderthal populations through the Middle and Late Pleistocene. Its deep divergence time allowed us to further constrain the lower boundary for the time of the proposed African mtDNA gene flow into Neanderthal populations. The temporal corridor for this introgression event between 460 ka and 219 ka is compatible with the evidence of archaeological similarities between Africa and western Eurasia during the Lower to Middle Paleolithic transition39 and potentially may explain the dissimilarities in Middle Paleolithic industries between eastern and western Eurasia. Environmental changes across this time span might have facilitated a hominin expansion out of Africa and potentially spread cultural innovations such as the Levallois technology into Eurasia40. Alternatively, other scenarios such as multiple inventions of similar technologies by various hominin groups may explain the complex tapestry of technological variability during the late Middle Pleistocene.
* [Schelbusch CM et al. (2017) Southern African ancient genomes estimate modern human divergence to 350,000 to 260,000 years ago. Science. 358(6363): 652-655.](https://science.sciencemag.org/content/358/6363/652)
* [Skoglund P et al. (2017) Reconstructing Prehistoric African Population Structure. Cell.1 71(1): 59-71.](https://doi.org/10.1016/j.cell.2017.08.049)
  * > Africa harbors more genetic diversity than any other part of the world (Cann et al., 1987, Tishkoff et al., 2009). This is reflected both in a higher average number of differences among sub-Saharan African genomes than among non-African genomes (Cann et al., 1987, Ramachandran et al., 2005) and in the fact that the ancestry found outside of Africa is largely a subset of that within it (Tishkoff et al., 2009).
* [Nielsen R et al. (2017) Tracing the peopling of the world through genomics. Nature. 541: 302-310.](https://www.nature.com/articles/nature21347) &mdash; Has a better map of possible migration routes which line up with indigenous histories. There's still some details missing, but it's the best 2017 had to offer.
* [Mallick S et al. (2016) The Simons Genome Diversity Project: 300 genomes from 142 diverse populations. Nature. 538: 201-206.](https://www.nature.com/articles/nature18964) &mdash; [PDF](The Simons Genome Diversity Project: 300 genomes from 142 diverse populations)
  * This has an interesting neighbor joining tree based on pairwise divergence showing relative relatedness and possible phylogeny for groups from Africa to the Americas. This also has some very intriguing connections between groups which could be interesting to explore.
* [Pagani L (2016) Genomic analyses inform on migration events during the peopling of Eurasia. Nature. 538: 238-242.](https://www.nature.com/articles/nature19792)
* [Fu W et al. (2016) Robust Inference of Identity by Descent from Exome-Sequencing Data. American Journal of Human Gentics. 99(5): 1106-1116.](https://doi.org/10.1016/j.ajhg.2016.09.011)
* [Harris K, Nielson R (2016) The Genetic Cost of Neanderthal Introgression. Genetic. 203(2): 881-891.](https://doi.org/10.1534/genetics.116.186890)
* [Kuhlwilm M et al. (2016) Ancient gene flow from early modern humans into Eastern Neanderthals. Nature. 530: 429-433.](https://www.nature.com/articles/nature16544)
* [Sankararaman, S., Mallick, S., Patterson, N., & Reich, D. (2016). The combined landscape of Denisovan and Neanderthal ancestry in present-day humans. Current Biology, 26(9), 1241-1247.](https://doi.org/10.1016/j.cub.2016.03.037)
* [Mathieson I et al. (2015) Eight thousand years of natural selection in Europe. bioRxiv preprint. 016477. doi: https://doi.org/10.1101/016477 Now published in Nature doi: 10.1038/nature16152](https://doi.org/10.1101/016477)
* [Haak W et al. (2015) Massive migration from the steppe was a source for Indo-European languages in Europe. Nature. 522(7555): 207-211](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5048219/)
* [The 1000 Genomes Project Consortium (2015) A global reference for human genetic variation. Nature. 526: 68-74.](https://www.nature.com/articles/nature15393)
* [Karmin M et al. (2015) A recent bottleneck of Y chromosome diversity coincides with a global change in culture. Genome Research. 25: 459-466.](https://genome.cshlp.org/content/25/4/459.full)
* [Fu Q et al. (2013) DNA analysis of an early modern human from Tianyuan Cave, China. PNAS. 110(6): 2223-2227.](https://doi.org/10.1073/pnas.1221359110)
* [Lachance J et al. (2012) Evolutionary History and Adaptation from High-Coverage Whole-Genome Sequences of Diverse African Hunter-Gatherers. Cell. 150(3): 457-469.](https://doi.org/10.1016/j.cell.2012.07.009)
* [Wilkins, J., Schoville, B. J., Brown, K. S., & Chazan, M. (2012). Evidence for early hafted hunting technology. Science, 338(6109), 942-946.](https://doi.org/10.1126/science.1227608)
* [Gronau I et al. (2011) Bayesian inference of ancient human demography from individual genome sequences. Nature Genetics. 43, 1031-1034.](https://www.nature.com/articles/ng.937)
* [Bramanti B et al. (2009) Genetic Discontinuity Between Local Hunter-Gatherers and Central Europe’s First Farmers. Science. 326(5949): 137-140.](https://science.sciencemag.org/content/326/5949/137.abstract)
* [Lalueza-Fox C, Römpler H, et al. (2007) A Melanocortin 1 Receptor Allele Suggests Varying Pigmentation Among Neanderthals. Science. 318(5855): 1453-1455.](https://science.sciencemag.org/content/318/5855/1453)
* [Burger, J., Kirchner, M., Bramanti, B., Haak, W., & Thomas, M. G. (2007). Absence of the lactase-persistence-associated allele in early Neolithic Europeans. Proceedings of the National Academy of Sciences, 104(10), 3736-3741.](https://dx.doi.org/10.1073%2Fpnas.0607187104)
* [Plagnol V, Wall JD (2006) Possible Ancestral Structure in Human Populations. PLoS Genet 2(7): e105.](https://doi.org/10.1371/journal.pgen.0020105)

## Early Hominid through Modern Human Archeology

* [Parkinson, J. A., Plummer, T. W., Oliver, J. S., & Bishop, L. C. (2022). Meat on the menu: GIS spatial distribution analysis of bone surface damage indicates that Oldowan hominins at Kanjera South, Kenya had early access to carcasses. Quaternary Science Reviews, 277, 107314.](https://doi.org/10.1016/j.quascirev.2021.107314)
* [Vidal, C. M., Lane, C. S., Asrat, A., Barfod, D. N., Mark, D. F., Tomlinson, E. L., ... & Oppenheimer, C. (2022). Age of the oldest known Homo sapiens from eastern Africa. Nature, 1-5.](https://doi.org/10.1038/s41586-021-04275-8)
* [Miller, J. M., & Wang, Y. V. (2022). Ostrich eggshell beads reveal 50,000-year-old social network in Africa. Nature, 601(7892), 234-239.](https://doi.org/10.1038/s41586-021-04227-2)
* [Prévost, M., Groman-Yaroslavski, I., Gershtein, K. M. C., Tejero, J. M., & Zaidner, Y. (2021). Early evidence for symbolic behavior in the Levantine Middle Paleolithic: A 120 ka old engraved aurochs bone shaft from the open-air site of Nesher Ramla, Israel. Quaternary International.](https://doi.org/10.1016/j.quaint.2021.01.002)
* [Sehasseh, E. M., Fernandez, P., Kuhn, S., Stiner, M., Mentzer, S., Colarossi, D., ... & Bouzouggar, A. (2021). Early Middle Stone Age personal ornaments from Bizmoune Cave, Essaouira, Morocco. Science Advances, 7(39), eabi8620.](https://doi.org/10.1126/sciadv.abi8620)
* [Lombard, M., & Shea, J. J. (2021). Did Pleistocene Africans use the spearthrower‐and‐dart?. Evolutionary Anthropology: Issues, News, and Reviews, 30(5), 307-315.](https://doi.org/10.1002/evan.21912)
* [Roebroeks, W., MacDonald, K., Scherjon, F., Bakels, C., Kindler, L., Nikulina, A., ... & Gaudzinski-Windheuser, S. (2021). Landscape modification by Last Interglacial Neanderthals. Science advances, 7(51), eabj5567.](https://doi.org/10.1126/sciadv.abj5567)
* [MacDonald, K., Scherjon, F., van Veen, E., Vaesen, K., & Roebroeks, W. (2021). Middle Pleistocene fire use: The first signal of widespread cultural diffusion in human evolution. Proceedings of the National Academy of Sciences, 118(31).](https://doi.org/10.1073/pnas.2101108118)
  * > Interaction with fire, including enhanced maintenance and production (1), is generally considered one of the most important processes within the cultural evolution of humankind. Fire afforded hominins protection against predators and cold, broadened the range of edible foods and the amount of energy that could be extracted from them through cooking (2), allowed manipulation of materials (3), extended the length of day, and impacted the character of social interactions (4). Fire gave hominins a means to increase the productivity of their habitats, over time significantly transforming natural landscapes (5, 6). Fire also came with costs (7⇓–9), as fire-using hominins had to collect fuel to be brought to a central place, calling for cooperation within a group (10). Given the impact pyrotechnology must have had, understanding the origins and development of fire use is relevant for our understanding of the development of the niche of hominin species through time, including—as we argue here—the importance of cultural behavior. Indeed, we hypothesize that the increasing evidence for the appearance of regular fire use within a (relatively) restricted time window (400 to 350 thousand y ago, ka) and across an extremely wide geographic region can be best explained by cultural diffusion of skills related to fire use. In fact, the fire record might provide the first clear evidence of the emergence of cultural diffusion in the evolution of humankind, indicating that a distinctive characteristic of the cultural behavior of Homo sapiens was in place at this time.
* [Reeves, J. S., Braun, D. R., Finestone, E. M., & Plummer, T. W. (2021). Ecological perspectives on technological diversity at Kanjera South. Journal of human evolution, 158, 103029.](https://doi.org/10.1016/j.jhevol.2021.103029)
* [McNutt, E. J., Hatala, K. G., Miller, C., Adams, J., Casana, J., Deane, A. S., ... & DeSilva, J. M. (2021). Footprint evidence of early hominin locomotor diversity at Laetoli, Tanzania. Nature, 600(7889), 468-471.](https://doi.org/10.1038/s41586-021-04187-7)
* [Shaar, R., Matmon, A., Horwitz, L. K., Ebert, Y., Chazan, M., Arnold, M., ... & Keddadouche, K. (2021). Magnetostratigraphy and cosmogenic dating of Wonderwerk Cave: New constraints for the chronology of the South African Earlier Stone Age. Quaternary Science Reviews, 259, 106907.](https://doi.org/10.1016/j.quascirev.2021.106907) &mdash; [PDF](http://huji.org.ar/wp-content/uploads/2021/04/Shaar_etal_2021.pdf)
* [Domínguez-Solera, S. D., Maíllo-Fernández, J. M., Baquedano, E., & Domínguez-Rodrigo, M. (2021). Equids can also make stone artefacts. Journal of Archaeological Science: Reports, 40, 103260.](https://doi.org/10.1016/j.jasrep.2021.103260)
  * > Here, we will show that in addition to capuchin monkeys, other non-primate mammals (equids) can certainly create lithic artefacts which display the same set of basic characteristics that have traditionally be used to identify intentional hominin stone tool knapping. Equid self-trimming can be differentiated from accidental trampling in the degree of reincidence in hitting cobbles (i.e., cores) and the resulting overlapping negative scars of the flaked products, as also produced through unintentional knapping through battering (as documented among capuchin monkeys).
* [Mayoral, E., Díaz-Martínez, I., Duveau, J., Santos, A., Ramírez, A. R., Morales, J. A., ... & Díaz-Delgado, R. (2021). Tracking late Pleistocene Neandertals on the Iberian coast. Scientific reports, 11(1), 1-12.](https://doi.org/10.1038/s41598-021-83413-8)
* [Wilkins, J., Schoville, B. J., Pickering, R., Gliganic, L., Collins, B., Brown, K. S., ... & Hatton, A. (2021). Innovative Homo sapiens behaviours 105,000 years ago in a wetter Kalahari. Nature, 592(7853), 248-252.](https://doi.org/10.1038/s41586-021-03419-0)
* [Barras, C. (2021). How did Neanderthals and other ancient humans learn to count?. Nature, 594(7861), 22-25.](https://doi.org/10.1038/d41586-021-01429-6)
* [Kirscher, U., El Atfy, H., Gärtner, A., Dallanave, E., Munz, P., Niedźwiedzki, G., ... & Böhme, M. (2021). Age constraints for the Trachilos footprints from Crete. Scientific reports, 11(1), 1-9.](https://doi.org/10.1038/s41598-021-98618-0)
  * > These footprints, which possess a suite of characteristic trace fossil features including expulsion rims, pull-up structures and toe drag marks, indicate the track-maker had a distinctive foot morphology. This morphology includes characters that are currently considered be unique to hominins such as the presence of a forefoot ball, a non-divergent and robust hallux placed alongside digit II on the distal margin of the sole and digits II through IV becoming progressively shorter. These are combined with generic primate traits such as the absence of a longitudinal medial arch, a proportionately shorter sole and a heel that is not bulbous. A straightforward comparative morphological analysis of the character suite suggests that the track-maker was a primitive hominin and a strong case has been made for it being phylogenetically basal to the Laetoli trackmaker, which had a longer, more human-like sole9. The morphometric analysis of Gierliński et al.6, shows that the Trachilos footprints cluster in the same anatomical space with other hominin footprints and are clearly separated from non-hominin primates.
  * [Kindy, D. (2021) New Research Suggests Human-Like Footprints in Crete Date to 6.05 Million Years Ago. Smart News. Smithsonian Magazine. 18 October.](https://www.smithsonianmag.com/smart-news/oldest-human-like-footprints-dated-to-605-million-years-ago-180978889/)
* [de León, M. S. P., Bienvenu, T., Marom, A., Engel, S., Tafforeau, P., Warren, J. L. A., ... & Zollikofer, C. P. (2021). The primitive brain of early Homo. Science, 372(6538), 165-171.](https://doi.org/10.1126/science.aaz0032)
* [Beaudet, A. (2021). The enigmatic origins of the human brain. Science, 372(6538), 124-125.](https://doi.org/10.1126/science.abi4661)
* [Will, M., Krapp, M., Stock, J. T., & Manica, A. (2021). Different environmental variables predict body and brain size evolution in Homo. Nature Communications, 12(1), 1-12.](https://doi.org/10.1038/s41467-021-24290-7)
* [Thompson, J. C., Wright, D. K., Ivory, S. J., Choi, J. H., Nightingale, S., Mackay, A., ... & Gomani-Chindebvu, E. (2021). Early human impacts and ecosystem reorganization in southern-central Africa. Science Advances, 7(19), eabf9776.](https://doi.org/10.1126/sciadv.abf9776)
  * > Human control of fire is well established in the archaeological record from before the start of the MSA (15), but its use as a landscape management tool has only so far been documented in a few Paleolithic contexts. These include in Australia ca. 40 ka (36), Highland New Guinea ca. 45 ka (37), and ca. 50 ka at Niah Cave in lowland Borneo (38). In the Americas, anthropogenic ignitions have been implicated as major factors in the reconfiguration of faunal and floral communities as humans first entered these ecosystems, especially within the past 20 ka (16). These conclusions are necessarily based on correlative evidence, but the argument for a cause-and-effect relationship is strengthened where there is direct overlap of archaeological, geochronological, geomorphic, and paleoenvironmental data. Although marine core data offshore of Africa have previously provided evidence of altered fire regimes over the past ~400 ka (9), here, we provide evidence of anthropogenic impacts that draw from correlated archaeological, paleoenvironmental, and geomorphic datasets.<br/><br/>Identifying anthropogenic fire in the paleoenvironmental record requires evidence of temporal or spatial changes in fire activity and vegetation, demonstration that these changes are not predicted by climate parameters alone, and temporal/spatial coincidence between fire regime changes and changes in the human record (29). Here, the first evidence for extensive MSA occupation and alluvial fan formation in the Lake Malawi basin occurred alongside a major reorganization of regional vegetation that began ca. 85 ka. Charcoal abundances in the MAL05-1B/1C core are reflective of regional trends in charcoal production and sedimentation that show substantial differences after ca. 150 ka when compared to the rest of the 636-ka record (figs. S5, S9, and S10). This transition shows an important contribution of fire for shaping ecosystem composition that cannot be explained by climate alone. In natural fire regimes, lightning ignitions typically occur at the end of the dry season (39). Anthropogenic fires, however, may be ignited at any time if fuels are sufficiently dry. On a site scale, humans can alter fire regimes continuously through collection of firewood from the understory. The net result of anthropogenic fire of any kind is that it has the potential to result in more consumption of woody vegetation, continuously throughout the year, and at a variety of scales.<br/><br/>In South Africa, fire was used in the heat treatment of stone for tool manufacture as early as 164 ka (12) and as a tool for cooking starchy tubers as early as 170 ka (40), taking advantage of resources that thrived in ancient fire-prone landscapes (41). Landscape fires reduce arboreal cover and are crucial tools in maintaining grassland and forest patch environments, which are defining elements of anthropogenically mediated ecosystems (13). If modification of vegetation or prey behavior was the intent of increased anthropogenic burning, then this behavior represents an increase in the complexity with which early modern humans controlled and deployed fire in comparison to earlier hominins and shows a transformed interdependency in our relationship with fire (7). Our analysis offers an additional avenue for understanding how human use of fire changed in the Late Pleistocene and what impacts these changes had on their landscapes and environments.
  * > Short sediment cores, such as that from the adjacent Lake Masoko basin in Tanzania, or shorter cores within Lake Malawi itself, show changes in the relative pollen abundances of grass to woodland taxa that have been attributed to natural climate variability over the past 45 ka (48–50). However, it is only with the longer perspective of the >600-ka pollen record of Lake Malawi, accompanied by the extensively dated archaeological landscape next to it, that it is possible to understand the longer-term associations between climate, vegetation, charcoal, and human activity. Although humans were likely present in the northern Lake Malawi basin before 85 ka, the density of archaeological sites after ca. 85 ka, and especially after 70 ka, indicates that the region was attractive for human occupation after the last major arid period ended. At this time, novel or more intensive/frequent usage of fire by humans apparently combined with natural climate shifts to restructure a >550-ka ecological relationship, ultimately generating an early preagricultural anthropogenic landscape (Fig. 4). Unlike during earlier time periods, the depositional nature of this landscape preserved MSA sites as a function of the recursive relationship between environment (resource distributions), human behavior (activity patterns), and fan activation (sedimentation/site burial).
* [Key, A. J., Jarić, I., & Roberts, D. L. (2021). Modelling the end of the Acheulean at global and continental levels suggests widespread persistence into the Middle Palaeolithic. Humanities and Social Sciences Communications, 8(1), 1-12.](https://doi.org/10.1057/s41599-021-00735-8)
  * > Acheulean stone tools were produced for more than 1.5 million years. Such an extended period of production is well established, with an age bracket of ~1.75 to 0.15 million years ago (Mya) widely cited as ‘the Acheulean period’ (Gowlett, 2015; de la Torre, 2016; Shea, 2017; Galway-Witham et al., 2019). Discoveries at Konso (Ethiopia), Olduvai Gorge (Tanzania), and West Turkana (Kenya) provide convincing evidence of the Acheulean’s origin in east Africa around 1.75 Mya (Lepre et al., 2011; Beyene et al., 2013; Diez-Martín et al., 2015). Multiple other sites support such an early occurrence in this region (de la Torre and Mora, 2014; de la Torre, 2016; Gallotti and Mussi, 2018), and no other countries claim evidence to the contrary (e.g. Dennell, 2018; Moncel and Ashton, 2018). The location and timing of the onset of the Acheulean therefore appears well supported.<br/><br/>In comparison, the end of the Acheulean is a relative unknown. No sites are widely recognised as displaying evidence of the ‘last Acheulean populations’, and no single region (nor continent) is convincingly argued to display the last stronghold of this technology. Instead, the Acheulean is broadly considered to have been replaced across the Old World between 0.3 and 0.15 Mya, but there is considerable debate on precisely when and where these transitions occurred, and how they varied between different regions (McBrearty and Tryon, 2006; Norton et al., 2009; Fontana et al., 2013; Akhilesh et al., 2018; Méndez-Quintas et al., 2020).
  * > Presented here are the most accurate estimates yet for when the Acheulean cultural tradition ended. We do so using optimal linear estimation, a modelling technique often used to estimate the extinction of faunal species, but novel to archaeological research. In Africa and the Near East the Acheulean is predicted to end 175–166 Kya. In Europe, the Acheulean is predicted to end 141–130 Kya. These dates are only slightly younger than current understanding on the end of the tradition in these regions. Asian estimates, however, range between 57 and 53 Kya. Thus, the Acheulean continues in this region long after it has ended elsewhere, and for the majority of the MP. The Acheulean stopped being a global tradition between 107 and 29 Kya, although we favour an age bracket of between 107 and 50 Kya. These estimates suggest the Acheulean to have remained a distinct cultural tradition long after the inception of MP technologies in multiple continental regions. Persistence of the tradition in Europe and Asia may be linked to each region’s geographic isolation, relative to the rest of the Acheulean world. In line with the increasingly dynamic nature of the Middle Pleistocene hominin fossil record, contemporaneous hominin populations are demonstrated to have been practicing distinct stone-tool behaviours, potentially due to cognitive and anatomical differences.
* [Bar-Yosef Mayer DE, Groman-Yaroslavski I, Bar-Yosef O, Hershkovitz I, Kampen-Hasday A, Vandermeersch B, et al. (2020) On holes and strings: Earliest displays of human adornment in the Middle Palaeolithic. PLoS ONE 15(7): e0234924.](https://doi.org/10.1371/journal.pone.0234924)
* [Stewart, B. A., Zhao, Y., Mitchell, P. J., Dewar, G., Gleason, J. D., & Blum, J. D. (2020). Ostrich eggshell bead strontium isotopes reveal persistent macroscale social networking across late Quaternary southern Africa. Proceedings of the National Academy of Sciences, 117(12), 6453-6462.](https://doi.org/10.1073/pnas.1921037117)
* [Larrasoaña, J. C., Waldmann, N., Mischke, S., Avni, Y., & Ginat, H. (2020). Magnetostratigraphy and paleoenvironments of the Kuntila Lake sediments, southern Israel: implications for Late Cenozoic climate variability at the northern fringe of the Saharo-Arabian desert belt. Frontiers in Earth Science, 8, 173.](https://doi.org/10.3389/feart.2020.00173)
* [Richter J, Litt T, Lehmkuhl F, Hense A, Hauck TC, Leder DF, et al. (2020) Al-Ansab and the Dead Sea: Mid-MIS 3 archaeology and environment of the early Ahmarian population of the Levantine corridor. PLoS ONE 15(10): e0239968. https://doi.org/10.1371/journal.pone.0239968](https://doi.org/10.1371/journal.pone.0239968)
  * > Seven 14C dates were obtained on charcoal samples in several locations in the excavated area (Table 2). The dates show a high level of concordance placing the occupation of Al-Ansab 1, layer 1 between 36,963 ± 610 and 37,890 ± 630 (cal BP). Both OSL and 14C results overlap within the 2σ range (Fig 14; see S3 Table for complete list of Early Ahmarian 14C dates).
  * > The establishment of the Ahmarian techno-cultural unit, during the pronounced global warm interval at 45–43 ka BP (Greenland Interstadial GIS 12) and the maximum expansion of the Early Ahmarian sub-groups NEA and SEA during the equally pronounced global warm interval at 39–36 ka BP (Greenland Interstadial GIS 8) represent the AMH demographic correlate of the extension of an environmental corridor that had come into action since 60 ka BP. The emergence and successful dispersal of the Early Ahmarian culture and the connected permanent establishment of AMH were both based on the environmental integration of the Levantine corridor. AMH populations migrating into the Levant during MIS 3, all profited from the refugial capacity of the relatively stable Levantine natural environment constituted by the Saharo-Arabian biome, the Mediterranean biome and the Irano-Turanian biome.<br/><br/>Since ~60 ka BP, the latter of them offered one (in terms of vegetation) quite homogenous and long-lasting, large south-to-north corridor from southern Jordan to the northern Lebanon mountains. Around 45 ka BP, and inside the same corridor, the introduction or invention, of hunting weapons with lithic projectile inserts, marked the onset of the Early Ahmarian techno-cultural unit, exclusively tied to AMH populations. We assume the Ahmarian techno-cultural unit came into play within a regional framework, at mid-MIS 3, of slowly lowering annual mean temperature and slow decrease of precipitation. This change, which is mirrored in the local sedimentary archives at Wadi Sabra is also reflected by the supraregional re-organisation of vegetational patterning within the three biomes which constitute the Levantine landscapes. In particular, the vegetational difference between southern desert environments (southern Jordan) and northern woodlands (Lebanon) areas had diminished after 60 ka BP, and again around 40 ka BP, in favor of Artemisia and grass steppe covering large parts of the whole Levant.
* [Haws, J. A., Benedetti, M. M., Talamo, S., Bicho, N., Cascalheira, J., Ellis, M. G., ... & Zinsious, B. K. (2020). The early Aurignacian dispersal of modern humans into westernmost Eurasia. Proceedings of the National Academy of Sciences, 117(41), 25414-25422.](https://doi.org/10.1073/pnas.2016062117)
  * > The stratigraphy, techno-typological analysis of the lithic artifacts, and radiocarbon dating demonstrate that the level GG-II assemblage represents a discrete occupation layer, wholly distinct from those in the levels above and below. The assemblage is small but consistent with the attributes of the early Aurignacian. The dates presented here conservatively place the level GG-II occupation at ∼41.1 to 38.1 ka cal BP.<br/><br/>Our results from Picareiro provide definitive evidence that modern humans were in western Iberia at a time when, if present at all, Neanderthal populations would have been extremely sparse. Our data offer some resolution to the implications of Wood et al. (29) scenarios for the ∼42 to 37 ka cal BP time frame. We can discount their scenario in which southern Iberia was abandoned by both Neanderthals and modern humans and confirm the one in which modern humans spread into the southern regions soon after they arrived in northern Iberia. As for the scenario in which Neanderthals were present in southern Iberia until very late, the Picareiro data alone cannot resolve this issue. Middle Paleolithic occupations at Picareiro ended by 42.5 ka cal BP, but they apparently continued until ∼36 ka cal BP at Gruta da Oliveira just 4.2 km away, based on current evidence (49, 50).<br/><br/>Our results also cast further doubt on the idea of a “hard border” or frontier between Neanderthal and modern human populations between ∼42 and 37 ka cal BP (22, 51). Instead, the Ebro river valley was likely a permeable landscape feature that facilitated dispersal (52). Modern humans may have encountered a few remnant Neanderthal groups but it appears that most of Iberia south of the Ebro was already depopulated (35). This pattern is evident in Portugal, where Middle Paleolithic end dates cluster at ∼45 to 42 ka at Foz do Enxarrique (30), Mira Nascente (53), Lapa do Picareiro, and Cardina (54), followed by a nearly total absence of evidence for Neanderthals on the landscape.
* [Cox, T. J., Fazenda, B. M., & Greaney, S. E. (2020). Using scale modelling to assess the prehistoric acoustics of stonehenge. Journal of Archaeological Science, 122, 105218.](https://doi.org/10.1016/j.jas.2020.105218)
* [Groucutt, H. S., Breeze, P. S., Guagnin, M., Stewart, M., Drake, N., Shipton, C., ... & Petraglia, M. D. (2020). Monumental landscapes of the Holocene humid period in Northern Arabia: The mustatil phenomenon. The Holocene, 0959683620950449.](https://doi.org/10.1177/0959683620950449)
* [Fewlass, H., Talamo, S., Wacker, L., Kromer, B., Tuna, T., Fagault, Y., ... & Martisius, N. L. (2020). A 14 C chronology for the Middle to Upper Palaeolithic transition at Bacho Kiro Cave, Bulgaria. Nature Ecology & Evolution, 1-8.](https://doi.org/10.1038/s41559-020-1136-3)
* [Curry, A. (2020). Ancient DNA tracks Vikings across Europe. Science, 369(6510)](https://doi.org/10.1126/science.369.6510.1416)
* [Stewart, M., Clark-Wilson, R., Breeze, P. S., Janulis, K., Candy, I., Armitage, S. J., ... & Petraglia, M. D. (2020). Human footprints provide snapshot of last interglacial ecology in the Arabian interior. Science advances, 6(38), eaba8940.](https://doi.org/10.1126/sciadv.aba8940) &mdash; Homo sapiens 120000 years ago in Levant and Arabia.
 * > Southwest Asia represents the main biogeographical gateway between Africa and Eurasia (1) and is, therefore, key for understanding hominin and faunal dispersal and evolution across continents. Fossils of Homo sapiens are first recorded outside of Africa ~210 and ~180 thousand years (ka) ago in southern Greece (2) and the Levant (3), respectively, and recent fossil evidence demonstrates the arrival of our species in the Arabian interior by at least ~85 ka (4). Understanding the nature of early H. sapiens out-of-Africa dispersals, however, has remained challenging owing to the poor resolution of paleoenvironmental and paleoecological data in direct association with remains of the earliest non-African people (5).
  * > Here, we report hominin and non-hominin mammal footprints and fossils from the Alathar lacustrine deposit in the western Nefud Desert, Saudi Arabia (Fig. 1). We argue that the footprints, dated to the last interglacial and therefore contemporaneous with an early H. sapiens out-of-Africa dispersal (Fig. 2) (6), most likely represent the earliest evidence of our species in the Arabian Peninsula. The unique setting and taphonomic factors affecting the long-term preservation of footprints mean that groups of footprints, and particularly those in similar states of preservation, can be assumed to have been generated within a very short window, typically within a few hours or days (see full discussion in text S5) (7–9). An experimental study of modern human footprints in mud flats found that fine details were lost within 2 days and prints were rendered unrecognizable within four (7), and similar observations have been made for other non-hominin mammal tracks (9). The findings presented here, therefore, provide a unique opportunity to examine the close ecological interplay between late Pleistocene humans, animals, and their environments as our species began to venture into Eurasia.
  * > The Alathar paleolake deposit lies as an inverted relief feature within an interdunal depression in the south-western portion of the Nefud Desert sand sea (Fig. 1). The sedimentary sequence comprises a ~1.8-m-thick deposit of sandy-silt diatomite (units 2 to 7) underlain by wind-blown sands (unit 1) (Fig. 3 and text S1). Two paleolake sections were recorded at the site and a composite section formed from both (Fig. 3). The first section (units 2 to 4) is exposed along the south-western margin of the deposit, and the footprints were found in the uppermost portion of unit 4b (i.e., the surface). A small number of fossils (n = 4) were also found eroding out of the surface of this unit. The second section (units 5 to 7) is located along parts of the western edge and toward the center of the modern-day surface of the paleolake deposit and stratigraphically overlies the footprint-bearing unit. The stratigraphic relationship between the two sections was visually correlated in the field based on lithostratigraphy. Samples for optically stimulated luminescence (OSL) dating were taken from sediments directly below (unit 2; PD61) and above (unit 5; PD62) the footprint-bearing unit and yielded ages of 121 ± 11 and 112 ± 10 ka, respectively, effectively bracketing the age of the prints (Fig. 3 and text S2). Diatom paleoecology and sedimentary analysis indicate that Alathar was an oligotrophic (nutrient-poor) and shallow freshwater lake for the majority of its existence (text S1). This is consistent with similar-aged nearby freshwater paleolake deposits (4, 10–11) situated in the southern reaches of a “freshwater corridor” that connected the Arabian interior to the Levant and northeast Africa at times during marine isotope stage 5 (MIS 5; ~130 to 80 ka) (12). The presence of freshwater lakes in the western Nefud Desert provided a vital resource and habitable landscape for hominins and animals.
* [Wadley, L., Esteban, I., de la Peña, P., Wojcieszak, M., Stratford, D., Lennox, S., ... & Sievers, C. (2020). Fire and grass-bedding construction 200 thousand years ago at Border Cave, South Africa. Science, 369(6505), 863-866.](https://doi.org/10.1126/science.abc7239)
* [Duveau J, et al. (2019) The composition of a Neandertal social group revealed by the hominin footprints at Le Rozel (Normandy, France). PNAS. 116(39): 19409-19414.](https://doi.org/10.1073/pnas.1901789116)
* [Faith JT, Rowan J, Du A (2019) Early hominins evolved within non-analog ecosystems. PNAS. 116(43): 21478-21483.](https://doi.org/10.1073/pnas.1909284116)
* [Schuster R (2019) Prehistoric Superglue May Reveal Neanderthal Experiments 200,000 Years Ago. Haaretz. November 13.](https://www.haaretz.com/archaeology/.premium.MAGAZINE-prehistoric-superglue-may-reveal-neanderthal-experiments-200-000-years-ago-1.8068691?fbclid=IwAR3Fb2ksOsyf8ftioHvT4pubyfF-TSoMG6rIW8ZkfZdr4La03fzZe1TnglY)
* [li Z et al. (2019) Engraved bones from the archaic hominin site of Lingjing, Henan Province. Antiquity. 93(370): 886-900.](https://doi.org/10.15184/aqy.2019.81)
* [Sahle Y, Brooks AS (2019) Assessment of complex projectiles in the early Late Pleistocene at Aduma, Ethiopia. PLoS ONE 14(5): e0216716.](https://doi.org/10.1371/journal.pone.0216716)
* [Braun DR et al. (2019) Earliest known Oldowan artifacts at >2.58 Ma from Ledi-Geraru, Ethiopia, highlight early technological diversity. PNAS. 116(24): 11712-11717.](https://doi.org/10.1073/pnas.1820177116)
* [Harvati K et al. (2019) Apidima Cave fossila provide earliest evidence of Homo sapiens in Eurasia. Nature. 571: 500-504.](https://www.nature.com/articles/s41586-019-1376-z)
* [Delson E (2019) An early dispersal of modern humans from Africa to Greece. Nature News. July 10.](https://www.nature.com/articles/d41586-019-02075-9) &mdash; Has a great map of locations and times for various known dispersals.
* [Wong K (2019) Cave That Housed Neandertals and Denisovans Challenges View of Cultural Evolution. Scientific American. January 30.](https://www.scientificamerican.com/article/cave-that-housed-neandertals-and-denisovans-challenges-view-of-cultural-evolution/)
* [Dennell R (2019) Dating hominin discoveries at Denisova. Nature News. January 30.](https://www.nature.com/articles/d41586-019-00264-0)
* [Morley MW et al. (2019) Hominin and animal activities in the microstratigraphic record from Denisova Cave (Altai Mountains, Russia). Scientific Reports. 9: 13785.](https://www.nature.com/articles/s41598-019-49930-3)
* [Jacobs A et al. (2019) Timing of archaic hominin occupation of Denisova Cave in southern Siberia. Nature. 565: 594-599.](https://www.nature.com/articles/s41586-018-0843-2)
* [Douka K et al. (2019) Age estimates for hominin fossils and the onset of the Upper Palaeolithic at Denisova Cave. Nature. 565: 640-644.](https://www.nature.com/articles/s41586-018-0870-z?_sg=HnHYAGx6TrgPF2A_usC7TI5caZ2-WjcdkmpxVWnaV0ZHwkkc-3JeeAzXd4rBtIcckyJpTe2WULuCVv4.fWw-pU6PDxJers3nN6eg1KwXhcPIuaX2eoGduCnTH7WaXIzyMy7SYCGkRQXyrTIAw85glW1XzhBx63O67-t8DQ)
* [Assaf, E. (2018). Paleolithic aesthetics: collecting colorful flint pebbles at Middle Pleistocene Qesem Cave, Israel. Journal of Lithic Studies, 5(1).](https://doi.org/10.2218/jls.2616)
* [Hoffmann, D. L., Angelucci, D. E., Villaverde, V., Zapata, J., & Zilhão, J. (2018). Symbolic use of marine shells and mineral pigments by Iberian Neandertals 115,000 years ago. Science advances, 4(2), eaar5255.](https://doi.org/10.1126/sciadv.aar5255)
* [Brooks AS et al. (2018) Long-distance stone transport and pigment use in the earliest Middle Stone Age. Science. 360(6384): 90-94.](https://science.sciencemag.org/content/360/6384/90)
  * > The Middle Stone Age (MSA) comprises a diverse group of African industries characterized by the absence or rarity of large cutting tools (LCTs, such as Acheulean handaxes and cleavers), the presence of prepared core (Levallois) technologies, and regional and/or temporal variation in weapon armatures (1–3). Later MSA industries contain varying frequencies of small retouched bladelets and geometric artifacts (4), small unifacially and bifacially worked points, and larger lanceolate and bone points (5, 6). African MSA sites are associated with the oldest known fossils attributed to Homo sapiens, which date between ~195 and ~350 thousand years (ka) ago (7–9). After 130 ka ago, late Pleistocene MSA sites, mostly from the north and south temperate zone extremities of the continent, document some of the oldest beads (10, 11), complex geometric designs (12, 13), lithic projectile armatures interpreted as arrow or dart tips (14, 15), paint production (16), heat treatment of lithics (17), and expanded resource use, including fishing and shellfish collecting (18, 19).
  * > Although these late Pleistocene behaviors are thought to reflect advanced cognitive abilities for learning, imitation, working memory, planning, and recursive technological innovation, their middle Pleistocene or earlier antecedents have remained obscure. Evolutionary explanations for the middle Pleistocene enlargement of relative brain size, largely overlapping the modern H. sapiens range, include a shift to larger social groups or networks (20); greater foraging, technological, and social adaptability in the face of more rapid and unpredictable climate fluctuations (21); and the development of symbolic expression (22).
* [Groucutt, H. S., Grün, R., Zalmout, I. A., Drake, N. A., Armitage, S. J., Candy, I., ... & Buck, L. T. (2018). Homo sapiens in Arabia by 85,000 years ago. Nature ecology & evolution, 2(5), 800-809.](https://doi.org/10.1038/s41559-018-0518-2) &mdash; [PDF](https://www.academia.edu/36508168/Homo_sapiens_in_Arabia_by_85_000_years_ago)
  * > Al Wusta-1 is the oldest directly dated H. sapiens fossil outside Africa and the Levant. It joins a small but growing corpus of evidence that the early dispersal of H. sapiens into Eurasia was much more widespread than previously thought. The site of Al Wusta is located in the Nefud desert more than 650 km southeast of Skhul and Qafzeh (Fig. 1A). This site establishes that H. sapiens were in Arabia in late MIS 5, rather than being restricted to Africa and the Levant as suggested by traditional models (Fig. 6). With Skhul dating to ~130-100 ka, Qafzeh to ~100-90 ka and Al Wusta to ~95-85 ka it is currently unclear if the southwest Asian record reflects multiple early dispersals out of Africa or a long occupation during MIS 5. The association of the Al Wusta site with a late MIS 5 humid phase (Fig. 6), suggests that significant aspects of this dispersal process were facilitated by enhanced monsoonal rainfall. While changes in behaviour and demography are crucial to understanding the dispersal process, climatic windows of opportunity were also key in allowing H. sapiens to cross the Saharo-Arabian arid belt, which often constituted a formidable barrier.
* [Tryon CA, Lewis JE, Ranhorn KL, Kwekason A, Alex B, Laird MF, et al. (2018) Middle and Later Stone Age chronology of Kisese II rockshelter (UNESCO World Heritage Kondoa Rock-Art Sites), Tanzania. PLoS ONE 13(2): e0192029.](https://doi.org/10.1371/journal.pone.0192029)
* [Hershkovitz, I., Weber, G. W., Quam, R., Duval, M., Grün, R., Kinsley, L., ... & Weinstein-Evron, M. (2018). The earliest modern humans outside Africa. Science, 359(6374), 456-459.](https://doi.org/10.1126/science.aap8369)
  * > Three independent numerical dating methods—U-series (U-Th), combined uranium series and electron spin resonance (US-ESR) series, and thermoluminescence (TL)—carried out in three different dating laboratories yielded consistent results (Fig. 2B, figs. S2 and S3, and tables S1 and S3). A series of nine TL dates on burnt flints from Square L10 and N12 in the vicinity of the human fossil (Fig. 1, A and B) provided a mean age of **179 ± 48 thousand years (ky) (2σ) (range = 212 to 140 ky)** (11). U-Th analyses of the dentine of the I2 from the maxilla and of the crust adhering directly to the maxilla yielded a minimum age of 70.2 ± 1.6 ky (2σ; table S1) and 185 ± 8.0 ky (2σ; Fig. 2B and table S2), respectively (9). The combined US-ESR dating of the enamel of the same tooth yielded a maximum age of 174 ± 20 ky (2σ) (Fig. 2B, fig. S3, and tables S1 and S3) (9). All these dates, except for the U-series dating of the dentine, which exclude the possibility of recent intrusion, fall within the time range for the Early Levantine Mousterian lithic industry (Tabun D-type) observed at Tabun, Hayonim, and Misliya caves (i.e., ~250 to ~140 ky) (11–13) and are older than the upper range defined for the EMP sequence in Misliya Cave (>165 ky) (11). Collectively, the evidence suggests an early marine isotope stage 6 (MIS 6) age for the Misliya-1 fossil. **The age range for Misliya-1, based on dates directly connected with the fossil (U-Th on crust providing the minimum boundary and the maximum boundary of US-ESR on the enamel of I2), is between 177 and 194 ky** [for details and calculation methods, see (9) and fig. S2].
* [Potts R et al. (2018) Environmental dynamics during the onset of the Middle Stone Age in eastern Africa. Science. 360(6384): 86-90.](https://science.sciencemag.org/content/360/6384/86) &mdash; Obsidian obtained from far away created a technological change in eastern Africa around 320 ka ago.
* [Deino AL et al. (2018) Chronology of the Acheulean to Middle Stone Age transition in eastern Africa. Science. 360(6384): 95-98.](https://science.sciencemag.org/content/360/6384/95)
* [Zhu Z et al. (2018) Hominin occupation of the Chinese Loess Plateau since about 2.1 million years ago. Nature. 559: 608-612.](https://www.nature.com/articles/s41586-018-0299-4)
* [Karakostis FA et al. (2018) Evidence for precision grasping in Neandertal daily activities. Science Advances. 4(9): eaat2369](http://advances.sciencemag.org/content/4/9/eaat2369)
* [Ohman J, Gomez-Olivencia A (2018) Neanderthals cared for each other and survived into old age – new research. HeritageDaily. April 11.](https://www.heritagedaily.com/2018/04/neanderthals-cared-for-each-other-and-survived-into-old-age-new-research/118812)
* [Gomez-Olivencia A et al. (2018) La Ferrassie 1: New perspectives on a “classic” Neandertal. Journal of Human Evolution. 117: 13-32.](https://doi.org/10.1016/j.jhevol.2017.12.004)
* [Daura J et al. (2017) New Middle Pleistocene hominin cranium from the Gruta da Aroeira (Portugal). PNAS. 114(13): 3397-3402.](https://doi.org/10.1073/pnas.1619040114)
* [Putt SS, Wijeakumar S, Franciscus RG, Spencer JP (2017) The functional brain networks that underlie Early Stone Age tool manufacture. Nature Human Behaviour. 1: 0102.](https://www.nature.com/articles/s41562-017-0102)
* [Scerri EML et al. (2017) Persistence of Middle Stone Age technology to the Pleistocene/Holocene transition supports a complex hominin evolutionary scenario in West Africa. Journal of Archeological Science: Reports. 11: 639-646.](https://doi.org/10.1016/j.jasrep.2017.01.003)
* [Hublin J-J et al. (2017) New fossils from Jebel Irhoud, Morocco and the pan-African origin of Homo sapiens. Nature. 546: 289–292.](https://www.nature.com/articles/nature22336)
* [Richter D (2017) The age of the hominin fossils from Jebel Irhoud, Morocco, and the origins of the Middle Stone Age. Nature. 546: 293-296.](https://doi.org/10.1038/nature22335) &mdash; Pushes back presence of modern humans to 286 ± 32 ka ago.
* [Platt DE, Haber M, Dagher-Kharrat MB, Douaihy B, Khazen G, Bonab MA, Salloum A, Mouzaya F, Luiselli D, Tyler-Smith C, Renfrew C, Matisoo-Smith E, Zalloua PA (2017) Mapping Post-Glacial expansions: The Peopling of Southwest Asia. Scientific Reports. 7.](https://www.nature.com/articles/srep40338)
* [Blegen, N. (2017). The earliest long-distance obsidian transport: Evidence from the∼ 200 ka Middle Stone Age Sibilo School Road Site, Baringo, Kenya. Journal of Human Evolution, 103, 1-19.](https://doi.org/10.1016/j.jhevol.2016.11.002)
* [Callaway E (2017) Oldest Homo sapiens fossil claim rewrites our species' history. Nature News. 07 Jun.](https://www.nature.com/news/oldest-homo-sapiens-fossil-claim-rewrites-our-species-history-1.22114)
* [Gibbons A (2017) World’s oldest Homo sapiens fossils found in Morocco. Science. 7 Jun.](http://www.sciencemag.org/news/2017/06/world-s-oldest-homo-sapiens-fossils-found-morocco)
* [Crevecoeur I et al. (2016) Late Stone Age human remains from Ishango (Democratic Republic of Congo): New insights on Late Pleistocene modern human diversity in Africa. Journal of Human Evolution. 96: 37-57.](https://doi.org/10.1016/j.jhevol.2016.04.003)
* [Stringer C (2016) The origin and evolution of Homo sapiens. Philosphical Transaction of Royal Society B. 371(1698)](https://doi.org/10.1098/rstb.2015.0237)
* [Harcourt, A. H. (2016). Human phylogeography and diversity. Proceedings of the National Academy of Sciences, 113(29), 8072-8078.](https://doi.org/10.1073/pnas.1601068113)
* [Qiu J (2016) How China is rewriting the book on human origins. Nature News. 12 Jul.](https://www.nature.com/news/how-china-is-rewriting-the-book-on-human-origins-1.20231)
* [Lahr MM (2016) The shaping of human diversity: filters, boundaries and transitions. Philosphical Transaction of Royal Society B. 371(1698)](https://doi.org/10.1098/rstb.2015.0241)
* [Hiscock, P., O’Connor, S., Balme, J., & Maloney, T. (2016). World’s earliest ground-edge axe production coincides with human colonisation of Australia. Australian Archaeology, 82(1), 2-11.](https://doi.org/10.1080/03122417.2016.1164379)
* [Goval, E., Hérisson, D., Locht, J. L., & Coudenneau, A. (2016). Levallois points and triangular flakes during the Middle Palaeolithic in northwestern Europe: Considerations on the status of these pieces in the Neanderthal hunting toolkit in northern France. Quaternary International, 411, 216-232.](http://dx.doi.org/10.1016/j.quaint.2015.04.039)
* [Roberts p et al. (2015) Direct evidence for human reliance on rainforest resources in late Pleistocene Sri Lanka. Science. 347(6227): 1246-1249.](https://science.sciencemag.org/content/347/6227/1246)
* [Harmand S et al. (2015) 3.3-million-year-old stone tools from Lomekwi 3, West Turkana, Kenya. Nature. 521: 310-315.](https://www.nature.com/articles/nature14464)
* [Joordens, J. C., d’Errico, F., Wesselingh, F. P., Munro, S., de Vos, J., Wallinga, J., ... & Roebroeks, W. (2015). Homo erectus at Trinil on Java used shells for tool production and engraving. Nature, 518(7538), 228-231.](https://doi.org/10.1038/nature13962)
* [Lemorini, C., Venditti, F., Assaf, E., Parush, Y., Barkai, R., & Gopher, A. (2015). The function of recycled lithic items at late Lower Paleolithic Qesem Cave, Israel: an overview of the use-wear data. Quaternary International, 361, 103-112.](https://doi.org/10.1016/j.quaint.2014.07.032)
* [Miyagawa S et al. (2014) The integration hypothesis of human language evolution and the nature of contemporary languages. Frontiers in Psychology. 5.](https://doi.org/10.3389/fpsyg.2014.00564)
* [Tryon, C. A., & Faith, J. T. (2013). Variability in the middle stone age of eastern Africa. Current Anthropology, 54(S8), S234-S254.](https://doi.org/10.1086/673752)
* [Texier P-J et al. (2013) The context, form and significance of the MSA engraved ostrich eggshell collection from Diepkloof Rock Shelter, Western Cape, South Africa. Journal of Archeological Science. 40(9): 3412-3431.](https://doi.org/10.1016/j.jas.2013.02.021)
* [Blome MW et al. (2012) The environmental context for the origins of modern human diversity: A synthesis of regional variability in African climate 150,000–30,000 years ago. Journal of Human Evolution. 62(5): 563-592.](https://doi.org/10.1016/j.jhevol.2012.01.011)
* [Lepre, C. J., Roche, H., Kent, D. V., Harmand, S., Quinn, R. L., Brugal, J. P., ... & Feibel, C. S. (2011). An earlier origin for the Acheulian. Nature, 477(7362), 82-85.](https://doi.org/10.1038/nature10372)
* [Armitage, S. J., Jasim, S. A., Marks, A. E., Parker, A. G., Usik, V. I., & Uerpmann, H. P. (2011). The southern route “out of Africa”: evidence for an early expansion of modern humans into Arabia. Science, 331(6016), 453-456.](https://doi.org/10.1126/science.1199113)
* [Texier P-J et al. (2010) A Howiesons Poort tradition of engraving ostrich eggshell containers dated to 60,000 years ago at Diepkloof Rock Shelter, South Africa. PNAS. 107(14): 6180-6185.](https://doi.org/10.1073/pnas.0913047107)
* [Hoffecker, J. F. (2009). The spread of modern humans in Europe. Proceedings of the National Academy of Sciences, 106(38), 16040-16045.](https://doi.org/10.1073/pnas.0903446106)
* [Plummer TW, Ditchfield PW, Bishop LC, Kingston JD, Ferraro JV, Braun DR, et al. (2009) Oldest Evidence of Toolmaking Hominins in a Grassland-Dominated Ecosystem. PLoS ONE 4(9): e7199.](https://doi.org/10.1371/journal.pone.0007199)
  * https://humanorigins.si.edu/research/east-african-research-projects/kanjera-kenya
* [Rots, V., & Van Peer, P. (2006). Early evidence of complexity in lithic economy: core-axe production, hafting and use at Late Middle Pleistocene site 8-B-11, Sai Island (Sudan). Journal of Archaeological Science, 33(3), 360-371.](https://doi.org/10.1016/j.jas.2005.08.002)
* [d'Errico, F., Henshilwood, C., Vanhaeren, M., & Van Niekerk, K. (2005). Nassarius kraussianus shell beads from Blombos Cave: evidence for symbolic behaviour in the Middle Stone Age. Journal of human evolution, 48(1), 3-24.](https://doi.org/10.1016/j.jhevol.2004.09.002)
* [Wiessner, P. (2002). Hunting, healing, and hxaro exchange: A long-term perspective on! Kung (Ju/'hoansi) large-game hunting. Evolution and Human Behavior, 23(6), 407-436.](https://doi.org/10.1016/S1090-5138(02)00096-X)
* [Water MR, Forman SL, Person JM (1997) Diring Yuriakh: A Lower Paleolithic Site in Central Siberia. Science. 275(5304): 1281-1284](https://science.sciencemag.org/content/275/5304/1281)
* [Hall DA (1996) Thermoluminescence Dating Proves Early Peopling of Siberia. Mammoth Trumpet. 11(1): 1.](https://csfa.tamu.edu/wp-content/uploads/2015/12/vol11_num1.pdf)
* [Shea, J. J. (1988). Spear points from the Middle Paleolithic of the Levant. Journal of Field Archaeology, 15(4), 441-450.](https://doi.org/10.2307/530046)
* [Mehlman, M. J. (1987). Provenience, age and associations of archaic Homo sapiens crania from Lake Eyasi, Tanzania. Journal of Archaeological Science, 14(2), 133-162.](https://doi.org/10.1016/0305-4403(87)90003-3)
* [Watson, W. N. (1939). Methods of fire making used by early man. Journal of Chemical Education, 16(1), 36.](https://doi.org/10.1021/ed016p36)

## Homind Evolution

* [Almécija, S., Hammond, A. S., Thompson, N. E., Pugh, K. D., Moyà-Solà, S., & Alba, D. M. (2021). Fossil apes and human evolution. Science, 372(6542).](https://doi.org/10.1126/science.abb4363)
* [Richerson, P. J., Gavrilets, S., & de Waal, F. B. (2021). Modern theories of human evolution foreshadowed by Darwin’s Descent of Man. Science, 372(6544).](https://doi.org/10.1126/science.aba3776)

## In Case You Are Interested in Dog Genetics

* [Grimm, D. (2021) Mysterious, extinct Japanese wolf may hold clues to origins of dogs. Science News. 18 October.](https://www.science.org/content/article/mysterious-extinct-japanese-wolf-may-hold-clues-origins-dogs)
  * [Gojobori, J., Arakawa, N., Xiayire, X., Matsumoto, Y., Matsumura, S., Hongo, H., ... & Terai, Y. (2021). The Japanese wolf is most closely related to modern dogs and its ancestral genome has been widely inherited by dogs throughout East Eurasia. bioRxiv.](https://www.biorxiv.org/content/10.1101/2021.10.10.463851v3.full)
* [Bergström, A., Frantz, L., Schmidt, R., Ersmark, E., Lebrasseur, O., Girdland-Flink, L., ... & Skoglund, P. (2020). Origins and genetic legacy of prehistoric dogs. Science, 370(6516), 557-564.](https://doi.org/10.1126/science.aba9572)
   * > Dogs were the first domestic animal, but little is known about their population history and to what extent it was linked to humans. We sequenced 27 ancient dog genomes and found that all dogs share a common ancestry distinct from present-day wolves, with limited gene flow from wolves since domestication but substantial dog-to-wolf gene flow. By 11,000 years ago, at least five major ancestry lineages had diversified, demonstrating a deep genetic history of dogs during the Paleolithic. Coanalysis with human genomes reveals aspects of dog population history that mirror humans, including Levant-related ancestry in Africa and early agricultural Europe. Other aspects differ, including the impacts of steppe pastoralist expansions in West and East Eurasia and a near-complete turnover of Neolithic European dog ancestry.
   * > Wolves were the first animal with which humans formed a mutualistic relationship, eventually giving rise to dogs. Although there is little consensus regarding when (1–9), where (2, 8–13), and how many times (1, 8, 9, 14) domestication took place, the archaeological record (9, 15) attests to a long-term and close relationship to humans (9, 16–18). Modern dog genomes have revealed a complex population structure (5, 8, 10, 12, 19, 20), but because only six ancient dog and wolf genomes are currently available (4, 9, 14, 21), the process by which this structure emerged remains largely unknown.
   * > To reconstruct dog population history, we sequenced 27 ancient dog genomes up to 10.9 thousand years (ka) old from Europe, the Near East, and Siberia (table S1) to a median of 1.5-fold coverage (range, 0.1- to 11-fold) (Fig. 1A and table S2) (30). To test the association with human population history, we compiled 17 sets of human genome-wide data (30) that matched the age, geographic location, and cultural contexts of the ancient dogs (table S4), and we directly compared genetic relationships within the two species.
   * > We modeled the genetic history underlying dog population structure for five populations that represent major ancestries and tested all 135,285 possible admixture graph models with up to two admixture events (30). Only one model fits the data, and features the Mesolithic Karelian dog (10.9 ka ago) as having received part of its ancestry from a lineage related to eastern dogs and part from the Levantine lineage (Fig. 1E) [(two highly similar models nearly fit (fig. S4)]. The model can be extended to feature the earliest Neolithic European dog (7 ka ago) (14) as a mixture of the Karelian and the Levantine branches without loss of fit (fig. S5), supporting the dual ancestry model for European dogs suggested by the ancient ancestry cline (Fig. 1C). The observed phylogenetic structure implies that all five ancestry lineages (Neolithic Levant, Mesolithic Karelia, Mesolithic Baikal, ancient America, and New Guinea singing dog) must have existed by 10.9 ka ago (the radiocarbon date of the Karelian dog) and thus most likely existed prior to the transition from the Pleistocene to the Holocene epoch ~11.6 ka ago.
   * > The extensive range of ancestry diversity among early European dogs is not preserved today, as modern European dogs are all symmetrically related to the ancient dogs in our dataset (Fig. 1C, fig. S13, and data file S1) (30). This suggests little to no contribution of most local Mesolithic and Neolithic populations to present-day diversity in Europe. Instead, we found that a single dog from a Neolithic megalithic context dated to 5 ka ago at the Frälsegården site in southwestern Sweden can be modeled as a single-source proxy for 90 to 100% of the ancestry of most modern European dogs, to the exclusion of all other ancient dogs (fig. S13 and data file S1). This implies that a population with ancestry similar to this individual, but not necessarily originating in Scandinavia, replaced other populations and erased the continent-wide genetic cline (Fig. 5B). This ancestry was in the middle of the cline (Fig. 1C), and so present-day European dogs can be modeled as having about-equal proportions of Karelian- and Levantine-related ancestries [54 and 46%, respectively, for German shepherd on the basis of the admixture graph (Fig. 1E)].
   * > More recently, this modern European ancestry has dispersed globally and today is a major component of most dog populations worldwide (Fig. 5A). Our ancestry models, however, reveal that some precolonial ancestry does survive in breeds such as the Mexican chihuahua (~4%) and Xoloitzcuintli (~3%) and the South African Rhodesian ridgeback (~4%) (data file S1).
   * > The diversification of at least five dog ancestry lineages by the onset of the Holocene was followed by a dynamic population history that in many ways tracked that of humans, likely reflecting how dogs migrated alongside human groups. However, in several instances, these histories do not align, suggesting that humans also dispersed without dogs, dogs moved between human groups, or that dogs were cultural and/or economic trade commodities.<br/><br/>Certain aspects of genetic relationships between dog populations, such as an east–west Eurasian differentiation, circumpolar connections, and possible basal lineages in the Near East, resemble features of human population history that were established before the earliest estimated dates of dog domestication. This superficial mirroring between the species may therefore instead point to recurrent population dynamics due to biogeographic or anthropological factors that remain to be understood. A key question is how dogs spread across Eurasia and the Americas by the Holocene, since no major human population movements have been identified after the initial out-of-Africa expansion that could have driven this global dispersal.<br/><br/>We find that the modern and ancient genomic data are consistent with a single origin for dogs, though a scenario involving multiple closely related wolf populations remains possible. However, in our view, the geographical origin of dogs remains unknown. Previously suggested points of origin based upon present-day patterns of genomic diversity (2, 8, 10) or affinities to modern wolf populations (12) are sensitive to the obscuring effects of more recent population dynamics and gene flow. Ultimately, integrating DNA from dogs and wolves even older than those analyzed here with archaeology, anthropology, ethology, and other disciplines is needed to determine where and in which environmental and cultural context the first dogs originated.
* [Norton, H.L., Quillen, E.E., Bigham, A.W. et al. Human races are not like dog breeds: refuting a racist analogy. Evo Edu Outreach 12, 17 (2019). https://doi.org/10.1186/s12052-019-0109-y](https://doi.org/10.1186/s12052-019-0109-y)
* [Leathlobhair, M. N., Perri, A. R., Irving-Pease, E. K., Witt, K. E., Linderholm, A., Haile, J., ... & Brace, S. (2018). The evolutionary history of dogs in the Americas. Science, 361(6397), 81-85.](https://doi.org/10.1126/science.aao4776)
* [Hecht, J. (2018) What a Dog Geneticist Wants You to Know about Dog Genetics. Dog Spies Blog. Scientific American.](https://blogs.scientificamerican.com/dog-spies/what-a-dog-geneticist-wants-you-to-know-about-dog-genetics/)
* [Parker, H. G., Dreger, D. L., Rimbault, M., Davis, B. W., Mullen, A. B., Carpintero-Ramirez, G., & Ostrander, E. A. (2017). Genomic analyses reveal the influence of geographic origin, migration, and hybridization on modern dog breed development. Cell reports, 19(4), 697-708.](https://doi.org/10.1016/j.celrep.2017.03.079)
* [Pennisi, E. (2013). Old dogs teach a new lesson about canine origins. Science, 342(6160), 785-786.](https://doi.org/10.1126/science.342.6160.785)
* [Larson, G., Karlsson, E. K., Perri, A., Webster, M. T., Ho, S. Y., Peters, J., ... & Comstock, K. E. (2012). Rethinking dog domestication by integrating genetics, archeology, and biogeography. Proceedings of the National Academy of Sciences, 109(23), 8878-8883.](https://doi.org/10.1073/pnas.1203005109)
* [Sutter, N. B., Bustamante, C. D., Chase, K., Gray, M. M., Zhao, K., Zhu, L., ... & Quignon, P. (2007). A single IGF1 allele is a major determinant of small size in dogs. Science, 316(5821), 112-115.](https://doi.org/10.1126/science.1137045)
* [Ostrander, E. A. (2007). Genetics and the shape of dogs: studying the new sequence of the canine genome shows how tiny genetic changes can create enormous variation within a single species. American Scientist, 95(5), 406-413.](https://www.americanscientist.org/article/genetics-and-the-shape-of-dogs)
* [Parker, H. G., & Ostrander, E. A. (2005). Canine genomics and genetics: running with the pack. PLoS Genet, 1(5), e58.](https://doi.org/10.1371/journal.pgen.0010058)

