# Genetics

Genetic studies are an area fraught with criticism. Some of these criticisms are for good reason, some from fear ([and there can be good reason behind the fear, too](https://en.wikipedia.org/wiki/Human_Genome_Diversity_Project#Potential_problems)). A proper scientific understanding of genetics is necessary to draw conclusions about the following papers. Any understanding mired in the nonsense of racism is bound to produce further nonscientific nonsense. Any studies which do not respect that ownership and copyright are not concepts recognized by indigenous peoples sovereign nations. The earth and all of creation "owns" the genetics. No one can retain ownership or copyright over genetics. Those concepts applied to genes and genetics are rendered as nonsense in many indigenous worldviews, including my own.

That being said, here is a the high-level overview of the very basics of human genetics:

* An average person has an estimated 37,200,000,000,000 cells containing some genetic material.
* Genetic material contains DNA and RNA.
* Only nucleated cells have DNA.
* Both RNA and DNA are passed on from parents to offspring, but the mechanism for some of the transfer is not fully understood.
* A complete DNA sequence has over 6,000,000,000 base pairs.
* Base pairs are two each of (A - adenine, C - cytosine, G - guanine, T - thymine). These have a particular way of matching up with each other so only certain combinations are possible.
* These base pairs form strands which are weakly bonded through the base pair bond. One stand is the sense strand, the other is the anti-sense strand.
* These two strands form a double helix.
* The DNA is stored in 23 pairs of chromosomes in the nucleus of a cell.
* Parts of chromosomes are broken down into genes, which code for specific molecules.
* The DNA is comprised of sequences which code for molecules (exons) and DNA which typically gets turned into RNA (introns).
* In exons, every three base pairs codes for a part of a protein. This unit is called a codon.
* Any gene coding for a protein can have mutations or several options (one might code for brown eyes, another for blue).
* The different possible genes (with mutations or changes between them) are called alleles.
* The differences between alleles are typically reduced to single nucleotide polymorphisms (SNPs). An SNP is the smallest change (mutation) one can make to a region that codes for molecules by swapping out one of the letters that code for a nucleotide. Many times, the alleles will have long sequences of SNPs.
* The exons (complete set of genetic material that codes for molecules) comprise about 1% of genetic material.
* Many genetic tests check a maximum of 10% of the exon SNPs.
* This means genetic tests only check a maximum of 0.1% of genetic material, unless they do full exon sequencing tests.
* The complete genetic sequence of any human being has not been sequenced. See the article: [Psst, the human genome was never completely sequenced. Some scientists say it should be](https://www.statnews.com/2017/06/20/human-genome-not-fully-sequenced/).
* Additionally, only populations which are sedentary and not traveling and continually mixing DNA develop distinct statistical differences. Those traveling together will likely receive similar mutations to SNPs through interacting with a similar environment.
* Additionally, there is heritable epigenetic coding. Most tests and research papers ignore this. Also, we don't fully understand how this happens since the epigenetic coding is wiped and reprogrammed through an as of yet undiscovered process.

You should take a look at [Whole genome sequencing](https://en.wikipedia.org/wiki/Whole_genome_sequencing) over at Wikipedia for more info.

## General Genetics

### Overview

The following Wikipedia articles contain the basics needed to progress to the next set of papers. You'll likely need to follow all of the links in the articles and fully explore the concepts.

* [Introduction to genetics](https://en.wikipedia.org/wiki/Introduction_to_genetics)
* [DNA](https://en.wikipedia.org/wiki/DNA)
* [RNA](https://en.wikipedia.org/wiki/RNA)
* [Genetic code](https://en.wikipedia.org/wiki/Genetic_code)
* [Genome](https://en.wikipedia.org/wiki/Genome)
* [Gene](https://en.wikipedia.org/wiki/Gene)
* [Gene expression](https://en.wikipedia.org/wiki/Gene_expression)
* [Gene–environment interaction](https://en.wikipedia.org/wiki/Gene%E2%80%93environment_interaction)
* [Epigenetics](https://en.wikipedia.org/wiki/Epigenetics)

### Not overview

The following journal papers contain most of the overview that one would need to understand genetics at a higher level necessary to understand the papers in the next section. In addition to these papers, a sound basis in math and statistics is needed to understand many of the methods used for creating the various graphs and relationship trees having the most parsimony.

* [Reich D (2016) The Genomic Ancient DNA Revolution. Edge. Feb.](https://www.edge.org/conversation/david_reich-the-genomic-ancient-dna-revolution)
* [Salem MSZ (2016) Formal genetic maps. Egyptian Journal of Medical Human Genetics. 16(2): 101-116.](https://doi.org/10.1016/j.ejmhg.2014.11.006)
* [Jo B, Chois SS (2015) Introns: The Functional Benefits of Introns in Genomes. Genomics & Informatics. 13(4): 112-118.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4742320/)
### Relatedness and Divergence over time.

* [Sethuraman A (2018) Estimating Genetic Relatedness in Admixed Populations. Genes Genomes Genetics. 8(10): 3203–3220.](https://dx.doi.org/10.1534%2Fg3.118.200485)
* [McIver LJ, McCormick JF, Martin A, Fondon III JW, Garner HR (2013) Population-scale analysis of human microsatellites reveals novel sources of exonic variation. Gene. 516(2)](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3815531/)
* [Kwong M, Pemberton TJ (2014) Sequence differences at orthologous microsatellites inflate estimates of human-chimpanzee differentiation. BMC Genomics. 15: 990.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4253012/)
* [Vaksman Z, Fonville NC, Tae H, Garner HR (2014) Exome-Wide Somatic Microsatellite Variation Is Altered in Cells with DNA Repair Deficiencies. PLoS ONE 9(11): e110263. https://doi.org/10.1371/journal.pone.0110263](https://doi.org/10.1371/journal.pone.0110263)
* [Pemberton TJ et al. (2013) Population Structure in a Comprehensive Genomic Data Set on Human Microsatellite Variation. Genes Genomes Genetics. 3(5): 891-907.](https://doi.org/10.1534/g3.113.005728)
* [López Herráez D, Bauchet M, Tang K, Theunert C, Pugach I, Li J, et al. (2009) Genetic Variation and Recent Positive Selection in Worldwide Human Populations: Evidence from Nearly 1 Million SNPs. PLoS ONE 4(11): e7888. https://doi.org/10.1371/journal.pone.0007888](https://doi.org/10.1371/journal.pone.0007888)
* [Rosenberg NA, Mahajan S, Ramachandran S, Zhao C, Pritchard JK, Feldman MW (2005) Clines, Clusters, and the Effect of Study Design on the Inference of Human Population Structure. PLoS Genet 1(6): e70. https://doi.org/10.1371/journal.pgen.0010070](https://doi.org/10.1371/journal.pgen.0010070)
* [Zhivotovsky LA, Rosenberg NA, Feldman MW (2003) Features of Evolution and Expansion of Modern Humans, Inferred from Genomewide Microsatellite Markers. American Journal of Human Genetics. 72(5): 1171–1186.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1180270/)
* [Driscoll CA et al. (2002) Genomic Microsatellites as Evolutionary Chronometers: A Test in Wild Cats. Genome Research. 12(3):414-23.](https://www.researchgate.net/publication/11487684_Genomic_Microsatellites_as_Evolutionary_Chronometers_A_Test_in_Wild_Cats)

### Accounting for Sequencing Errors

* [Ma X et al. (2019) Analysis of error profiles in deep next-generation sequencing data. Genome Biology. 20.](https://doi.org/10.1186/s13059-019-1659-6)
* [Bilton TP et al. (2018) Accounting for Errors in Low Coverage High-Throughput Sequencing Data When Constructing Genetic Maps Using Biparental Outcrossed Populations. 209(1): 65-76.](https://doi.org/10.1534/genetics.117.300627 )
* [Pfeiffer F et al. (2018) Systematic evaluation of error rates and causes in short samples in next-generation sequencing. Scientific Reports. 8(1): doi: 10.1038/s41598-018-29325-6](https://doi.org/10.1038/s41598-018-29325-6)
* [Wall JD et al. (2014) Estimating genotype error rates from high-coverage next-generation sequence data. Genome Research. 24(11): 1734-1739.](https://dx.doi.org/10.1101%2Fgr.168393.113)
* [Trimble WL et al. (2012) Short-read reading-frame predictors are not created equal: sequence error causes loss of signal. BMC Informatics. 13: 183. doi: 10.1186/1471-2105-13-183](https://doi.org/10.1186/1471-2105-13-183)
* [Wang XV et al. (2012) Estimation of sequencing error rates in short reads. BMC Bioinformatics. 13: 185. doi: 10.1186/1471-2105-13-185](https://dx.doi.org/10.1186%2F1471-2105-13-185)
* [Richterich T (1998) Estimation of Errors in "Raw" DNA Sequences: A Validation Study. Genome Research. 8: 251-259. doi: 10.1101/gr.8.3.251](https://genome.cshlp.org/content/8/3/251.full.html)

### The Myth of "Race"

* [Norton, H.L., Quillen, E.E., Bigham, A.W. et al. Human races are not like dog breeds: refuting a racist analogy. Evo Edu Outreach 12, 17 (2019). https://doi.org/10.1186/s12052-019-0109-y](https://doi.org/10.1186/s12052-019-0109-y)
* [Carja O et al. (2017) Worldwide patterns of human epigenetic variation. Nature Ecology & Evolution. 1: 1577–1583.](https://www.nature.com/articles/s41559-017-0299-z) &mdash; [bioRXiv](https://www.biorxiv.org/content/early/2015/07/05/021931)
* [Rosenberg NA (2011) A Population-Genetic Perspective on the Similarities and Differences among Worldwide Human Populations. Human Biology. 83(6): 659-684.](https://doi.org/10.3378/027.083.0601) &mdash; [Open Access](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3531797/)
* [Li JZ, Absher DD, et al. (2008) Worldwide Human Relationships Inferred from Genome-Wide Patterns of Variation. Science. 319(5866): 1100-1104.](https://science.sciencemag.org/content/319/5866/1100) &mdash; [PDF](https://pdfs.semanticscholar.org/b8de/1df47f30b54c57dce03fb14a4a94459b2e46.pdf) &mdash; Re-iterates the above finding.
  * `Together, these results reaffirm that within-population variation accounts for most of the genetic diversity in humans. However, the between-population variance is sufficient to reveal consistent population structure because subtle but nonrandom differences between populations accumulate over a large number of loci and yield principal components that can account for a major portion of the variation (21).`
* [Race, Ethnicity, and Genetics Working Group (2005) The Use of Racial, Ethnic, and Ancestral Categories in Human Genetics Research. American Journal of Human Genetics. 77(4): 519–532.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1275602/)
* [Excoffier, L., & Hamilton, G. (2003). Comment on genetic structure of human populations. Science, 300(5627), 1877. DOI: 10.1126/science.1083411](https://doi.or/10.1126/science.1083411)
* [Rosenberg NA et al. (2002) Genetic Structure of Human Populations. Science. 298(5602): 2381-2385.](https://science.sciencemag.org/content/298/5602/2381.full) &mdash; [PDF](https://www.researchgate.net/publication/10980474_Genetic_Structure_of_Human_Populations) &mdash; This is the original paper that started the meme that humans are roughly 99.9% the same. The biggest take away from the paper is that self reporting would likely be a reliable proxy for what genetic conditions they might get. The second take away was the intra-group genetic variation is greater than inter-group genetic variation.

## Genetic Evidence for the Peopling of the Americas

* [Ioannidis, A.G., Blanco-Portillo, J., Sandoval, K. et al. (2020) Native American gene flow into Polynesia predating Easter Island settlement. Nature. https://doi.org/10.1038/s41586-020-2487-2]
* [Blell M, Hunter MA (2019) Direct-to-Consumer Genetic Testing's Red Herring: “Genetic Ancestry” and Personalized Medicine. Frontiers in Medicine. 29 March.](https://doi.org/10.3389/fmed.2019.00048)
* [Walajahi H, Wilson DR, Hull SC (2019) Constructing Identities: The Implications of DTC Ancestry Testing for Tribal Communities. Genetic Medicine. 21(8): 1744-1750.](https://dx.doi.org/10.1038%2Fs41436-018-0429-2)
* [Pinotti T et al. (2019) Y Chromosome Sequences Reveal a Short Beringian Standstill, Rapid Expansion, and early Population structure of Native American Founders. Current Biology. 29(1): 149-157.](https://doi.org/10.1016/j.cub.2018.11.029)
* [Sikora, M., Pitulko, V. V., Sousa, V. C., Allentoft, M. E., Vinner, L., Rasmussen, S., ... & Yang, M. A. (2019). The population history of northeastern Siberia since the Pleistocene. Nature, 570(7760), 182-188.](https://doi.org/10.1038/s41586-019-1279-z)
* [Flegontov, P., Altınışık, N. E., Changmai, P., Rohland, N., Mallick, S., Adamski, N., ... & Flegontova, O. (2019). Palaeo-Eskimo genetic ancestry and the peopling of Chukotka and North America. Nature, 570(7760), 236-240.](https://doi.org/10.1038/s41586-019-1251-y)
* [Gnecchi-Ruscone GA et al. (2019) Dissecting the Pre-Columbian Genomic Ancestry of Native Americans along the Andes–Amazonia Divide, Molecular Biology and Evolution. 36(6): 1254–1269](https://doi.org/10.1093/molbev/msz066)
* [Grugni V et al. (2019) Analysis of the human Y-chromosome haplogroup Q characterizes ancient population movements in Eurasia and the Americas. BMC Biology. 17, 3.](https://bmcbiol.biomedcentral.com/articles/10.1186/s12915-018-0622-4)
* [Wright JL et al. (2018) Ancient nuclear genomes enable repatriation of Indigenous human remains. Science Advances. 4(12): eaau5064.](https://dx.doi.org/10.1126%2Fsciadv.aau5064)
* [Wade L (2018) Ancient DNA confirms Native Americans’ deep roots in North and South America. Science Magazine.](https://www.sciencemag.org/news/2018/11/ancient-dna-confirms-native-americans-deep-roots-north-and-south-america?fbclid=IwAR21oJI6ch48j-xRr5OlOn7Cyco9GKaDEqOpNAsrdhcfF3bD2tfGqpG6jx8)
* [Barbieri C et al. (2018) The genomic landscape of western South America: Andes, Amazonia and Pacific Coast. bioRXiv preprint.](https://www.biorxiv.org/content/10.1101/505628v1.full)
* [Posth C et al. (2018) Reconstructing the Deep Population History of Central and South America. Cell. DOI:https://doi.org/10.1016/j.cell.2018.10.027](https://www.cell.com/cell/fulltext/S0092-8674(18)31380-1)
* [Messina F, Di Corcia T, Ragazzo M, Sanchez Mellado C, Contini I, Malaspina P, et al. (2018) Signs of continental ancestry in urban populations of Peru through autosomal STR loci and mitochondrial DNA typing. PLoS ONE 13(7): e0200796.](https://doi.org/10.1371/journal.pone.0200796)
* [Moreno-Mayar JV et al. (2018) Early human dispersals within the Americas. Science. 10.1126/science.aav2621](https://www.researchgate.net/publication/328815661_Early_human_dispersals_within_the_Americas) &mdash; This is a fantastic paper. I'd love to apply it's techniques to a broader sample of populations and early DNA sequences.
* [Scheib CL et al. (2018) Ancient human parallel lineages within North America contributed to a coastal expansion. Science. 360(6392): 1024-1027.](https://science.sciencemag.org/content/360/6392/1024) &mdash; [PDF](https://www.academia.edu/36866318/Ancient_human_parallel_lineages_within_North_America_contributed_to_a_coastal_expansion)
* [Gómez-Carballa A et al. (2018) The peopling of South America and the trans-Andean gene flow of the first settlers. Genome Research. 28: 767-779.](https://genome.cshlp.org/content/28/6/767.full)
* [Brandini S et al. (2018) The Paleo-Indian Entry into South America According to Mitogenomes. Molecular Biology and Evolution. 35(2): 299-311.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5850732/)
* [Hamilton-Brehm SD, Hristova LT, Edwards SR, Wedding JR, Snow M, Kruger BR, et al. (2018) Ancient human mitochondrial DNA and radiocarbon analysis of archived quids from the Mule Spring Rockshelter, Nevada, USA. PLoS ONE 13(3): e0194223. https://doi.org/10.1371/journal.pone.0194223](https://doi.org/10.1371/journal.pone.0194223)
* [KINY Radio (2018) DNA of Indigenous peoples explained. KINY Radio.](http://www.kinyradio.com/news/news-of-the-north/dna-of-indigenous-peoples-explained/?fbclid=IwAR0_VYafbGYaKQsvQaGldUY5lLVn3g9uhYzhy1tgWiN-xHupxOoG80Q4k1A)
* [Hohenstatt B (2018) DNA, oral tradition indicate 10,000 years of Native history in Southeast. Juneau Empire. November 13.](https://www.juneauempire.com/news/dna-oral-tradition-indicate-10000-years-of-native-history-in-southeast/)
* [de Saint Pierre M (2017) Antiquity of mtDNA lineage D1g from the southern cone of South America supports pre-Clovis migration. Quaternary International. 444 B: 19-25.](https://doi.org/10.1016/j.quaint.2017.05.054)
* [Morrow JE (2017) After Anzick: Reconciling new genomic data and models with the archaeological evidence for peopling of the Americas. Quaternary International. Part B. 444, 1-3.](https://doi.org/10.1016/j.quaint.2017.07.002) &mdash; [PDF](https://www.researchgate.net/publication/318442263_After_Anzick_Reconciling_new_genomic_data_and_models_with_the_archaeological_evidence_for_peopling_of_the_Americas)
* [Lindo J et al. (2017) Ancient individuals from the North American Northwest Coast reveal 10,000 years of regional genetic continuity. PNAS. 114(16): 4093-4098.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5402414/)
* [González-Oliver A, Garfias-Morales E, Smith DG, Quinto-Sánchez M (2017) Mitochondrial DNA Analysis of Mazahua and Otomi Indigenous Populations from Estado de México Suggests a Distant Common Ancestry. Human Biology. 89(3): 3.](https://digitalcommons.wayne.edu/humbiol/vol89/iss3/3/)
* [Ochoa-Lugo MI, de Lourdez Muñoz M, et al. (2016) Genetic Affiliation of Pre-Hispanic and Contemporary Mayas through Maternal Linage. Human Biology. 88(2):136-167.](https://www.researchgate.net/publication/313483707_Genetic_Affiliation_of_Pre-Hispanic_and_Contemporary_Mayas_Through_Maternal_Linage)
* [López, John Emilio William (2016) Genetic Diversity in Urban and Rural Indigenous Mexico. Master's thesis, Harvard Extension School.](https://dash.harvard.edu/handle/1/33797263)
* [Rasmussen M, Sikora M, Albrechtsen A, Korneliussen TS, Moreno-Mayar JV, Poznik GD, et al. (2015) The ancestry and affiliations of Kennewick Man. Nature. 523: 455–458. 10.1038/nature14625](https://www.nature.com/articles/nature14625)
* [González-Martín A, Gorostiza A, Regalado-Liu L, Arroyo-Peña S, Tirado S, Nuño-Arana I, et al. (2015) Demographic History of Indigenous Populations in Mesoamerica Based on mtDNA Sequence Data. PLoS ONE 10(8): e0131791. https://doi.org/10.1371/journal.pone.0131791](https://doi.org/10.1371/journal.pone.0131791)
* [Vilar MG et al. (2014) Genetic diversity in Puerto Rico and its implications for the peopling of the Island and the West Indies. American Journal of Physical Anthropology. 155: 352-368.](https://www.academia.edu/20083458/Genetic_diversity_in_Puerto_Rico_and_its_implications_for_the_peopling_of_the_Island_and_the_West_Indies)
* [Brooks DR (2014) Genetic Diversity in Totonacan-Speaking Populations from Veracruz and Puebla States of Mexico. Proceedings of the National Conference of Undergraduate Research.](http://www.ncurproceedings.org/ojs/index.php/NCUR2014/article/view/1087)
* [Raghavan M, Skoglund P, et al. (2014) Upper Palaeolithic Siberian genome reveals dual ancestry of Native Americans. Nature. 505(7481): 87-91.](https://dx.doi.org/10.1038%2Fnature12736) &mdash; [Open Access](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4105016/)
* [Gravel S, Zakharia F, Moreno-Estrada A, Byrnes JK, Muzzio M, Rodriguez-Flores JL, et al. (2013) Reconstructing Native American Migrations from Whole-Genome and Whole-Exome Data. PLoS Genet 9(12): e1004023. https://doi.org/10.1371/journal.pgen.1004023](https://doi.org/10.1371/journal.pgen.1004023)
* [Amorim CEG, Bisso-Machado R, Ramallo V, Bortolini MC, Bonatto SL, Salzano FM, et al. (2013) A Bayesian Approach to Genome/Linguistic Relationships in Native South Americans. PLoS ONE 8(5): e64099. https://doi.org/10.1371/journal.pone.0064099](https://doi.org/10.1371/journal.pone.0064099)
* [Schurr TG et al. (2012) Clan, Language, and Migration History Has Shaped Genetic Diversity in Haida and Tlingit Populations From Southeast Alaska. American Journal of Physical Anthropology. 148(3): 422-435.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4335652/)
* [de Saint Pierre M, Bravi CM, Motti JMB, Fuku N, Tanaka M, Llop E, et al. (2012) An Alternative Model for the Early Peopling of Southern South America Revealed by Analyses of Three Mitochondrial DNA Haplogroups. PLoS ONE 7(9): e43486. https://doi.org/10.1371/journal.pone.0043486](https://doi.org/10.1371/journal.pone.0043486)
* [Gorostiza A, Acunha-Alonzo V, Regalado-Liu L, Tirado S, Granados J, Sámano D, et al. (2012) Reconstructing the History of Mesoamerican Populations through the Study of the Mitochondrial DNA Control Region. PLoS ONE 7(9): e44666. https://doi.org/10.1371/journal.pone.0044666](https://doi.org/10.1371/journal.pone.0044666)
* [Kemp BM, González-Oliver A, et al. (2010) Evaluating the Farming/Language Dispersal Hypothesis with genetic variation exhibited by populations in the Southwest and Mesoamerica. PNAS. 107(15): 6759-6764.](https://doi.org/10.1073/pnas.0905753107)
* [Rasmussen M, Li Y, et al. (2010) Ancient human genome sequence of an extinct Palaeo-Eskimo. Nature. 463: 757-762.](https://rdcu.be/bOjbe)
* [Achilli A, Perego UA, Bravi CM, Coble MD, Kong Q-P, Woodward SR, et al. (2008) The Phylogeny of the Four Pan-American MtDNA Haplogroups: Implications for Evolutionary and Disease Studies. PLoS ONE 3(3): e1764. https://doi.org/10.1371/journal.pone.0001764](https://doi.org/10.1371/journal.pone.0001764)
* [Melton, Philip Edward (2008) Genetic history and pre-Columbian Diaspora of Chibchan speaking populations: Molecular genetic evidence. Ph.D. thesis, University of Kansas.](https://kuscholarworks.ku.edu/handle/1808/4508?show=full)
* [Tamm E, Kivisild T, Reidla M, Metspalu M, Smith DG, Mulligan CJ, et al. (2007) Beringian Standstill and Spread of Native American Founders. PLoS ONE 2(9): e829. https://doi.org/10.1371/journal.pone.0000829](https://doi.org/10.1371/journal.pone.0000829)
* [Kemp BM et al. (2007), Genetic analysis of early holocene skeletal remains from Alaska and its implications for the settlement of the Americas. Am. J. Phys. Anthropol., 132: 605-621. doi:10.1002/ajpa.20543](https://doi.org/10.1002/ajpa.20543) &mdash; [PDF](https://www.igb.illinois.edu/sites/default/files/Kempetal2007.pdf)
* [Reséndez A, Kemp BM, et al. (2005) "An Analysis of Ancient Aztec mtDNA from Tlatelolco: Pre-Columbian Relations and the Spread of Uto-Aztecan." Biomolecular Archeology: Genetic Approaches to the Past. Center for Archeological Investigations, Ocassional Paper No. 32.](https://migrationcluster.ucdavis.edu/research/national-identity-shifting-border-texas-and-new-mexico-age-transition-1821-1848-0)
* [Malhi RS et al. (2003) Native American mtDNA prehistory in the American Southwest. American Journal of Physical Anthropology. 120(2): 108-124. <http://hdl.handle.net/2027.42/34277>](http://hdl.handle.net/2027.42/34277)
* [Hurles, M. E., Matisoo-Smith, E., Gray, R. D., & Penny, D. (2003). Untangling Oceanic settlement: the edge of the knowable. Trends in Ecology & Evolution, 18(10), 531-540.](https://doi.org/10.1016/S0169-5347(03)00245-3)
* [Torroni A, Schurr TG, et al. (1993) Asian affinities and continental radiation of the four founding Native American mtDNAs. American Journal of Human Genetics. American Journal of Human Genetics. 53(3): 563-590.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1682412/)
* [Torroni A, Sukernik RI, Schurr TG, et al. (1993). mtDNA variation of aboriginal Siberians reveals distinct genetic affinities with Native Americans. American journal of human genetics, 53(3), 591-608.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1682407/)
* [Shields GF, Schmiechen AM, Frazier BL, Redd A Voevoda MI, Reed JK, Ward RH (1993) mtDNA sequences suggest a recent evolutionary divergence for Beringian and northern North American populations. American Journal of Human Genetics. 53(3): 549-562.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1682422/)
* [Ward RH, Redd A, Valencia D, Pääbo S (1993) Genetic and linguistic differentiation in the Americas. PNAS. 90(22): 10663-10667.](https://dx.doi.org/10.1073%2Fpnas.90.22.10663)
* [Ward RH, Frazier BL, Dew-Jager K, Pääbo S (1991) Extensive mitochondrial diversity within a single Amerindian tribe. PNAS. 88(19): 8720-8724.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC52581/)
* [Schurr TG et al. (1990) Amerindian mitochondrial DNAs have rare Asian mutations at high frequencies, suggesting they derived from four primary maternal lineages. American Journal of Human Genetics. 46(3): 613-623.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1683611/)

* [Genetics and the History of Latin America](https://migrationcluster.ucdavis.edu/research/genetics-and-history-latin-america)
* [Genetics, Linguistics, and Prehistoric Migrations: An Analysis of California Indian Mitochondrial DNA Lineages](https://escholarship.org/uc/item/6g135010)
* [Genome-wide insights into the genetic history of human populations](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4381409/)
* [Genomic insights on the ethno-history of the Maya and the ‘Ladinos’ from Guatemala](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4422311/)
* [High-resolution mitochondrial DNA analysis sheds light on human diversity, cultural interactions, and population mobility in Northwestern Amazonia](https://www.researchgate.net/publication/320654352_High-resolution_mitochondrial_DNA_analysis_sheds_light_on_human_diversity_cultural_interactions_and_population_mobility_in_Northwestern_Amazonia)
* [HLA Class II Alleles in the Otomi Population of the Mezquital Valley: A Genetic Approach to the History of Interethnic Migrations in the Mexican Central Plateau](https://www.researchgate.net/publication/274396125_HLA_Class_II_Alleles_in_the_Otomi_Population_of_the_Mezquital_Valley_A_Genetic_Approach_to_the_History_of_Interethnic_Migrations_in_the_Mexican_Central_Plateau)
* [Large scale mitochondrial sequencing in Mexican Americans suggests a reappraisal of Native American origins](https://doi.org/10.1186/1471-2148-11-293)
* [Mitochondrial DNA and Prehistoric Settlements: Native Migrations on the Western Edge of North America](https://www.researchgate.net/publication/8485114_Mitochondrial_DNA_and_Prehistoric_Settlements_Native_Migrations_on_the_Western_Edge_of_North_America)
* [Mitochondrial DNA Studies of Native Americans: Conceptions and Misconceptions of the Population Prehistory of the Americas](https://pdfs.semanticscholar.org/a40f/fbea9601cea8b7c973a8cedbea297a6de20a.pdf)
* [Mitochondrial DNA Diversity in Indigenous Populations of the Southern Extent of Siberia, and the Origins of Native American Haplogroups](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3905771/)
* [Mitochondrial Population Genomics Supports a Single Pre-Clovis Origin with a Coastal Route for the Peopling of the Americas](https://doi.org/10.1016/j.ajhg.2007.11.013)
* [mtDNA haplogroup X: An ancient link between Europe/Western Asia and North America?](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1377656/)
* [mtDNA Variation among Greenland Eskimos: The Edge of the Beringian Expansion.](https://doi.org/10.1086/303038)
* [Origins and genetic legacies of the Caribbean Taino](https://doi.org/10.1073/pnas.1716839115)
* [Patterns of mtDNA Diversity in Northwestern North America](https://www.researchgate.net/publication/8485113_Patterns_of_mtDNA_Diversity_in_Northwestern_North_America)
* [The Genetic History Of The Otomi In The Central Mexican Valley](https://repository.upenn.edu/anthro_seniortheses/133/)
* [The Genetics of Mexico Recapitulates Native American Substructure and Affects Biomedical Traits](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4156478/)

## Early Hominid through Modern Human

This includes archeological and genetics papers to build a perspective on the diffusion of Hominids and modern humans.

* [Duveau J, et al. (2019) The composition of a Neandertal social group revealed by the hominin footprints at Le Rozel (Normandy, France). PNAS. 116(39): 19409-19414.](https://doi.org/10.1073/pnas.1901789116)
* [Faith JT, Rowan J, Du A (2019) Early hominins evolved within non-analog ecosystems. PNAS. 116(43): 21478-21483.](https://doi.org/10.1073/pnas.1909284116)
* [Schuster R (2019) Prehistoric Superglue May Reveal Neanderthal Experiments 200,000 Years Ago. Haaretz. November 13.](https://www.haaretz.com/archaeology/.premium.MAGAZINE-prehistoric-superglue-may-reveal-neanderthal-experiments-200-000-years-ago-1.8068691?fbclid=IwAR3Fb2ksOsyf8ftioHvT4pubyfF-TSoMG6rIW8ZkfZdr4La03fzZe1TnglY)
* [Chan EKF, et al. (2019) Human origins in a southern African palaeo-wetland and first migrations. Nature. 575: 185-189.](https://www.nature.com/articles/s41586-019-1714-1) &mdash; [PDF](https://www.nature.com/articles/s41586-019-1714-1.epdf?referrer_access_token=iAfJZZ_lpZl3huBpr5zq1NRgN0jAjWel9jnR3ZoTv0Ow72lSjAIzdDOJcHUPoWIRGEOjMmthCa5BpP_dciaujFlTFsWPygPnm_eHl2AcrLFjeSO9PGAR2vyJhXma8rECK6S40cpYYTwflvHD4NiZ7HKsXBLLsVGkx8YcO8B0aOZu7MlerKh-hv7q75L8HMWR8nuZQfJdUEeusZGlVVZJe91EQF82zbGiMzbevXVwLkpyiENHRd_a17eb9Y1IDkDOLfVDnQfqUdMXlojsECkV6esUOl3P3GBBWSs3U57dPLc3Yz7vKowijwOo0uwLwoNK&tracking_referrer=www.theguardian.com)
* [li Z et al. (2019) Engraved bones from the archaic hominin site of Lingjing, Henan Province. Antiquity. 93(370): 886-900.](https://doi.org/10.15184/aqy.2019.81)
* [Price M (2019) Ancient DNA puts a face on the mysterious Denisovans, extinct cousins of Neanderthals. Science News. September 19.](https://www.sciencemag.org/news/2019/09/ancient-dna-puts-face-mysterious-denisovans-extinct-cousins-neanderthals)
* [Roberts P, Amano N (2019) Plastic pioneers: Hominin biogeography east of the Movius Line during the Pleistocene. Archaeological Research in Asia. 17: 181-192.](https://www.sciencedirect.com/science/article/pii/S2352226718300989?via%3Dihub)
* [Sahle Y, Brooks AS (2019) Assessment of complex projectiles in the early Late Pleistocene at Aduma, Ethiopia. PLoS ONE 14(5): e0216716.](https://doi.org/10.1371/journal.pone.0216716)
* [Braun DR et al. (2019) Earliest known Oldowan artifacts at >2.58 Ma from Ledi-Geraru, Ethiopia, highlight early technological diversity. PNAS. 116(24): 11712-11717.](https://doi.org/10.1073/pnas.1820177116)
* [Shinde V et al. (2019) An Ancient Harappan Genome Lacks Ancestry from Steppe Pastoralists or Iranian Farmers. Cell.](https://doi.org/10.1016/j.cell.2019.08.048)
* [Narasimhan VM et al. (2019) The formation of human populations in South and Central Asia. Science. 35(6457): eaat7487](https://science.sciencemag.org/content/365/6457/eaat7487.full)
* [Schaefer NK, Shapiro B (2019) New middle chapter in the story of human evolution. Science. 365(6457): 981-982.](https://science.sciencemag.org/content/365/6457/981) &mdash; Has great references for an updated and more complete phylogenetic tree.
* National Geographic has a nice looking, but crude, map which traces the general flow of modern genetics out of Africa and into the rest of the world: [Map of Human Migration](https://genographic.nationalgeographic.com/human-journey/)
* [Ragsdale AP, Gravel S (2019) Models of archaic admixture and recent history from two-locus statistics. PLoS Genet 15(6): e1008204.](https://doi.org/10.1371/journal.pgen.1008204)
* [Bokelmann L et al. (2019) A genetic analysis of the Gibralter Neanderthals. PNAS. 116(31): 15610-15615.](https://doi.org/10.1073/pnas.1903984116)
* [Harvati K et al. (2019) Apidima Cave fossila provide earliest evidence of Homo sapiens in Eurasia. Nature. 571: 500-504.](https://www.nature.com/articles/s41586-019-1376-z)
* [Delson E (2019) An early dispersal of modern humans from Africa to Greece. Nature News. July 10.](https://www.nature.com/articles/d41586-019-02075-9) &mdash; Has a great map of locations and times for various known dispersals.
* [Cepelewicz, J (2019) Fossil DNA Reveals New Twists in Modern Human Origins. Quanta Magazine. August 29. Web. Accessed September 7, 2019.](https://www.quantamagazine.org/fossil-dna-reveals-new-twists-in-modern-human-origins-20190829/)
* [Roger AR, Harris NS, Achenbach AA (2019) Neanderthal-Denisovan ancestors interbred with a distantly-related hominin. bioRxiv preprint. 657247.](https://doi.org/10.1101/657247)
* [Potts R et al. (2018) Environmental dynamics during the onset of the Middle Stone Age in eastern Africa. Science. 360(6384): 86-90.](https://science.sciencemag.org/content/360/6384/86) &mdash; Obsidian obtained from far away created a technological change in eastern Africa around 320 ka ago.
* [Brooks AS et al. (2018) Long-distance stone transport and pigment use in the earliest Middle Stone Age. Science. 360(6384): 90-94.](https://science.sciencemag.org/content/360/6384/90)
  * `The Middle Stone Age (MSA) comprises a diverse group of African industries characterized by the absence or rarity of large cutting tools (LCTs, such as Acheulean handaxes and cleavers), the presence of prepared core (Levallois) technologies, and regional and/or temporal variation in weapon armatures (1–3). Later MSA industries contain varying frequencies of small retouched bladelets and geometric artifacts (4), small unifacially and bifacially worked points, and larger lanceolate and bone points (5, 6). African MSA sites are associated with the oldest known fossils attributed to Homo sapiens, which date between ~195 and ~350 thousand years (ka) ago (7–9). After 130 ka ago, late Pleistocene MSA sites, mostly from the north and south temperate zone extremities of the continent, document some of the oldest beads (10, 11), complex geometric designs (12, 13), lithic projectile armatures interpreted as arrow or dart tips (14, 15), paint production (16), heat treatment of lithics (17), and expanded resource use, including fishing and shellfish collecting (18, 19).`
  * `Although these late Pleistocene behaviors are thought to reflect advanced cognitive abilities for learning, imitation, working memory, planning, and recursive technological innovation, their middle Pleistocene or earlier antecedents have remained obscure. Evolutionary explanations for the middle Pleistocene enlargement of relative brain size, largely overlapping the modern H. sapiens range, include a shift to larger social groups or networks (20); greater foraging, technological, and social adaptability in the face of more rapid and unpredictable climate fluctuations (21); and the development of symbolic expression (22).`
* [Deino AL et al. (2018) Chronology of the Acheulean to Middle Stone Age transition in eastern Africa. Science. 360(6384): 95-98.](https://science.sciencemag.org/content/360/6384/95)
* [Zhu Z et al. (2018) Hominin occupation of the Chinese Loess Plateau since about 2.1 million years ago. Nature. 559: 608-612.](https://www.nature.com/articles/s41586-018-0299-4)
* [Karakostis FA et al. (2018) Evidence for precision grasping in Neandertal daily activities. Science Advances. 4(9): eaat2369](http://advances.sciencemag.org/content/4/9/eaat2369)
* [Hoffman DL et al. (2018) Symbolic use of marine shells and mineral pigments by Iberian Neandertals 115,000 years ago. Science Advances. 4(2): eaar5255](http://advances.sciencemag.org/content/4/2/eaar5255)
* [Browning SR et al. (2018) Analysis of Human Sequence Data Reveals Two Pulses of Archaic Denisovan Admixture. Cell. 173(1): 53-61.](https://doi.org/10.1016/j.cell.2018.02.031)
* [Wolf AB, Akey JM (2018) Outstanding questions in the study of archaic hominin admixture. PLoS Genet 14(5): e1007349.](https://doi.org/10.1371/journal.pgen.1007349)
* [Slon V et al. (2018) The genome of the offspring of a Neanderthal mother and a Denisovan father. Nature. 561: 113-116.](https://www.nature.com/articles/s41586-018-0455-x)
* [Groucutt HS, Grün R, Zalmout IAS, Drake NA, Armitage SJ, Candy I, et al. (2018). Homo sapiens in Arabia by 85,000 years ago. Nature Ecology & Evolution. doi:10.1038/s41559-018-0518-2](https://doi.org/10.1038/s41559-018-0518-2)
* [Scerri EML et al. (2018) Did Our Species Evolve in Subdivided Populations across Africa, and Why Does It Matter? Trends in Ecology & Evolution. 33(8): 582-594.](https://doi.org/10.1016/j.tree.2018.05.005)
* [Ohman J, Gomez-Olivencia A (2018) Neanderthals cared for each other and survived into old age – new research. HeritageDaily. April 11.](https://www.heritagedaily.com/2018/04/neanderthals-cared-for-each-other-and-survived-into-old-age-new-research/118812)
* [Gomez-Olivencia A et al. (2018) La Ferrassie 1: New perspectives on a “classic” Neandertal. Journal of Human Evolution. 117: 13-32.](https://doi.org/10.1016/j.jhevol.2017.12.004)
* [Daura J et al. (2017) New Middle Pleistocene hominin cranium from the Gruta da Aroeira (Portugal). PNAS. 114(13): 3397-3402.](https://doi.org/10.1073/pnas.1619040114)
* [Posth C et al. (2017) Deeply divergent archaic mitochondrial genome provides lower time boundary for African gene flow into Neanderthals. Nature Communications. 8: 16046.](https://www.nature.com/articles/ncomms16046)
* [Putt SS, Wijeakumar S, Franciscus RG, Spencer JP (2017) The functional brain networks that underlie Early Stone Age tool manufacture. Nature Human Behaviour. 1: 0102.](https://www.nature.com/articles/s41562-017-0102)
* [Scerri EML et al. (2017) Persistence of Middle Stone Age technology to the Pleistocene/Holocene transition supports a complex hominin evolutionary scenario in West Africa. Journal of Archeological Science: Reports. 11: 639-646.](https://doi.org/10.1016/j.jasrep.2017.01.003)
* [Schelbusch CM et al. (2017) Southern African ancient genomes estimate modern human divergence to 350,000 to 260,000 years ago. Science. 358(6363): 652-655.](https://science.sciencemag.org/content/358/6363/652)
* [Skoglund P et al. (2017) Reconstructing Prehistoric African Population Structure. Cell.1 71(1): 59-71.](https://doi.org/10.1016/j.cell.2017.08.049)
  * `Africa harbors more genetic diversity than any other part of the world (Cann et al., 1987, Tishkoff et al., 2009). This is reflected both in a higher average number of differences among sub-Saharan African genomes than among non-African genomes (Cann et al., 1987, Ramachandran et al., 2005) and in the fact that the ancestry found outside of Africa is largely a subset of that within it (Tishkoff et al., 2009).`
* [Hublin J-J et al. (2017) New fossils from Jebel Irhoud, Morocco and the pan-African origin of Homo sapiens. Nature. 546: 289–292.](https://www.nature.com/articles/nature22336)
* [Richter D (2017) The age of the hominin fossils from Jebel Irhoud, Morocco, and the origins of the Middle Stone Age. Nature. 546: 293-296.](https://doi.org/10.1038/nature22335) &mdash; Pushes back presence of modern humans to 286 ± 32 ka ago.
* [Langley M (2017) World’s scientists turn to Asia and Australia to rewrite human history. The Conversation. 7 Dec. 2017.](http://theconversation.com/worlds-scientists-turn-to-asia-and-australia-to-rewrite-human-history-88697)
* [Gibbens S (2017) Humans Arrived in Australia 18,000 Years Earlier Than We Thought. National Geographic. 20 Jul.](https://news.nationalgeographic.com/2017/07/australia-aboriginal-early-human-evolution-spd/)
* [Gibbons A (2017) A find in Australia hints at very early human exit from Africa. 19 Jul.](http://www.sciencemag.org/news/2017/07/find-australia-hints-very-early-human-exit-africa)
* [Callaway E (2017) Oldest Homo sapiens fossil claim rewrites our species' history. Nature News. 07 Jun.](https://www.nature.com/news/oldest-homo-sapiens-fossil-claim-rewrites-our-species-history-1.22114)
* [Gibbons A (2017) World’s oldest Homo sapiens fossils found in Morocco. Science. 7 Jun.](http://www.sciencemag.org/news/2017/06/world-s-oldest-homo-sapiens-fossils-found-morocco)
* [Nielsen R et al. (2017) Tracing the peopling of the world through genomics. Nature. 541: 302-310.](https://www.nature.com/articles/nature21347) &mdash; Has a better map of possible migration routes which line up with indigenous histories. There's still some details missing, but it's the best 2017 had to offer.
* [Crevecoeur I et al. (2016) Late Stone Age human remains from Ishango (Democratic Republic of Congo): New insights on Late Pleistocene modern human diversity in Africa. Journal of Human Evolution. 96: 37-57.](https://doi.org/10.1016/j.jhevol.2016.04.003)
* [Stringer C (2016) The origin and evolution of Homo sapiens. Philosphical Transaction of Royal Society B. 371(1698)](https://doi.org/10.1098/rstb.2015.0237)
* [Mallick S et al. (2016) The Simons Genome Diversity Project: 300 genomes from 142 diverse populations. Nature. 538: 201-206.](https://www.nature.com/articles/nature18964) &mdash; This has an interesting neighbor joining tree based on pairwise divergence showing .
* [Malaspinas A-S et al. (2016) A genomic history of Aboriginal Australia. Nature. 538: 207–214.](https://www.nature.com/articles/nature18299)
* [Pagani L (2016) Genomic analyses inform on migration events during the peopling of Eurasia. Nature. 538: 238-242.](https://www.nature.com/articles/nature19792)
* [Qiu J (2016) How China is rewriting the book on human origins. Nature News. 12 Jul.](https://www.nature.com/news/how-china-is-rewriting-the-book-on-human-origins-1.20231)
* [(2016) Unprecedented study of Aboriginal Australians points to one shared Out of Africa migration for modern humans. 21 Sep.](https://www.cam.ac.uk/research/news/unprecedented-study-of-aboriginal-australians-points-to-one-shared-out-of-africa-migration-for)
* [Fu W et al. (2016) Robust Inference of Identity by Descent from Exome-Sequencing Data. American Journal of Human Gentics. 99(5): 1106-1116.](https://doi.org/10.1016/j.ajhg.2016.09.011)
* [Harris K, Nielson R (2016) The Genetic Cost of Neanderthal Introgression. Genetic. 203(2): 881-891.](https://doi.org/10.1534/genetics.116.186890)
* [Kuhlwilm M et al. (2016) Ancient gene flow from early modern humans into Eastern Neanderthals. Nature. 530: 429-433.](https://www.nature.com/articles/nature16544)
* [Lahr MM (2016) The shaping of human diversity: filters, boundaries and transitions. Philosphical Transaction of Royal Society B. 371(1698)](https://doi.org/10.1098/rstb.2015.0241)
* [Mathieson I et al. (2015) Eight thousand years of natural selection in Europe. bioRxiv preprint. 016477. doi: https://doi.org/10.1101/016477 Now published in Nature doi: 10.1038/nature16152](https://doi.org/10.1101/016477)
* [Roberts p et al. (2015) Direct evidence for human reliance on rainforest resources in late Pleistocene Sri Lanka. Science. 347(6227): 1246-1249.](https://science.sciencemag.org/content/347/6227/1246)
* [Haak W et al. (2015) Massive migration from the steppe was a source for Indo-European languages in Europe. Nature. 522(7555): 207-211](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5048219/)
* [Harmand S et al. (2015) 3.3-million-year-old stone tools from Lomekwi 3, West Turkana, Kenya. Nature. 521: 310-315.](https://www.nature.com/articles/nature14464)
* [Karmin M et al. (2015) A recent bottleneck of Y chromosome diversity coincides with a global change in culture. Genome Research. 25: 459-466.](https://genome.cshlp.org/content/25/4/459.full)
* [The 1000 Genomes Project Consortium (2015) A global reference for human genetic variation. Nature. 526: 68-74.](https://www.nature.com/articles/nature15393)
* [Miyagawa S et al. (2014) The integration hypothesis of human language evolution and the nature of contemporary languages. Frontiers in Psychology. 5.](https://doi.org/10.3389/fpsyg.2014.00564)
* [Fu Q et al. (2013) DNA analysis of an early modern human from Tianyuan Cave, China. PNAS. 110(6): 2223-2227.](https://doi.org/10.1073/pnas.1221359110)
* [Texier P-J et al. (2013) The context, form and significance of the MSA engraved ostrich eggshell collection from Diepkloof Rock Shelter, Western Cape, South Africa. Journal of Archeological Science. 40(9): 3412-3431.](https://doi.org/10.1016/j.jas.2013.02.021)
* [Blome MW et al. (2012) The environmental context for the origins of modern human diversity: A synthesis of regional variability in African climate 150,000–30,000 years ago. Journal of Human Evolution. 62(5): 563-592.](https://doi.org/10.1016/j.jhevol.2012.01.011)
* [Lachance J et al. (2012) Evolutionary History and Adaptation from High-Coverage Whole-Genome Sequences of Diverse African Hunter-Gatherers. Cell. 150(3): 457-469.](https://doi.org/10.1016/j.cell.2012.07.009)
* [Gronau I et al. (2011) Bayesian inference of ancient human demography from individual genome sequences. Nature Genetics. 43, 1031-1034.](https://www.nature.com/articles/ng.937)
* [McEvoy B et al. (2010) Whole-Genome Genetic Diversity in a Sample of Australians with Deep Aboriginal Ancestry. American Journal of Human Genetics. 13 Aug.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2917718/)
* [Texier P-J et al. (2010) A Howiesons Poort tradition of engraving ostrich eggshell containers dated to 60,000 years ago at Diepkloof Rock Shelter, South Africa. PNAS. 107(14): 6180-6185.](https://doi.org/10.1073/pnas.0913047107)
* [Bramanti B et al. (2009) Genetic Discontinuity Between Local Hunter-Gatherers and Central Europe’s First Farmers. Science. 326(5949): 137-140.](https://science.sciencemag.org/content/326/5949/137.abstract)
* [Lalueza-Fox C, Römpler H, et al. (2007) A Melanocortin 1 Receptor Allele Suggests Varying Pigmentation Among Neanderthals. Science. 318(5855): 1453-1455.](https://science.sciencemag.org/content/318/5855/1453)
* [Plagnol V, Wall JD (2006) Possible Ancestral Structure in Human Populations. PLoS Genet 2(7): e105.](https://doi.org/10.1371/journal.pgen.0020105)

## In Case You Are Interested in Dog Genetics

* [Norton, H.L., Quillen, E.E., Bigham, A.W. et al. Human races are not like dog breeds: refuting a racist analogy. Evo Edu Outreach 12, 17 (2019). https://doi.org/10.1186/s12052-019-0109-y](https://doi.org/10.1186/s12052-019-0109-y)
* [Leathlobhair, M. N., Perri, A. R., Irving-Pease, E. K., Witt, K. E., Linderholm, A., Haile, J., ... & Brace, S. (2018). The evolutionary history of dogs in the Americas. Science, 361(6397), 81-85.](https://doi.org/10.1126/science.aao4776)
* [Hecht, J. (2018) What a Dog Geneticist Wants You to Know about Dog Genetics. Dog Spies Blog. Scientific American.](https://blogs.scientificamerican.com/dog-spies/what-a-dog-geneticist-wants-you-to-know-about-dog-genetics/)
* [Parker, H. G., Dreger, D. L., Rimbault, M., Davis, B. W., Mullen, A. B., Carpintero-Ramirez, G., & Ostrander, E. A. (2017). Genomic analyses reveal the influence of geographic origin, migration, and hybridization on modern dog breed development. Cell reports, 19(4), 697-708.](https://doi.org/10.1016/j.celrep.2017.03.079)
* [Larson, G., Karlsson, E. K., Perri, A., Webster, M. T., Ho, S. Y., Peters, J., ... & Comstock, K. E. (2012). Rethinking dog domestication by integrating genetics, archeology, and biogeography. Proceedings of the National Academy of Sciences, 109(23), 8878-8883.](https://doi.org/10.1073/pnas.1203005109)
* [Sutter, N. B., Bustamante, C. D., Chase, K., Gray, M. M., Zhao, K., Zhu, L., ... & Quignon, P. (2007). A single IGF1 allele is a major determinant of small size in dogs. Science, 316(5821), 112-115.](https://doi.org/10.1126/science.1137045)
* [Ostrander, E. A. (2007). Genetics and the shape of dogs: studying the new sequence of the canine genome shows how tiny genetic changes can create enormous variation within a single species. American Scientist, 95(5), 406-413.](https://www.americanscientist.org/article/genetics-and-the-shape-of-dogs)
* [Parker, H. G., & Ostrander, E. A. (2005). Canine genomics and genetics: running with the pack. PLoS Genet, 1(5), e58.](https://doi.org/10.1371/journal.pgen.0010058)

## Was farming a mistake? (Should this go somewhere else?)

Certain people have notice that in industrialized societies our jaw is getting smaller (and so are certain other features of our skulls). This is an extention of the process seen historically through evolution starting with our jaws becoming less pronounced and our skulls bigger. But since the adoption of farming it has accelerated. This is thought to be due to not using our jaw muscles and not having to work to eat processed food. However the mechanism, it poses existential risks for humanity.

* [Antrosio J (2011) Agriculture as ‘Worst Mistake in the History of the Human Race’? Living Anthropologically. June 1.](https://www.livinganthropologically.com/archaeology/agriculture-worst-mistake/)
* [Antrosio J (2011) Many Ways of Gathering and Hunting. Living Anthropologically. June 9.](https://www.livinganthropologically.com/archaeology/gathering-and-hunting/)
* [Diamond J (1987) The Worst Mistake in the History of the Human Race. Discover Magazine. May. Published online May 1999.](http://discovermagazine.com/1987/may/02-the-worst-mistake-in-the-history-of-the-human-race) &mdash; please read the last two articles first for proper context.
* [Pennisi E (2004) Weak Jaw, Big Brain. Sicence News. March 24.](https://www.sciencemag.org/news/2004/03/weak-jaw-big-brain)
* [Hopkin M (2004) Jaw-dropping theory of human evolution. Nature News. March 25.](https://www.nature.com/news/2004/040322/full/news040322-9.html)
* [Stedman H et al. (2004) Myosin gene mutation correlates with anatomical changes in the human lineage. Nature. 428: 415-418.](https://www.nature.com/articles/nature02358)
* [Ungar P (2017) It's not that your teeth are too big: your jaw is too small. Aeon. Marina Benjamin, editor. June 30.](https://aeon.co/ideas/its-not-that-your-teeth-are-too-big-your-jaw-is-too-small)
* [von Cramon-Taubadel N (2011) Global human mandibular variation reflects differences in agricultural and hunter-gatherer subsistence strategies. PNAS. 108(49) 19546-19551.](https://doi.org/10.1073/pnas.1113050108)
* [Crompton AW (1963) The Evolution of the Mammalian Jaw. Evolution. 17(4): 431-439.](http://www.jstor.org/stable/2407093) &mdash; [PDF](https://crompton.oeb.harvard.edu/files/crompton/files/crompt1963_evolmammjaw.pdf)
* [Alex B (2019) The Human Brain Has been Getting Smaller Since the Stone Age. The Crux Blog. Discover Magazine. April 8.](blogs.discovermagazine.com/crux/2019/04/08/human-brains-have-shrunk-since-the-stone-age/)
* [McAuliffe K (2010) If Modern Humans Are So Smart, Why Are Our Brains Shrinking? Discover Magazine. September.](http://discovermagazine.com/2010/sep/25-modern-humans-smart-why-brain-shrinking)
* [Henneberg M, Steyn M (1993) Trends in cranial capacity and cranial index in Subsaharan Africa during the Holocene. American Journal of Human Biology. 5(4): 473-479.](https://doi.org/10.1002/ajhb.1310050411)
* [Brown P, Maeda T (2004) Post-Pleistocene diachronic change in East Asian facial skeletons: the size, shape and volume of the orbits. Anthropological Science. 112(1): 29-40.](https://doi.org/10.1537/ase.00072)
* [Brown P (1987) Pleistocene Homogeneity and Holocene Size Reduction: The Australian Human Skeletal Evidence. Archeology in Oceania. 22(2): 41-67.](https://www.jstor.org/stable/40386774)
* [Stringer, C (2014) Why Have Our Brains Started to Shrink? Scientific American Mind. November 1.](https://www.scientificamerican.com/article/why-have-our-brains-started-to-shrink/)
* [Henneberg M (1988) Decrease of Human Skull Size in the Holocene. Human Biology. 60(3): 395-405.](https://www.jstor.org/stable/41464021)
* [Thompson H (2014) Switching to Farming Made Human Joint Bones Lighter. Smithsonian. December 22.](https://www.smithsonianmag.com/science-nature/switching-farming-made-human-joint-bones-lighter-180953711/#rzByrssq22fhuOva.99)
* [Pinhasi R, Eshed V, von Cramon-Taubadel N (2015) Incongruity between Affinity Patterns Based on Mandibular and Lower Dental Dimensions following the Transition to Agriculture in the Near East, Anatolia and Europe. PLoS ONE 10(2): e0117301.](https://doi.org/10.1371/journal.pone.0117301)
* [Carpenter J (2011) Jawbones are 'shaped by diet', a study finds. BBC News November 23.](https://www.bbc.com/news/science-environment-15823276)
* [University of Kent. (2011, November 23). Jaw size linked to diet: Could too soft a diet cause lower jaw to stay too short and cause orthodontic problems?. ScienceDaily. Retrieved October 2, 2019 from www.sciencedaily.com/releases/2011/11/111122112032.htm](www.sciencedaily.com/releases/2011/11/111122112032.htm)
* [Lewis KR (2019) Our Skulls Are Out-Evolving Us. OneZero. September 19.](https://onezero.medium.com/our-skulls-are-out-evolving-us-and-that-could-mean-a-public-health-crisis-f950faed696d) &mdash; This seems to conflate many issues. It is the articles that made me look up the rest of this.
* [Hidaka BH (2012) Depression as a disease of modernity: explanations for increasing prevalence. J Affect Disord. 140(3): 205-214.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3330161/)
* [Sadeh A, Gruber R, Raviv A (2003) The Effects of Sleep Restriction and Extension on School‐Age Children: What a Difference an Hour Makes. Child Development. 74(2): 444-455.](https://doi.org/10.1111/1467-8624.7402008)
* [Galland B et al. (2015) Sleep Disordered Breathing and Academic Performance: A Meta-analysis. Pediatrics. 136(4): e934-e946.](http://pediatrics.aappublications.org/cgi/pmidlookup?view=long&pmid=26347434)


Does this have anything to do with farming, or just general stress induced by modernity?
* [Berreby, D (2013) The obesity era. Aeon. June 19.](https://aeon.co/essays/blaming-individuals-for-obesity-may-be-altogether-wrong)
* [Shell ER (2019) A New Theory of Obesity. Scientific American. October.](https://www.scientificamerican.com/article/a-new-theory-of-obesity/)
