# Genetics

Genetic studies are an area fraught with criticism. Some of these criticisms are for good reason, some from fear ([and there can be good reason behind the fear, too](https://en.wikipedia.org/wiki/Human_Genome_Diversity_Project#Potential_problems)). A proper scientific understanding of genetics is necessary to draw conclusions about the following papers. Any understanding mired in the nonsense of racism is bound to produce further nonscientific nonsense. Any studies which do not respect that ownership and copyright are not concepts recognized by indigenous peoples sovereign nations. The earth and all of creation "owns" the genetics. No one can retain ownership or copyright over genetics. Those concepts applied to genes and genetics are rendered as nonsense in many indigenous worldviews, including my own.

That being said, here is a the high-level overview of the very basics:

* An average person has an estimated 37,200,000,000,000 cells containing some genetic material.
* Genetic material contains DNA and RNA.
* Only nucleated cells have DNA.
* Both RNA and DNA are passed on from parents to offspring, but the mechanism for some of the transfer is not fully understood.
* A complete DNA sequence has over 6,000,000,000 base pairs.
* Base pairs are two each of (A - adenine, C - cytosine, G - guanine, T - thymine). These have a particular way of matching up with each other so only certain combinations are possible.
* These base pairs form strands which are weakly bonded through the base pair bond. One stand is the sense strand, the other is the anti-sense strand.
* These two strands form a double helix.
* The DNA is stored in 23 pairs of chromosomes in the nucleus of a cell.
* Parts of chromosomes are broken down into genes, which code for specific molecules.
* The DNA is comprised of sequences which code for molecules (exons) and DNA which typically gets turned into RNA (introns).
* In exons, every three base pairs codes for a part of a protein. This unit is called a codon.
* Any gene coding for a protein can have mutations or several options (one might code for brown eyes, another for blue).
* The different possible genes (with mutations or changes between them) are called alleles.
* The differences between alleles are typically reduced to single nucleotide polymorphisms (SNPs). An SNP is the smallest change (mutation) one can make to a region that codes for molecules by swapping out one of the letters that code for a nucleotide. Many times, the alleles will have long sequences of SNPs.
* The exons (complete set of genetic material that codes for molecules) comprise about 1% of genetic material.
* Many genetic tests check a maximum of 10% of the exon SNPs.
* This means genetic tests only check a maximum of 0.1% of genetic material, unless they do full exon sequencing tests.
* The complete genetic sequence of any human being has not been sequenced. See the article: [Psst, the human genome was never completely sequenced. Some scientists say it should be](https://www.statnews.com/2017/06/20/human-genome-not-fully-sequenced/).
* Additionally, only populations which are sedentary and not traveling and continually mixing DNA develop distinct statistical differences. Those traveling together will likely receive similar mutations to SNPs through interacting with a similar environment.
* Additionally, there is heritable epigenetic coding. Most tests and research papers ignore this. Also, we don't fully understand how this happens since the epigenetic coding is wiped and reprogrammed through an as of yet undiscovered process.

You should take a look at [Whole genome sequencing](https://en.wikipedia.org/wiki/Whole_genome_sequencing) over at Wikipedia for more info.

## General Genetics

### Overview

The following Wikipedia articles contain the basics needed to progress to the next set of papers. You'll likely need to follow all of the links in the articles and fully explore the concepts.

* [Introduction to genetics](https://en.wikipedia.org/wiki/Introduction_to_genetics)
* [DNA](https://en.wikipedia.org/wiki/DNA)
* [RNA](https://en.wikipedia.org/wiki/RNA)
* [Genetic code](https://en.wikipedia.org/wiki/Genetic_code)
* [Genome](https://en.wikipedia.org/wiki/Genome)
* [Gene](https://en.wikipedia.org/wiki/Gene)
* [Gene expression](https://en.wikipedia.org/wiki/Gene_expression)
* [Gene–environment interaction](https://en.wikipedia.org/wiki/Gene%E2%80%93environment_interaction)
* [Epigenetics](https://en.wikipedia.org/wiki/Epigenetics)

### Not overview

The following journal papers contain most of the overview that one would need to understand genetics at a higher level necessary to understand the papers in the next section. In addition to these papers, a sound basis in math and statistics is needed to understand many of the methods used for creating the various graphs and relationship trees having the most parsimony.

* [Reich D (2016) The Genomic Ancient DNA Revolution. Edge. Feb.](https://www.edge.org/conversation/david_reich-the-genomic-ancient-dna-revolution)
* [Salem MSZ (2016) Formal genetic maps. Egyptian Journal of Medical Human Genetics. 16(2): 101-116.](https://doi.org/10.1016/j.ejmhg.2014.11.006)
* [Jo B, Chois SS (2015) Introns: The Functional Benefits of Introns in Genomes. Genomics & Informatics. 13(4): 112-118.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4742320/)
* [McIver LJ, McCormick JF, Martin A, Fondon III JW, Garner HR (2013) Population-scale analysis of human microsatellites reveals novel sources of exonic variation. Gene. 516(2)](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3815531/)
* [Kwong M, Pemberton TJ (2014) Sequence differences at orthologous microsatellites inflate estimates of human-chimpanzee differentiation. BMC Genomics. 15: 990.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4253012/)
* [Vaksman Z, Fonville NC, Tae H, Garner HR (2014) Exome-Wide Somatic Microsatellite Variation Is Altered in Cells with DNA Repair Deficiencies. PLoS ONE 9(11): e110263. https://doi.org/10.1371/journal.pone.0110263](https://doi.org/10.1371/journal.pone.0110263)
* [Race, Ethnicity, and Genetics Working Group (2005) The Use of Racial, Ethnic, and Ancestral Categories in Human Genetics Research. American Journal of Human Genetics. 77(4): 519–532.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1275602/)
* [Rosenberg NA (2011) A Population-Genetic Perspective on the Similarities and Differences among Worldwide Human Populations. Human Biology. 83(6): 659-684.](https://doi.org/10.3378/027.083.0601) &mdash; [Open Access](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3531797/)
* [Carja O et al. (2017) Worldwide patterns of human epigenetic variation. Nature Ecology & Evolution. 1: 1577–1583.](https://www.nature.com/articles/s41559-017-0299-z) &mdash; [bioRXiv](https://www.biorxiv.org/content/early/2015/07/05/021931)
* [Pemberton TJ et al. (2013) Population Structure in a Comprehensive Genomic Data Set on Human Microsatellite Variation. Genes Genomes Genetics. 3(5): 891-907.](https://doi.org/10.1534/g3.113.005728)
* [Rosenberg NA, Mahajan S, Ramachandran S, Zhao C, Pritchard JK, Feldman MW (2005) Clines, Clusters, and the Effect of Study Design on the Inference of Human Population Structure. PLoS Genet 1(6): e70. https://doi.org/10.1371/journal.pgen.0010070](https://doi.org/10.1371/journal.pgen.0010070)
* [Driscoll CA et al. (2002) Genomic Microsatellites as Evolutionary Chronometers: A Test in Wild Cats. Genome Research. 12(3):414-23.](https://www.researchgate.net/publication/11487684_Genomic_Microsatellites_as_Evolutionary_Chronometers_A_Test_in_Wild_Cats)
* [Rosenberg NA et al. (2002) Genetic Structure of Human Populations. Science. 298(5602): 2381-2385.](https://science.sciencemag.org/content/298/5602/2381.full) &mdash; [PDF](https://www.researchgate.net/publication/10980474_Genetic_Structure_of_Human_Populations) &mdash; This is the original paper that started the meme that humans are roughly 99.9% the same. The biggest take away from the paper is that self reporting would likely be a reliable proxy for what genetic conditions they might get. The second take away was the intra-group genetic variation is greater than inter-group genetic variation.
* [Li JZ, Absher DD, et al. (2008) Worldwide Human Relationships Inferred from Genome-Wide Patterns of Variation. Science. 319(5866): 1100-1104.](https://science.sciencemag.org/content/319/5866/1100) &mdash; [PDF](https://pdfs.semanticscholar.org/b8de/1df47f30b54c57dce03fb14a4a94459b2e46.pdf) &mdash; Re-iterates the above finding.
  * `Together, these results reaffirm that within-population variation accounts for most of the genetic diversity in humans. However, the between-population variance is sufficient to reveal consistent population structure because subtle but nonrandom differences between populations accumulate over a large number of loci and yield principal components that can account for a major portion of the variation (21).`
* [López Herráez D, Bauchet M, Tang K, Theunert C, Pugach I, Li J, et al. (2009) Genetic Variation and Recent Positive Selection in Worldwide Human Populations: Evidence from Nearly 1 Million SNPs. PLoS ONE 4(11): e7888. https://doi.org/10.1371/journal.pone.0007888](https://doi.org/10.1371/journal.pone.0007888)
* [Zhivotovsky LA, Rosenberg NA, Feldman MW (2003) Features of Evolution and Expansion of Modern Humans, Inferred from Genomewide Microsatellite Markers. American Journal of Human Genetics. 72(5): 1171–1186.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1180270/)
* [Sethuraman A (2018) Estimating Genetic Relatedness in Admixed Populations. Genes Genomes Genetics. 8(10): 3203–3220.](https://dx.doi.org/10.1534%2Fg3.118.200485)

## Genetic Evidence for the Peopling of the Americas

* [Blell M, Hunter MA (2019) Direct-to-Consumer Genetic Testing's Red Herring: “Genetic Ancestry” and Personalized Medicine. Frontiers in Medicine. 29 March.](https://doi.org/10.3389/fmed.2019.00048)
* [Walajahi H, Wilson DR, Hull SC (2019) Constructing Identities: The Implications of DTC Ancestry Testing for Tribal Communities. Genetic Medicine. 21(8): 1744-1750.](https://dx.doi.org/10.1038%2Fs41436-018-0429-2)
* [Pinotti T et al. (2019) Y Chromosome Sequences Reveal a Short Beringian Standstill, Rapid Expansion, and early Population structure of Native American Founders. Current Biology. 29(1): 149-157.](https://doi.org/10.1016/j.cub.2018.11.029)
* [Gnecchi-Ruscone GA et al. (2019) Dissecting the Pre-Columbian Genomic Ancestry of Native Americans along the Andes–Amazonia Divide, Molecular Biology and Evolution. 36(6): 1254–1269](https://doi.org/10.1093/molbev/msz066)
* [Grugni V et al. (2019) Analysis of the human Y-chromosome haplogroup Q characterizes ancient population movements in Eurasia and the Americas. BMC Biology. 17, 3.](https://bmcbiol.biomedcentral.com/articles/10.1186/s12915-018-0622-4)
* [Wright JL et al. (2018) Ancient nuclear genomes enable repatriation of Indigenous human remains. Science Advances. 4(12): eaau5064.](https://dx.doi.org/10.1126%2Fsciadv.aau5064)
* [Wade L (2018) Ancient DNA confirms Native Americans’ deep roots in North and South America. Science Magazine.](https://www.sciencemag.org/news/2018/11/ancient-dna-confirms-native-americans-deep-roots-north-and-south-america?fbclid=IwAR21oJI6ch48j-xRr5OlOn7Cyco9GKaDEqOpNAsrdhcfF3bD2tfGqpG6jx8)
* [Barbieri C et al. (2018) The genomic landscape of western South America: Andes, Amazonia and Pacific Coast. bioRXiv preprint.](https://www.biorxiv.org/content/10.1101/505628v1.full)
* [Posth C et al. (2018) Reconstructing the Deep Population History of Central and South America. Cell. DOI:https://doi.org/10.1016/j.cell.2018.10.027](https://www.cell.com/cell/fulltext/S0092-8674(18)31380-1)
* [Messina F, Di Corcia T, Ragazzo M, Sanchez Mellado C, Contini I, Malaspina P, et al. (2018) Signs of continental ancestry in urban populations of Peru through autosomal STR loci and mitochondrial DNA typing. PLoS ONE 13(7): e0200796.](https://doi.org/10.1371/journal.pone.0200796)
* [Moreno-Mayar JV et al. (2018) Early human dispersals within the Americas. Science. 10.1126/science.aav2621](https://www.researchgate.net/publication/328815661_Early_human_dispersals_within_the_Americas) &mdash; This is a fantastic paper. I'd love to apply it's techniques to a broader sample of populations and early DNA sequences.
* [Scheib CL et al. (2018) Ancient human parallel lineages within North America contributed to a coastal expansion. Science. 360(6392): 1024-1027.](https://science.sciencemag.org/content/360/6392/1024) &mdash; [PDF](https://www.academia.edu/36866318/Ancient_human_parallel_lineages_within_North_America_contributed_to_a_coastal_expansion)
* [Gómez-Carballa A et al. (2018) The peopling of South America and the trans-Andean gene flow of the first settlers. Genome Research. 28: 767-779.](https://genome.cshlp.org/content/28/6/767.full)
* [Brandini S et al. (2018) The Paleo-Indian Entry into South America According to Mitogenomes. Molecular Biology and Evolution. 35(2): 299-311.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5850732/)
* [Hamilton-Brehm SD, Hristova LT, Edwards SR, Wedding JR, Snow M, Kruger BR, et al. (2018) Ancient human mitochondrial DNA and radiocarbon analysis of archived quids from the Mule Spring Rockshelter, Nevada, USA. PLoS ONE 13(3): e0194223. https://doi.org/10.1371/journal.pone.0194223](https://doi.org/10.1371/journal.pone.0194223)
* [KINY Radio (2018) DNA of Indigenous peoples explained. KINY Radio.](http://www.kinyradio.com/news/news-of-the-north/dna-of-indigenous-peoples-explained/?fbclid=IwAR0_VYafbGYaKQsvQaGldUY5lLVn3g9uhYzhy1tgWiN-xHupxOoG80Q4k1A)
* [Hohenstatt B (2018) DNA, oral tradition indicate 10,000 years of Native history in Southeast. Juneau Empire. November 13.](https://www.juneauempire.com/news/dna-oral-tradition-indicate-10000-years-of-native-history-in-southeast/)
* [de Saint Pierre M (2017) Antiquity of mtDNA lineage D1g from the southern cone of South America supports pre-Clovis migration. Quaternary International. 444 B: 19-25.](https://doi.org/10.1016/j.quaint.2017.05.054)
* [Morrow JE (2017) After Anzick: Reconciling new genomic data and models with the archaeological evidence for peopling of the Americas. Quaternary International. Part B. 444, 1-3.](https://doi.org/10.1016/j.quaint.2017.07.002) &mdash; [PDF](https://www.researchgate.net/publication/318442263_After_Anzick_Reconciling_new_genomic_data_and_models_with_the_archaeological_evidence_for_peopling_of_the_Americas)
* [Lindo J et al. (2017) Ancient individuals from the North American Northwest Coast reveal 10,000 years of regional genetic continuity. PNAS. 114(16): 4093-4098.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5402414/)
* [González-Oliver A, Garfias-Morales E, Smith DG, Quinto-Sánchez M (2017) Mitochondrial DNA Analysis of Mazahua and Otomi Indigenous Populations from Estado de México Suggests a Distant Common Ancestry. Human Biology. 89(3): 3.](https://digitalcommons.wayne.edu/humbiol/vol89/iss3/3/)
* [Ochoa-Lugo MI, de Lourdez Muñoz M, et al. (2016) Genetic Affiliation of Pre-Hispanic and Contemporary Mayas through Maternal Linage. Human Biology. 88(2):136-167.](https://www.researchgate.net/publication/313483707_Genetic_Affiliation_of_Pre-Hispanic_and_Contemporary_Mayas_Through_Maternal_Linage)
* [López, John Emilio William (2016) Genetic Diversity in Urban and Rural Indigenous Mexico. Master's thesis, Harvard Extension School.](https://dash.harvard.edu/handle/1/33797263)
* [Rasmussen M, Sikora M, Albrechtsen A, Korneliussen TS, Moreno-Mayar JV, Poznik GD, et al. (2015) The ancestry and affiliations of Kennewick Man. Nature. 523: 455–458. 10.1038/nature14625](https://www.nature.com/articles/nature14625)
* [González-Martín A, Gorostiza A, Regalado-Liu L, Arroyo-Peña S, Tirado S, Nuño-Arana I, et al. (2015) Demographic History of Indigenous Populations in Mesoamerica Based on mtDNA Sequence Data. PLoS ONE 10(8): e0131791. https://doi.org/10.1371/journal.pone.0131791](https://doi.org/10.1371/journal.pone.0131791)
* [Vilar MG et al. (2014) Genetic diversity in Puerto Rico and its implications for the peopling of the Island and the West Indies. American Journal of Physical Anthropology. 155: 352-368.](https://www.academia.edu/20083458/Genetic_diversity_in_Puerto_Rico_and_its_implications_for_the_peopling_of_the_Island_and_the_West_Indies)
* [Brooks DR (2014) Genetic Diversity in Totonacan-Speaking Populations from Veracruz and Puebla States of Mexico. Proceedings of the National Conference of Undergraduate Research.](http://www.ncurproceedings.org/ojs/index.php/NCUR2014/article/view/1087)
* [Raghavan M, Skoglund P, et al. (2014) Upper Palaeolithic Siberian genome reveals dual ancestry of Native Americans. Nature. 505(7481): 87-91.](https://dx.doi.org/10.1038%2Fnature12736) &mdash; [Open Access](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4105016/)
* [Gravel S, Zakharia F, Moreno-Estrada A, Byrnes JK, Muzzio M, Rodriguez-Flores JL, et al. (2013) Reconstructing Native American Migrations from Whole-Genome and Whole-Exome Data. PLoS Genet 9(12): e1004023. https://doi.org/10.1371/journal.pgen.1004023](https://doi.org/10.1371/journal.pgen.1004023)
* [Amorim CEG, Bisso-Machado R, Ramallo V, Bortolini MC, Bonatto SL, Salzano FM, et al. (2013) A Bayesian Approach to Genome/Linguistic Relationships in Native South Americans. PLoS ONE 8(5): e64099. https://doi.org/10.1371/journal.pone.0064099](https://doi.org/10.1371/journal.pone.0064099)
* [Schurr TG et al. (2012) Clan, Language, and Migration History Has Shaped Genetic Diversity in Haida and Tlingit Populations From Southeast Alaska. American Journal of Physical Anthropology. 148(3): 422-435.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4335652/)
* [de Saint Pierre M, Bravi CM, Motti JMB, Fuku N, Tanaka M, Llop E, et al. (2012) An Alternative Model for the Early Peopling of Southern South America Revealed by Analyses of Three Mitochondrial DNA Haplogroups. PLoS ONE 7(9): e43486. https://doi.org/10.1371/journal.pone.0043486](https://doi.org/10.1371/journal.pone.0043486)
* [Gorostiza A, Acunha-Alonzo V, Regalado-Liu L, Tirado S, Granados J, Sámano D, et al. (2012) Reconstructing the History of Mesoamerican Populations through the Study of the Mitochondrial DNA Control Region. PLoS ONE 7(9): e44666. https://doi.org/10.1371/journal.pone.0044666](https://doi.org/10.1371/journal.pone.0044666)
* [Kemp BM, González-Oliver A, et al. (2010) Evaluating the Farming/Language Dispersal Hypothesis with genetic variation exhibited by populations in the Southwest and Mesoamerica. PNAS. 107(15): 6759-6764.](https://doi.org/10.1073/pnas.0905753107)
* [Rasmussen M, Li Y, et al. (2010) Ancient human genome sequence of an extinct Palaeo-Eskimo. Nature. 463: 757-762.](https://rdcu.be/bOjbe)
* [Achilli A, Perego UA, Bravi CM, Coble MD, Kong Q-P, Woodward SR, et al. (2008) The Phylogeny of the Four Pan-American MtDNA Haplogroups: Implications for Evolutionary and Disease Studies. PLoS ONE 3(3): e1764. https://doi.org/10.1371/journal.pone.0001764](https://doi.org/10.1371/journal.pone.0001764)
* [Melton, Philip Edward (2008) Genetic history and pre-Columbian Diaspora of Chibchan speaking populations: Molecular genetic evidence. Ph.D. thesis, University of Kansas.](https://kuscholarworks.ku.edu/handle/1808/4508?show=full)
* [Tamm E, Kivisild T, Reidla M, Metspalu M, Smith DG, Mulligan CJ, et al. (2007) Beringian Standstill and Spread of Native American Founders. PLoS ONE 2(9): e829. https://doi.org/10.1371/journal.pone.0000829](https://doi.org/10.1371/journal.pone.0000829)
* [Kemp BM et al. (2007), Genetic analysis of early holocene skeletal remains from Alaska and its implications for the settlement of the Americas. Am. J. Phys. Anthropol., 132: 605-621. doi:10.1002/ajpa.20543](https://doi.org/10.1002/ajpa.20543) &mdash; [PDF](https://www.igb.illinois.edu/sites/default/files/Kempetal2007.pdf)
* [Reséndez A, Kemp BM, et al. (2005) "An Analysis of Ancient Aztec mtDNA from Tlatelolco: Pre-Columbian Relations and the Spread of Uto-Aztecan." Biomolecular Archeology: Genetic Approaches to the Past. Center for Archeological Investigations, Ocassional Paper No. 32.](https://migrationcluster.ucdavis.edu/research/national-identity-shifting-border-texas-and-new-mexico-age-transition-1821-1848-0)
* [Malhi RS et al. (2003) Native American mtDNA prehistory in the American Southwest. American Journal of Physical Anthropology. 120(2): 108-124. <http://hdl.handle.net/2027.42/34277>](http://hdl.handle.net/2027.42/34277)
* [Torroni A, Schurr TG, et al. (1993) Asian affinities and continental radiation of the four founding Native American mtDNAs. American Journal of Human Genetics. American Journal of Human Genetics. 53(3): 563-590.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1682412/)
* [Torroni A, Sukernik RI, Schurr TG, et al. (1993). mtDNA variation of aboriginal Siberians reveals distinct genetic affinities with Native Americans. American journal of human genetics, 53(3), 591-608.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1682407/)
* [Ward RH, Redd A, Valencia D, Pääbo S (1993) Genetic and linguistic differentiation in the Americas. PNAS. 90(22): 10663-10667.](https://dx.doi.org/10.1073%2Fpnas.90.22.10663)
* [Ward RH, Frazier BL, Dew-Jager K, Pääbo S (1991) Extensive mitochondrial diversity within a single Amerindian tribe. PNAS. 88(19): 8720-8724.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC52581/)
* [Schurr TG et al. (1990) Amerindian mitochondrial DNAs have rare Asian mutations at high frequencies, suggesting they derived from four primary maternal lineages. American Journal of Human Genetics. 46(3): 613-623.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1683611/)

* [Genetics and the History of Latin America](https://migrationcluster.ucdavis.edu/research/genetics-and-history-latin-america)
* [Genetics, Linguistics, and Prehistoric Migrations: An Analysis of California Indian Mitochondrial DNA Lineages](https://escholarship.org/uc/item/6g135010)
* [Genome-wide insights into the genetic history of human populations](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4381409/)
* [Genomic insights on the ethno-history of the Maya and the ‘Ladinos’ from Guatemala](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4422311/)
* [High-resolution mitochondrial DNA analysis sheds light on human diversity, cultural interactions, and population mobility in Northwestern Amazonia](https://www.researchgate.net/publication/320654352_High-resolution_mitochondrial_DNA_analysis_sheds_light_on_human_diversity_cultural_interactions_and_population_mobility_in_Northwestern_Amazonia)
* [HLA Class II Alleles in the Otomi Population of the Mezquital Valley: A Genetic Approach to the History of Interethnic Migrations in the Mexican Central Plateau](https://www.researchgate.net/publication/274396125_HLA_Class_II_Alleles_in_the_Otomi_Population_of_the_Mezquital_Valley_A_Genetic_Approach_to_the_History_of_Interethnic_Migrations_in_the_Mexican_Central_Plateau)
* [Large scale mitochondrial sequencing in Mexican Americans suggests a reappraisal of Native American origins](https://doi.org/10.1186/1471-2148-11-293)
* [Mitochondrial DNA and Prehistoric Settlements: Native Migrations on the Western Edge of North America](https://www.researchgate.net/publication/8485114_Mitochondrial_DNA_and_Prehistoric_Settlements_Native_Migrations_on_the_Western_Edge_of_North_America)
* [Mitochondrial DNA Studies of Native Americans: Conceptions and Misconceptions of the Population Prehistory of the Americas](https://pdfs.semanticscholar.org/a40f/fbea9601cea8b7c973a8cedbea297a6de20a.pdf)
* [Mitochondrial DNA Diversity in Indigenous Populations of the Southern Extent of Siberia, and the Origins of Native American Haplogroups](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3905771/)
* [Mitochondrial Population Genomics Supports a Single Pre-Clovis Origin with a Coastal Route for the Peopling of the Americas](https://doi.org/10.1016/j.ajhg.2007.11.013)
* [mtDNA haplogroup X: An ancient link between Europe/Western Asia and North America?](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1377656/)
* [mtDNA sequences suggest a recent evolutionary divergence for Beringian and northern North American populations.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1682422/)
* [mtDNA Variation among Greenland Eskimos: The Edge of the Beringian Expansion.](https://doi.org/10.1086/303038)
* [Origins and genetic legacies of the Caribbean Taino](https://doi.org/10.1073/pnas.1716839115)
* [Patterns of mtDNA Diversity in Northwestern North America](https://www.researchgate.net/publication/8485113_Patterns_of_mtDNA_Diversity_in_Northwestern_North_America)
* [The Genetic History Of The Otomi In The Central Mexican Valley](https://repository.upenn.edu/anthro_seniortheses/133/)
* [The Genetics of Mexico Recapitulates Native American Substructure and Affects Biomedical Traits](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4156478/)

## Early Hominid through Modern Human

* [Shinde V et al. (2019) An Ancient Harappan Genome Lacks Ancestry from Steppe Pastoralists or Iranian Farmers. Cell.](https://doi.org/10.1016/j.cell.2019.08.048)
* [Narasimhan VM et al. (2019) The formation of human populations in South and Central Asia. Science. 35(6457): eaat7487](https://science.sciencemag.org/content/365/6457/eaat7487.full)
* [Schaefer NK, Shapiro B (2019) New middle chapter in the story of human evolution. Science. 365(6457): 981-982.](https://science.sciencemag.org/content/365/6457/981) &mdash; Has great references for an updated and more complete phylogenetic tree.
* National Geographic has a nice looking, but crude, map which traces the general flow of modern genetics out of Africa and into the rest of the world: [Map of Human Migration](https://genographic.nationalgeographic.com/human-journey/)
* [Ragsdale AP, Gravel S (2019) Models of archaic admixture and recent history from two-locus statistics. PLoS Genet 15(6): e1008204.](https://doi.org/10.1371/journal.pgen.1008204)
* [Bokelmann L et al. (2019) A genetic analysis of the Gibralter Neanderthals. PNAS. 116(31): 15610-15615.](https://doi.org/10.1073/pnas.1903984116)
* [Harvati K et al. (2019) Apidima Cave fossila provide earliest evidence of Homo sapiens in Eurasia. Nature. 571: 500-504.](https://www.nature.com/articles/s41586-019-1376-z)
* [Delson E (2019) An early dispersal of modern humans from Africa to Greece. Nature News. July 10.](https://www.nature.com/articles/d41586-019-02075-9) &mdash; Has a great map of locations and times for various known dispersals.
* [Cepelewicz, J (2019) Fossil DNA Reveals New Twists in Modern Human Origins. Quanta Magazine. August 29. Web. Accessed September 7, 2019.](https://www.quantamagazine.org/fossil-dna-reveals-new-twists-in-modern-human-origins-20190829/)
* [Browning SR et al. (2018) Analysis of Human Sequence Data Reveals Two Pulses of Archaic Denisovan Admixture. Cell. 173(1): 53-61.](https://doi.org/10.1016/j.cell.2018.02.031)
* [Wolf AB, Akey JM (2018) Outstanding questions in the study of archaic hominin admixture. PLoS Genet 14(5): e1007349.](https://doi.org/10.1371/journal.pgen.1007349)
* [Slon V et al. (2018) The genome of the offspring of a Neanderthal mother and a Denisovan father. Nature. 561: 113-116.](https://www.nature.com/articles/s41586-018-0455-x)
* [Groucutt HS, Grün R, Zalmout IAS, Drake NA, Armitage SJ, Candy I, et al. (2018). Homo sapiens in Arabia by 85,000 years ago. Nature Ecology & Evolution. doi:10.1038/s41559-018-0518-2](https://doi.org/10.1038/s41559-018-0518-2)
* [Scerri EML et al. (2018) Did Our Species Evolve in Subdivided Populations across Africa, and Why Does It Matter? Trends in Ecology & Evolution. 33(8): 582-594.](https://doi.org/10.1016/j.tree.2018.05.005)
* [Langley M. (2017) World’s scientists turn to Asia and Australia to rewrite human history. The Conversation. 7 Dec. 2017.](http://theconversation.com/worlds-scientists-turn-to-asia-and-australia-to-rewrite-human-history-88697)
* [Gibbens S (2017) Humans Arrived in Australia 18,000 Years Earlier Than We Thought. National Geographic. 20 Jul.](https://news.nationalgeographic.com/2017/07/australia-aboriginal-early-human-evolution-spd/)
* [Gibbons A (2017) A find in Australia hints at very early human exit from Africa. 19 Jul.](http://www.sciencemag.org/news/2017/07/find-australia-hints-very-early-human-exit-africa)
* [Callaway E (2017) Oldest Homo sapiens fossil claim rewrites our species' history. Nature News. 07 Jun.](https://www.nature.com/news/oldest-homo-sapiens-fossil-claim-rewrites-our-species-history-1.22114)
* [Gibbons A (2017) World’s oldest Homo sapiens fossils found in Morocco. Science. 7 Jun.](http://www.sciencemag.org/news/2017/06/world-s-oldest-homo-sapiens-fossils-found-morocco)
* [Nielsen R et al. (2017) Tracing the peopling of the world through genomics. Nature. 541: 302-310.](https://www.nature.com/articles/nature21347) &mdash; Has a better map of possible migration routes which line up with indigenous histories. There's still some details missing, but it's the best 2017 had to offer.
* [Mallick S et al. (2016) The Simons Genome Diversity Project: 300 genomes from 142 diverse populations. Nature. 538: 201-206.](https://www.nature.com/articles/nature18964) &mdash; This has an interesting neighbor joining tree based on pairwise divergence showing .
* [Malaspinas A-S et al. (2016) A genomic history of Aboriginal Australia. Nature. 538: 207–214.](https://www.nature.com/articles/nature18299)
* [Pagani L (2016) Genomic analyses inform on migration events during the peopling of Eurasia. Nature. 538: 238-242.](https://www.nature.com/articles/nature19792)
* [Qiu J (2016) How China is rewriting the book on human origins. Nature News. 12 Jul.](https://www.nature.com/news/how-china-is-rewriting-the-book-on-human-origins-1.20231)
* [(2016) Unprecedented study of Aboriginal Australians points to one shared Out of Africa migration for modern humans. 21 Sep.](https://www.cam.ac.uk/research/news/unprecedented-study-of-aboriginal-australians-points-to-one-shared-out-of-africa-migration-for)
* [Fu W et al. (2016) Robust Inference of Identity by Descent from Exome-Sequencing Data. American Journal of Human Gentics. 99(5): 1106-1116.](https://doi.org/10.1016/j.ajhg.2016.09.011)
* [Kuhlwilm M et al. (2016) Ancient gene flow from early modern humans into Eastern Neanderthals. Nature. 530: 429-433.](https://www.nature.com/articles/nature16544)
* [Karmin M et al. (2015) A recent bottleneck of Y chromosome diversity coincides with a global change in culture. Genome Research. 25: 459-466.](https://genome.cshlp.org/content/25/4/459.full)
* [The 1000 Genomes Project Consortium (2015) A global reference for human genetic variation. Nature. 526: 68-74.](https://www.nature.com/articles/nature15393)
* [Fu Q et al. (2013) DNA analysis of an early modern human from Tianyuan Cave, China. PNAS. 110(6): 2223-2227.](https://doi.org/10.1073/pnas.1221359110)
* [Gronau I et al. (2011) Bayesian inference of ancient human demography from individual genome sequences. Nature Genetics. 43, 1031-1034.](https://www.nature.com/articles/ng.937)
* [McEvoy B et al. (2010) Whole-Genome Genetic Diversity in a Sample of Australians with Deep Aboriginal Ancestry. American Journal of Human Genetics. 13 Aug.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2917718/)

* [Scientists discover evidence of early human innovation, pushing back evolutionary timeline](https://unews.utah.edu/scientists-discover-evidence-of-early-human-innovation-pushing-back-evolutionary-timeline/)
* [Neanderthals cared for each other and survived into old age – new research](https://www.heritagedaily.com/2018/04/neanderthals-cared-for-each-other-and-survived-into-old-age-new-research/118812)
* [First human migration out of Africa much more geographically widespread than previously thought](https://www.shh.mpg.de/891823/wusta)
* [DNA of early Neanderthal gives timeline for new modern human-related dispersal from Africa](https://www.heritagedaily.com/2017/07/dna-early-neanderthal-gives-timeline-new-modern-human-related-dispersal-africa/115826)
* [‘Humanlike’ ways of thinking evolved 1.8 million years ago, suggests new study](https://www.heritagedaily.com/2017/05/humanlike-ways-of-thinking-evolved-1-8-million-years-ago-suggests-new-study/114789)
* [400,000-year-old fossil human cranium is oldest ever found in Portugal](https://www.heritagedaily.com/2017/03/400000-year-old-fossil-human-cranium-is-oldest-ever-found-in-portugal/114346)
* [Ancient Humans Lived in China 2.1 Million Years Ago](https://www.theatlantic.com/science/archive/2018/07/early-humans-left-africa-250000-years-earlier-than-thought/564896/)
* [Hominin occupation of the Chinese Loess Plateau since about 2.1 million years ago](https://www.nature.com/articles/s41586-018-0299-4)
* [Evidence for precision grasping in Neandertal daily activities](http://advances.sciencemag.org/content/4/9/eaat2369)
* [Symbolic use of marine shells and mineral pigments by Iberian Neandertals 115,000 years ago](http://advances.sciencemag.org/content/4/2/eaar5255)
* [Traces of mystery ancient humans found lurking in our genomes](https://www.newscientist.com/article/mg24031992-600-traces-of-mystery-ancient-humans-found-lurking-in-our-genomes/)
